<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.9.23">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">

<meta name="author" content="Tanya Strydom">
<meta name="author" content="Baran Karapunar">
<meta name="author" content="Andrew P. Beckerman">
<meta name="author" content="Alexander Dunhill">
<meta name="dcterms.date" content="2026-02-25">
<meta name="keywords" content="Ecological networks, Biotic interactions, Community assembly, Environmental gradients, Interaction turnover, Trophic organisation, Ecosystem resilience, Macroecology">

<title>Model structure conditions ecological inference in food web reconstruction</title>
<style>
/* Default styles provided by pandoc.
** See https://pandoc.org/MANUAL.html#variables-for-html for config info.
*/
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
/* CSS for citations */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging-indent div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}</style>


<script src="site_libs/clipboard/clipboard.min.js"></script>
<script src="site_libs/quarto-html/quarto.js" type="module"></script>
<script src="site_libs/quarto-html/tabsets/tabsets.js" type="module"></script>
<script src="site_libs/quarto-html/axe/axe-check.js" type="module"></script>
<script src="site_libs/quarto-html/popper.min.js"></script>
<script src="site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="site_libs/quarto-html/anchor.min.js"></script>
<link href="site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="site_libs/quarto-html/quarto-syntax-highlighting-076ecbd647e1f0418c5051713cd9b730.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="site_libs/bootstrap/bootstrap.min.js"></script>
<link href="site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="site_libs/bootstrap/bootstrap-cb5849899318f24a43522ae45f8874d0.min.css" rel="stylesheet" append-hash="true" id="quarto-bootstrap" data-mode="light">
<script src="site_libs/quarto-contrib/glightbox/glightbox.min.js"></script>
<link href="site_libs/quarto-contrib/glightbox/glightbox.min.css" rel="stylesheet">
<link href="site_libs/quarto-contrib/glightbox/lightbox.css" rel="stylesheet">
<script async="" src="https://hypothes.is/embed.js"></script>
<script>
  window.document.addEventListener("DOMContentLoaded", function (_event) {
    document.body.classList.add('hypothesis-enabled');
  });
</script>

  <script src="https://cdnjs.cloudflare.com/polyfill/v3/polyfill.min.js?features=es6"></script>
  <script defer="" src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-chtml-full.js" type="text/javascript"></script>

<script type="text/javascript">
const typesetMath = (el) => {
  if (window.MathJax) {
    // MathJax Typeset
    window.MathJax.typeset([el]);
  } else if (window.katex) {
    // KaTeX Render
    var mathElements = el.getElementsByClassName("math");
    var macros = [];
    for (var i = 0; i < mathElements.length; i++) {
      var texText = mathElements[i].firstChild;
      if (mathElements[i].tagName == "SPAN" && texText && texText.data) {
        window.katex.render(texText.data, mathElements[i], {
          displayMode: mathElements[i].classList.contains('display'),
          throwOnError: false,
          macros: macros,
          fleqn: false
        });
      }
    }
  }
}
window.Quarto = {
  typesetMath
};
</script>

<meta name="citation_title" content="Model structure conditions ecological inference in food web reconstruction">
<meta name="citation_abstract" content="Aim

Ecological networks are widely used to compare community structure, stability, and responses to disturbance across environmental gradients. However, many networks (particularly those assembled from incomplete interaction data) require model-based reconstruction. We test how alternative reconstruction frameworks condition ecological inference by quantifying their effects on network structure and disturbance dynamics.

Location

Cleveland Basin, United Kingdom.

Time period

Toarcian extinction event (Early Jurassic, late Pliensbachian–late Toarcian, ~183 Ma).

Major taxa studied

Marine animal communities.

Methods

We reconstructed four successive communities from an identical species pool using six contrasting food-web models spanning feasible (trait-based), realised (allometric and energetic), and structural (topological) network representations. For each community and model, 100 replicate networks were generated. We quantified macro-, meso-, and micro-scale network properties and assessed differences among models using multivariate analyses. Pairwise interaction turnover was measured using link-based beta diversity. We then simulated species loss under multiple disturbance scenarios, allowing cascading extinctions, and compared predicted community states using mean absolute differences and rank concordance metrics between models.

Results

Reconstruction framework strongly influenced inferred network topology (MANOVA, p &amp;amp;lt; 0.001), generating distinct structural signatures independent of species composition. Models that were similar in global metrics often diverged in species-level interactions, with high β-turnover among inferred link sets. During disturbance simulations, species-level vulnerability rankings were broadly consistent across models, but interaction-level outcomes and cascade dynamics varied substantially. Concordance in extinction-scenario rankings was scale dependent, with higher agreement at the species level than at the interaction level.

Main conclusions

Network reconstruction functions as a structural prior that conditions ecological inference. While some aggregate patterns are robust across modelling frameworks, detailed interaction-level dynamics are highly model contingent. Comparative network studies across spatial or environmental gradients should therefore align reconstruction framework with inferential goals and explicitly evaluate sensitivity to modelling assumptions.
">
<meta name="citation_keywords" content="Ecological networks,Biotic interactions,Community assembly,Environmental gradients,Interaction turnover,Trophic organisation,Ecosystem resilience,Macroecology">
<meta name="citation_author" content="Tanya Strydom">
<meta name="citation_author" content="Baran Karapunar">
<meta name="citation_author" content="Andrew P. Beckerman">
<meta name="citation_author" content="Alexander Dunhill">
<meta name="citation_publication_date" content="2026-02-25">
<meta name="citation_cover_date" content="2026-02-25">
<meta name="citation_year" content="2026">
<meta name="citation_online_date" content="2026-02-25">
<meta name="citation_language" content="en">
<meta name="citation_journal_title" content="TBD">
<meta name="citation_reference" content="citation_title=Extinction cascades, community collapse, and recovery across a mesozoic hyperthermal event;,citation_author=Alexander M. Dunhill;,citation_author=Karolina Zarzyczny;,citation_author=Jack O. Shaw;,citation_author=Jed W. Atkinson;,citation_author=Crispin T. S. Little;,citation_author=Andrew P. Beckerman;,citation_publication_date=2024-10-04;,citation_cover_date=2024-10-04;,citation_year=2024;,citation_fulltext_html_url=https://www.nature.com/articles/s41467-024-53000-2;,citation_issue=1;,citation_doi=10.1038/s41467-024-53000-2;,citation_volume=15;,citation_language=en;,citation_journal_title=Nature Communications;">
<meta name="citation_reference" content="citation_title=Shortcomings of reusing species interaction networks created by different sets of researchers;,citation_author=Chris Brimacombe;,citation_author=Korryn Bodner;,citation_author=Matthew Michalska-Smith;,citation_author=Timothée Poisot;,citation_author=Marie-Josée Fortin;,citation_publication_date=2023-03;,citation_cover_date=2023-03;,citation_year=2023;,citation_fulltext_html_url=https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.3002068;,citation_issue=4;,citation_doi=10.1371/journal.pbio.3002068;,citation_volume=21;,citation_language=en;,citation_journal_title=PLOS Biology;">
<meta name="citation_reference" content="citation_title=A framework for reconstructing ancient food webs using functional trait data;,citation_abstract=Food webs provide quantitative insights into the structure and dynamics of ecological communities. Previous work has shown their utility in understanding community responses to modern and ancient perturbations, including anthropogenic change and mass extinctions. However, few ancient food webs have been reconstructed due to difficulties assessing trophic interactions amongst extinct species derived from an incomplete fossil record. We present and assess the Paleo Food web Inference Model (PFIM). PFIM uses functional trait data–predictive of interactions in modern ecosystems and commonly available for fossil organisms–to reconstruct ancient food webs. We test the model by (i) applying it to four modern ecosystems with empirical constrained food webs to directly compare PFIM-constructed networks to their empirical counterparts, (ii) by carefully comparing discrepancies between PFIM-inferred and empirical webs in one of those systems, and (iii) by comparing networks describing feasible trophic interactions (&amp;amp;amp;quot;feasible webs&amp;quot;) with networks to which we superimpose characteristic interaction distributions derived from modern theory (&quot;realized webs&quot;). As a proof of concept, we then apply the method to faunal data from two Cambrian fossil deposits to reconstruct ancient trophic systems. PFIM-inferred feasible food webs successfully predict ~70% of trophic interactions across four modern systems. Furthermore, inferred food webs with enforced interaction distributions (i.e., realized webs) accurately predict ~90% of interactions. Comparisons with a global database of trophic interactions and other food web models, suggest that under sampling of empirical webs accounts for up to 21% of the remaining differences between PFIM and empirical food webs. Food webs can be reasonably approximated by inferring trophic interactions based upon life habit traits. This study provides the foundation to use trait-based inference models across the fossil record to examine ancient food webs and community evolution.;,citation_author=Jack O. Shaw;,citation_author=Alexander M. Dunhill;,citation_author=Andrew P. Beckerman;,citation_author=Jennifer A. Dunne;,citation_author=Pincelli M. Hull;,citation_publication_date=2024-01;,citation_cover_date=2024-01;,citation_year=2024;,citation_doi=10.1101/2024.01.30.578036;,citation_language=en-US;,citation_publisher=bioRxiv;">
<meta name="citation_reference" content="citation_title=Extinction cascades and catastrophe in ancient food webs;,citation_author=Peter D. Roopnarine;,citation_publication_date=2006;,citation_cover_date=2006;,citation_year=2006;,citation_fulltext_html_url=https://www.jstor.org/stable/4096814;,citation_issue=1;,citation_volume=32;,citation_journal_title=Paleobiology;">
<meta name="citation_reference" content="citation_title=Size, foraging, and food web structure;,citation_author=Owen L. Petchey;,citation_author=Andrew P. Beckerman;,citation_author=Jens O. Riede;,citation_author=Philip H. Warren;,citation_publication_date=2008-03-18;,citation_cover_date=2008-03-18;,citation_year=2008;,citation_fulltext_html_url=https://pnas.org/doi/full/10.1073/pnas.0710672105;,citation_issue=11;,citation_doi=10.1073/pnas.0710672105;,citation_volume=105;,citation_language=en;,citation_journal_title=Proceedings of the National Academy of Sciences;">
<meta name="citation_reference" content="citation_title=Modeling food webs: Exploring unexplained structure using latent traits.;,citation_author=Rudolf&nbsp;Philippe Rohr;,citation_author=Heike Scherer;,citation_author=Patrik Kehrli;,citation_author=Christian Mazza;,citation_author=Louis-Félix Bersier;,citation_publication_date=2010-08;,citation_cover_date=2010-08;,citation_year=2010;,citation_fulltext_html_url=https://www.journals.uchicago.edu/doi/10.1086/653667;,citation_issue=2;,citation_doi=10.1086/653667;,citation_volume=176;,citation_journal_title=The American Naturalist;">
<meta name="citation_reference" content="citation_title=Collapse of an ecological network in ancient egypt;,citation_author=Justin D. Yeakel;,citation_author=Mathias M. Pires;,citation_author=Lars Rudolf;,citation_author=Nathaniel J. Dominy;,citation_author=Paul L. Koch;,citation_author=Paulo R. Guimarães;,citation_author=Thilo Gross;,citation_publication_date=2014-10;,citation_cover_date=2014-10;,citation_year=2014;,citation_issue=40;,citation_doi=10.1073/pnas.1408471111;,citation_volume=111;,citation_journal_title=PNAS;">
<meta name="citation_reference" content="citation_title=Simple rules yield complex food webs;,citation_author=Richard J. Williams;,citation_author=Neo D. Martinez;,citation_publication_date=2000-03;,citation_cover_date=2000-03;,citation_year=2000;,citation_fulltext_html_url=https://www.nature.com/articles/35004572;,citation_issue=6774;,citation_doi=10.1038/35004572;,citation_volume=404;,citation_language=en;,citation_journal_title=Nature;">
<meta name="citation_reference" content="citation_title=Success and its limits among structural models of complex food webs;,citation_author=Richard J. Williams;,citation_author=Neo D. Martinez;,citation_publication_date=2008-05;,citation_cover_date=2008-05;,citation_year=2008;,citation_issue=3;,citation_doi=10.1111/j.1365-2656.2008.01362.x;,citation_volume=77;,citation_language=eng;,citation_journal_title=The Journal of Animal Ecology;">
<meta name="citation_reference" content="citation_title=Graph embedding and transfer learning can help predict potential species interaction networks despite data limitations;,citation_author=Tanya Strydom;,citation_author=Salomé Bouskila;,citation_author=Francis Banville;,citation_author=Ceres Barros;,citation_author=Dominique Caron;,citation_author=Maxwell J. Farrell;,citation_author=Marie-Josée Fortin;,citation_author=Benjamin Mercier;,citation_author=Laura J. Pollock;,citation_author=Rogini Runghen;,citation_author=Giulio V. Dalla Riva;,citation_author=Timothée Poisot;,citation_publication_date=2023;,citation_cover_date=2023;,citation_year=2023;,citation_fulltext_html_url=https://onlinelibrary.wiley.com/doi/abs/10.1111/2041-210X.14228;,citation_issue=12;,citation_doi=10.1111/2041-210X.14228;,citation_volume=14;,citation_language=en;,citation_journal_title=Methods in Ecology and Evolution;">
<meta name="citation_reference" content="citation_title=Addressing the eltonian shortfall with trait-based interaction models;,citation_author=Dominique Caron;,citation_author=Luigi Maiorano;,citation_author=Wilfried Thuiller;,citation_author=Laura J. Pollock;,citation_publication_date=2022;,citation_cover_date=2022;,citation_year=2022;,citation_fulltext_html_url=https://onlinelibrary.wiley.com/doi/abs/10.1111/ele.13966;,citation_issue=4;,citation_doi=10.1111/ele.13966;,citation_volume=25;,citation_language=en;,citation_journal_title=Ecology Letters;">
<meta name="citation_reference" content="citation_title=Collapse of terrestrial mammal food webs since the late pleistocene;,citation_author=Evan C. Fricke;,citation_author=Chia Hsieh;,citation_author=Owen Middleton;,citation_author=Daniel Gorczynski;,citation_author=Caroline D. Cappello;,citation_author=Oscar Sanisidro;,citation_author=John Rowan;,citation_author=Jens-Christian Svenning;,citation_author=Lydia Beaudrot;,citation_publication_date=2022-08-26;,citation_cover_date=2022-08-26;,citation_year=2022;,citation_fulltext_html_url=http://dx.doi.org/10.1126/science.abn4012;,citation_issue=6609;,citation_doi=10.1126/science.abn4012;,citation_volume=377;,citation_language=en;,citation_journal_title=Science;">
<meta name="citation_reference" content="citation_title=Ecological modelling of paleocommunity food webs;,citation_author=Peter D Roopnarine;,citation_publication_date=2017;,citation_cover_date=2017;,citation_year=2017;,citation_language=en;,citation_inbook_title=undefined;">
<meta name="citation_reference" content="citation_title=Autecology and the filling of ecospace: Key metazoan radiations;,citation_author=Richard K. Bambach;,citation_author=Andrew M. Bush;,citation_author=Douglas H. Erwin;,citation_publication_date=2007;,citation_cover_date=2007;,citation_year=2007;,citation_fulltext_html_url=https://onlinelibrary.wiley.com/doi/abs/10.1111/j.1475-4983.2006.00611.x;,citation_issue=1;,citation_doi=10.1111/j.1475-4983.2006.00611.x;,citation_volume=50;,citation_language=en;,citation_journal_title=Palaeontology;">
<meta name="citation_reference" content="citation_title=Inferring predator-prey interactions in food webs;,citation_author=Justin PF Pomeranz;,citation_author=Ross M. Thompson;,citation_author=Timothée Poisot;,citation_author=Jon S. Harding;,citation_publication_date=2018;,citation_cover_date=2018;,citation_year=2018;,citation_fulltext_html_url=https://besjournals.onlinelibrary.wiley.com/doi/abs/10.1111/2041-210X.13125;,citation_issue=ja;,citation_doi=10.1111/2041-210X.13125;,citation_volume=0;,citation_language=en;,citation_journal_title=Methods in Ecology and Evolution;">
<meta name="citation_reference" content="citation_title=SVD entropy reveals the high complexity of ecological networks;,citation_author=Tanya Strydom;,citation_author=Giulio V. Dalla Riva;,citation_author=Timothée Poisot;,citation_publication_date=2021;,citation_cover_date=2021;,citation_year=2021;,citation_fulltext_html_url=https://www.frontiersin.org/articles/10.3389/fevo.2021.623141/full;,citation_doi=10.3389/fevo.2021.623141;,citation_volume=9;,citation_language=en-US;,citation_journal_title=Frontiers in Ecology and Evolution;">
<meta name="citation_reference" content="citation_title=Network motifs: Simple building blocks of complex networks;,citation_author=R. Milo;,citation_author=S. Shen-Orr;,citation_author=S. Itzkovitz;,citation_author=N. Kashtan;,citation_author=D. Chklovskii;,citation_author=U. Alon;,citation_publication_date=2002-10-25;,citation_cover_date=2002-10-25;,citation_year=2002;,citation_fulltext_html_url=https://www.science.org/doi/10.1126/science.298.5594.824;,citation_issue=5594;,citation_doi=10.1126/science.298.5594.824;,citation_volume=298;,citation_journal_title=Science;">
<meta name="citation_reference" content="citation_title=Evidence for the existence of a robust pattern of prey selection in food webs;,citation_author=Daniel B Stouffer;,citation_author=Juan Camacho;,citation_author=Wenxin Jiang;,citation_author=Luís A Nunes Amaral;,citation_publication_date=2007-06;,citation_cover_date=2007-06;,citation_year=2007;,citation_issue=1621;,citation_doi=10.1098/rspb.2007.0571;,citation_volume=274;,citation_journal_title=Proceedings of the Royal Society B: Biological Sciences;">
<meta name="citation_reference" content="citation_title=Success and its limits among structural models of complex food webs;,citation_author=Richard J. Williams;,citation_author=Neo D. Martinez;,citation_publication_date=2008-05;,citation_cover_date=2008-05;,citation_year=2008;,citation_issue=3;,citation_doi=10.1111/j.1365-2656.2008.01362.x;,citation_volume=77;,citation_language=eng;,citation_journal_title=The Journal of Animal Ecology;">
<meta name="citation_reference" content="citation_title=Size, foraging, and food web structure;,citation_author=Owen L. Petchey;,citation_author=Andrew P. Beckerman;,citation_author=Jens O. Riede;,citation_author=Philip H. Warren;,citation_publication_date=2008-03-18;,citation_cover_date=2008-03-18;,citation_year=2008;,citation_fulltext_html_url=https://pnas.org/doi/full/10.1073/pnas.0710672105;,citation_issue=11;,citation_doi=10.1073/pnas.0710672105;,citation_volume=105;,citation_language=en;,citation_journal_title=Proceedings of the National Academy of Sciences;">
<meta name="citation_reference" content="citation_title=Constant connectance in community food webs;,citation_author=Neo D. Martinez;,citation_publication_date=1992;,citation_cover_date=1992;,citation_year=1992;,citation_fulltext_html_url=http://www.jstor.org/stable/2462337;,citation_issue=6;,citation_volume=139;,citation_journal_title=The American Naturalist;">
<meta name="citation_reference" content="citation_title=Analysing ecological networks of species interactions;,citation_author=Eva Delmas;,citation_author=Mathilde Besson;,citation_author=Marie-Hélène Brice;,citation_author=Laura A. Burkle;,citation_author=Giulio V. Dalla Riva;,citation_author=Marie-Josée Fortin;,citation_author=Dominique Gravel;,citation_author=Paulo R. Guimarães;,citation_author=David H. Hembry;,citation_author=Erica A. Newman;,citation_author=Jens M. Olesen;,citation_author=Mathias M. Pires;,citation_author=Justin D. Yeakel;,citation_author=Timothée Poisot;,citation_publication_date=2018-06-20;,citation_cover_date=2018-06-20;,citation_year=2018;,citation_fulltext_html_url=http://doi.wiley.com/10.1111/brv.12433;,citation_doi=10.1111/brv.12433;,citation_language=en;,citation_journal_title=Biological Reviews;">
<meta name="citation_reference" content="citation_title=A general model for food web structure;,citation_author=Stefano Allesina;,citation_author=David Alonso;,citation_author=Mercedes Pascual;,citation_publication_date=2008-05-02;,citation_cover_date=2008-05-02;,citation_year=2008;,citation_fulltext_html_url=https://www.science.org/doi/full/10.1126/science.1156269;,citation_issue=5876;,citation_doi=10.1126/science.1156269;,citation_volume=320;,citation_journal_title=Science;">
<meta name="citation_reference" content="citation_title=A roadmap towards predicting species interaction networks (across space and time);,citation_author=Tanya Strydom;,citation_author=Michael D. Catchen;,citation_author=Francis Banville;,citation_author=Dominique Caron;,citation_author=Gabriel Dansereau;,citation_author=Philippe Desjardins-Proulx;,citation_author=Norma R. Forero-Muñoz;,citation_author=Gracielle Higino;,citation_author=Benjamin Mercier;,citation_author=Andrew Gonzalez;,citation_author=Dominique Gravel;,citation_author=Laura Pollock;,citation_author=Timothée Poisot;,citation_publication_date=2021-11;,citation_cover_date=2021-11;,citation_year=2021;,citation_issue=1837;,citation_doi=10.1098/rstb.2021.0063;,citation_volume=376;,citation_journal_title=Philosophical Transactions of the Royal Society B: Biological Sciences;,citation_publisher=Royal Society;">
<meta name="citation_reference" content="citation_title=A comparative study of ecological specialization estimators;,citation_author=Timothée Poisot;,citation_author=Elsa Canard;,citation_author=Nicolas Mouquet;,citation_author=Michael E. Hochberg;,citation_publication_date=2012-01-19;,citation_cover_date=2012-01-19;,citation_year=2012;,citation_fulltext_html_url=http://dx.doi.org/10.1111/j.2041-210X.2011.00174.x;,citation_issue=3;,citation_doi=10.1111/j.2041-210x.2011.00174.x;,citation_volume=3;,citation_language=en;,citation_journal_title=Methods in Ecology and Evolution;">
<meta name="citation_reference" content="citation_title=Food webs from the small to the large: The robert h. MacArthur award lecture;,citation_author=Thomas W. Schoener;,citation_publication_date=1989-12;,citation_cover_date=1989-12;,citation_year=1989;,citation_fulltext_html_url=http://dx.doi.org/10.2307/1938088;,citation_issue=6;,citation_doi=10.2307/1938088;,citation_volume=70;,citation_language=en;,citation_journal_title=Ecology;">
<meta name="citation_reference" content="citation_title=ATNr: Allometric trophic network models in r;,citation_author=Benoit Gauzens;,citation_author=Ulrich Brose;,citation_author=Eva Delmas;,citation_author=Emilio Berti;,citation_publication_date=2023;,citation_cover_date=2023;,citation_year=2023;,citation_fulltext_html_url=https://onlinelibrary.wiley.com/doi/abs/10.1111/2041-210X.14212;,citation_issue=11;,citation_doi=10.1111/2041-210X.14212;,citation_volume=14;,citation_language=en;,citation_journal_title=Methods in Ecology and Evolution;">
<meta name="citation_reference" content="citation_title=Animal diversity and ecosystem functioning in dynamic food webs;,citation_author=Florian D. Schneider;,citation_author=Ulrich Brose;,citation_author=Björn C. Rall;,citation_author=Christian Guill;,citation_publication_date=2016-10-05;,citation_cover_date=2016-10-05;,citation_year=2016;,citation_fulltext_html_url=https://www.nature.com/articles/ncomms12718;,citation_issue=1;,citation_doi=10.1038/ncomms12718;,citation_volume=7;,citation_language=en;,citation_journal_title=Nature Communications;">
<meta name="citation_reference" content="citation_title=The reliability of R50 as a measure of vulnerability of food webs to sequential species deletions;,citation_author=Tomas Jonsson;,citation_author=Sofia Berg;,citation_author=Alexander Pimenov;,citation_author=Catherine Palmer;,citation_author=Mark Emmerson;,citation_publication_date=2015;,citation_cover_date=2015;,citation_year=2015;,citation_fulltext_html_url=https://onlinelibrary.wiley.com/doi/abs/10.1111/oik.01588;,citation_issue=4;,citation_doi=10.1111/oik.01588;,citation_volume=124;,citation_language=en;,citation_journal_title=Oikos;">
<meta name="citation_reference" content="citation_title=On random graphs. i.;,citation_author=P. Erdős;,citation_author=A. Rényi;,citation_publication_date=1959-07-01;,citation_cover_date=1959-07-01;,citation_year=1959;,citation_fulltext_html_url=http://dx.doi.org/10.5486/PMD.1959.6.3-4.12;,citation_issue=3-4;,citation_doi=10.5486/pmd.1959.6.3-4.12;,citation_volume=6;,citation_journal_title=Publicationes Mathematicae Debrecen;">
<meta name="citation_reference" content="citation_title=The ghost of nestedness in ecological networks;,citation_author=Phillip P. A. Staniczenko;,citation_author=Jason C. Kopp;,citation_author=Stefano Allesina;,citation_publication_date=2013-06;,citation_cover_date=2013-06;,citation_year=2013;,citation_fulltext_html_url=http://www.nature.com/articles/ncomms2422;,citation_issue=1;,citation_doi=10.1038/ncomms2422;,citation_volume=4;,citation_language=en;,citation_journal_title=Nature Communications;">
<meta name="citation_reference" content="citation_title=Stabilization of chaotic and non-permanent food-web dynamics;,citation_author=R. J. Williams;,citation_author=N. D. Martinez;,citation_publication_date=2004-03-01;,citation_cover_date=2004-03-01;,citation_year=2004;,citation_fulltext_html_url=http://link.springer.com/10.1140/epjb/e2004-00122-1;,citation_issue=2;,citation_doi=10.1140/epjb/e2004-00122-1;,citation_volume=38;,citation_language=en;,citation_journal_title=The European Physical Journal B - Condensed Matter;">
<meta name="citation_reference" content="citation_title=Simultaneously estimating food web connectance and structure with uncertainty;,citation_author=Anubhav Gupta;,citation_author=Reinhard Furrer;,citation_author=Owen L. Petchey;,citation_publication_date=2022;,citation_cover_date=2022;,citation_year=2022;,citation_fulltext_html_url=https://onlinelibrary.wiley.com/doi/abs/10.1002/ece3.8643;,citation_issue=3;,citation_doi=10.1002/ece3.8643;,citation_volume=12;,citation_language=en;,citation_journal_title=Ecology and Evolution;">
<meta name="citation_reference" content="citation_title=Highly resolved early eocene food webs show development of modern trophic structure after the end-cretaceous extinction;,citation_author=Jennifer A. Dunne;,citation_author=Conrad C. Labandeira;,citation_author=Richard J. Williams;,citation_publication_date=2014-05-07;,citation_cover_date=2014-05-07;,citation_year=2014;,citation_fulltext_html_url=https://royalsocietypublishing.org/doi/10.1098/rspb.2013.3280;,citation_issue=1782;,citation_doi=10.1098/rspb.2013.3280;,citation_volume=281;,citation_journal_title=Proceedings of the Royal Society B: Biological Sciences;">
<meta name="citation_reference" content="citation_title=Extinction cascades and catastrophe in ancient food webs;,citation_author=Peter D. Roopnarine;,citation_publication_date=2006;,citation_cover_date=2006;,citation_year=2006;,citation_fulltext_html_url=https://www.jstor.org/stable/4096814;,citation_issue=1;,citation_volume=32;,citation_journal_title=Paleobiology;">
<meta name="citation_reference" content="citation_title=Tailoring interaction network types to answer different ecological questions;,citation_author=Benoit Gauzens;,citation_author=Lise Thouvenot;,citation_author=Diane S. Srivastava;,citation_author=Pavel Kratina;,citation_author=Gustavo Q. Romero;,citation_author=Emilio Berti;,citation_author=Eoin J. O’Gorman;,citation_author=Angélica L. González;,citation_author=Olivier Dézerald;,citation_author=Nico Eisenhauer;,citation_author=Mathias Pires;,citation_author=Remo Ryser;,citation_author=Vinicius F. Farjalla;,citation_author=Pierre Rogy;,citation_author=Ulrich Brose;,citation_author=Jana S. Petermann;,citation_author=Benoît Geslin;,citation_author=Jes Hines;,citation_publication_date=2025-06-13;,citation_cover_date=2025-06-13;,citation_year=2025;,citation_fulltext_html_url=https://www.nature.com/articles/s44358-025-00056-7;,citation_doi=10.1038/s44358-025-00056-7;,citation_language=en;,citation_journal_title=Nature Reviews Biodiversity;">
<meta name="citation_reference" content="citation_title=Addressing priority questions of conservation science with palaeontological data;,citation_author=Wolfgang Kiessling;,citation_author=Nussaïbah B. Raja;,citation_author=Vanessa Julie Roden;,citation_author=Samuel T. Turvey;,citation_author=Erin E. Saupe;,citation_publication_date=2019-11-04;,citation_cover_date=2019-11-04;,citation_year=2019;,citation_fulltext_html_url=https://royalsocietypublishing.org/doi/10.1098/rstb.2019.0222;,citation_issue=1788;,citation_doi=10.1098/rstb.2019.0222;,citation_volume=374;,citation_journal_title=Philosophical Transactions of the Royal Society B: Biological Sciences;">
<meta name="citation_reference" content="citation_title=What is conservation paleobiology? Tracking 20 years of research and development;,citation_author=Erin M. Dillon;,citation_author=Jaleigh Q. Pier;,citation_author=Jansen A. Smith;,citation_author=Nussaïbah B. Raja;,citation_author=Danijela Dimitrijević;,citation_author=Elizabeth L. Austin;,citation_author=Jonathan D. Cybulski;,citation_author=Julia De Entrambasaguas;,citation_author=Stephen R. Durham;,citation_author=Carolin M. Grether;,citation_author=Himadri Sekhar Haldar;,citation_author=Kristína Kocáková;,citation_author=Chien-Hsiang Lin;,citation_author=Ilaria Mazzini;,citation_author=Alexis M. Mychajliw;,citation_author=Amy L. Ollendorf;,citation_author=Catalina Pimiento;,citation_author=Omar R. Regalado Fernández;,citation_author=Isaiah E. Smith;,citation_author=Gregory P. Dietl;,citation_publication_date=2022-12-07;,citation_cover_date=2022-12-07;,citation_year=2022;,citation_fulltext_html_url=https://www.frontiersin.org/journals/ecology-and-evolution/articles/10.3389/fevo.2022.1031483/full;,citation_doi=10.3389/fevo.2022.1031483;,citation_volume=10;,citation_language=en-US;,citation_journal_title=Frontiers in Ecology and Evolution;">
<meta name="citation_reference" content="citation_title=Global projection of terrestrial vertebrate food webs under future climate and land-use changes;,citation_author=Xiyang Hao;,citation_author=Marcel Holyoak;,citation_author=Zhicheng Zhang;,citation_author=Chuan Yan;,citation_publication_date=2025;,citation_cover_date=2025;,citation_year=2025;,citation_fulltext_html_url=https://onlinelibrary.wiley.com/doi/abs/10.1111/gcb.70061;,citation_issue=2;,citation_doi=10.1111/gcb.70061;,citation_volume=31;,citation_language=en;,citation_journal_title=Global Change Biology;">
<meta name="citation_reference" content="citation_title=Chasing ecological interactions;,citation_author=Pedro Jordano;,citation_publication_date=2016-09-15;,citation_cover_date=2016-09-15;,citation_year=2016;,citation_fulltext_html_url=https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002559;,citation_issue=9;,citation_doi=10.1371/journal.pbio.1002559;,citation_volume=14;,citation_language=en;,citation_journal_title=PLOS Biology;">
<meta name="citation_reference" content="citation_title=Inferring biotic interactions from proxies;,citation_author=Ignacio Morales-Castilla;,citation_author=Miguel G. Matias;,citation_author=Dominique Gravel;,citation_author=Miguel B. Araújo;,citation_publication_date=2015-06;,citation_cover_date=2015-06;,citation_year=2015;,citation_fulltext_html_url=https://linkinghub.elsevier.com/retrieve/pii/S0169534715000774;,citation_issue=6;,citation_doi=10.1016/j.tree.2015.03.014;,citation_volume=30;,citation_language=en;,citation_journal_title=Trends in Ecology &amp;amp;amp; Evolution;">
<meta name="citation_reference" content="citation_title=Machine learning and deep learninga review for ecologists;,citation_author=Maximilian Pichler;,citation_author=Florian Hartig;,citation_publication_date=2023;,citation_cover_date=2023;,citation_year=2023;,citation_fulltext_html_url=https://onlinelibrary.wiley.com/doi/abs/10.1111/2041-210X.14061;,citation_issue=4;,citation_doi=10.1111/2041-210X.14061;,citation_volume=14;,citation_language=en;,citation_journal_title=Methods in Ecology and Evolution;">
<meta name="citation_reference" content="citation_title=A roadmap towards predicting species interaction networks (across space and time);,citation_author=Tanya Strydom;,citation_author=Michael D. Catchen;,citation_author=Francis Banville;,citation_author=Dominique Caron;,citation_author=Gabriel Dansereau;,citation_author=Philippe Desjardins-Proulx;,citation_author=Norma R. Forero-Muñoz;,citation_author=Gracielle Higino;,citation_author=Benjamin Mercier;,citation_author=Andrew Gonzalez;,citation_author=Dominique Gravel;,citation_author=Laura Pollock;,citation_author=Timothée Poisot;,citation_publication_date=2021-11-08;,citation_cover_date=2021-11-08;,citation_year=2021;,citation_fulltext_html_url=https://royalsocietypublishing.org/doi/abs/10.1098/rstb.2021.0063;,citation_issue=1837;,citation_doi=10.1098/rstb.2021.0063;,citation_volume=376;,citation_journal_title=Philosophical Transactions of the Royal Society B: Biological Sciences;">
<meta name="citation_reference" content="citation_title=A general model for food web structure;,citation_author=Stefano Allesina;,citation_author=David Alonso;,citation_author=Mercedes Pascual;,citation_publication_date=2008-05-02;,citation_cover_date=2008-05-02;,citation_year=2008;,citation_fulltext_html_url=https://www.science.org/doi/full/10.1126/science.1156269;,citation_issue=5876;,citation_doi=10.1126/science.1156269;,citation_volume=320;,citation_journal_title=Science;">
<meta name="citation_reference" content="citation_title=Collapse of an ecological network in ancient egypt;,citation_author=Justin D. Yeakel;,citation_author=Mathias M. Pires;,citation_author=Lars Rudolf;,citation_author=Nathaniel J. Dominy;,citation_author=Paul L. Koch;,citation_author=Paulo R. Guimarães;,citation_author=Thilo Gross;,citation_publication_date=2014-10;,citation_cover_date=2014-10;,citation_year=2014;,citation_issue=40;,citation_doi=10.1073/pnas.1408471111;,citation_volume=111;,citation_journal_title=PNAS;">
<meta name="citation_reference" content="citation_title=Food web reconstruction through phylogenetic transfer of low-rank network representation;,citation_author=Tanya Strydom;,citation_author=Salomé Bouskila;,citation_author=Francis Banville;,citation_author=Ceres Barros;,citation_author=Dominique Caron;,citation_author=Maxwell J. Farrell;,citation_author=Marie-Josée Fortin;,citation_author=Victoria Hemming;,citation_author=Benjamin Mercier;,citation_author=Laura J. Pollock;,citation_author=Rogini Runghen;,citation_author=Giulio V. Dalla Riva;,citation_author=Timothée Poisot;,citation_publication_date=2022-12;,citation_cover_date=2022-12;,citation_year=2022;,citation_fulltext_html_url=https://onlinelibrary.wiley.com/doi/abs/10.1111/2041-210X.13835;,citation_issue=12;,citation_doi=10.1111/2041-210X.13835;,citation_volume=13;,citation_language=en;,citation_journal_title=Methods in Ecology and Evolution;">
<meta name="citation_reference" content="citation_title=Navigating the integration of biotic interactions in biogeography;,citation_author=Wilfried Thuiller;,citation_author=Irene Calderon-Sanou;,citation_author=Loïc Chalmandrier;,citation_author=Pierre Gaüzere;,citation_author=Marc Ohlmann;,citation_author=Giovanni Poggiato;,citation_author=Tamara Münkemüller;,citation_language=en;">
<meta name="citation_reference" content="citation_title=Robustness to secondary extinctions: Comparing trait-based sequential deletions in static and dynamic food webs;,citation_author=Alva Curtsdotter;,citation_author=Amrei Binzer;,citation_author=Ulrich Brose;,citation_author=Francisco De Castro;,citation_author=Bo Ebenman;,citation_author=Anna Eklöf;,citation_author=Jens O. Riede;,citation_author=Aaron Thierry;,citation_author=Björn C. Rall;,citation_publication_date=2011-11;,citation_cover_date=2011-11;,citation_year=2011;,citation_fulltext_html_url=https://linkinghub.elsevier.com/retrieve/pii/S1439179111001216;,citation_issue=7;,citation_doi=10.1016/j.baae.2011.09.008;,citation_volume=12;,citation_language=en;,citation_journal_title=Basic and Applied Ecology;">
<meta name="citation_reference" content="citation_title=Tailoring interaction network types to answer different ecological questions;,citation_author=Benoit Gauzens;,citation_author=Lise Thouvenot;,citation_author=Diane S. Srivastava;,citation_author=Pavel Kratina;,citation_author=Gustavo Q. Romero;,citation_author=Emilio Berti;,citation_author=Eoin J. O’Gorman;,citation_author=Angélica L. González;,citation_author=Olivier Dézerald;,citation_author=Nico Eisenhauer;,citation_author=Mathias Pires;,citation_author=Remo Ryser;,citation_author=Vinicius F. Farjalla;,citation_author=Pierre Rogy;,citation_author=Ulrich Brose;,citation_author=Jana S. Petermann;,citation_author=Benoît Geslin;,citation_author=Jes Hines;,citation_publication_date=2025-06-13;,citation_cover_date=2025-06-13;,citation_year=2025;,citation_fulltext_html_url=https://www.nature.com/articles/s44358-025-00056-7;,citation_doi=10.1038/s44358-025-00056-7;,citation_language=en;,citation_journal_title=Nature Reviews Biodiversity;">
<meta name="citation_reference" content="citation_title=Navigating the integration of biotic interactions in biogeography;,citation_author=Wilfried Thuiller;,citation_author=Irene Calderón-Sanou;,citation_author=Loïc Chalmandrier;,citation_author=Pierre Gaüzère;,citation_author=Louise M. J. O’Connor;,citation_author=Marc Ohlmann;,citation_author=Giovanni Poggiato;,citation_author=Tamara Münkemüller;,citation_publication_date=2024;,citation_cover_date=2024;,citation_year=2024;,citation_fulltext_html_url=https://onlinelibrary.wiley.com/doi/abs/10.1111/jbi.14734;,citation_issue=4;,citation_doi=10.1111/jbi.14734;,citation_volume=51;,citation_language=en;,citation_journal_title=Journal of Biogeography;">
<meta name="citation_reference" content="citation_title=Ecological networks–beyond food webs;,citation_author=Thomas C. Ings;,citation_author=José M. Montoya;,citation_author=Jordi Bascompte;,citation_author=Nico Blüthgen;,citation_author=Lee Brown;,citation_author=Carsten F. Dormann;,citation_author=François Edwards;,citation_author=David Figueroa;,citation_author=Ute Jacob;,citation_author=J. Iwan Jones;,citation_author=Rasmus B. Lauridsen;,citation_author=Mark E. Ledger;,citation_author=Hannah M. Lewis;,citation_author=Jens M. Olesen;,citation_author=F. J. Frank Veen;,citation_author=Phil H. Warren;,citation_author=Guy Woodward;,citation_publication_date=2009-01;,citation_cover_date=2009-01;,citation_year=2009;,citation_issue=1;,citation_doi=10.1111/j.1365-2656.2008.01460.x;,citation_volume=78;,citation_language=eng;,citation_journal_title=The Journal of Animal Ecology;">
<meta name="citation_reference" content="citation_title=Compilation and network analyses of cambrian food webs;,citation_author=Jennifer A. Dunne;,citation_author=Richard J. Williams;,citation_author=Neo D. Martinez;,citation_author=Rachel A. Wood;,citation_author=Douglas H. Erwin;,citation_publication_date=2008-04-29;,citation_cover_date=2008-04-29;,citation_year=2008;,citation_fulltext_html_url=https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.0060102;,citation_issue=4;,citation_doi=10.1371/journal.pbio.0060102;,citation_volume=6;,citation_language=en;,citation_journal_title=PLOS Biology;">
<meta name="citation_reference" content="citation_title=Predatory behaviour and taphonomy of a jurassic belemnoid coleoid (diplobelida, cephalopoda);,citation_author=Dominique Jenny;,citation_author=Dirk Fuchs;,citation_author=Alexander I. Arkhipkin;,citation_author=Rolf B. Hauff;,citation_author=Barbara Fritschi;,citation_author=Christian Klug;,citation_publication_date=2019-05-28;,citation_cover_date=2019-05-28;,citation_year=2019;,citation_fulltext_html_url=https://www.nature.com/articles/s41598-019-44260-w;,citation_issue=1;,citation_doi=10.1038/s41598-019-44260-w;,citation_volume=9;,citation_language=en;,citation_journal_title=Scientific Reports;">
<meta name="citation_reference" content="citation_title=Direct evidence of hybodont shark predation on late jurassic ammonites;,citation_author=Romain Vullo;,citation_publication_date=2011-06-01;,citation_cover_date=2011-06-01;,citation_year=2011;,citation_fulltext_html_url=https://doi.org/10.1007/s00114-011-0789-9;,citation_issue=6;,citation_doi=10.1007/s00114-011-0789-9;,citation_volume=98;,citation_language=en;,citation_journal_title=Naturwissenschaften;">
<meta name="citation_reference" content="citation_title=Chasing ecological interactions;,citation_author=Pedro Jordano;,citation_publication_date=2016-09-15;,citation_cover_date=2016-09-15;,citation_year=2016;,citation_fulltext_html_url=https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002559;,citation_issue=9;,citation_doi=10.1371/journal.pbio.1002559;,citation_volume=14;,citation_language=en;,citation_journal_title=PLOS Biology;">
<meta name="citation_reference" content="citation_title=Sampling networks of ecological interactions;,citation_author=Pedro Jordano;,citation_editor=Daniel Stouffer;,citation_publication_date=2016-12;,citation_cover_date=2016-12;,citation_year=2016;,citation_fulltext_html_url=http://doi.wiley.com/10.1111/1365-2435.12763;,citation_issue=12;,citation_doi=10.1111/1365-2435.12763;,citation_volume=30;,citation_language=en;,citation_journal_title=Functional Ecology;">
<meta name="citation_reference" content="citation_title=Analysing ecological networks of species interactions;,citation_author=Eva Delmas;,citation_author=Mathilde Besson;,citation_author=Marie-Hélène Brice;,citation_author=Laura A. Burkle;,citation_author=Giulio V. Dalla Riva;,citation_author=Marie-Josée Fortin;,citation_author=Dominique Gravel;,citation_author=Paulo R. Guimarães;,citation_author=David H. Hembry;,citation_author=Erica A. Newman;,citation_author=Jens M. Olesen;,citation_author=Mathias M. Pires;,citation_author=Justin D. Yeakel;,citation_author=Timothée Poisot;,citation_publication_date=2019;,citation_cover_date=2019;,citation_year=2019;,citation_fulltext_html_url=http://onlinelibrary.wiley.com/doi/abs/10.1111/brv.12433;,citation_issue=1;,citation_doi=10.1111/brv.12433;,citation_volume=94;,citation_language=en;,citation_journal_title=Biological Reviews;">
<meta name="citation_reference" content="citation_title=Stability criteria for complex ecosystems;,citation_author=Stefano Allesina;,citation_author=Si Tang;,citation_publication_date=2012-03;,citation_cover_date=2012-03;,citation_year=2012;,citation_fulltext_html_url=https://www.nature.com/articles/nature10832;,citation_issue=7388;,citation_doi=10.1038/nature10832;,citation_volume=483;,citation_language=en;,citation_journal_title=Nature;">
<meta name="citation_reference" content="citation_title=Coral reefs in crisis: The reliability of deep-time food web reconstructions as analogs for the present;,citation_author=Peter D. Roopnarine;,citation_author=Ashley A. Dineen;,citation_editor=Carrie L. Tyler;,citation_editor=Chris L. Schneider;,citation_publication_date=2018;,citation_cover_date=2018;,citation_year=2018;,citation_fulltext_html_url=https://doi.org/10.1007/978-3-319-73795-9_6;,citation_language=en;,citation_inbook_title=undefined;">
<meta name="citation_reference" content="citation_title=Complexity and fragility in ecological networks;,citation_author=Ricard V. Solé;,citation_author=M. Montoya;,citation_publication_date=2001-10-07;,citation_cover_date=2001-10-07;,citation_year=2001;,citation_fulltext_html_url=https://royalsocietypublishing.org/doi/10.1098/rspb.2001.1767;,citation_issue=1480;,citation_doi=10.1098/rspb.2001.1767;,citation_volume=268;,citation_language=en;,citation_journal_title=Proceedings of the Royal Society of London. Series B: Biological Sciences;">
<meta name="citation_reference" content="citation_title=Network structure and biodiversity loss in food webs: Robustness increases with connectance;,citation_author=Jennifer A. Dunne;,citation_author=Richard J Williams;,citation_author=Néo D Martinez;,citation_publication_date=2002;,citation_cover_date=2002;,citation_year=2002;,citation_issue=4;,citation_volume=5;,citation_journal_title=Ecol. Lett.;">
<meta name="citation_reference" content="citation_title=Approaching a state shift in earth’s biosphere;,citation_author=Anthony D. Barnosky;,citation_author=Elizabeth A. Hadly;,citation_author=Jordi Bascompte;,citation_author=Eric L. Berlow;,citation_author=James H. Brown;,citation_author=Mikael Fortelius;,citation_author=Wayne M. Getz;,citation_author=John Harte;,citation_author=Alan Hastings;,citation_author=Pablo A. Marquet;,citation_author=Neo D. Martinez;,citation_author=Arne Mooers;,citation_author=Peter Roopnarine;,citation_author=Geerat Vermeij;,citation_author=John W. Williams;,citation_author=Rosemary Gillespie;,citation_author=Justin Kitzes;,citation_author=Charles Marshall;,citation_author=Nicholas Matzke;,citation_author=David P. Mindell;,citation_author=Eloy Revilla;,citation_author=Adam B. Smith;,citation_publication_date=2012-06;,citation_cover_date=2012-06;,citation_year=2012;,citation_fulltext_html_url=https://www.nature.com/articles/nature11018;,citation_issue=7401;,citation_doi=10.1038/nature11018;,citation_volume=486;,citation_language=en;,citation_journal_title=Nature;">
<meta name="citation_reference" content="citation_title=Scaling from metawebs to realised webs: A hierarchical approach to network ecology;,citation_author=Tanya Strydom;,citation_author=Alexander M. Dunhill;,citation_author=Jennifer A. Dunne;,citation_author=Timothée Poisot;,citation_author=Andrew P. Beckerman;,citation_publication_date=2026-01-21;,citation_cover_date=2026-01-21;,citation_year=2026;,citation_fulltext_html_url=https://ecoevorxiv.org/repository/view/11514/;,citation_doi=10.32942/X2JW8K;,citation_language=en;,citation_journal_title=EcoEvoRxiv;">
<meta name="citation_reference" content="citation_title=Animal diversity and ecosystem functioning in dynamic food webs;,citation_author=Florian D. Schneider;,citation_author=Ulrich Brose;,citation_author=Björn C. Rall;,citation_author=Christian Guill;,citation_publication_date=2016-10-05;,citation_cover_date=2016-10-05;,citation_year=2016;,citation_fulltext_html_url=https://www.nature.com/articles/ncomms12718;,citation_issue=1;,citation_doi=10.1038/ncomms12718;,citation_volume=7;,citation_language=en;,citation_journal_title=Nature Communications;">
<meta name="citation_reference" content="citation_title=The effects of defaunation on plants’ capacity to track climate change;,citation_author=Evan C. Fricke;,citation_author=Alejandro Ordonez;,citation_author=Haldre S. Rogers;,citation_author=Jens-Christian Svenning;,citation_publication_date=2022-01-14;,citation_cover_date=2022-01-14;,citation_year=2022;,citation_fulltext_html_url=http://www.science.org/doi/full/10.1126/science.abk3510;,citation_issue=6577;,citation_doi=10.1126/science.abk3510;,citation_volume=375;,citation_journal_title=Science;">
<meta name="citation_reference" content="citation_title=Rescuing ecosystems from extinction cascades through compensatory perturbations;,citation_author=Sagar Sahasrabudhe;,citation_author=Adilson E. Motter;,citation_publication_date=2011-01-25;,citation_cover_date=2011-01-25;,citation_year=2011;,citation_fulltext_html_url=https://www.nature.com/articles/ncomms1163;,citation_issue=1;,citation_doi=10.1038/ncomms1163;,citation_volume=2;,citation_language=en;,citation_journal_title=Nature Communications;">
<meta name="citation_reference" content="citation_title=Stability criteria for complex ecosystems;,citation_author=Stefano Allesina;,citation_author=Si Tang;,citation_publication_date=2012-03;,citation_cover_date=2012-03;,citation_year=2012;,citation_fulltext_html_url=https://www.nature.com/articles/nature10832;,citation_issue=7388;,citation_doi=10.1038/nature10832;,citation_volume=483;,citation_language=en;,citation_journal_title=Nature;">
<meta name="citation_reference" content="citation_title=All ecological models are wrong, but some are useful;,citation_author=Daniel B. Stouffer;,citation_publication_date=2019;,citation_cover_date=2019;,citation_year=2019;,citation_fulltext_html_url=http://besjournals.onlinelibrary.wiley.com/doi/abs/10.1111/1365-2656.12949;,citation_issue=2;,citation_doi=https://doi.org/10.1111/1365-2656.12949;,citation_volume=88;,citation_language=en;,citation_journal_title=Journal of Animal Ecology;">
<meta name="citation_reference" content="citation_title=Consumerresource body-size relationships in natural food webs;,citation_author=Ulrich Brose;,citation_author=Tomas Jonsson;,citation_author=Eric L. Berlow;,citation_author=Philip Warren;,citation_author=Carolin Banasek-Richter;,citation_author=Louis-Félix Bersier;,citation_author=Julia L. Blanchard;,citation_author=Thomas Brey;,citation_author=Stephen R. Carpenter;,citation_author=Marie-France Cattin Blandenier;,citation_author=Lara Cushing;,citation_author=Hassan Ali Dawah;,citation_author=Tony Dell;,citation_author=Francois Edwards;,citation_author=Sarah Harper-Smith;,citation_author=Ute Jacob;,citation_author=Mark E. Ledger;,citation_author=Neo D. Martinez;,citation_author=Jane Memmott;,citation_author=Katja Mintenbeck;,citation_author=John K. Pinnegar;,citation_author=Björn C. Rall;,citation_author=Thomas S. Rayner;,citation_author=Daniel C. Reuman;,citation_author=Liliane Ruess;,citation_author=Werner Ulrich;,citation_author=Richard J. Williams;,citation_author=Guy Woodward;,citation_author=Joel E. Cohen;,citation_publication_date=2006;,citation_cover_date=2006;,citation_year=2006;,citation_fulltext_html_url=http://esajournals.onlinelibrary.wiley.com/doi/abs/10.1890/0012-9658%282006%2987%5B2411%3ACBRINF%5D2.0.CO%3B2;,citation_issue=10;,citation_doi=https://doi.org/10.1890/0012-9658(2006)87[2411:CBRINF]2.0.CO;2;,citation_volume=87;,citation_language=en;,citation_journal_title=Ecology;">
<meta name="citation_reference" content="citation_title=Collapse of an ecological network in ancient egypt;,citation_author=Justin D. Yeakel;,citation_author=Mathias M. Pires;,citation_author=Lars Rudolf;,citation_author=Nathaniel J. Dominy;,citation_author=Paul L. Koch;,citation_author=Paulo R. Guimarães;,citation_author=Thilo Gross;,citation_publication_date=2014-10;,citation_cover_date=2014-10;,citation_year=2014;,citation_issue=40;,citation_doi=10.1073/pnas.1408471111;,citation_volume=111;,citation_journal_title=PNAS;">
<meta name="citation_reference" content="citation_title=Extinction cascades, community collapse, and recovery across a mesozoic hyperthermal event;,citation_author=Alexander M. Dunhill;,citation_author=Karolina Zarzyczny;,citation_author=Jack O. Shaw;,citation_author=Jed W. Atkinson;,citation_author=Crispin T. S. Little;,citation_author=Andrew P. Beckerman;,citation_publication_date=2024-10-04;,citation_cover_date=2024-10-04;,citation_year=2024;,citation_fulltext_html_url=https://www.nature.com/articles/s41467-024-53000-2;,citation_issue=1;,citation_doi=10.1038/s41467-024-53000-2;,citation_volume=15;,citation_language=en;,citation_journal_title=Nature Communications;">
<meta name="citation_reference" content="citation_title=The structure of probabilistic networks;,citation_author=Timothée Poisot;,citation_author=Alyssa Cirtwill;,citation_author=Kévin Cazelles;,citation_author=Dominique Gravel;,citation_author=Marie-Josée Fortin;,citation_author=Daniel Stouffer;,citation_publication_date=2016;,citation_cover_date=2016;,citation_year=2016;,citation_issue=3;,citation_doi=10;,citation_volume=7;,citation_journal_title=Methods in Ecology and Evolution;">
<meta name="citation_reference" content="citation_title=Beyond species: Why ecological interaction networks vary through space and time;,citation_author=Timothée Poisot;,citation_author=Daniel B. Stouffer;,citation_author=Dominique Gravel;,citation_publication_date=2015;,citation_cover_date=2015;,citation_year=2015;,citation_fulltext_html_url=http://onlinelibrary.wiley.com/doi/abs/10.1111/oik.01719;,citation_issue=3;,citation_doi=10.1111/oik.01719;,citation_volume=124;,citation_language=en;,citation_journal_title=Oikos;">
<meta name="citation_reference" content="citation_title=Bringing elton and grinnell together: A quantitative framework to represent the biogeography of ecological interaction networks;,citation_author=Dominique Gravel;,citation_author=Benjamin Baiser;,citation_author=Jennifer A. Dunne;,citation_author=Jens-Peter Kopelke;,citation_author=Neo D. Martinez;,citation_author=Tommi Nyman;,citation_author=Timothée Poisot;,citation_author=Daniel B. Stouffer;,citation_author=Jason M. Tylianakis;,citation_author=Spencer A. Wood;,citation_author=Tomas Roslin;,citation_publication_date=2019;,citation_cover_date=2019;,citation_year=2019;,citation_fulltext_html_url=https://onlinelibrary.wiley.com/doi/abs/10.1111/ecog.04006;,citation_issue=3;,citation_doi=https://doi.org/10.1111/ecog.04006;,citation_volume=42;,citation_language=en;,citation_journal_title=Ecography;">
<meta name="citation_reference" content="citation_title=Quantitative patterns in the structure of model and empirical food webs;,citation_author=D. B. Stouffer;,citation_author=J. Camacho;,citation_author=R. Guimerà;,citation_author=C. A. Ng;,citation_author=L. A. Nunes Amaral;,citation_publication_date=2005;,citation_cover_date=2005;,citation_year=2005;,citation_fulltext_html_url=https://onlinelibrary.wiley.com/doi/abs/10.1890/04-0957;,citation_issue=5;,citation_doi=10.1890/04-0957;,citation_volume=86;,citation_language=en;,citation_journal_title=Ecology;">
<meta name="citation_reference" content="citation_title=Robustness to secondary extinctions: Comparing trait-based sequential deletions in static and dynamic food webs;,citation_author=Alva Curtsdotter;,citation_author=Amrei Binzer;,citation_author=Ulrich Brose;,citation_author=Francisco De Castro;,citation_author=Bo Ebenman;,citation_author=Anna Eklöf;,citation_author=Jens O. Riede;,citation_author=Aaron Thierry;,citation_author=Björn C. Rall;,citation_publication_date=2011-11;,citation_cover_date=2011-11;,citation_year=2011;,citation_fulltext_html_url=https://linkinghub.elsevier.com/retrieve/pii/S1439179111001216;,citation_issue=7;,citation_doi=10.1016/j.baae.2011.09.008;,citation_volume=12;,citation_language=en;,citation_journal_title=Basic and Applied Ecology;">
<meta name="citation_reference" content="citation_title=Collapse of terrestrial mammal food webs since the late pleistocene;,citation_author=Evan C. Fricke;,citation_author=Chia Hsieh;,citation_author=Owen Middleton;,citation_author=Daniel Gorczynski;,citation_author=Caroline D. Cappello;,citation_author=Oscar Sanisidro;,citation_author=John Rowan;,citation_author=Jens-Christian Svenning;,citation_author=Lydia Beaudrot;,citation_publication_date=2022-08-26;,citation_cover_date=2022-08-26;,citation_year=2022;,citation_fulltext_html_url=http://dx.doi.org/10.1126/science.abn4012;,citation_issue=6609;,citation_doi=10.1126/science.abn4012;,citation_volume=377;,citation_language=en;,citation_journal_title=Science;">
<meta name="citation_reference" content="citation_title=Collapse of terrestrial mammal food webs since the late pleistocene;,citation_author=Evan C. Fricke;,citation_author=Chia Hsieh;,citation_author=Owen Middleton;,citation_author=Daniel Gorczynski;,citation_author=Caroline D. Cappello;,citation_author=Oscar Sanisidro;,citation_author=John Rowan;,citation_author=Jens-Christian Svenning;,citation_author=Lydia Beaudrot;,citation_publication_date=2022-08-26;,citation_cover_date=2022-08-26;,citation_year=2022;,citation_fulltext_html_url=http://dx.doi.org/10.1126/science.abn4012;,citation_issue=6609;,citation_doi=10.1126/science.abn4012;,citation_volume=377;,citation_language=en;,citation_journal_title=Science;">
<meta name="citation_reference" content="citation_title=The dissimilarity of species interaction networks;,citation_author=Timothée Poisot;,citation_author=Elsa Canard;,citation_author=David Mouillot;,citation_author=Nicolas Mouquet;,citation_author=Dominique Gravel;,citation_editor=Ferenc Jordan;,citation_publication_date=2012-09-20;,citation_cover_date=2012-09-20;,citation_year=2012;,citation_fulltext_html_url=http://dx.doi.org/10.1111/ele.12002;,citation_issue=12;,citation_doi=10.1111/ele.12002;,citation_volume=15;,citation_language=en;,citation_journal_title=Ecology Letters;">
<meta name="citation_reference" content="citation_title=Telling ecological networks apart by their structure: A computational challenge;,citation_author=Matthew J. Michalska-Smith;,citation_author=Stefano Allesina;,citation_editor=Tobias Bollenbach;,citation_publication_date=2019-06-27;,citation_cover_date=2019-06-27;,citation_year=2019;,citation_fulltext_html_url=http://dx.doi.org/10.1371/journal.pcbi.1007076;,citation_issue=6;,citation_doi=10.1371/journal.pcbi.1007076;,citation_volume=15;,citation_language=en;,citation_journal_title=PLOS Computational Biology;">
<meta name="citation_reference" content="citation_title=Telling ecological networks apart by their structure: A computational challenge;,citation_author=Matthew J. Michalska-Smith;,citation_author=Stefano Allesina;,citation_editor=Tobias Bollenbach;,citation_publication_date=2019-06-27;,citation_cover_date=2019-06-27;,citation_year=2019;,citation_fulltext_html_url=http://dx.doi.org/10.1371/journal.pcbi.1007076;,citation_issue=6;,citation_doi=10.1371/journal.pcbi.1007076;,citation_volume=15;,citation_language=en;,citation_journal_title=PLOS Computational Biology;">
<meta name="citation_reference" content="citation_title=When is an ecological network complex? Connectance drives degree distribution and emerging network properties;,citation_author=Timothée Poisot;,citation_author=Dominique Gravel;,citation_publication_date=2014-02-04;,citation_cover_date=2014-02-04;,citation_year=2014;,citation_fulltext_html_url=https://peerj.com/articles/251;,citation_doi=10.7717/peerj.251;,citation_volume=2;,citation_language=en;,citation_journal_title=PeerJ;">
<meta name="citation_reference" content="citation_title=Spatial guilds in the serengeti food web revealed by a bayesian group model;,citation_author=Edward B. Baskerville;,citation_author=Andy P. Dobson;,citation_author=Trevor Bedford;,citation_author=Stefano Allesina;,citation_author=T. Michael Anderson;,citation_author=Mercedes Pascual;,citation_publication_date=2011-12-29;,citation_cover_date=2011-12-29;,citation_year=2011;,citation_fulltext_html_url=https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1002321;,citation_issue=12;,citation_doi=10.1371/journal.pcbi.1002321;,citation_volume=7;,citation_language=en;,citation_journal_title=PLOS Computational Biology;">
<meta name="citation_reference" content="citation_title=A hierarchical bayesian model for predicting ecological interactions using scaled evolutionary relationships;,citation_author=Mohamad Elmasri;,citation_author=Maxwell J. Farrell;,citation_author=T. Jonathan Davies;,citation_author=David A. Stephens;,citation_publication_date=2020-03;,citation_cover_date=2020-03;,citation_year=2020;,citation_fulltext_html_url=https://projecteuclid.org/euclid.aoas/1587002672;,citation_issue=1;,citation_doi=10.1214/19-AOAS1296;,citation_volume=14;,citation_language=en;,citation_journal_title=The Annals of Applied Statistics;">
<meta name="citation_reference" content="citation_title=The multilayer nature of ecological networks;,citation_author=Shai Pilosof;,citation_author=Mason A. Porter;,citation_author=Mercedes Pascual;,citation_author=Sonia Kéfi;,citation_publication_date=2017-03-23;,citation_cover_date=2017-03-23;,citation_year=2017;,citation_issue=4;,citation_doi=10.1038/s41559-017-0101;,citation_volume=1;,citation_language=eng;,citation_journal_title=Nature Ecology &amp;amp;amp; Evolution;">
<meta name="citation_reference" content="citation_title=Deciphering probabilistic species interaction networks;,citation_author=Francis Banville;,citation_author=Tanya Strydom;,citation_author=Penelope S. A. Blyth;,citation_author=Chris Brimacombe;,citation_author=Michael D. Catchen;,citation_author=Gabriel Dansereau;,citation_author=Gracielle Higino;,citation_author=Thomas Malpas;,citation_author=Hana Mayall;,citation_author=Kari Norman;,citation_author=Dominique Gravel;,citation_author=Timothée Poisot;,citation_publication_date=2025;,citation_cover_date=2025;,citation_year=2025;,citation_fulltext_html_url=https://onlinelibrary.wiley.com/doi/abs/10.1111/ele.70161;,citation_issue=6;,citation_doi=10.1111/ele.70161;,citation_volume=28;,citation_language=en;,citation_journal_title=Ecology Letters;">
<meta name="citation_reference" content="citation_title=Darwin’s entangled bank through deep time: Structural stability of mutualistic networks over large geographic and temporal scales;,citation_author=Benoît Perez-Lamarque;,citation_author=Jérémy Andréoletti;,citation_author=Baptiste Morillon;,citation_author=Orane Pion-Piola;,citation_author=Amaury Lambert;,citation_author=Hélène Morlon;,citation_publication_date=2026;,citation_cover_date=2026;,citation_year=2026;,citation_doi=10.1101/2025.10.08.681159;,citation_language=en;,citation_journal_title=EcoEvoRxiv;">
<meta name="citation_reference" content="citation_title=What constrains food webs? A maximum entropy framework for predicting their structure with minimal biases;,citation_author=Francis Banville;,citation_author=Dominique Gravel;,citation_author=Timothée Poisot;,citation_publication_date=2023-09-05;,citation_cover_date=2023-09-05;,citation_year=2023;,citation_fulltext_html_url=https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1011458;,citation_issue=9;,citation_doi=10.1371/journal.pcbi.1011458;,citation_volume=19;,citation_language=en;,citation_journal_title=PLOS Computational Biology;">
<meta name="citation_reference" content="citation_title=Ecological networks across environmental gradients;,citation_author=Jason M. Tylianakis;,citation_author=Rebecca J. Morris;,citation_publication_date=2017-11-02;,citation_cover_date=2017-11-02;,citation_year=2017;,citation_fulltext_html_url=http://dx.doi.org/10.1146/annurev-ecolsys-110316-022821;,citation_issue=1;,citation_doi=10.1146/annurev-ecolsys-110316-022821;,citation_volume=48;,citation_language=en;,citation_journal_title=Annual Review of Ecology, Evolution, and Systematics;">
<meta name="citation_reference" content="citation_title=Global knowledge gaps in species interaction networks data;,citation_author=Timothée Poisot;,citation_author=Gabriel Bergeron;,citation_author=Kevin Cazelles;,citation_author=Tad Dallas;,citation_author=Dominique Gravel;,citation_author=Andrew MacDonald;,citation_author=Benjamin Mercier;,citation_author=Clément Violet;,citation_author=Steve Vissault;,citation_publication_date=2021-04-29;,citation_cover_date=2021-04-29;,citation_year=2021;,citation_fulltext_html_url=https://onlinelibrary.wiley.com/doi/10.1111/jbi.14127;,citation_doi=10.1111/jbi.14127;,citation_language=en;,citation_journal_title=Journal of Biogeography;">
<meta name="citation_reference" content="citation_title=Scaling from metawebs to realised webs: A hierarchical approach to network ecology;,citation_author=Tanya Strydom;,citation_author=Alexander M. Dunhill;,citation_author=Jennifer A. Dunne;,citation_author=Timothée Poisot;,citation_author=Andrew P. Beckerman;,citation_publication_date=2026-01-21;,citation_cover_date=2026-01-21;,citation_year=2026;,citation_fulltext_html_url=https://ecoevorxiv.org/repository/view/11514/;,citation_doi=10.32942/X2JW8K;,citation_language=en;,citation_journal_title=EcoEvoRxiv;">
<meta name="citation_reference" content="citation_title=Tailoring interaction network types to answer different ecological questions;,citation_author=Benoit Gauzens;,citation_author=Lise Thouvenot;,citation_author=Diane S. Srivastava;,citation_author=Pavel Kratina;,citation_author=Gustavo Q. Romero;,citation_author=Emilio Berti;,citation_author=Eoin J. O’Gorman;,citation_author=Angélica L. González;,citation_author=Olivier Dézerald;,citation_author=Nico Eisenhauer;,citation_author=Mathias Pires;,citation_author=Remo Ryser;,citation_author=Vinicius F. Farjalla;,citation_author=Pierre Rogy;,citation_author=Ulrich Brose;,citation_author=Jana S. Petermann;,citation_author=Benoît Geslin;,citation_author=Jes Hines;,citation_publication_date=2025-06-13;,citation_cover_date=2025-06-13;,citation_year=2025;,citation_fulltext_html_url=https://www.nature.com/articles/s44358-025-00056-7;,citation_doi=10.1038/s44358-025-00056-7;,citation_language=en;,citation_journal_title=Nature Reviews Biodiversity;">
<meta name="citation_reference" content="citation_title=Improving ecological connectivity assessments with transfer learning and function approximation;,citation_author=Michael D. Catchen;,citation_author=Michelle Lin;,citation_author=Timothée Poisot;,citation_author=David Rolnick;,citation_author=Andrew Gonzalez;,citation_publication_date=2023-05-04;,citation_cover_date=2023-05-04;,citation_year=2023;,citation_fulltext_html_url=https://ecoevorxiv.org/repository/view/5348/;,citation_language=en;">
<meta name="citation_reference" content="citation_title=Sampling biases across interaction types affect the robustness of ecological multilayer networks;,citation_author=Hervías-Parejo Sandra;,citation_author=Anna Traveset;,citation_author=Manuel Nogales;,citation_author=Ruben Heleno;,citation_author=John Llewelyn;,citation_author=Giovanni Strona;,citation_publication_date=2025-05-02;,citation_cover_date=2025-05-02;,citation_year=2025;,citation_fulltext_html_url=https://www.sciencedirect.com/science/article/pii/S157495412500192X;,citation_doi=10.1016/j.ecoinf.2025.103183;,citation_journal_title=Ecological Informatics;">
<meta name="citation_reference" content="citation_title=When is an ecological network complex? Connectance drives degree distribution and emerging network properties;,citation_author=Timothée Poisot;,citation_author=Dominique Gravel;,citation_publication_date=2014-02-04;,citation_cover_date=2014-02-04;,citation_year=2014;,citation_fulltext_html_url=https://peerj.com/articles/251;,citation_doi=10.7717/peerj.251;,citation_volume=2;,citation_language=en;,citation_journal_title=PeerJ;">
<meta name="citation_reference" content="citation_title=How to make more out of community data? A conceptual framework and its implementation as models and software;,citation_author=Otso Ovaskainen;,citation_author=Gleb Tikhonov;,citation_author=Anna Norberg;,citation_author=F. Guillaume Blanchet;,citation_author=Leo Duan;,citation_author=David Dunson;,citation_author=Tomas Roslin;,citation_author=Nerea Abrego;,citation_publication_date=2017;,citation_cover_date=2017;,citation_year=2017;,citation_fulltext_html_url=https://onlinelibrary.wiley.com/doi/abs/10.1111/ele.12757;,citation_issue=5;,citation_doi=10.1111/ele.12757;,citation_volume=20;,citation_language=en;,citation_journal_title=Ecology Letters;">
<meta name="citation_reference" content="citation_title=A common framework for identifying linkage rules across different types of interactions;,citation_author=Ignasi Bartomeus;,citation_author=Dominique Gravel;,citation_author=Jason M. Tylianakis;,citation_author=Marcelo A. Aizen;,citation_author=Ian A. Dickie;,citation_author=Maud Bernard-Verdier;,citation_publication_date=2016;,citation_cover_date=2016;,citation_year=2016;,citation_fulltext_html_url=https://besjournals.onlinelibrary.wiley.com/doi/abs/10.1111/1365-2435.12666;,citation_issue=12;,citation_doi=10.1111/1365-2435.12666;,citation_volume=30;,citation_language=en;,citation_journal_title=Functional Ecology;">
<meta name="citation_reference" content="citation_title=Inferring food web structure from predatorprey body size relationships;,citation_author=Dominique Gravel;,citation_author=Timothée Poisot;,citation_author=Camille Albouy;,citation_author=Laure Velez;,citation_author=David Mouillot;,citation_publication_date=2013;,citation_cover_date=2013;,citation_year=2013;,citation_fulltext_html_url=https://besjournals.onlinelibrary.wiley.com/doi/abs/10.1111/2041-210X.12103;,citation_issue=11;,citation_doi=https://doi.org/10.1111/2041-210X.12103;,citation_volume=4;,citation_language=en;,citation_journal_title=Methods in Ecology and Evolution;">
<meta name="citation_reference" content="citation_title=Fit, efficiency, and biology: Some thoughts on judging food web models;,citation_author=Owen L. Petchey;,citation_author=Andrew P. Beckerman;,citation_author=Jens O. Riede;,citation_author=Philip H. Warren;,citation_publication_date=2011-06-21;,citation_cover_date=2011-06-21;,citation_year=2011;,citation_fulltext_html_url=https://www.sciencedirect.com/science/article/pii/S0022519311001652;,citation_issue=1;,citation_doi=10.1016/j.jtbi.2011.03.019;,citation_volume=279;,citation_journal_title=Journal of Theoretical Biology;">
<meta name="citation_reference" content="citation_title=The structure of ecological networks across levels of organization;,citation_author=Paulo R. Guimarães;,citation_publication_date=2020-11-02;,citation_cover_date=2020-11-02;,citation_year=2020;,citation_fulltext_html_url=https://www.annualreviews.org/doi/10.1146/annurev-ecolsys-012220-120819;,citation_issue=1;,citation_doi=10.1146/annurev-ecolsys-012220-120819;,citation_volume=51;,citation_language=en;,citation_journal_title=Annual Review of Ecology, Evolution, and Systematics;">
<meta name="citation_reference" content="citation_title=Ecological networks in motion: Micro- and macroscopic variability across scales;,citation_author=Kristian Trøjelsgaard;,citation_author=Jens M. Olesen;,citation_publication_date=2016;,citation_cover_date=2016;,citation_year=2016;,citation_fulltext_html_url=https://besjournals.onlinelibrary.wiley.com/doi/abs/10.1111/1365-2435.12710;,citation_issue=12;,citation_doi=https://doi.org/10.1111/1365-2435.12710;,citation_volume=30;,citation_language=en;,citation_journal_title=Functional Ecology;">
<meta name="citation_reference" content="citation_title=Beta diversity of plant-pollinator networks and the spatial turnover of pairwise interactions;,citation_author=Daniel W. Carstensen;,citation_author=Malena Sabatino;,citation_author=Kristian Trøjelsgaard;,citation_author=Leonor Patricia C. Morellato;,citation_publication_date=2014-11-10;,citation_cover_date=2014-11-10;,citation_year=2014;,citation_fulltext_html_url=https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0112903;,citation_issue=11;,citation_doi=10.1371/journal.pone.0112903;,citation_volume=9;,citation_language=en;,citation_journal_title=PLOS ONE;">
<meta name="citation_reference" content="citation_title=Sampling networks of ecological interactions;,citation_author=Pedro Jordano;,citation_publication_date=2016-09;,citation_cover_date=2016-09;,citation_year=2016;,citation_fulltext_html_url=http://doi.wiley.com/10.1111/1365-2435.12763;,citation_doi=10.1111/1365-2435.12763;,citation_language=en;,citation_journal_title=Functional Ecology;">
<meta name="citation_reference" content="citation_title=Sampling bias is a challenge for quantifying specialization and network structure: Lessons from a quantitative niche model;,citation_author=Jochen Fründ;,citation_author=Kevin S. McCann;,citation_author=Neal M. Williams;,citation_publication_date=2016;,citation_cover_date=2016;,citation_year=2016;,citation_fulltext_html_url=https://onlinelibrary.wiley.com/doi/abs/10.1111/oik.02256;,citation_issue=4;,citation_doi=10.1111/oik.02256;,citation_volume=125;,citation_language=en;,citation_journal_title=Oikos;">
<meta name="citation_reference" content="citation_title=Global hydroclimate perturbations during the toarcian oceanic anoxic event;,citation_author=David B. Kemp;,citation_author=Zhong Han;,citation_author=Xiumian Hu;,citation_author=Wenhan Chen;,citation_author=Simin Jin;,citation_author=Kentaro Izumi;,citation_author=Qing Yan;,citation_author=Viktória Baranyi;,citation_author=Xin Jin;,citation_author=Jacopo Dal Corso;,citation_author=Yuzhu Ge;,citation_publication_date=2024-11-01;,citation_cover_date=2024-11-01;,citation_year=2024;,citation_fulltext_html_url=https://www.sciencedirect.com/science/article/pii/S0012825224002745;,citation_doi=10.1016/j.earscirev.2024.104946;,citation_volume=258;,citation_journal_title=Earth-Science Reviews;">
</head>

<body class="quarto-light">

<header id="title-block-header" class="quarto-title-block default toc-left page-columns page-full">
  <div class="quarto-title-banner page-columns page-full">
    <div class="quarto-title column-body">
      <h1 class="title">Model structure conditions ecological inference in food web reconstruction</h1>
          </div>

    
    <div class="quarto-title-meta-container">
      <div class="quarto-title-meta-column-start">
            <div class="quarto-title-meta-author">
          <div class="quarto-title-meta-heading">Authors</div>
          <div class="quarto-title-meta-heading">Affiliations</div>
          
                <div class="quarto-title-meta-contents">
            <p class="author">Tanya Strydom <a href="mailto:t.strydom@sheffield.ac.uk" class="quarto-title-author-email"><i class="bi bi-envelope"></i></a> <a href="https://orcid.org/0000-0001-6067-1349" class="quarto-title-author-orcid"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAA2ZpVFh0WE1MOmNvbS5hZG9iZS54bXAAAAAAADw/eHBhY2tldCBiZWdpbj0i77u/IiBpZD0iVzVNME1wQ2VoaUh6cmVTek5UY3prYzlkIj8+IDx4OnhtcG1ldGEgeG1sbnM6eD0iYWRvYmU6bnM6bWV0YS8iIHg6eG1wdGs9IkFkb2JlIFhNUCBDb3JlIDUuMC1jMDYwIDYxLjEzNDc3NywgMjAxMC8wMi8xMi0xNzozMjowMCAgICAgICAgIj4gPHJkZjpSREYgeG1sbnM6cmRmPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5LzAyLzIyLXJkZi1zeW50YXgtbnMjIj4gPHJkZjpEZXNjcmlwdGlvbiByZGY6YWJvdXQ9IiIgeG1sbnM6eG1wTU09Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC9tbS8iIHhtbG5zOnN0UmVmPSJodHRwOi8vbnMuYWRvYmUuY29tL3hhcC8xLjAvc1R5cGUvUmVzb3VyY2VSZWYjIiB4bWxuczp4bXA9Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC8iIHhtcE1NOk9yaWdpbmFsRG9jdW1lbnRJRD0ieG1wLmRpZDo1N0NEMjA4MDI1MjA2ODExOTk0QzkzNTEzRjZEQTg1NyIgeG1wTU06RG9jdW1lbnRJRD0ieG1wLmRpZDozM0NDOEJGNEZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wTU06SW5zdGFuY2VJRD0ieG1wLmlpZDozM0NDOEJGM0ZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wOkNyZWF0b3JUb29sPSJBZG9iZSBQaG90b3Nob3AgQ1M1IE1hY2ludG9zaCI+IDx4bXBNTTpEZXJpdmVkRnJvbSBzdFJlZjppbnN0YW5jZUlEPSJ4bXAuaWlkOkZDN0YxMTc0MDcyMDY4MTE5NUZFRDc5MUM2MUUwNEREIiBzdFJlZjpkb2N1bWVudElEPSJ4bXAuZGlkOjU3Q0QyMDgwMjUyMDY4MTE5OTRDOTM1MTNGNkRBODU3Ii8+IDwvcmRmOkRlc2NyaXB0aW9uPiA8L3JkZjpSREY+IDwveDp4bXBtZXRhPiA8P3hwYWNrZXQgZW5kPSJyIj8+84NovQAAAR1JREFUeNpiZEADy85ZJgCpeCB2QJM6AMQLo4yOL0AWZETSqACk1gOxAQN+cAGIA4EGPQBxmJA0nwdpjjQ8xqArmczw5tMHXAaALDgP1QMxAGqzAAPxQACqh4ER6uf5MBlkm0X4EGayMfMw/Pr7Bd2gRBZogMFBrv01hisv5jLsv9nLAPIOMnjy8RDDyYctyAbFM2EJbRQw+aAWw/LzVgx7b+cwCHKqMhjJFCBLOzAR6+lXX84xnHjYyqAo5IUizkRCwIENQQckGSDGY4TVgAPEaraQr2a4/24bSuoExcJCfAEJihXkWDj3ZAKy9EJGaEo8T0QSxkjSwORsCAuDQCD+QILmD1A9kECEZgxDaEZhICIzGcIyEyOl2RkgwAAhkmC+eAm0TAAAAABJRU5ErkJggg=="></a></p>
          </div>
                <div class="quarto-title-meta-contents">
                    <p class="affiliation">
                        School of Biosciences, University of Sheffield, Sheffield, UK
                      </p>
                  </div>
                      <div class="quarto-title-meta-contents">
            <p class="author">Baran Karapunar <a href="https://orcid.org/0000-0001-9711-1492" class="quarto-title-author-orcid"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAA2ZpVFh0WE1MOmNvbS5hZG9iZS54bXAAAAAAADw/eHBhY2tldCBiZWdpbj0i77u/IiBpZD0iVzVNME1wQ2VoaUh6cmVTek5UY3prYzlkIj8+IDx4OnhtcG1ldGEgeG1sbnM6eD0iYWRvYmU6bnM6bWV0YS8iIHg6eG1wdGs9IkFkb2JlIFhNUCBDb3JlIDUuMC1jMDYwIDYxLjEzNDc3NywgMjAxMC8wMi8xMi0xNzozMjowMCAgICAgICAgIj4gPHJkZjpSREYgeG1sbnM6cmRmPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5LzAyLzIyLXJkZi1zeW50YXgtbnMjIj4gPHJkZjpEZXNjcmlwdGlvbiByZGY6YWJvdXQ9IiIgeG1sbnM6eG1wTU09Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC9tbS8iIHhtbG5zOnN0UmVmPSJodHRwOi8vbnMuYWRvYmUuY29tL3hhcC8xLjAvc1R5cGUvUmVzb3VyY2VSZWYjIiB4bWxuczp4bXA9Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC8iIHhtcE1NOk9yaWdpbmFsRG9jdW1lbnRJRD0ieG1wLmRpZDo1N0NEMjA4MDI1MjA2ODExOTk0QzkzNTEzRjZEQTg1NyIgeG1wTU06RG9jdW1lbnRJRD0ieG1wLmRpZDozM0NDOEJGNEZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wTU06SW5zdGFuY2VJRD0ieG1wLmlpZDozM0NDOEJGM0ZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wOkNyZWF0b3JUb29sPSJBZG9iZSBQaG90b3Nob3AgQ1M1IE1hY2ludG9zaCI+IDx4bXBNTTpEZXJpdmVkRnJvbSBzdFJlZjppbnN0YW5jZUlEPSJ4bXAuaWlkOkZDN0YxMTc0MDcyMDY4MTE5NUZFRDc5MUM2MUUwNEREIiBzdFJlZjpkb2N1bWVudElEPSJ4bXAuZGlkOjU3Q0QyMDgwMjUyMDY4MTE5OTRDOTM1MTNGNkRBODU3Ii8+IDwvcmRmOkRlc2NyaXB0aW9uPiA8L3JkZjpSREY+IDwveDp4bXBtZXRhPiA8P3hwYWNrZXQgZW5kPSJyIj8+84NovQAAAR1JREFUeNpiZEADy85ZJgCpeCB2QJM6AMQLo4yOL0AWZETSqACk1gOxAQN+cAGIA4EGPQBxmJA0nwdpjjQ8xqArmczw5tMHXAaALDgP1QMxAGqzAAPxQACqh4ER6uf5MBlkm0X4EGayMfMw/Pr7Bd2gRBZogMFBrv01hisv5jLsv9nLAPIOMnjy8RDDyYctyAbFM2EJbRQw+aAWw/LzVgx7b+cwCHKqMhjJFCBLOzAR6+lXX84xnHjYyqAo5IUizkRCwIENQQckGSDGY4TVgAPEaraQr2a4/24bSuoExcJCfAEJihXkWDj3ZAKy9EJGaEo8T0QSxkjSwORsCAuDQCD+QILmD1A9kECEZgxDaEZhICIzGcIyEyOl2RkgwAAhkmC+eAm0TAAAAABJRU5ErkJggg=="></a></p>
          </div>
                <div class="quarto-title-meta-contents">
                    <p class="affiliation">
                        School of Earth and Environment, University of Leeds, Leeds, UK
                      </p>
                  </div>
                      <div class="quarto-title-meta-contents">
            <p class="author">Andrew P. Beckerman <a href="https://orcid.org/0000-0002-7859-8394" class="quarto-title-author-orcid"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAA2ZpVFh0WE1MOmNvbS5hZG9iZS54bXAAAAAAADw/eHBhY2tldCBiZWdpbj0i77u/IiBpZD0iVzVNME1wQ2VoaUh6cmVTek5UY3prYzlkIj8+IDx4OnhtcG1ldGEgeG1sbnM6eD0iYWRvYmU6bnM6bWV0YS8iIHg6eG1wdGs9IkFkb2JlIFhNUCBDb3JlIDUuMC1jMDYwIDYxLjEzNDc3NywgMjAxMC8wMi8xMi0xNzozMjowMCAgICAgICAgIj4gPHJkZjpSREYgeG1sbnM6cmRmPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5LzAyLzIyLXJkZi1zeW50YXgtbnMjIj4gPHJkZjpEZXNjcmlwdGlvbiByZGY6YWJvdXQ9IiIgeG1sbnM6eG1wTU09Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC9tbS8iIHhtbG5zOnN0UmVmPSJodHRwOi8vbnMuYWRvYmUuY29tL3hhcC8xLjAvc1R5cGUvUmVzb3VyY2VSZWYjIiB4bWxuczp4bXA9Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC8iIHhtcE1NOk9yaWdpbmFsRG9jdW1lbnRJRD0ieG1wLmRpZDo1N0NEMjA4MDI1MjA2ODExOTk0QzkzNTEzRjZEQTg1NyIgeG1wTU06RG9jdW1lbnRJRD0ieG1wLmRpZDozM0NDOEJGNEZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wTU06SW5zdGFuY2VJRD0ieG1wLmlpZDozM0NDOEJGM0ZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wOkNyZWF0b3JUb29sPSJBZG9iZSBQaG90b3Nob3AgQ1M1IE1hY2ludG9zaCI+IDx4bXBNTTpEZXJpdmVkRnJvbSBzdFJlZjppbnN0YW5jZUlEPSJ4bXAuaWlkOkZDN0YxMTc0MDcyMDY4MTE5NUZFRDc5MUM2MUUwNEREIiBzdFJlZjpkb2N1bWVudElEPSJ4bXAuZGlkOjU3Q0QyMDgwMjUyMDY4MTE5OTRDOTM1MTNGNkRBODU3Ii8+IDwvcmRmOkRlc2NyaXB0aW9uPiA8L3JkZjpSREY+IDwveDp4bXBtZXRhPiA8P3hwYWNrZXQgZW5kPSJyIj8+84NovQAAAR1JREFUeNpiZEADy85ZJgCpeCB2QJM6AMQLo4yOL0AWZETSqACk1gOxAQN+cAGIA4EGPQBxmJA0nwdpjjQ8xqArmczw5tMHXAaALDgP1QMxAGqzAAPxQACqh4ER6uf5MBlkm0X4EGayMfMw/Pr7Bd2gRBZogMFBrv01hisv5jLsv9nLAPIOMnjy8RDDyYctyAbFM2EJbRQw+aAWw/LzVgx7b+cwCHKqMhjJFCBLOzAR6+lXX84xnHjYyqAo5IUizkRCwIENQQckGSDGY4TVgAPEaraQr2a4/24bSuoExcJCfAEJihXkWDj3ZAKy9EJGaEo8T0QSxkjSwORsCAuDQCD+QILmD1A9kECEZgxDaEZhICIzGcIyEyOl2RkgwAAhkmC+eAm0TAAAAABJRU5ErkJggg=="></a></p>
          </div>
                <div class="quarto-title-meta-contents">
                    <p class="affiliation">
                        School of Biosciences, University of Sheffield, Sheffield, UK
                      </p>
                  </div>
                      <div class="quarto-title-meta-contents">
            <p class="author">Alexander Dunhill <a href="https://orcid.org/0000-0002-8680-9163" class="quarto-title-author-orcid"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAA2ZpVFh0WE1MOmNvbS5hZG9iZS54bXAAAAAAADw/eHBhY2tldCBiZWdpbj0i77u/IiBpZD0iVzVNME1wQ2VoaUh6cmVTek5UY3prYzlkIj8+IDx4OnhtcG1ldGEgeG1sbnM6eD0iYWRvYmU6bnM6bWV0YS8iIHg6eG1wdGs9IkFkb2JlIFhNUCBDb3JlIDUuMC1jMDYwIDYxLjEzNDc3NywgMjAxMC8wMi8xMi0xNzozMjowMCAgICAgICAgIj4gPHJkZjpSREYgeG1sbnM6cmRmPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5LzAyLzIyLXJkZi1zeW50YXgtbnMjIj4gPHJkZjpEZXNjcmlwdGlvbiByZGY6YWJvdXQ9IiIgeG1sbnM6eG1wTU09Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC9tbS8iIHhtbG5zOnN0UmVmPSJodHRwOi8vbnMuYWRvYmUuY29tL3hhcC8xLjAvc1R5cGUvUmVzb3VyY2VSZWYjIiB4bWxuczp4bXA9Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC8iIHhtcE1NOk9yaWdpbmFsRG9jdW1lbnRJRD0ieG1wLmRpZDo1N0NEMjA4MDI1MjA2ODExOTk0QzkzNTEzRjZEQTg1NyIgeG1wTU06RG9jdW1lbnRJRD0ieG1wLmRpZDozM0NDOEJGNEZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wTU06SW5zdGFuY2VJRD0ieG1wLmlpZDozM0NDOEJGM0ZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wOkNyZWF0b3JUb29sPSJBZG9iZSBQaG90b3Nob3AgQ1M1IE1hY2ludG9zaCI+IDx4bXBNTTpEZXJpdmVkRnJvbSBzdFJlZjppbnN0YW5jZUlEPSJ4bXAuaWlkOkZDN0YxMTc0MDcyMDY4MTE5NUZFRDc5MUM2MUUwNEREIiBzdFJlZjpkb2N1bWVudElEPSJ4bXAuZGlkOjU3Q0QyMDgwMjUyMDY4MTE5OTRDOTM1MTNGNkRBODU3Ii8+IDwvcmRmOkRlc2NyaXB0aW9uPiA8L3JkZjpSREY+IDwveDp4bXBtZXRhPiA8P3hwYWNrZXQgZW5kPSJyIj8+84NovQAAAR1JREFUeNpiZEADy85ZJgCpeCB2QJM6AMQLo4yOL0AWZETSqACk1gOxAQN+cAGIA4EGPQBxmJA0nwdpjjQ8xqArmczw5tMHXAaALDgP1QMxAGqzAAPxQACqh4ER6uf5MBlkm0X4EGayMfMw/Pr7Bd2gRBZogMFBrv01hisv5jLsv9nLAPIOMnjy8RDDyYctyAbFM2EJbRQw+aAWw/LzVgx7b+cwCHKqMhjJFCBLOzAR6+lXX84xnHjYyqAo5IUizkRCwIENQQckGSDGY4TVgAPEaraQr2a4/24bSuoExcJCfAEJihXkWDj3ZAKy9EJGaEo8T0QSxkjSwORsCAuDQCD+QILmD1A9kECEZgxDaEZhICIzGcIyEyOl2RkgwAAhkmC+eAm0TAAAAABJRU5ErkJggg=="></a></p>
          </div>
                <div class="quarto-title-meta-contents">
                    <p class="affiliation">
                        School of Earth and Environment, University of Leeds, Leeds, UK
                      </p>
                  </div>
                    </div>
        
        <div class="quarto-title-meta">

                      
                <div>
            <div class="quarto-title-meta-heading">Published</div>
            <div class="quarto-title-meta-contents">
              <p class="date">February 25, 2026</p>
            </div>
          </div>
          
                
              </div>
      </div>
      <div class="quarto-title-meta-column-end quarto-other-formats-target">
      <div class="quarto-alternate-formats"><div class="quarto-title-meta-heading">Other Formats</div><div class="quarto-title-meta-contents"><p><a href="index.docx"><i class="bi bi-file-word"></i>MS Word</a></p></div><div class="quarto-title-meta-contents"><p><a href="index.pdf"><i class="bi bi-file-pdf"></i>PDF (preprint)</a></p></div></div></div>
    </div>

    <div>
      <div class="abstract">
        <div class="block-title">Abstract</div>
        <p>Aim</p>
        <p>Ecological networks are widely used to compare community structure, stability, and responses to disturbance across environmental gradients. However, many networks (particularly those assembled from incomplete interaction data) require model-based reconstruction. We test how alternative reconstruction frameworks condition ecological inference by quantifying their effects on network structure and disturbance dynamics.</p>
        <p>Location</p>
        <p>Cleveland Basin, United Kingdom.</p>
        <p>Time period</p>
        <p>Toarcian extinction event (Early Jurassic, late Pliensbachian–late Toarcian, ~183 Ma).</p>
        <p>Major taxa studied</p>
        <p>Marine animal communities.</p>
        <p>Methods</p>
        <p>We reconstructed four successive communities from an identical species pool using six contrasting food-web models spanning feasible (trait-based), realised (allometric and energetic), and structural (topological) network representations. For each community and model, 100 replicate networks were generated. We quantified macro-, meso-, and micro-scale network properties and assessed differences among models using multivariate analyses. Pairwise interaction turnover was measured using link-based beta diversity. We then simulated species loss under multiple disturbance scenarios, allowing cascading extinctions, and compared predicted community states using mean absolute differences and rank concordance metrics between models.</p>
        <p>Results</p>
        <p>Reconstruction framework strongly influenced inferred network topology (MANOVA, p &lt; 0.001), generating distinct structural signatures independent of species composition. Models that were similar in global metrics often diverged in species-level interactions, with high β-turnover among inferred link sets. During disturbance simulations, species-level vulnerability rankings were broadly consistent across models, but interaction-level outcomes and cascade dynamics varied substantially. Concordance in extinction-scenario rankings was scale dependent, with higher agreement at the species level than at the interaction level.</p>
        <p>Main conclusions</p>
        <p>Network reconstruction functions as a structural prior that conditions ecological inference. While some aggregate patterns are robust across modelling frameworks, detailed interaction-level dynamics are highly model contingent. Comparative network studies across spatial or environmental gradients should therefore align reconstruction framework with inferential goals and explicitly evaluate sensitivity to modelling assumptions.</p>
      </div>
    </div>

    <div>
      <div class="keywords">
        <div class="block-title">Keywords</div>
        <p>Ecological networks, Biotic interactions, Community assembly, Environmental gradients, Interaction turnover, Trophic organisation, Ecosystem resilience, Macroecology</p>
      </div>
    </div>

    <div class="quarto-other-links-text-target">
    <div class="quarto-code-links"><div class="quarto-title-meta-heading">Code Links</div><div class="quarto-title-meta-contents"><span><a href="https://github.com/TanyaS08/ms_paleo_webs/tree/main/code"><i class="bi bi-github"></i>GitHub</a></span></div></div></div>  </div>
</header><div id="quarto-content" class="page-columns page-rows-contents page-layout-article toc-left">
<div id="quarto-sidebar-toc-left" class="sidebar toc-left">
  <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">Table of contents</h2>
   
  <ul>
  <li><a href="#introduction" id="toc-introduction" class="nav-link active" data-scroll-target="#introduction"><span class="header-section-number">1</span> Introduction</a></li>
  <li><a href="#methods" id="toc-methods" class="nav-link" data-scroll-target="#methods"><span class="header-section-number">2</span> Methods</a>
  <ul class="collapse">
  <li><a href="#study-system-and-fossil-data" id="toc-study-system-and-fossil-data" class="nav-link" data-scroll-target="#study-system-and-fossil-data"><span class="header-section-number">2.1</span> Study system and fossil data</a></li>
  <li><a href="#network-reconstruction-approaches" id="toc-network-reconstruction-approaches" class="nav-link" data-scroll-target="#network-reconstruction-approaches"><span class="header-section-number">2.2</span> Network reconstruction approaches</a>
  <ul class="collapse">
  <li><a href="#conceptual-classification-of-network-types" id="toc-conceptual-classification-of-network-types" class="nav-link" data-scroll-target="#conceptual-classification-of-network-types"><span class="header-section-number">2.2.1</span> Conceptual classification of network types</a></li>
  <li><a href="#network-generation-and-replication" id="toc-network-generation-and-replication" class="nav-link" data-scroll-target="#network-generation-and-replication"><span class="header-section-number">2.2.2</span> Network generation and replication</a></li>
  </ul></li>
  <li><a href="#network-metrics-and-structural-analyses" id="toc-network-metrics-and-structural-analyses" class="nav-link" data-scroll-target="#network-metrics-and-structural-analyses"><span class="header-section-number">2.3</span> Network metrics and structural analyses</a></li>
  <li><a href="#extinction-simulations-and-model-evaluation" id="toc-extinction-simulations-and-model-evaluation" class="nav-link" data-scroll-target="#extinction-simulations-and-model-evaluation"><span class="header-section-number">2.4</span> Extinction simulations and model evaluation</a></li>
  </ul></li>
  <li><a href="#results" id="toc-results" class="nav-link" data-scroll-target="#results"><span class="header-section-number">3</span> Results</a>
  <ul class="collapse">
  <li><a href="#network-structure-differs-among-reconstruction-approaches" id="toc-network-structure-differs-among-reconstruction-approaches" class="nav-link" data-scroll-target="#network-structure-differs-among-reconstruction-approaches"><span class="header-section-number">3.1</span> Network structure differs among reconstruction approaches</a>
  <ul class="collapse">
  <li><a href="#variance-partitioning-of-network-structure" id="toc-variance-partitioning-of-network-structure" class="nav-link" data-scroll-target="#variance-partitioning-of-network-structure"><span class="header-section-number">3.1.1</span> Variance partitioning of network structure</a></li>
  <li><a href="#inferred-pairwise-interactions-vary-widely-among-models" id="toc-inferred-pairwise-interactions-vary-widely-among-models" class="nav-link" data-scroll-target="#inferred-pairwise-interactions-vary-widely-among-models"><span class="header-section-number">3.1.2</span> Inferred pairwise interactions vary widely among models</a></li>
  </ul></li>
  <li><a href="#model-choice-influences-inferred-extinction-dynamics" id="toc-model-choice-influences-inferred-extinction-dynamics" class="nav-link" data-scroll-target="#model-choice-influences-inferred-extinction-dynamics"><span class="header-section-number">3.2</span> Model choice influences inferred extinction dynamics</a></li>
  </ul></li>
  <li><a href="#discussion" id="toc-discussion" class="nav-link" data-scroll-target="#discussion"><span class="header-section-number">4</span> Discussion</a>
  <ul class="collapse">
  <li><a href="#network-reconstruction-is-not-neutral-structural-priors-shape-ecological-theory" id="toc-network-reconstruction-is-not-neutral-structural-priors-shape-ecological-theory" class="nav-link" data-scroll-target="#network-reconstruction-is-not-neutral-structural-priors-shape-ecological-theory"><span class="header-section-number">4.1</span> Network reconstruction is not neutral: structural priors shape ecological theory</a></li>
  <li><a href="#scale-dependent-robustness-in-network-based-inference" id="toc-scale-dependent-robustness-in-network-based-inference" class="nav-link" data-scroll-target="#scale-dependent-robustness-in-network-based-inference"><span class="header-section-number">4.2</span> Scale-dependent robustness in network-based inference</a></li>
  <li><a href="#implications-for-comparative-biogeography-and-global-change-research" id="toc-implications-for-comparative-biogeography-and-global-change-research" class="nav-link" data-scroll-target="#implications-for-comparative-biogeography-and-global-change-research"><span class="header-section-number">4.3</span> Implications for comparative biogeography and global change research</a></li>
  <li><a href="#toward-a-more-explicit-modelling-paradigm-in-food-web-ecology" id="toc-toward-a-more-explicit-modelling-paradigm-in-food-web-ecology" class="nav-link" data-scroll-target="#toward-a-more-explicit-modelling-paradigm-in-food-web-ecology"><span class="header-section-number">4.4</span> Toward a more explicit modelling paradigm in food web ecology</a></li>
  </ul></li>
  <li><a href="#conclusions" id="toc-conclusions" class="nav-link" data-scroll-target="#conclusions"><span class="header-section-number">5</span> Conclusions</a></li>
  <li><a href="#references" id="toc-references" class="nav-link" data-scroll-target="#references">References</a></li>
  </ul>
<div class="quarto-alternate-notebooks"><h2>Notebooks</h2><ul><li><a href="notebooks/model_summaries-preview.html"><i class="bi bi-journal-code"></i>SuppMat 1: Expanded descriptions of the food web models</a></li><li><a href="notebooks/results_expanded-preview.html"><i class="bi bi-journal-code"></i>SuppMat 2: Effects of Network Reconstruction on Food-Web Structure</a></li></ul></div></nav>
</div>
<div id="quarto-margin-sidebar" class="sidebar margin-sidebar zindex-bottom">
</div>
<main class="content quarto-banner-title-block" id="quarto-document-content">



  


<section id="introduction" class="level1" data-number="1">
<h1 data-number="1"><span class="header-section-number">1</span> Introduction</h1>
<p>Understanding how biodiversity is organised across space and time is a central goal of macroecology and biogeography. While early efforts focused primarily on species richness and composition, there is growing recognition that ecological communities are structured not only by which species occur, but by how they interact <span class="citation" data-cites="thuiller2024">(<a href="#ref-thuiller2024" role="doc-biblioref">Thuiller et al. 2024</a>)</span>. Interaction networks are increasingly treated as macroecological state variables where they are used to compare community organisation across environmental gradients, to quantify β-diversity in interaction structure, to evaluate stability–complexity relationships, and to infer vulnerability under global change <span class="citation" data-cites="tylianakis2017 poisot2015 gravel2019 trøjelsgaard2016 carstensen2014">(<a href="#ref-tylianakis2017" role="doc-biblioref">Tylianakis and Morris 2017</a>; <a href="#ref-poisot2015" role="doc-biblioref">Poisot et al. 2015</a>; <a href="#ref-gravel2019" role="doc-biblioref">Gravel et al. 2019</a>; <a href="#ref-trøjelsgaard2016" role="doc-biblioref">Trøjelsgaard and Olesen 2016</a>; <a href="#ref-carstensen2014" role="doc-biblioref">Carstensen et al. 2014</a>)</span>.</p>
<p>As a result, ecological networks now play a central role in comparative analyses spanning latitudinal gradients, disturbance regimes, and deep-time environmental transitions <span class="citation" data-cites="hao2025 michalska-smith2019 dunhill2024 roopnarine2006 poisot2014">(<a href="#ref-hao2025" role="doc-biblioref">Hao et al. 2025</a>; <a href="#ref-michalska-smith2019" role="doc-biblioref">Michalska-Smith and Allesina 2019</a>; <a href="#ref-dunhill2024" role="doc-biblioref">Dunhill et al. 2024</a>; <a href="#ref-roopnarine2006" role="doc-biblioref">Roopnarine 2006</a>; <a href="#ref-poisot2014" role="doc-biblioref">Poisot and Gravel 2014</a>)</span>. Implicit in this expansion is the critical assumption that network properties estimated across systems are structurally comparable, and that differences among them reflect ecological signal rather than methodological artefact <span class="citation" data-cites="jordano2016 fründ2016">(<a href="#ref-jordano2016" role="doc-biblioref">Jordano 2016</a>; <a href="#ref-fründ2016" role="doc-biblioref">Fründ et al. 2016</a>)</span>.</p>
<p>Yet most ecological networks are not fully observed as interaction data are incomplete and sampling is uneven across historical and biogeographic contexts, across both contemporary and deep-time <span class="citation" data-cites="catchen2023 sandra2025 poisot2021">(<a href="#ref-catchen2023" role="doc-biblioref">Catchen et al. 2023</a>; <a href="#ref-sandra2025" role="doc-biblioref">Sandra et al. 2025</a>; <a href="#ref-poisot2021" role="doc-biblioref">Poisot et al. 2021</a>)</span>. Interactions must often be inferred indirectly from traits, phylogeny, body size, co-occurrence, or theoretical constraints <span class="citation" data-cites="strydom2021 morales-castilla2015">(<a href="#ref-strydom2021" role="doc-biblioref">Strydom et al. 2021</a>; <a href="#ref-morales-castilla2015" role="doc-biblioref">Morales-Castilla et al. 2015</a>)</span>. Network construction therefore constitutes a model-based inference step rather than a purely descriptive exercise. Different reconstruction frameworks encode distinct ecological assumptions about how interactions arise - whether as biologically feasible combinations of traits, energetically optimised realised diets, or topological structures constrained by macroecological regularities. These assumptions act as structural priors over network architecture <span class="citation" data-cites="strydom2026 gauzens2025 petchey2011 guimarães2020">(<a href="#ref-strydom2026" role="doc-biblioref">Strydom et al. 2026</a>; <a href="#ref-gauzens2025" role="doc-biblioref">Gauzens et al. 2025</a>; <a href="#ref-petchey2011" role="doc-biblioref">Petchey et al. 2011</a>; <a href="#ref-guimarães2020" role="doc-biblioref">Guimarães 2020</a>)</span>. If alternative reconstruction models systematically generate different trophic configurations, then comparative analyses risk conflating ecological differences among communities with artefacts introduced by modelling choice. The reliability of macroecological inference therefore depends not only on ecological data, but on the structural assumptions embedded in network reconstruction.</p>
<p>Despite rapid methodological development in interaction inference, few studies have directly evaluated how alternative reconstruction frameworks condition macroecological conclusions when applied to the same species pool. This gap is particularly consequential for comparative research, where network metrics are routinely interpreted as indicators of environmental filtering, disturbance intensity, evolutionary history, or community stability <span class="citation" data-cites="delmas2018 allesina2012 poisot2015">(<a href="#ref-delmas2018" role="doc-biblioref">Delmas et al. 2018</a>; <a href="#ref-allesina2012" role="doc-biblioref">Allesina and Tang 2012</a>; <a href="#ref-poisot2015" role="doc-biblioref">Poisot et al. 2015</a>)</span>. If reconstruction models encode distinct structural priors over interaction topology, then differences among communities may reflect modelling assumptions rather than ecological processes. We therefore test whether macroecological inference derived from ecological networks is robust to variation in reconstruction framework, asking which aspects of network-based inference are stable across plausible representations of interaction structure and which are intrinsically model dependent.</p>
<p>Deep-time ecosystems provide an especially stringent test of this issue because interactions are not observed directly and must be reconstructed explicitly <span class="citation" data-cites="dunne2008 dunne2014 dunhill2024 roopnarine2006">(<a href="#ref-dunne2008" role="doc-biblioref">Dunne et al. 2008</a>; <a href="#ref-dunne2014" role="doc-biblioref">Dunne et al. 2014</a>; <a href="#ref-dunhill2024" role="doc-biblioref">Dunhill et al. 2024</a>; <a href="#ref-roopnarine2006" role="doc-biblioref">Roopnarine 2006</a>)</span>, rendering modelling assumptions transparent. Against this stringency, here we re-evaluate inferences made by <span class="citation" data-cites="dunhill2024">Dunhill et al. (<a href="#ref-dunhill2024" role="doc-biblioref">2024</a>)</span> on community structure and extinction dynamics during the early Toarcian extinction event (~183 Ma), a volcanic-driven hyperthermal and marine crisis in the Early Jurassic <span class="citation" data-cites="kemp2024">(<a href="#ref-kemp2024" role="doc-biblioref">Kemp et al. 2024</a>)</span>. Using four successive communities drawn from an identical taxon pool, we reconstruct ecological networks under six contrasting model classes spanning feasible, realised, and structural representations. For each reconstruction framework, we quantify emergent topology across scales, measure interaction turnover, and simulate disturbance-driven collapse. By holding species composition constant while varying the food web model used, this design isolates the influence of model constrained structure on inferred food web organisation and extinction dynamics, allowing us to distinguish ecological signals that are robust from those that are reconstruction contingent.</p>
</section>
<section id="methods" class="level1" data-number="2">
<h1 data-number="2"><span class="header-section-number">2</span> Methods</h1>
<section id="study-system-and-fossil-data" class="level2" data-number="2.1">
<h2 data-number="2.1" class="anchored" data-anchor-id="study-system-and-fossil-data"><span class="header-section-number">2.1</span> Study system and fossil data</h2>
<p>We used fossil occurrence data from the Cleveland Basin spanning the upper Pliensbachian to the upper Toarcian. This interval encompasses a major volcanic-driven hyperthermal and marine extinction event. To capture network dynamics across this transition, we defined four successive paleo-communities: Pre-extinction (Pliensbachian), Post-extinction (Lower Toarcian), Early recovery, and Late recovery (Middle/Upper Toarcian). Each taxon was characterized using their size and Bambach’s ecospace framework <span class="citation" data-cites="bambach2007">(<a href="#ref-bambach2007" role="doc-biblioref">Bambach et al. 2007</a>)</span>, coding for tiering, motility, and feeding mode as per <span class="citation" data-cites="dunhill2024">Dunhill et al. (<a href="#ref-dunhill2024" role="doc-biblioref">2024</a>)</span>. Each assemblage was treated as a community of potentially interacting taxa. The dataset includes 57 taxa across diverse groups (<em>e.g.,</em> cephalopods, bivalves, and gastropods). By restricting our analysis to a single basin with consistent lithofacies, we controlled for biogeographic noise, ensuring that structural shifts across the four time-bins reflect localised ecological responses to environmental stress rather than sampling or facies bias.</p>
</section>
<section id="network-reconstruction-approaches" class="level2" data-number="2.2">
<h2 data-number="2.2" class="anchored" data-anchor-id="network-reconstruction-approaches"><span class="header-section-number">2.2</span> Network reconstruction approaches</h2>
<section id="conceptual-classification-of-network-types" class="level3" data-number="2.2.1">
<h3 data-number="2.2.1" class="anchored" data-anchor-id="conceptual-classification-of-network-types"><span class="header-section-number">2.2.1</span> Conceptual classification of network types</h3>
<p>Most paleo-specific research (<em>e.g.,</em> <span class="citation" data-cites="fricke2022a">Fricke et al. (<a href="#ref-fricke2022a" role="doc-biblioref">2022</a>)</span>; <span class="citation" data-cites="roopnarine2006">Roopnarine (<a href="#ref-roopnarine2006" role="doc-biblioref">2006</a>)</span>; <span class="citation" data-cites="shaw2024">Shaw et al. (<a href="#ref-shaw2024" role="doc-biblioref">2024</a>)</span>) currently uses models from within the feasibility space. That is, the model reconstructions identify and encode the entire feasible diet of a species to build the network. These methods, however, represent only a subset of the broader spectrum of network construction approaches. Here, we present a suite of methods (<a href="#tbl-models" class="quarto-xref">Table&nbsp;1</a>) that enable the construction of a wider range of ecological networks and the exploration of a broader set of ecological questions, provided that their underlying assumptions are compatible with the constraints of fossil data. The methods include a) structural models that create species agnostic networks that are structurally ‘correct’ by assigning links between nodes based on assumptions of link distributions; and b) realised models that create networks where links between species are constrained based on some form of ‘species choice’ <em>e.g.,</em> maximising energy gain.</p>
<div id="tbl-models" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-tbl figure">
<figcaption class="quarto-float-caption-top quarto-float-caption quarto-float-tbl" id="tbl-models-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Table&nbsp;1: Six different models that can be used to construct food webs for both this specific community but are also broadly suited to paleo network prediction. These models span all facets of the network representation space (feasibility, realised, and structural network) and are suitable for an array of different paleo communities as the data requirements fall within the limitations set by the fossil record.
</figcaption>
<div aria-describedby="tbl-models-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<table class="caption-top table">
<colgroup>
<col style="width: 14%">
<col style="width: 14%">
<col style="width: 14%">
<col style="width: 14%">
<col style="width: 14%">
<col style="width: 14%">
<col style="width: 14%">
</colgroup>
<thead>
<tr class="header">
<th>Model</th>
<th>Assumptions</th>
<th>Data needs</th>
<th>Limitation</th>
<th>Network type</th>
<th>Key reference</th>
<th>Usage examples</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Random</td>
<td>Links assigned randomly</td>
<td>Species richness, number of links</td>
<td>Parameter assumptions, species agnostic</td>
<td>Structural</td>
<td><span class="citation" data-cites="erdos1959">Erdős and Rényi (<a href="#ref-erdos1959" role="doc-biblioref">1959</a>)</span></td>
<td>Null-model comparisons; testing whether observed network structure (connectance, motifs) deviates from random expectations</td>
</tr>
<tr class="even">
<td>Niche</td>
<td>Species ordered along a ‘niche axis’; interactions interval-constrained</td>
<td>Species richness, connectance</td>
<td>Parameter assumptions, species agnostic</td>
<td>Structural</td>
<td><span class="citation" data-cites="williams2008">Williams and Martinez (<a href="#ref-williams2008" role="doc-biblioref">2008</a>)</span></td>
<td>Evaluating trophic hierarchy and motif structure; baseline structural predictions</td>
</tr>
<tr class="odd">
<td>Allometric diet breadth model (ADBM)</td>
<td>Energy-maximizing predator diets</td>
<td>Body mass, abundance/biomass</td>
<td>Assumes optimal foraging; does not account for forbidden links</td>
<td>Realised</td>
<td><span class="citation" data-cites="petchey2008">Petchey et al. (<a href="#ref-petchey2008" role="doc-biblioref">2008</a>)</span></td>
<td>Predicting realized predator diets; exploring secondary extinctions</td>
</tr>
<tr class="even">
<td>Allometric trophic network (ATN)</td>
<td>Links constrained by body-size ratios and functional response</td>
<td>Body mass, number of basal species</td>
<td>Assumes only mechanical/energetic constraints</td>
<td>Realised</td>
<td><span class="citation" data-cites="brose2006">Brose et al. (<a href="#ref-brose2006" role="doc-biblioref">2006</a>)</span>; <span class="citation" data-cites="gauzens2023">Gauzens et al. (<a href="#ref-gauzens2023" role="doc-biblioref">2023</a>)</span></td>
<td>Simulating species loss; evaluating network collapse dynamics</td>
</tr>
<tr class="odd">
<td>Paleo food web inference model (PFIM)</td>
<td>Interactions inferred using trait-based mechanistic rules</td>
<td>Feeding traits</td>
<td>Assumes feeding mechanisms; trait resolution required</td>
<td>Feasibility</td>
<td><span class="citation" data-cites="shaw2024">Shaw et al. (<a href="#ref-shaw2024" role="doc-biblioref">2024</a>)</span></td>
<td>Mapping feasible trophic interactions; assessing secondary extinctions</td>
</tr>
<tr class="even">
<td>Body-size ratio model</td>
<td>Probabilistic assignment of links based on predator–prey size ratios</td>
<td>Body mass</td>
<td>Does not account for forbidden links</td>
<td>Realised</td>
<td><span class="citation" data-cites="rohr2010">Rohr et al. (<a href="#ref-rohr2010" role="doc-biblioref">2010</a>)</span></td>
<td>Estimating likely interactions; simulating cascading effects.</td>
</tr>
</tbody>
</table>
</div>
</figure>
</div>
<p>The three body mass-based models (ADBM, ATN, Body-size ratio) differ primarily in their underlying ecological assumptions. Although all three models use body mass to infer food web structure, they differ in their ecological assumptions. The ADBM is based on energy maximization under optimal foraging theory, the ATN constrains interactions via mechanically optimal consumer–resource size ratios, and the Body-size ratio model defines links probabilistically within a fixed allometric niche. Together, these approaches span bioenergetic, mechanical, and statistical interpretations of size-structured interactions.</p>
</section>
<section id="network-generation-and-replication" class="level3" data-number="2.2.2">
<h3 data-number="2.2.2" class="anchored" data-anchor-id="network-generation-and-replication"><span class="header-section-number">2.2.2</span> Network generation and replication</h3>
<p>We evaluated six models spanning this space <a href="#tbl-models" class="quarto-xref">Table&nbsp;1</a>: random and niche models (structural networks); allometric diet breadth (ADBM), allometric trophic network (ATN), and Body-size ratio models (realised networks); and a paleo food web inference model (PFIM; feasibility network). Expanded descriptions of model assumptions, parameterisation, and link-generation rules are provided in Supplementary Material S1. For each community, 100 networks were generated per model per successive community (n = 2400) to capture stochastic variation in link assignment. Where models required species body mass or trait values, these were sampled within biologically reasonable ranges to preserve relative differences among species. We adopted uniform sampling by default, as alternative distributions (lognormal, truncated lognormal) have negligible impact on topology (Supplementary Material S2; Figure S1). Structural models were parameterized using connectance values drawn from an empirically realistic range (0.07 – 0.34), with species richness held constant. Identical parameter draws were applied across comparable models within each replicate to ensure comparability. For the Body-size ratio model, we followed the approach of <span class="citation" data-cites="yeakel2014">Yeakel et al. (<a href="#ref-yeakel2014" role="doc-biblioref">2014</a>)</span> and excluded latent trait terms as opposed fitting the full model, which introduces additional inference and assumptions.</p>
</section>
</section>
<section id="network-metrics-and-structural-analyses" class="level2" data-number="2.3">
<h2 data-number="2.3" class="anchored" data-anchor-id="network-metrics-and-structural-analyses"><span class="header-section-number">2.3</span> Network metrics and structural analyses</h2>
<p>We quantified network structure using a suite of macro-, meso-, and micro-scale metrics <a href="#tbl-properties" class="quarto-xref">Table&nbsp;2</a>, capturing global properties, motif structure, and species-level variability. Differences among reconstruction approaches were assessed using a multivariate analysis of variance (MANOVA), with model identity as a fixed factor and the full set of network metrics as response variables. Pairwise interaction turnover was quantified using link‑based beta diversity, which measures dissimilarity in the identity of trophic links between networks, capturing differences due to species turnover or changes in interactions among shared species <span class="citation" data-cites="poisot2012a">(<a href="#ref-poisot2012a" role="doc-biblioref">Poisot et al. 2012</a>)</span>.</p>
<div id="tbl-properties" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-tbl figure">
<figcaption class="quarto-float-caption-top quarto-float-caption quarto-float-tbl" id="tbl-properties-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Table&nbsp;2: Network properties used for analysis.
</figcaption>
<div aria-describedby="tbl-properties-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<table class="caption-top table">
<colgroup>
<col style="width: 24%">
<col style="width: 26%">
<col style="width: 24%">
<col style="width: 24%">
</colgroup>
<thead>
<tr class="header">
<th>Metric</th>
<th>Definition</th>
<th>Scale</th>
<th>Reference (for maths), can make footnotes probs</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Richness</td>
<td>Number of nodes in the network</td>
<td>Macro</td>
<td></td>
</tr>
<tr class="even">
<td>Links</td>
<td>Normalized standard deviation of links (number of consumers plus resources per taxon)</td>
<td>Micro</td>
<td></td>
</tr>
<tr class="odd">
<td>Connectance</td>
<td><span class="math inline">\(L/S^2\)</span>, where <span class="math inline">\(S\)</span> is the number of species and <span class="math inline">\(L\)</span> the number of links</td>
<td>Macro</td>
<td></td>
</tr>
<tr class="even">
<td>Max trophic level</td>
<td>Prey-weighted trophic level averaged across taxa</td>
<td>Macro</td>
<td><span class="citation" data-cites="williams2004">Williams and Martinez (<a href="#ref-williams2004" role="doc-biblioref">2004</a>)</span></td>
</tr>
<tr class="odd">
<td>S1</td>
<td>Number of linear chains, normalised</td>
<td>Meso</td>
<td><span class="citation" data-cites="milo2002">Milo et al. (<a href="#ref-milo2002" role="doc-biblioref">2002</a>)</span>; <span class="citation" data-cites="stouffer2007">Stouffer et al. (<a href="#ref-stouffer2007" role="doc-biblioref">2007</a>)</span></td>
</tr>
<tr class="even">
<td>S2</td>
<td>Number of omnivory motifs, normalised</td>
<td>Meso</td>
<td><span class="citation" data-cites="milo2002">Milo et al. (<a href="#ref-milo2002" role="doc-biblioref">2002</a>)</span>; <span class="citation" data-cites="stouffer2007">Stouffer et al. (<a href="#ref-stouffer2007" role="doc-biblioref">2007</a>)</span></td>
</tr>
<tr class="odd">
<td>S4</td>
<td>Number of apparent competition motifs, normalised</td>
<td>Meso</td>
<td><span class="citation" data-cites="milo2002">Milo et al. (<a href="#ref-milo2002" role="doc-biblioref">2002</a>)</span>; <span class="citation" data-cites="stouffer2007">Stouffer et al. (<a href="#ref-stouffer2007" role="doc-biblioref">2007</a>)</span></td>
</tr>
<tr class="even">
<td>S5</td>
<td>Number of direct competition motifs, normalised</td>
<td>Meso</td>
<td><span class="citation" data-cites="milo2002">Milo et al. (<a href="#ref-milo2002" role="doc-biblioref">2002</a>)</span>; <span class="citation" data-cites="stouffer2007">Stouffer et al. (<a href="#ref-stouffer2007" role="doc-biblioref">2007</a>)</span></td>
</tr>
<tr class="odd">
<td>Generality</td>
<td>Normalized standard deviation of generality of a species, standardised by <span class="math inline">\(L/S\)</span></td>
<td>Micro</td>
<td><span class="citation" data-cites="williams2000">Williams and Martinez (<a href="#ref-williams2000" role="doc-biblioref">2000</a>)</span></td>
</tr>
<tr class="even">
<td>Vulnerability</td>
<td>Normalized standard deviation of vulnerability of a species, standardised by <span class="math inline">\(L/S\)</span></td>
<td>Micro</td>
<td><span class="citation" data-cites="williams2000">Williams and Martinez (<a href="#ref-williams2000" role="doc-biblioref">2000</a>)</span></td>
</tr>
</tbody>
</table>
</div>
</figure>
</div>
</section>
<section id="extinction-simulations-and-model-evaluation" class="level2" data-number="2.4">
<h2 data-number="2.4" class="anchored" data-anchor-id="extinction-simulations-and-model-evaluation"><span class="header-section-number">2.4</span> Extinction simulations and model evaluation</h2>
<p>Following <span class="citation" data-cites="dunhill2024">Dunhill et al. (<a href="#ref-dunhill2024" role="doc-biblioref">2024</a>)</span> and using the pre-extinction and post-extinction networks, we simulated species loss under multiple extinction scenarios including trait-based, network-position-based, and random removals, allowing cascading extinctions to propagate (cascading secondary extinctions). Simulated outcomes were compared to observed or expected community states using mean absolute differences (MAD) of food web metrics and modified true skill statistics (TSS) at node and link levels. Kendall’s rank correlation coefficient was used to evaluate concordance in scenario rankings across reconstruction models, providing a measure of robustness in inferred community responses.</p>
</section>
</section>
<section id="results" class="level1" data-number="3">
<h1 data-number="3"><span class="header-section-number">3</span> Results</h1>
<p>Results show that reconstruction approaches that appear similar when evaluated using global network metrics can yield fundamentally different ecological narratives when interrogated at the level of interactions and extinction dynamics. Across six network reconstruction approaches, inferred food web structure, species interactions, and extinction dynamics differed consistently. Multivariate analyses showed pronounced separation among models in network metric space. Reconstruction approach explained most of the variance in structural properties, leaving a distinct signature independent of community composition. Notably, agreement among models depended on the scale at which the comparison was made- approaches that were statistically similar in multivariate structural space (scale 1) often diverged in inferred interactions (scale 2) or extinction dynamics (scale 3). This demonstrates that structural similarity does not guarantee concordance in species-level diets or trophic roles.</p>
<p>Model choice substantially influenced inferred extinction dynamics. Temporal trajectories of network collapse, interaction loss, and motif reorganization differed among approaches. Although species-level extinction rankings were often broadly consistent, link-level outcomes and extinction inferences were highly sensitive to reconstruction assumptions. Together, these results show that ecological inferences drawn from networks depend critically on the reconstruction framework employed.</p>
<section id="network-structure-differs-among-reconstruction-approaches" class="level2" data-number="3.1">
<h2 data-number="3.1" class="anchored" data-anchor-id="network-structure-differs-among-reconstruction-approaches"><span class="header-section-number">3.1</span> Network structure differs among reconstruction approaches</h2>
<p>Across six reconstruction approaches, network structure (network properties listed in <a href="#tbl-properties" class="quarto-xref">Table&nbsp;2</a>) differed significantly (MANOVA, Pillai’s trace = 3.84, approximate <span class="math inline">\(F_{40,11955}\)</span> = 987.35, p &lt; 0.001), indicating that model choice systematically alters inferred food web topology. Canonical discriminant analysis identified two dominant axes of variation, explaining 86% of between-model variance. LD1 correlated with vulnerability, direct competition motifs, and connectance. LD2 correlated with maximum trophic level and apparent competition motifs, reflecting vertical trophic structure (Figure 1; Table S1, Figure S1). All higher-order canonical variates each explained less than 9% of the remaining variance.</p>
<div id="fig-structure" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-structure-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="figures/MANOVA_lda.png" class="lightbox" data-gallery="quarto-lightbox-gallery-1" title="Figure&nbsp;1: Linear discriminant analysis (LDA) of ecological network metrics for six model types. Each point represents a replicate, and ellipses indicate 95% confidence regions for each model. The second column represents the correlation of the various network metrics with the respective LDA axes."><img src="figures/MANOVA_lda.png" class="img-fluid figure-img"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-structure-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;1: Linear discriminant analysis (LDA) of ecological network metrics for six model types. Each point represents a replicate, and ellipses indicate 95% confidence regions for each model. The second column represents the correlation of the various network metrics with the respective LDA axes.
</figcaption>
</figure>
</div>
<section id="variance-partitioning-of-network-structure" class="level3" data-number="3.1.1">
<h3 data-number="3.1.1" class="anchored" data-anchor-id="variance-partitioning-of-network-structure"><span class="header-section-number">3.1.1</span> Variance partitioning of network structure</h3>
<p>Permutational multivariate analysis of variance revealed that reconstruction framework accounted for the majority of variation in multivariate network structure (R² = 0.795, p &lt; 0.001), whereas temporal turnover across extinction phases explained a comparatively small proportion of variance (R² = 0.064, p &lt; 0.001). The model × time interaction contributed a further 7.1% of variance (R² = 0.071, p &lt; 0.001), indicating limited but significant time-dependent divergence among reconstruction frameworks. Thus, differences among modelling approaches were more than an order of magnitude greater than structural differences associated with ecological turnover through the extinction sequence, even if the Toarcian dataset was characterized with a significant community turnover.</p>
<p>To determine whether the dominance of the reconstruction framework reflected absolute mean shifts among time bins, we repeated the analysis after centring network metrics within each extinction phase. This procedure removes between-phase differences while retaining within-phase structural variation. Even after temporal bin-standardised centring, the reconstruction framework explained 84.8% of multivariate variance (R² = 0.848, p &lt; 0.001). These results demonstrate that the influence of model choice is not driven by temporal mean differences, but reflects intrinsic divergence among reconstruction frameworks in how ecological interactions are organised.</p>
</section>
<section id="inferred-pairwise-interactions-vary-widely-among-models" class="level3" data-number="3.1.2">
<h3 data-number="3.1.2" class="anchored" data-anchor-id="inferred-pairwise-interactions-vary-widely-among-models"><span class="header-section-number">3.1.2</span> Inferred pairwise interactions vary widely among models</h3>
<p>Despite some models showing similar global metrics, specific pairwise interactions often differed. Pairwise β-turnover revealed that certain model pairs shared very few links <a href="#fig-beta_div" class="quarto-xref">Figure&nbsp;2</a>. Size-based models (ADBM, ATN) were broadly similar due to shared sole reliance on body-size constraints, whereas the Body-size ratio model exhibited consistently higher differences to other models. PFIM showed intermediate overlap with theoretical models. These results demonstrate that agreement in global network structure does not guarantee concordance in species-level interactions.</p>
<div id="fig-beta_div" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-beta_div-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="figures/beta_div.png" class="lightbox" data-gallery="quarto-lightbox-gallery-2" title="Figure&nbsp;2: Pairwise β-turnover in species interactions among four ecological network models (ADBM, ATN, Body-size ratio, and PFIM). Each cell represents the mean turnover value between a pair of models, with warmer colours indicating greater dissimilarity in inferred interactions. The diagonal is omitted. High turnover values (yellow) indicate strong disagreement in network structure between models, whereas lower values (blue–purple) indicate greater similarity."><img src="figures/beta_div.png" class="img-fluid figure-img"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-beta_div-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;2: Pairwise β-turnover in species interactions among four ecological network models (ADBM, ATN, Body-size ratio, and PFIM). Each cell represents the mean turnover value between a pair of models, with warmer colours indicating greater dissimilarity in inferred interactions. The diagonal is omitted. High turnover values (yellow) indicate strong disagreement in network structure between models, whereas lower values (blue–purple) indicate greater similarity.
</figcaption>
</figure>
</div>
</section>
</section>
<section id="model-choice-influences-inferred-extinction-dynamics" class="level2" data-number="3.2">
<h2 data-number="3.2" class="anchored" data-anchor-id="model-choice-influences-inferred-extinction-dynamics"><span class="header-section-number">3.2</span> Model choice influences inferred extinction dynamics</h2>
<p>To quantify how network structure changed over time during extinction simulations and whether these dynamics differed among reconstruction models, we fit generalized additive models (GAMs) to time series of network-level metrics. GAMs capture non-linear temporal trajectories, allowing formal tests of whether the shape of these trajectories differs among models. These model-specific temporal trajectories are shown in <a href="#fig-gam" class="quarto-xref">Figure&nbsp;3</a>. For all metrics examined, the inclusion of model-specific smooth terms significantly improved model fit (ANOVA model comparison: <span class="math inline">\(p\)</span> &lt; 0.001 for all metrics). Model-specific smooths differed not only in magnitude but also in the timing and abruptness of change, indicating distinct modes of collapse across reconstruction approaches (Tables S3–S4). Deterministic, data-driven approaches (PFIM) and allometric models (ADBM, ATN) exhibited highly non-linear trajectories, showing structural shifts in connectivity and motif frequency. In contrast, the Niche model produced the most consistent and gradual trajectories, effectively smoothing the perceived magnitude of structural change during community collapse. These results demonstrate that inferred pathways of collapse, trophic bottlenecks, and secondary extinctions are highly sensitive to model choice. Corresponding raw temporal trajectories are shown in Figure S2.</p>
<div id="fig-gam" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-gam-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="figures/GAM_predictions.png" class="lightbox" data-gallery="quarto-lightbox-gallery-3" title="Figure&nbsp;3: GAM-predicted trajectories of network structure during extinction simulations reveal pronounced differences in the timing and magnitude of change across reconstruction models. Lines show model-specific smooths and shaded areas indicate 95% confidence intervals. Deterministic approaches produce smoother, more consistent dynamics, whereas stochastic models exhibit greater variability, underscoring the sensitivity of inferred collapse pathways to reconstruction assumptions."><img src="figures/GAM_predictions.png" class="img-fluid figure-img"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-gam-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;3: GAM-predicted trajectories of network structure during extinction simulations reveal pronounced differences in the timing and magnitude of change across reconstruction models. Lines show model-specific smooths and shaded areas indicate 95% confidence intervals. Deterministic approaches produce smoother, more consistent dynamics, whereas stochastic models exhibit greater variability, underscoring the sensitivity of inferred collapse pathways to reconstruction assumptions.
</figcaption>
</figure>
</div>
<p>To evaluate how model choice affects inferred extinction dynamics, we compared simulated post-extinction networks to observed networks using mean absolute differences (MAD) for network-level metrics and true skills statistics (TSS) for node- and link-level outcomes <a href="#fig-mad" class="quarto-xref">Figure&nbsp;4</a>. Across models, MAD-based rankings were generally positively correlated (Kendall’s <span class="math inline">\(\tau\)</span> ≈ 0.13 across structural metrics), indicating broad agreement on the relative importance of extinction drivers despite substantial differences in reconstructed network structure. However, agreement within the allometric models differed from patterns observed for reconstructed network structure.</p>
<p>Node-level TSS rankings were similarly consistent across models (<span class="math inline">\(\tau\)</span> = 0.26–0.90), reflecting broadly comparable species removal sequences. In contrast, link-level outcomes were far more variable (<span class="math inline">\(\tau\)</span> = −0.48–0.29), highlighting that inferences about which interactions are lost, retained, or re-established during collapse and recovery are highly model contingent. Together, these results suggest that while alternative models converge on similar species-level extinction patterns, the inferred pathways of interaction loss and cascading dynamics depend strongly on both reconstruction approaches.</p>
<div id="fig-mad" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-mad-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="figures/kendal_tau.png" class="lightbox" data-gallery="quarto-lightbox-gallery-4" title="Figure&nbsp;4: Heatmaps showing pairwise Kendall rank correlation coefficients (\tau) between models for each network metric. Each panel corresponds to a different metric and displays the degree of agreement in extinction-scenario rankings across models based on mean absolute differences (MAD) between observed and predicted network values. Positive \tau values (blue) indicate concordant rankings between models, whereas negative \tau values (red) indicate opposing rankings. Warmer colours approaching zero represent little or no agreement. Panels illustrate how consistently different modelling approaches evaluate the relative realism of extinction scenarios across multiple network properties."><img src="figures/kendal_tau.png" class="img-fluid figure-img"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-mad-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;4: Heatmaps showing pairwise Kendall rank correlation coefficients (<span class="math inline">\(\tau\)</span>) between models for each network metric. Each panel corresponds to a different metric and displays the degree of agreement in extinction-scenario rankings across models based on mean absolute differences (MAD) between observed and predicted network values. Positive <span class="math inline">\(\tau\)</span> values (blue) indicate concordant rankings between models, whereas negative <span class="math inline">\(\tau\)</span> values (red) indicate opposing rankings. Warmer colours approaching zero represent little or no agreement. Panels illustrate how consistently different modelling approaches evaluate the relative realism of extinction scenarios across multiple network properties.
</figcaption>
</figure>
</div>
</section>
</section>
<section id="discussion" class="level1" data-number="4">
<h1 data-number="4"><span class="header-section-number">4</span> Discussion</h1>
<section id="network-reconstruction-is-not-neutral-structural-priors-shape-ecological-theory" class="level2" data-number="4.1">
<h2 data-number="4.1" class="anchored" data-anchor-id="network-reconstruction-is-not-neutral-structural-priors-shape-ecological-theory"><span class="header-section-number">4.1</span> Network reconstruction is not neutral: structural priors shape ecological theory</h2>
<p>Food web ecology has long treated network reconstruction as a technical step preceding ecological analysis. Once a network is assembled (whether from observation, inference, or simulation) its properties are typically analysed as reflections of underlying ecological organisation. Implicit in this workflow is a powerful assumption - that reconstructed networks provide structurally comparable representations of ecological communities, such that differences in connectance, trophic structure, motif composition, or robustness primarily reflect biological variation.</p>
<p>Our results challenge this assumption and reveal strong sensitivity of inference about secondary extinction dynamics in paleo-foodwebs. Across an identical species pool, reconstruction frameworks explained the majority of variance among reconstructed network topologies, generating distinct structural signatures independent of community composition. Although some model classes converged on similar global metrics, pairwise interaction turnover revealed substantial divergence in inferred trophic links. Structural similarity at the level of connectance or trophic height therefore does not guarantee concordance in species roles or interaction architecture.</p>
<p>Reconstruction framework explained the majority of variance in inferred food web topology, far outweighing the influence of temporal turnover across extinction phases. These patterns demonstrate that the divergence among reconstruction approaches reflects intrinsic differences in how models organise interactions, not temporal shifts in network structure. In other words, different reconstruction approaches impose distinct structural priors on inference of ecological organisation. These priors are not subtle - they propagate into emergent topology, species roles, and dynamical predictions. Network structure is therefore not solely a property of ecological communities; it is jointly determined by ecological data, modelling assumptions, and level of organisation <span class="citation" data-cites="pichler2023 strydom2021 strydom2026 gauzens2025 guimarães2020">(<a href="#ref-pichler2023" role="doc-biblioref">Pichler and Hartig 2023</a>; <a href="#ref-strydom2021" role="doc-biblioref">Strydom et al. 2021</a>; <a href="#ref-strydom2026" role="doc-biblioref">Strydom et al. 2026</a>; <a href="#ref-gauzens2025" role="doc-biblioref">Gauzens et al. 2025</a>; <a href="#ref-guimarães2020" role="doc-biblioref">Guimarães 2020</a>)</span>.</p>
<p>This has direct implications for the interpretation of comparative network studies. Feasible, realised, and structural models encode different assumptions about constraint, optimisation, and topology, and these assumptions propagate into emergent metrics and dynamical predictions <span class="citation" data-cites="allesina2012 solé2001 dunne2002 poisot2014 michalska-smith2019">(<a href="#ref-allesina2012" role="doc-biblioref">Allesina and Tang 2012</a>; <a href="#ref-solé2001" role="doc-biblioref">Solé and Montoya 2001</a>; <a href="#ref-dunne2002" role="doc-biblioref">Dunne et al. 2002</a>; <a href="#ref-poisot2014" role="doc-biblioref">Poisot and Gravel 2014</a>; <a href="#ref-michalska-smith2019" role="doc-biblioref">Michalska-Smith and Allesina 2019</a>)</span>. When networks reconstructed under different model classes are compared across spatial gradients, disturbance regimes, or evolutionary transitions, part of the observed variation may derive from structural priors rather than ecological process. Without explicit standardisation or sensitivity analysis, methodological heterogeneity can be mistaken for biological signal. Food web ecology has devoted substantial effort to understanding how topology shapes dynamics; comparatively less attention has been paid to how reconstruction method shapes topology. Our findings indicate that these two questions cannot be separated.</p>
</section>
<section id="scale-dependent-robustness-in-network-based-inference" class="level2" data-number="4.2">
<h2 data-number="4.2" class="anchored" data-anchor-id="scale-dependent-robustness-in-network-based-inference"><span class="header-section-number">4.2</span> Scale-dependent robustness in network-based inference</h2>
<p>Importantly, reconstruction sensitivity was not uniform across network scales (macro-, mesio-, micro- level properties). Species-level extinction rankings were broadly consistent among model classes, whereas interaction-level outcomes and cascade trajectories were highly contingent on structural prior. The predominance of reconstruction framework over temporal turnover (~80% vs.&nbsp;6% variance explained) illustrates why coarse-grained patterns like species-level extinction rankings are more robust. Model-imposed structure dominates the overall topology, leaving finer-scale interaction dynamics highly contingent on framework choice. This asymmetry reveals a context-dependent pattern of robustness. Coarse-grained macroecological patterns (such as the vulnerability of a community to collapse) can emerge from multiple plausible interaction architectures. By contrast, fine-grained inferences about which links are lost, retained, or reorganised depend strongly on how interactions are inferred.</p>
<p>This distinction challenges a central ambition of food web ecology - the use of detailed interaction structure to diagnose mechanisms of stability and collapse <span class="citation" data-cites="dunhill2024">(<em>e.g.,</em> <a href="#ref-dunhill2024" role="doc-biblioref">Dunhill et al. 2024</a>)</span>. If interaction-level cascade pathways vary substantially across equally plausible reconstructions, then mechanistic narratives derived from a single inferred topology may overstate their precision <span class="citation" data-cites="dunne2002 allesina2012 curtsdotter2011">(<a href="#ref-dunne2002" role="doc-biblioref">Dunne et al. 2002</a>; <a href="#ref-allesina2012" role="doc-biblioref">Allesina and Tang 2012</a>; <a href="#ref-curtsdotter2011" role="doc-biblioref">Curtsdotter et al. 2011</a>)</span>. The apparent determinism of extinction cascades or trophic bottlenecks may partly reflect model-imposed structure rather than ecological inevitability.</p>
<p>For macroecology, this metric dependence clarifies where network-based inference is accurate. Aggregate properties may be comparatively robust to reconstruction assumptions, whereas conclusions about interaction turnover, motif reorganisation, or fine-scale trophic dynamics are intrinsically uncertain. Recognising this asymmetry is essential if network analyses are to inform comparative synthesis across space and time.</p>
<p>Taken together, these results underscore that network reconstruction is not a neutral preprocessing step but part of a hypothesis-generating process in which each model encodes a distinct set of ecological assumptions. The inferred topology and dynamics of a food web therefore reflect not only ecological data, but the theoretical assumptions embedded in the reconstruction framework. Disagreement among models does not imply that any single approach is ‘wrong’, but rather that different models capture different facets of ecological reality <span class="citation" data-cites="stouffer2019 petchey2011">(<a href="#ref-stouffer2019" role="doc-biblioref">Stouffer 2019</a>; <a href="#ref-petchey2011" role="doc-biblioref">Petchey et al. 2011</a>)</span>. Disagreement among models does not imply that any single approach is ‘incorrect’. Rather, different models capture different facets of ecological constraint—trait compatibility, energetic optimisation, or topological regularity. The critical point is that these facets are not interchangeable.</p>
<p>This perspective reframes reconstruction choice as part of hypothesis specification. Researchers must align reconstruction approaches with the ecological signals of interest (whether potential interactions, realised diets, or macro-scale structural properties) rather than treating model selection as a technical convenience. Viewed through the lens of accuracy and precision, our results indicate that some network-based inferences are relatively robust across reconstruction approaches, whereas others remain intrinsically uncertain. High-level extinction rankings were broadly convergent, suggesting relative accuracy at coarse resolution, but interaction-level details and temporal cascade dynamics diverged substantially, indicating limited precision in reconstructing the fine structure of collapse. Recognising and explicitly accounting for this distinction is essential if food web ecology is to move beyond descriptive reconstruction toward rigorous comparative inference.</p>
</section>
<section id="implications-for-comparative-biogeography-and-global-change-research" class="level2" data-number="4.3">
<h2 data-number="4.3" class="anchored" data-anchor-id="implications-for-comparative-biogeography-and-global-change-research"><span class="header-section-number">4.3</span> Implications for comparative biogeography and global change research</h2>
<p>Network approaches are increasingly applied to examine how ecological organisation varies across latitudinal gradients, environmental filters, disturbance regimes, and climate-driven transitions. These studies frequently interpret variation in connectance, trophic height, interaction β-diversity, or robustness as indicators of ecological differentiation among regions or time intervals. Yet reconstruction frameworks often differ among systems due to variation in sampling intensity, trait resolution, or modelling preference. Our results show that such differences can systematically alter inferred topology and disturbance dynamics even when species composition is held constant.</p>
<p>This finding has consequences for interaction biogeography. Apparent differences in network turnover across spatial gradients may reflect divergence in structural priors rather than genuine ecological reorganisation. Similarly, projected differences in vulnerability under climate change may depend on how realised interactions are modelled, not solely on community composition. Without explicit treatment of reconstruction assumptions, comparative network analyses risk conflating modelling artefact with macroecological process.</p>
<p>If ecological networks are to function as reliable macroecological state variables, reconstruction framework must be treated as a formal component of hypothesis specification. Cross-system comparisons should standardise modelling approaches wherever possible, or explicitly evaluate sensitivity to alternative structural priors. Ensemble reconstruction—treating networks as distributions rather than single topologies—offers a principled pathway toward incorporating structural uncertainty into comparative synthesis.</p>
</section>
<section id="toward-a-more-explicit-modelling-paradigm-in-food-web-ecology" class="level2" data-number="4.4">
<h2 data-number="4.4" class="anchored" data-anchor-id="toward-a-more-explicit-modelling-paradigm-in-food-web-ecology"><span class="header-section-number">4.4</span> Toward a more explicit modelling paradigm in food web ecology</h2>
<p>The broader implication is not that any single reconstruction framework is ‘correct’ or ‘incorrect’. Rather, each model class represents a distinct hypothesis about how interactions are constrained—by trait compatibility, energetic optimisation, or topological regularity <span class="citation" data-cites="petchey2011">(<a href="#ref-petchey2011" role="doc-biblioref">Petchey et al. 2011</a>)</span>. Food web reconstruction is therefore theory-laden. Making this explicit shifts reconstruction from a preparatory step to a central component of ecological modelling.</p>
<p>A mature modelling paradigm in food web ecology would treat structural priors as testable assumptions, incorporate probabilistic link inference where possible, and quantify the sensitivity of macroecological conclusions to alternative representations of interaction structure. Such an approach aligns with recent advances in probabilistic and ensemble network modelling and would strengthen the interpretability of network-based inference under global change <span class="citation" data-cites="baskerville2011 elmasri2020 banville2025 poisot2016 perez-lamarque2026">(<a href="#ref-baskerville2011" role="doc-biblioref">Baskerville et al. 2011</a>; <a href="#ref-elmasri2020" role="doc-biblioref">Elmasri et al. 2020</a>; <a href="#ref-banville2025" role="doc-biblioref">Banville et al. 2025</a>; <a href="#ref-poisot2016" role="doc-biblioref">Poisot et al. 2016</a>; <a href="#ref-perez-lamarque2026" role="doc-biblioref">Perez-Lamarque et al. 2026</a>)</span>.</p>
</section>
</section>
<section id="conclusions" class="level1" data-number="5">
<h1 data-number="5"><span class="header-section-number">5</span> Conclusions</h1>
<p>Ecological network reconstruction is not a neutral technical procedure but a theoretical act that shapes ecological inference. By applying six contrasting reconstruction frameworks to an identical species pool, we show that structural priors systematically influence inferred food-web topology, interaction identity, and disturbance dynamics. Some coarse-grained patterns, such as relative species vulnerability, are comparatively robust across representations. In contrast, fine-scale interaction structure and cascade pathways are highly contingent on modelling assumptions. The reliability of network-based inference is therefore scale dependent.</p>
<p>These results challenge the implicit assumption that reconstructed networks are structurally commensurable across systems. When reconstruction frameworks differ, variation in connectance, trophic organisation, robustness, or interaction turnover may reflect embedded modelling assumptions as much as ecological processes. Network reconstruction should thus be treated as an explicit component of hypothesis specification in comparative macroecology and biogeography.</p>
<p>No single model captures the full complexity of ecological organisation, but neither are alternative models interchangeable. Aligning reconstruction framework with inferential goals, standardising approaches across comparative studies, and incorporating ensemble or probabilistic representations will be essential for strengthening the interpretability of network analyses across spatial and temporal gradients. As ecological networks increasingly inform global change research, recognising structural priors as fundamental determinants of inference is critical for advancing food web ecology from descriptive reconstruction toward rigorous comparative synthesis.</p>
</section>
<section id="references" class="level1 unnumbered">


</section>

<div id="quarto-appendix" class="default"><section class="quarto-appendix-contents" role="doc-bibliography" id="quarto-bibliography"><h2 class="anchored quarto-appendix-heading">References</h2><div id="refs" class="references csl-bib-body hanging-indent" role="list">
<div id="ref-allesina2012" class="csl-entry" role="listitem">
Allesina, Stefano, and Si Tang. 2012. <span>“Stability Criteria for Complex Ecosystems.”</span> <em>Nature</em> 483 (7388): 205–8. <a href="https://doi.org/10.1038/nature10832">https://doi.org/10.1038/nature10832</a>.
</div>
<div id="ref-bambach2007" class="csl-entry" role="listitem">
Bambach, Richard K., Andrew M. Bush, and Douglas H. Erwin. 2007. <span>“Autecology and the Filling of Ecospace: Key Metazoan Radiations.”</span> <em>Palaeontology</em> 50 (1): 1–22. <a href="https://doi.org/10.1111/j.1475-4983.2006.00611.x">https://doi.org/10.1111/j.1475-4983.2006.00611.x</a>.
</div>
<div id="ref-banville2025" class="csl-entry" role="listitem">
Banville, Francis, Tanya Strydom, Penelope S. A. Blyth, et al. 2025. <span>“Deciphering Probabilistic Species Interaction Networks.”</span> <em>Ecology Letters</em> 28 (6): e70161. <a href="https://doi.org/10.1111/ele.70161">https://doi.org/10.1111/ele.70161</a>.
</div>
<div id="ref-baskerville2011" class="csl-entry" role="listitem">
Baskerville, Edward B., Andy P. Dobson, Trevor Bedford, Stefano Allesina, T. Michael Anderson, and Mercedes Pascual. 2011. <span>“Spatial Guilds in the Serengeti Food Web Revealed by a Bayesian Group Model.”</span> <em>PLOS Computational Biology</em> 7 (12): e1002321. <a href="https://doi.org/10.1371/journal.pcbi.1002321">https://doi.org/10.1371/journal.pcbi.1002321</a>.
</div>
<div id="ref-brose2006" class="csl-entry" role="listitem">
Brose, Ulrich, Tomas Jonsson, Eric L. Berlow, et al. 2006. <span>“Consumer<span></span>resource Body-Size Relationships in Natural Food Webs.”</span> <em>Ecology</em> 87 (10): 2411–17. https://doi.org/<a href="https://doi.org/10.1890/0012-9658(2006)87[2411:CBRINF]2.0.CO;2">https://doi.org/10.1890/0012-9658(2006)87[2411:CBRINF]2.0.CO;2</a>.
</div>
<div id="ref-carstensen2014" class="csl-entry" role="listitem">
Carstensen, Daniel W., Malena Sabatino, Kristian Trøjelsgaard, and Leonor Patricia C. Morellato. 2014. <span>“Beta Diversity of Plant-Pollinator Networks and the Spatial Turnover of Pairwise Interactions.”</span> <em>PLOS ONE</em> 9 (11): e112903. <a href="https://doi.org/10.1371/journal.pone.0112903">https://doi.org/10.1371/journal.pone.0112903</a>.
</div>
<div id="ref-catchen2023" class="csl-entry" role="listitem">
Catchen, Michael D., Michelle Lin, Timothée Poisot, David Rolnick, and Andrew Gonzalez. 2023. <em>Improving Ecological Connectivity Assessments with Transfer Learning and Function Approximation</em>. May 4. <a href="https://ecoevorxiv.org/repository/view/5348/">https://ecoevorxiv.org/repository/view/5348/</a>.
</div>
<div id="ref-curtsdotter2011" class="csl-entry" role="listitem">
Curtsdotter, Alva, Amrei Binzer, Ulrich Brose, et al. 2011. <span>“Robustness to Secondary Extinctions: Comparing Trait-Based Sequential Deletions in Static and Dynamic Food Webs.”</span> <em>Basic and Applied Ecology</em> 12 (7): 571–80. <a href="https://doi.org/10.1016/j.baae.2011.09.008">https://doi.org/10.1016/j.baae.2011.09.008</a>.
</div>
<div id="ref-delmas2018" class="csl-entry" role="listitem">
Delmas, Eva, Mathilde Besson, Marie-Hélène Brice, et al. 2018. <span>“Analysing Ecological Networks of Species Interactions.”</span> <em>Biological Reviews</em>, June 20, 112540. <a href="https://doi.org/10.1111/brv.12433">https://doi.org/10.1111/brv.12433</a>.
</div>
<div id="ref-dunhill2024" class="csl-entry" role="listitem">
Dunhill, Alexander M., Karolina Zarzyczny, Jack O. Shaw, Jed W. Atkinson, Crispin T. S. Little, and Andrew P. Beckerman. 2024. <span>“Extinction Cascades, Community Collapse, and Recovery Across a Mesozoic Hyperthermal Event.”</span> <em>Nature Communications</em> 15 (1): 8599. <a href="https://doi.org/10.1038/s41467-024-53000-2">https://doi.org/10.1038/s41467-024-53000-2</a>.
</div>
<div id="ref-dunne2014" class="csl-entry" role="listitem">
Dunne, Jennifer A., Conrad C. Labandeira, and Richard J. Williams. 2014. <span>“Highly Resolved Early Eocene Food Webs Show Development of Modern Trophic Structure After the End-Cretaceous Extinction.”</span> <em>Proceedings of the Royal Society B: Biological Sciences</em> 281 (1782): 20133280. <a href="https://doi.org/10.1098/rspb.2013.3280">https://doi.org/10.1098/rspb.2013.3280</a>.
</div>
<div id="ref-dunne2008" class="csl-entry" role="listitem">
Dunne, Jennifer A., Richard J. Williams, Neo D. Martinez, Rachel A. Wood, and Douglas H. Erwin. 2008. <span>“Compilation and Network Analyses of Cambrian Food Webs.”</span> <em>PLOS Biology</em> 6 (4): e102. <a href="https://doi.org/10.1371/journal.pbio.0060102">https://doi.org/10.1371/journal.pbio.0060102</a>.
</div>
<div id="ref-dunne2002" class="csl-entry" role="listitem">
Dunne, Jennifer A., Richard J Williams, and Néo D Martinez. 2002. <span>“Network Structure and Biodiversity Loss in Food Webs: Robustness Increases with Connectance.”</span> <em>Ecol. Lett.</em> 5 (4): 558–67.
</div>
<div id="ref-elmasri2020" class="csl-entry" role="listitem">
Elmasri, Mohamad, Maxwell J. Farrell, T. Jonathan Davies, and David A. Stephens. 2020. <span>“A Hierarchical Bayesian Model for Predicting Ecological Interactions Using Scaled Evolutionary Relationships.”</span> <em>The Annals of Applied Statistics</em> 14 (1): 221–40. <a href="https://doi.org/10.1214/19-AOAS1296">https://doi.org/10.1214/19-AOAS1296</a>.
</div>
<div id="ref-erdos1959" class="csl-entry" role="listitem">
Erdős, P., and A. Rényi. 1959. <span>“On Random Graphs. i.”</span> <em>Publicationes Mathematicae Debrecen</em> 6 (3-4): 290–97. <a href="https://doi.org/10.5486/pmd.1959.6.3-4.12">https://doi.org/10.5486/pmd.1959.6.3-4.12</a>.
</div>
<div id="ref-fricke2022a" class="csl-entry" role="listitem">
Fricke, Evan C., Chia Hsieh, Owen Middleton, et al. 2022. <span>“Collapse of Terrestrial Mammal Food Webs Since the Late Pleistocene.”</span> <em>Science</em> 377 (6609): 1008–11. <a href="https://doi.org/10.1126/science.abn4012">https://doi.org/10.1126/science.abn4012</a>.
</div>
<div id="ref-fründ2016" class="csl-entry" role="listitem">
Fründ, Jochen, Kevin S. McCann, and Neal M. Williams. 2016. <span>“Sampling Bias Is a Challenge for Quantifying Specialization and Network Structure: Lessons from a Quantitative Niche Model.”</span> <em>Oikos</em> 125 (4): 502–13. <a href="https://doi.org/10.1111/oik.02256">https://doi.org/10.1111/oik.02256</a>.
</div>
<div id="ref-gauzens2023" class="csl-entry" role="listitem">
Gauzens, Benoit, Ulrich Brose, Eva Delmas, and Emilio Berti. 2023. <span>“ATNr: Allometric Trophic Network Models in r.”</span> <em>Methods in Ecology and Evolution</em> 14 (11): 2766–73. <a href="https://doi.org/10.1111/2041-210X.14212">https://doi.org/10.1111/2041-210X.14212</a>.
</div>
<div id="ref-gauzens2025" class="csl-entry" role="listitem">
Gauzens, Benoit, Lise Thouvenot, Diane S. Srivastava, et al. 2025. <span>“Tailoring Interaction Network Types to Answer Different Ecological Questions.”</span> <em>Nature Reviews Biodiversity</em>, June 13, 1–10. <a href="https://doi.org/10.1038/s44358-025-00056-7">https://doi.org/10.1038/s44358-025-00056-7</a>.
</div>
<div id="ref-gravel2019" class="csl-entry" role="listitem">
Gravel, Dominique, Benjamin Baiser, Jennifer A. Dunne, et al. 2019. <span>“Bringing Elton and Grinnell Together: A Quantitative Framework to Represent the Biogeography of Ecological Interaction Networks.”</span> <em>Ecography</em> 42 (3): 401–15. https://doi.org/<a href="https://doi.org/10.1111/ecog.04006">https://doi.org/10.1111/ecog.04006</a>.
</div>
<div id="ref-guimarães2020" class="csl-entry" role="listitem">
Guimarães, Paulo R. 2020. <span>“The Structure of Ecological Networks Across Levels of Organization.”</span> <em>Annual Review of Ecology, Evolution, and Systematics</em> 51 (1). <a href="https://doi.org/10.1146/annurev-ecolsys-012220-120819">https://doi.org/10.1146/annurev-ecolsys-012220-120819</a>.
</div>
<div id="ref-hao2025" class="csl-entry" role="listitem">
Hao, Xiyang, Marcel Holyoak, Zhicheng Zhang, and Chuan Yan. 2025. <span>“Global Projection of Terrestrial Vertebrate Food Webs Under Future Climate and Land-Use Changes.”</span> <em>Global Change Biology</em> 31 (2): e70061. <a href="https://doi.org/10.1111/gcb.70061">https://doi.org/10.1111/gcb.70061</a>.
</div>
<div id="ref-jordano2016" class="csl-entry" role="listitem">
Jordano, Pedro. 2016. <span>“Sampling Networks of Ecological Interactions.”</span> <em>Functional Ecology</em>, ahead of print, September. <a href="https://doi.org/10.1111/1365-2435.12763">https://doi.org/10.1111/1365-2435.12763</a>.
</div>
<div id="ref-kemp2024" class="csl-entry" role="listitem">
Kemp, David B., Zhong Han, Xiumian Hu, et al. 2024. <span>“Global Hydroclimate Perturbations During the Toarcian Oceanic Anoxic Event.”</span> <em>Earth-Science Reviews</em> 258 (November): 104946. <a href="https://doi.org/10.1016/j.earscirev.2024.104946">https://doi.org/10.1016/j.earscirev.2024.104946</a>.
</div>
<div id="ref-michalska-smith2019" class="csl-entry" role="listitem">
Michalska-Smith, Matthew J., and Stefano Allesina. 2019. <span>“Telling Ecological Networks Apart by Their Structure: A Computational Challenge.”</span> <em>PLOS Computational Biology</em> 15 (6): e1007076. <a href="https://doi.org/10.1371/journal.pcbi.1007076">https://doi.org/10.1371/journal.pcbi.1007076</a>.
</div>
<div id="ref-milo2002" class="csl-entry" role="listitem">
Milo, R., S. Shen-Orr, S. Itzkovitz, N. Kashtan, D. Chklovskii, and U. Alon. 2002. <span>“Network Motifs: Simple Building Blocks of Complex Networks.”</span> <em>Science</em> 298 (5594): 824–27. <a href="https://doi.org/10.1126/science.298.5594.824">https://doi.org/10.1126/science.298.5594.824</a>.
</div>
<div id="ref-morales-castilla2015" class="csl-entry" role="listitem">
Morales-Castilla, Ignacio, Miguel G. Matias, Dominique Gravel, and Miguel B. Araújo. 2015. <span>“Inferring Biotic Interactions from Proxies.”</span> <em>Trends in Ecology &amp; Evolution</em> 30 (6): 347–56. <a href="https://doi.org/10.1016/j.tree.2015.03.014">https://doi.org/10.1016/j.tree.2015.03.014</a>.
</div>
<div id="ref-perez-lamarque2026" class="csl-entry" role="listitem">
Perez-Lamarque, Benoît, Jérémy Andréoletti, Baptiste Morillon, Orane Pion-Piola, Amaury Lambert, and Hélène Morlon. 2026. <span>“Darwin<span>’</span>s Entangled Bank Through Deep Time: Structural Stability of Mutualistic Networks over Large Geographic and Temporal Scales.”</span> <em>EcoEvoRxiv</em>, ahead of print. <a href="https://doi.org/10.1101/2025.10.08.681159">https://doi.org/10.1101/2025.10.08.681159</a>.
</div>
<div id="ref-petchey2008" class="csl-entry" role="listitem">
Petchey, Owen L., Andrew P. Beckerman, Jens O. Riede, and Philip H. Warren. 2008. <span>“Size, Foraging, and Food Web Structure.”</span> <em>Proceedings of the National Academy of Sciences</em> 105 (11): 4191–96. <a href="https://doi.org/10.1073/pnas.0710672105">https://doi.org/10.1073/pnas.0710672105</a>.
</div>
<div id="ref-petchey2011" class="csl-entry" role="listitem">
Petchey, Owen L., Andrew P. Beckerman, Jens O. Riede, and Philip H. Warren. 2011. <span>“Fit, Efficiency, and Biology: Some Thoughts on Judging Food Web Models.”</span> <em>Journal of Theoretical Biology</em> 279 (1): 169–71. <a href="https://doi.org/10.1016/j.jtbi.2011.03.019">https://doi.org/10.1016/j.jtbi.2011.03.019</a>.
</div>
<div id="ref-pichler2023" class="csl-entry" role="listitem">
Pichler, Maximilian, and Florian Hartig. 2023. <span>“Machine Learning and Deep Learning<span></span>a Review for Ecologists.”</span> <em>Methods in Ecology and Evolution</em> 14 (4): 994–1016. <a href="https://doi.org/10.1111/2041-210X.14061">https://doi.org/10.1111/2041-210X.14061</a>.
</div>
<div id="ref-poisot2021" class="csl-entry" role="listitem">
Poisot, Timothée, Gabriel Bergeron, Kevin Cazelles, et al. 2021. <span>“Global Knowledge Gaps in Species Interaction Networks Data.”</span> <em>Journal of Biogeography</em>, April 29, jbi.14127. <a href="https://doi.org/10.1111/jbi.14127">https://doi.org/10.1111/jbi.14127</a>.
</div>
<div id="ref-poisot2012a" class="csl-entry" role="listitem">
Poisot, Timothée, Elsa Canard, David Mouillot, Nicolas Mouquet, and Dominique Gravel. 2012. <span>“The Dissimilarity of Species Interaction Networks.”</span> <em>Ecology Letters</em> 15 (12): 1353–61. <a href="https://doi.org/10.1111/ele.12002">https://doi.org/10.1111/ele.12002</a>.
</div>
<div id="ref-poisot2016" class="csl-entry" role="listitem">
Poisot, Timothée, Alyssa Cirtwill, Kévin Cazelles, Dominique Gravel, Marie-Josée Fortin, and Daniel Stouffer. 2016. <span>“The Structure of Probabilistic Networks.”</span> <em>Methods in Ecology and Evolution</em> 7 (3): 303312. <a href="https://doi.org/10">https://doi.org/10</a>.
</div>
<div id="ref-poisot2014" class="csl-entry" role="listitem">
Poisot, Timothée, and Dominique Gravel. 2014. <span>“When Is an Ecological Network Complex? Connectance Drives Degree Distribution and Emerging Network Properties.”</span> <em>PeerJ</em> 2 (February): e251. <a href="https://doi.org/10.7717/peerj.251">https://doi.org/10.7717/peerj.251</a>.
</div>
<div id="ref-poisot2015" class="csl-entry" role="listitem">
Poisot, Timothée, Daniel B. Stouffer, and Dominique Gravel. 2015. <span>“Beyond Species: Why Ecological Interaction Networks Vary Through Space and Time.”</span> <em>Oikos</em> 124 (3): 243–51. <a href="https://doi.org/10.1111/oik.01719">https://doi.org/10.1111/oik.01719</a>.
</div>
<div id="ref-rohr2010" class="csl-entry" role="listitem">
Rohr, Rudolf&nbsp;Philippe, Heike Scherer, Patrik Kehrli, Christian Mazza, and Louis-Félix Bersier. 2010. <span>“Modeling Food Webs: Exploring Unexplained Structure Using Latent Traits.”</span> <em>The American Naturalist</em> 176 (2): 170–77. <a href="https://doi.org/10.1086/653667">https://doi.org/10.1086/653667</a>.
</div>
<div id="ref-roopnarine2006" class="csl-entry" role="listitem">
Roopnarine, Peter D. 2006. <span>“Extinction Cascades and Catastrophe in Ancient Food Webs.”</span> <em>Paleobiology</em> 32 (1): 1–19. <a href="https://www.jstor.org/stable/4096814">https://www.jstor.org/stable/4096814</a>.
</div>
<div id="ref-sandra2025" class="csl-entry" role="listitem">
Sandra, Hervías-Parejo, Anna Traveset, Manuel Nogales, Ruben Heleno, John Llewelyn, and Giovanni Strona. 2025. <span>“Sampling Biases Across Interaction Types Affect the Robustness of Ecological Multilayer Networks.”</span> <em>Ecological Informatics</em>, May 2, 103183. <a href="https://doi.org/10.1016/j.ecoinf.2025.103183">https://doi.org/10.1016/j.ecoinf.2025.103183</a>.
</div>
<div id="ref-shaw2024" class="csl-entry" role="listitem">
Shaw, Jack O., Alexander M. Dunhill, Andrew P. Beckerman, Jennifer A. Dunne, and Pincelli M. Hull. 2024. <em>A Framework for Reconstructing Ancient Food Webs Using Functional Trait Data</em>. bioRxiv. <a href="https://doi.org/10.1101/2024.01.30.578036">https://doi.org/10.1101/2024.01.30.578036</a>.
</div>
<div id="ref-solé2001" class="csl-entry" role="listitem">
Solé, Ricard V., and M. Montoya. 2001. <span>“Complexity and Fragility in Ecological Networks.”</span> <em>Proceedings of the Royal Society of London. Series B: Biological Sciences</em> 268 (1480): 2039–45. <a href="https://doi.org/10.1098/rspb.2001.1767">https://doi.org/10.1098/rspb.2001.1767</a>.
</div>
<div id="ref-stouffer2019" class="csl-entry" role="listitem">
Stouffer, Daniel B. 2019. <span>“All Ecological Models Are Wrong, but Some Are Useful.”</span> <em>Journal of Animal Ecology</em> 88 (2): 192–95. https://doi.org/<a href="https://doi.org/10.1111/1365-2656.12949">https://doi.org/10.1111/1365-2656.12949</a>.
</div>
<div id="ref-stouffer2007" class="csl-entry" role="listitem">
Stouffer, Daniel B, Juan Camacho, Wenxin Jiang, and Luís A Nunes Amaral. 2007. <span>“Evidence for the Existence of a Robust Pattern of Prey Selection in Food Webs.”</span> <em>Proceedings of the Royal Society B: Biological Sciences</em> 274 (1621): 1931–40. <a href="https://doi.org/10.1098/rspb.2007.0571">https://doi.org/10.1098/rspb.2007.0571</a>.
</div>
<div id="ref-strydom2021" class="csl-entry" role="listitem">
Strydom, Tanya, Michael D. Catchen, Francis Banville, et al. 2021. <span>“A Roadmap Towards Predicting Species Interaction Networks (Across Space and Time).”</span> <em>Philosophical Transactions of the Royal Society B: Biological Sciences</em> 376 (1837): 20210063. <a href="https://doi.org/10.1098/rstb.2021.0063">https://doi.org/10.1098/rstb.2021.0063</a>.
</div>
<div id="ref-strydom2026" class="csl-entry" role="listitem">
Strydom, Tanya, Alexander M. Dunhill, Jennifer A. Dunne, Timothée Poisot, and Andrew P. Beckerman. 2026. <span>“Scaling from Metawebs to Realised Webs: A Hierarchical Approach to Network Ecology.”</span> <em>EcoEvoRxiv</em>, ahead of print, January 21. <a href="https://doi.org/10.32942/X2JW8K">https://doi.org/10.32942/X2JW8K</a>.
</div>
<div id="ref-thuiller2024" class="csl-entry" role="listitem">
Thuiller, Wilfried, Irene Calderón-Sanou, Loïc Chalmandrier, et al. 2024. <span>“Navigating the Integration of Biotic Interactions in Biogeography.”</span> <em>Journal of Biogeography</em> 51 (4): 550–59. <a href="https://doi.org/10.1111/jbi.14734">https://doi.org/10.1111/jbi.14734</a>.
</div>
<div id="ref-trøjelsgaard2016" class="csl-entry" role="listitem">
Trøjelsgaard, Kristian, and Jens M. Olesen. 2016. <span>“Ecological Networks in Motion: Micro- and Macroscopic Variability Across Scales.”</span> <em>Functional Ecology</em> 30 (12): 1926–35. https://doi.org/<a href="https://doi.org/10.1111/1365-2435.12710">https://doi.org/10.1111/1365-2435.12710</a>.
</div>
<div id="ref-tylianakis2017" class="csl-entry" role="listitem">
Tylianakis, Jason M., and Rebecca J. Morris. 2017. <span>“Ecological Networks Across Environmental Gradients.”</span> <em>Annual Review of Ecology, Evolution, and Systematics</em> 48 (1): 25–48. <a href="https://doi.org/10.1146/annurev-ecolsys-110316-022821">https://doi.org/10.1146/annurev-ecolsys-110316-022821</a>.
</div>
<div id="ref-williams2004" class="csl-entry" role="listitem">
Williams, R. J., and N. D. Martinez. 2004. <span>“Stabilization of Chaotic and Non-Permanent Food-Web Dynamics.”</span> <em>The European Physical Journal B - Condensed Matter</em> 38 (2): 297–303. <a href="https://doi.org/10.1140/epjb/e2004-00122-1">https://doi.org/10.1140/epjb/e2004-00122-1</a>.
</div>
<div id="ref-williams2000" class="csl-entry" role="listitem">
Williams, Richard J., and Neo D. Martinez. 2000. <span>“Simple Rules Yield Complex Food Webs.”</span> <em>Nature</em> 404 (6774): 180–83. <a href="https://doi.org/10.1038/35004572">https://doi.org/10.1038/35004572</a>.
</div>
<div id="ref-williams2008" class="csl-entry" role="listitem">
Williams, Richard J., and Neo D. Martinez. 2008. <span>“Success and its limits among structural models of complex food webs.”</span> <em>The Journal of Animal Ecology</em> 77 (3): 512–19. <a href="https://doi.org/10.1111/j.1365-2656.2008.01362.x">https://doi.org/10.1111/j.1365-2656.2008.01362.x</a>.
</div>
<div id="ref-yeakel2014" class="csl-entry" role="listitem">
Yeakel, Justin D., Mathias M. Pires, Lars Rudolf, et al. 2014. <span>“Collapse of an Ecological Network in Ancient Egypt.”</span> <em>PNAS</em> 111 (40): 14472–77. <a href="https://doi.org/10.1073/pnas.1408471111">https://doi.org/10.1073/pnas.1408471111</a>.
</div>
</div></section><section class="quarto-appendix-contents" id="quarto-citation"><h2 class="anchored quarto-appendix-heading">Citation</h2><div><div class="quarto-appendix-secondary-label">BibTeX citation:</div><pre class="sourceCode code-with-copy quarto-appendix-bibtex"><code class="sourceCode bibtex">@online{strydom2026,
  author = {Strydom, Tanya and Karapunar, Baran and P. Beckerman, Andrew
    and Dunhill, Alexander},
  title = {Model Structure Conditions Ecological Inference in Food Web
    Reconstruction},
  date = {2026-02-25},
  langid = {en},
  abstract = {Aim Ecological networks are widely used to compare
    community structure, stability, and responses to disturbance across
    environmental gradients. However, many networks (particularly those
    assembled from incomplete interaction data) require model-based
    reconstruction. We test how alternative reconstruction frameworks
    condition ecological inference by quantifying their effects on
    network structure and disturbance dynamics. Location Cleveland
    Basin, United Kingdom. Time period Toarcian extinction event (Early
    Jurassic, late Pliensbachian–late Toarcian, \textasciitilde183 Ma).
    Major taxa studied Marine animal communities. Methods We
    reconstructed four successive communities from an identical species
    pool using six contrasting food-web models spanning feasible
    (trait-based), realised (allometric and energetic), and structural
    (topological) network representations. For each community and model,
    100 replicate networks were generated. We quantified macro-, meso-,
    and micro-scale network properties and assessed differences among
    models using multivariate analyses. Pairwise interaction turnover
    was measured using link-based beta diversity. We then simulated
    species loss under multiple disturbance scenarios, allowing
    cascading extinctions, and compared predicted community states using
    mean absolute differences and rank concordance metrics between
    models. Results Reconstruction framework strongly influenced
    inferred network topology (MANOVA, p \textless{} 0.001), generating
    distinct structural signatures independent of species composition.
    Models that were similar in global metrics often diverged in
    species-level interactions, with high β-turnover among inferred link
    sets. During disturbance simulations, species-level vulnerability
    rankings were broadly consistent across models, but
    interaction-level outcomes and cascade dynamics varied
    substantially. Concordance in extinction-scenario rankings was scale
    dependent, with higher agreement at the species level than at the
    interaction level. Main conclusions Network reconstruction functions
    as a structural prior that conditions ecological inference. While
    some aggregate patterns are robust across modelling frameworks,
    detailed interaction-level dynamics are highly model contingent.
    Comparative network studies across spatial or environmental
    gradients should therefore align reconstruction framework with
    inferential goals and explicitly evaluate sensitivity to modelling
    assumptions.}
}
</code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre><div class="quarto-appendix-secondary-label">For attribution, please cite this work as:</div><div id="ref-strydom2026" class="csl-entry quarto-appendix-citeas" role="listitem">
Strydom, Tanya, Baran Karapunar, Andrew P. Beckerman, and Alexander
Dunhill. 2026. <span>“Model Structure Conditions Ecological Inference in
Food Web Reconstruction.”</span> TBD, February 25.
</div></div></section></div></main>
<!-- /main column -->
<script id="quarto-html-after-body" type="application/javascript">
  window.document.addEventListener("DOMContentLoaded", function (event) {
    const icon = "";
    const anchorJS = new window.AnchorJS();
    anchorJS.options = {
      placement: 'right',
      icon: icon
    };
    anchorJS.add('.anchored');
    const isCodeAnnotation = (el) => {
      for (const clz of el.classList) {
        if (clz.startsWith('code-annotation-')) {                     
          return true;
        }
      }
      return false;
    }
    const onCopySuccess = function(e) {
      // button target
      const button = e.trigger;
      // don't keep focus
      button.blur();
      // flash "checked"
      button.classList.add('code-copy-button-checked');
      var currentTitle = button.getAttribute("title");
      button.setAttribute("title", "Copied!");
      let tooltip;
      if (window.bootstrap) {
        button.setAttribute("data-bs-toggle", "tooltip");
        button.setAttribute("data-bs-placement", "left");
        button.setAttribute("data-bs-title", "Copied!");
        tooltip = new bootstrap.Tooltip(button, 
          { trigger: "manual", 
            customClass: "code-copy-button-tooltip",
            offset: [0, -8]});
        tooltip.show();    
      }
      setTimeout(function() {
        if (tooltip) {
          tooltip.hide();
          button.removeAttribute("data-bs-title");
          button.removeAttribute("data-bs-toggle");
          button.removeAttribute("data-bs-placement");
        }
        button.setAttribute("title", currentTitle);
        button.classList.remove('code-copy-button-checked');
      }, 1000);
      // clear code selection
      e.clearSelection();
    }
    const getTextToCopy = function(trigger) {
      const outerScaffold = trigger.parentElement.cloneNode(true);
      const codeEl = outerScaffold.querySelector('code');
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
    }
    const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
      text: getTextToCopy
    });
    clipboard.on('success', onCopySuccess);
    if (window.document.getElementById('quarto-embedded-source-code-modal')) {
      const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
        text: getTextToCopy,
        container: window.document.getElementById('quarto-embedded-source-code-modal')
      });
      clipboardModal.on('success', onCopySuccess);
    }
      var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
      var mailtoRegex = new RegExp(/^mailto:/);
        var filterRegex = new RegExp('/' + window.location.host + '/');
      var isInternal = (href) => {
          return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
      }
      // Inspect non-navigation links and adorn them if external
     var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
      for (var i=0; i<links.length; i++) {
        const link = links[i];
        if (!isInternal(link.href)) {
          // undo the damage that might have been done by quarto-nav.js in the case of
          // links that we want to consider external
          if (link.dataset.originalHref !== undefined) {
            link.href = link.dataset.originalHref;
          }
        }
      }
    function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
      const config = {
        allowHTML: true,
        maxWidth: 500,
        delay: 100,
        arrow: false,
        appendTo: function(el) {
            return el.parentElement;
        },
        interactive: true,
        interactiveBorder: 10,
        theme: 'quarto',
        placement: 'bottom-start',
      };
      if (contentFn) {
        config.content = contentFn;
      }
      if (onTriggerFn) {
        config.onTrigger = onTriggerFn;
      }
      if (onUntriggerFn) {
        config.onUntrigger = onUntriggerFn;
      }
      window.tippy(el, config); 
    }
    const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
    for (var i=0; i<noterefs.length; i++) {
      const ref = noterefs[i];
      tippyHover(ref, function() {
        // use id or data attribute instead here
        let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
        try { href = new URL(href).hash; } catch {}
        const id = href.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note) {
          return note.innerHTML;
        } else {
          return "";
        }
      });
    }
    const xrefs = window.document.querySelectorAll('a.quarto-xref');
    const processXRef = (id, note) => {
      // Strip column container classes
      const stripColumnClz = (el) => {
        el.classList.remove("page-full", "page-columns");
        if (el.children) {
          for (const child of el.children) {
            stripColumnClz(child);
          }
        }
      }
      stripColumnClz(note)
      if (id === null || id.startsWith('sec-')) {
        // Special case sections, only their first couple elements
        const container = document.createElement("div");
        if (note.children && note.children.length > 2) {
          container.appendChild(note.children[0].cloneNode(true));
          for (let i = 1; i < note.children.length; i++) {
            const child = note.children[i];
            if (child.tagName === "P" && child.innerText === "") {
              continue;
            } else {
              container.appendChild(child.cloneNode(true));
              break;
            }
          }
          if (window.Quarto?.typesetMath) {
            window.Quarto.typesetMath(container);
          }
          return container.innerHTML
        } else {
          if (window.Quarto?.typesetMath) {
            window.Quarto.typesetMath(note);
          }
          return note.innerHTML;
        }
      } else {
        // Remove any anchor links if they are present
        const anchorLink = note.querySelector('a.anchorjs-link');
        if (anchorLink) {
          anchorLink.remove();
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        if (note.classList.contains("callout")) {
          return note.outerHTML;
        } else {
          return note.innerHTML;
        }
      }
    }
    for (var i=0; i<xrefs.length; i++) {
      const xref = xrefs[i];
      tippyHover(xref, undefined, function(instance) {
        instance.disable();
        let url = xref.getAttribute('href');
        let hash = undefined; 
        if (url.startsWith('#')) {
          hash = url;
        } else {
          try { hash = new URL(url).hash; } catch {}
        }
        if (hash) {
          const id = hash.replace(/^#\/?/, "");
          const note = window.document.getElementById(id);
          if (note !== null) {
            try {
              const html = processXRef(id, note.cloneNode(true));
              instance.setContent(html);
            } finally {
              instance.enable();
              instance.show();
            }
          } else {
            // See if we can fetch this
            fetch(url.split('#')[0])
            .then(res => res.text())
            .then(html => {
              const parser = new DOMParser();
              const htmlDoc = parser.parseFromString(html, "text/html");
              const note = htmlDoc.getElementById(id);
              if (note !== null) {
                const html = processXRef(id, note);
                instance.setContent(html);
              } 
            }).finally(() => {
              instance.enable();
              instance.show();
            });
          }
        } else {
          // See if we can fetch a full url (with no hash to target)
          // This is a special case and we should probably do some content thinning / targeting
          fetch(url)
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.querySelector('main.content');
            if (note !== null) {
              // This should only happen for chapter cross references
              // (since there is no id in the URL)
              // remove the first header
              if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
                note.children[0].remove();
              }
              const html = processXRef(null, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      }, function(instance) {
      });
    }
        let selectedAnnoteEl;
        const selectorForAnnotation = ( cell, annotation) => {
          let cellAttr = 'data-code-cell="' + cell + '"';
          let lineAttr = 'data-code-annotation="' +  annotation + '"';
          const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
          return selector;
        }
        const selectCodeLines = (annoteEl) => {
          const doc = window.document;
          const targetCell = annoteEl.getAttribute("data-target-cell");
          const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
          const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
          const lines = annoteSpan.getAttribute("data-code-lines").split(",");
          const lineIds = lines.map((line) => {
            return targetCell + "-" + line;
          })
          let top = null;
          let height = null;
          let parent = null;
          if (lineIds.length > 0) {
              //compute the position of the single el (top and bottom and make a div)
              const el = window.document.getElementById(lineIds[0]);
              top = el.offsetTop;
              height = el.offsetHeight;
              parent = el.parentElement.parentElement;
            if (lineIds.length > 1) {
              const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
              const bottom = lastEl.offsetTop + lastEl.offsetHeight;
              height = bottom - top;
            }
            if (top !== null && height !== null && parent !== null) {
              // cook up a div (if necessary) and position it 
              let div = window.document.getElementById("code-annotation-line-highlight");
              if (div === null) {
                div = window.document.createElement("div");
                div.setAttribute("id", "code-annotation-line-highlight");
                div.style.position = 'absolute';
                parent.appendChild(div);
              }
              div.style.top = top - 2 + "px";
              div.style.height = height + 4 + "px";
              div.style.left = 0;
              let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
              if (gutterDiv === null) {
                gutterDiv = window.document.createElement("div");
                gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
                gutterDiv.style.position = 'absolute';
                const codeCell = window.document.getElementById(targetCell);
                const gutter = codeCell.querySelector('.code-annotation-gutter');
                gutter.appendChild(gutterDiv);
              }
              gutterDiv.style.top = top - 2 + "px";
              gutterDiv.style.height = height + 4 + "px";
            }
            selectedAnnoteEl = annoteEl;
          }
        };
        const unselectCodeLines = () => {
          const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
          elementsIds.forEach((elId) => {
            const div = window.document.getElementById(elId);
            if (div) {
              div.remove();
            }
          });
          selectedAnnoteEl = undefined;
        };
          // Handle positioning of the toggle
      window.addEventListener(
        "resize",
        throttle(() => {
          elRect = undefined;
          if (selectedAnnoteEl) {
            selectCodeLines(selectedAnnoteEl);
          }
        }, 10)
      );
      function throttle(fn, ms) {
      let throttle = false;
      let timer;
        return (...args) => {
          if(!throttle) { // first call gets through
              fn.apply(this, args);
              throttle = true;
          } else { // all the others get throttled
              if(timer) clearTimeout(timer); // cancel #2
              timer = setTimeout(() => {
                fn.apply(this, args);
                timer = throttle = false;
              }, ms);
          }
        };
      }
        // Attach click handler to the DT
        const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
        for (const annoteDlNode of annoteDls) {
          annoteDlNode.addEventListener('click', (event) => {
            const clickedEl = event.target;
            if (clickedEl !== selectedAnnoteEl) {
              unselectCodeLines();
              const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
              if (activeEl) {
                activeEl.classList.remove('code-annotation-active');
              }
              selectCodeLines(clickedEl);
              clickedEl.classList.add('code-annotation-active');
            } else {
              // Unselect the line
              unselectCodeLines();
              clickedEl.classList.remove('code-annotation-active');
            }
          });
        }
    const findCites = (el) => {
      const parentEl = el.parentElement;
      if (parentEl) {
        const cites = parentEl.dataset.cites;
        if (cites) {
          return {
            el,
            cites: cites.split(' ')
          };
        } else {
          return findCites(el.parentElement)
        }
      } else {
        return undefined;
      }
    };
    var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
    for (var i=0; i<bibliorefs.length; i++) {
      const ref = bibliorefs[i];
      const citeInfo = findCites(ref);
      if (citeInfo) {
        tippyHover(citeInfo.el, function() {
          var popup = window.document.createElement('div');
          citeInfo.cites.forEach(function(cite) {
            var citeDiv = window.document.createElement('div');
            citeDiv.classList.add('hanging-indent');
            citeDiv.classList.add('csl-entry');
            var biblioDiv = window.document.getElementById('ref-' + cite);
            if (biblioDiv) {
              citeDiv.innerHTML = biblioDiv.innerHTML;
            }
            popup.appendChild(citeDiv);
          });
          return popup.innerHTML;
        });
      }
    }
  });
  </script>
</div> <!-- /content -->
<script>var lightboxQuarto = GLightbox({"closeEffect":"zoom","descPosition":"bottom","loop":false,"openEffect":"zoom","selector":".lightbox"});
(function() {
  let previousOnload = window.onload;
  window.onload = () => {
    if (previousOnload) {
      previousOnload();
    }
    lightboxQuarto.on('slide_before_load', (data) => {
      const { slideIndex, slideNode, slideConfig, player, trigger } = data;
      const href = trigger.getAttribute('href');
      if (href !== null) {
        const imgEl = window.document.querySelector(`a[href="${href}"] img`);
        if (imgEl !== null) {
          const srcAttr = imgEl.getAttribute("src");
          if (srcAttr && srcAttr.startsWith("data:")) {
            slideConfig.href = srcAttr;
          }
        }
      } 
    });
  
    lightboxQuarto.on('slide_after_load', (data) => {
      const { slideIndex, slideNode, slideConfig, player, trigger } = data;
      if (window.Quarto?.typesetMath) {
        window.Quarto.typesetMath(slideNode);
      }
    });
  
  };
  
})();
          </script>




</body></html>