<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.9.2">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">

<meta name="author" content="Tanya Strydom">
<meta name="author" content="Andrew P. Beckerman">
<meta name="dcterms.date" content="2025-09-29">
<meta name="keywords" content="food web, network construction">

<title>Methods for Reconstructing Paleo Food Webs</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
/* CSS for citations */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
  margin-bottom: 0em;
}
.hanging-indent div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}</style>


<script src="site_libs/clipboard/clipboard.min.js"></script>
<script src="site_libs/quarto-html/quarto.js" type="module"></script>
<script src="site_libs/quarto-html/tabsets/tabsets.js" type="module"></script>
<script src="site_libs/quarto-html/axe/axe-check.js" type="module"></script>
<script src="site_libs/quarto-html/popper.min.js"></script>
<script src="site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="site_libs/quarto-html/anchor.min.js"></script>
<link href="site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="site_libs/quarto-html/quarto-syntax-highlighting-c6982bbdd90c704a9b15aa23d87ad911.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="site_libs/bootstrap/bootstrap.min.js"></script>
<link href="site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="site_libs/bootstrap/bootstrap-0a61af393cc41664e89a7addda21b354.min.css" rel="stylesheet" append-hash="true" id="quarto-bootstrap" data-mode="light">
<script src="site_libs/quarto-contrib/glightbox/glightbox.min.js"></script>
<link href="site_libs/quarto-contrib/glightbox/glightbox.min.css" rel="stylesheet">
<link href="site_libs/quarto-contrib/glightbox/lightbox.css" rel="stylesheet">
<script async="" src="https://hypothes.is/embed.js"></script>
<script>
  window.document.addEventListener("DOMContentLoaded", function (_event) {
    document.body.classList.add('hypothesis-enabled');
  });
</script>

  <script src="https://cdnjs.cloudflare.com/polyfill/v3/polyfill.min.js?features=es6"></script>
  <script src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-chtml-full.js" type="text/javascript"></script>

<script type="text/javascript">
const typesetMath = (el) => {
  if (window.MathJax) {
    // MathJax Typeset
    window.MathJax.typeset([el]);
  } else if (window.katex) {
    // KaTeX Render
    var mathElements = el.getElementsByClassName("math");
    var macros = [];
    for (var i = 0; i < mathElements.length; i++) {
      var texText = mathElements[i].firstChild;
      if (mathElements[i].tagName == "SPAN" && texText && texText.data) {
        window.katex.render(texText.data, mathElements[i], {
          displayMode: mathElements[i].classList.contains('display'),
          throwOnError: false,
          macros: macros,
          fleqn: false
        });
      }
    }
  }
}
window.Quarto = {
  typesetMath
};
</script>

<meta name="citation_title" content="Methods for Reconstructing Paleo Food Webs">
<meta name="citation_abstract" content="Food webs represent the feeding relationships between species and can help infer ecosystem-level processes. Alongside the development of food web theory, methods for constructing food webs have been developed to infer species interactions when empirical data is lacking. Food web construction methods are diverse, each utilising different approaches to infer species interactions —such as the use of traits to infer mechanistic relationships vs using gut content as a proxy for species diets. These methods have distinct theories, mechanisms, and data requirements. In paleoecology, where direct evidence of feeding interactions are rare, food web construction methods are especially valuable and affords us the opportunity to make inferences about paleo communities beyond simply a record of species composition. However, the limitations of paleontological data (e.g., information of species traits is limited to that which can be preserved) restrict which methods can reliably be used. By considering both ecological theory and the constraints of what can be derived from the fossil record, we identify the methods best suited for the construction of paleo food webs. Specifically, we focus on how these methods differ in the networks they produce and what these networks can reveal about species interactions. In doing so we hope to clarify the ecological nuances of network prediction and help prevent the accidental misuse or misinterpretation of paleo food webs.
">
<meta name="citation_keywords" content="food web,network construction">
<meta name="citation_author" content="Tanya Strydom">
<meta name="citation_author" content="Andrew P. Beckerman">
<meta name="citation_publication_date" content="2025-09-29">
<meta name="citation_cover_date" content="2025-09-29">
<meta name="citation_year" content="2025">
<meta name="citation_online_date" content="2025-09-29">
<meta name="citation_language" content="en">
<meta name="citation_journal_title" content="TBD">
<meta name="citation_reference" content="citation_title=Extinction cascades, community collapse, and recovery across a Mesozoic hyperthermal event;,citation_author=Alexander M. Dunhill;,citation_author=Karolina Zarzyczny;,citation_author=Jack O. Shaw;,citation_author=Jed W. Atkinson;,citation_author=Crispin T. S. Little;,citation_author=Andrew P. Beckerman;,citation_publication_date=2024-10-04;,citation_cover_date=2024-10-04;,citation_year=2024;,citation_fulltext_html_url=https://www.nature.com/articles/s41467-024-53000-2;,citation_issue=1;,citation_doi=10.1038/s41467-024-53000-2;,citation_volume=15;,citation_language=en;,citation_journal_title=Nature Communications;">
<meta name="citation_reference" content="citation_title=Shortcomings of reusing species interaction networks created by different sets of researchers;,citation_author=Chris Brimacombe;,citation_author=Korryn Bodner;,citation_author=Matthew Michalska-Smith;,citation_author=Timothée Poisot;,citation_author=Marie-Josée Fortin;,citation_publication_date=2023-03;,citation_cover_date=2023-03;,citation_year=2023;,citation_fulltext_html_url=https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.3002068;,citation_issue=4;,citation_doi=10.1371/journal.pbio.3002068;,citation_volume=21;,citation_language=en;,citation_journal_title=PLOS Biology;">
<meta name="citation_reference" content="citation_title=A framework for reconstructing ancient food webs using functional trait data;,citation_abstract=Food webs provide quantitative insights into the structure and dynamics of ecological communities. Previous work has shown their utility in understanding community responses to modern and ancient perturbations, including anthropogenic change and mass extinctions. However, few ancient food webs have been reconstructed due to difficulties assessing trophic interactions amongst extinct species derived from an incomplete fossil record. We present and assess the Paleo Food web Inference Model (PFIM). PFIM uses functional trait data–predictive of interactions in modern ecosystems and commonly available for fossil organisms–to reconstruct ancient food webs. We test the model by (i) applying it to four modern ecosystems with empirical constrained food webs to directly compare PFIM-constructed networks to their empirical counterparts, (ii) by carefully comparing discrepancies between PFIM-inferred and empirical webs in one of those systems, and (iii) by comparing networks describing feasible trophic interactions (&amp;amp;amp;quot;feasible webs&amp;quot;) with networks to which we superimpose characteristic interaction distributions derived from modern theory (&quot;realized webs&quot;). As a proof of concept, we then apply the method to faunal data from two Cambrian fossil deposits to reconstruct ancient trophic systems. PFIM-inferred feasible food webs successfully predict ~70% of trophic interactions across four modern systems. Furthermore, inferred food webs with enforced interaction distributions (i.e., realized webs) accurately predict ~90% of interactions. Comparisons with a global database of trophic interactions and other food web models, suggest that under sampling of empirical webs accounts for up to 21% of the remaining differences between PFIM and empirical food webs. Food webs can be reasonably approximated by inferring trophic interactions based upon life habit traits. This study provides the foundation to use trait-based inference models across the fossil record to examine ancient food webs and community evolution.;,citation_author=Jack O. Shaw;,citation_author=Alexander M. Dunhill;,citation_author=Andrew P. Beckerman;,citation_author=Jennifer A. Dunne;,citation_author=Pincelli M. Hull;,citation_publication_date=2024-01;,citation_cover_date=2024-01;,citation_year=2024;,citation_doi=10.1101/2024.01.30.578036;,citation_language=en-US;,citation_publisher=bioRxiv;">
<meta name="citation_reference" content="citation_title=Extinction cascades and catastrophe in ancient food webs;,citation_author=Peter D. Roopnarine;,citation_publication_date=2006;,citation_cover_date=2006;,citation_year=2006;,citation_fulltext_html_url=https://www.jstor.org/stable/4096814;,citation_issue=1;,citation_volume=32;,citation_journal_title=Paleobiology;">
<meta name="citation_reference" content="citation_title=Size, foraging, and food web structure;,citation_author=Owen L. Petchey;,citation_author=Andrew P. Beckerman;,citation_author=Jens O. Riede;,citation_author=Philip H. Warren;,citation_publication_date=2008-03-18;,citation_cover_date=2008-03-18;,citation_year=2008;,citation_fulltext_html_url=https://pnas.org/doi/full/10.1073/pnas.0710672105;,citation_issue=11;,citation_doi=10.1073/pnas.0710672105;,citation_volume=105;,citation_language=en;,citation_journal_title=Proceedings of the National Academy of Sciences;">
<meta name="citation_reference" content="citation_title=Modeling food webs: Exploring unexplained structure using latent traits.;,citation_author=Rudolf&nbsp;Philippe Rohr;,citation_author=Heike Scherer;,citation_author=Patrik Kehrli;,citation_author=Christian Mazza;,citation_author=Louis-Félix Bersier;,citation_publication_date=2010-08;,citation_cover_date=2010-08;,citation_year=2010;,citation_fulltext_html_url=https://www.journals.uchicago.edu/doi/10.1086/653667;,citation_issue=2;,citation_doi=10.1086/653667;,citation_volume=176;,citation_journal_title=The American Naturalist;">
<meta name="citation_reference" content="citation_title=Collapse of an ecological network in ancient egypt;,citation_author=Justin D. Yeakel;,citation_author=Mathias M. Pires;,citation_author=Lars Rudolf;,citation_author=Nathaniel J. Dominy;,citation_author=Paul L. Koch;,citation_author=Paulo R. Guimarães;,citation_author=Thilo Gross;,citation_publication_date=2014-10;,citation_cover_date=2014-10;,citation_year=2014;,citation_issue=40;,citation_doi=10.1073/pnas.1408471111;,citation_volume=111;,citation_journal_title=PNAS;">
<meta name="citation_reference" content="citation_title=Simple rules yield complex food webs;,citation_author=Richard J. Williams;,citation_author=Neo D. Martinez;,citation_publication_date=2000-03;,citation_cover_date=2000-03;,citation_year=2000;,citation_fulltext_html_url=https://www.nature.com/articles/35004572;,citation_issue=6774;,citation_doi=10.1038/35004572;,citation_volume=404;,citation_language=en;,citation_journal_title=Nature;">
<meta name="citation_reference" content="citation_title=Success and its limits among structural models of complex food webs;,citation_author=Richard J. Williams;,citation_author=Neo D. Martinez;,citation_publication_date=2008-05;,citation_cover_date=2008-05;,citation_year=2008;,citation_issue=3;,citation_doi=10.1111/j.1365-2656.2008.01362.x;,citation_volume=77;,citation_language=eng;,citation_journal_title=The Journal of Animal Ecology;">
<meta name="citation_reference" content="citation_title=Graph embedding and transfer learning can help predict potential species interaction networks despite data limitations;,citation_author=Tanya Strydom;,citation_author=Salomé Bouskila;,citation_author=Francis Banville;,citation_author=Ceres Barros;,citation_author=Dominique Caron;,citation_author=Maxwell J. Farrell;,citation_author=Marie-Josée Fortin;,citation_author=Benjamin Mercier;,citation_author=Laura J. Pollock;,citation_author=Rogini Runghen;,citation_author=Giulio V. Dalla Riva;,citation_author=Timothée Poisot;,citation_publication_date=2023;,citation_cover_date=2023;,citation_year=2023;,citation_fulltext_html_url=https://onlinelibrary.wiley.com/doi/abs/10.1111/2041-210X.14228;,citation_issue=12;,citation_doi=10.1111/2041-210X.14228;,citation_volume=14;,citation_language=en;,citation_journal_title=Methods in Ecology and Evolution;">
<meta name="citation_reference" content="citation_title=Addressing the Eltonian shortfall with trait-based interaction models;,citation_author=Dominique Caron;,citation_author=Luigi Maiorano;,citation_author=Wilfried Thuiller;,citation_author=Laura J. Pollock;,citation_publication_date=2022;,citation_cover_date=2022;,citation_year=2022;,citation_fulltext_html_url=https://onlinelibrary.wiley.com/doi/abs/10.1111/ele.13966;,citation_issue=4;,citation_doi=10.1111/ele.13966;,citation_volume=25;,citation_language=en;,citation_journal_title=Ecology Letters;">
<meta name="citation_reference" content="citation_title=Collapse of terrestrial mammal food webs since the Late Pleistocene;,citation_author=Evan C. Fricke;,citation_author=Chia Hsieh;,citation_author=Owen Middleton;,citation_author=Daniel Gorczynski;,citation_author=Caroline D. Cappello;,citation_author=Oscar Sanisidro;,citation_author=John Rowan;,citation_author=Jens-Christian Svenning;,citation_author=Lydia Beaudrot;,citation_publication_date=2022-08-26;,citation_cover_date=2022-08-26;,citation_year=2022;,citation_fulltext_html_url=http://dx.doi.org/10.1126/science.abn4012;,citation_issue=6609;,citation_doi=10.1126/science.abn4012;,citation_volume=377;,citation_language=en;,citation_journal_title=Science;">
<meta name="citation_reference" content="citation_title=Ecological Modelling of Paleocommunity Food Webs;,citation_author=Peter D Roopnarine;,citation_publication_date=2017;,citation_cover_date=2017;,citation_year=2017;,citation_language=en;,citation_inbook_title=undefined;">
<meta name="citation_reference" content="citation_title=Autecology and the Filling of Ecospace: Key Metazoan Radiations;,citation_author=Richard K. Bambach;,citation_author=Andrew M. Bush;,citation_author=Douglas H. Erwin;,citation_publication_date=2007;,citation_cover_date=2007;,citation_year=2007;,citation_fulltext_html_url=https://onlinelibrary.wiley.com/doi/abs/10.1111/j.1475-4983.2006.00611.x;,citation_issue=1;,citation_doi=10.1111/j.1475-4983.2006.00611.x;,citation_volume=50;,citation_language=en;,citation_journal_title=Palaeontology;">
<meta name="citation_reference" content="citation_title=Inferring predator-prey interactions in food webs;,citation_author=Justin PF Pomeranz;,citation_author=Ross M. Thompson;,citation_author=Timothée Poisot;,citation_author=Jon S. Harding;,citation_publication_date=2018;,citation_cover_date=2018;,citation_year=2018;,citation_fulltext_html_url=https://besjournals.onlinelibrary.wiley.com/doi/abs/10.1111/2041-210X.13125;,citation_issue=ja;,citation_doi=10.1111/2041-210X.13125;,citation_volume=0;,citation_language=en;,citation_journal_title=Methods in Ecology and Evolution;">
<meta name="citation_reference" content="citation_title=SVD entropy reveals the high complexity of ecological networks;,citation_author=Tanya Strydom;,citation_author=Giulio V. Dalla Riva;,citation_author=Timothée Poisot;,citation_publication_date=2021;,citation_cover_date=2021;,citation_year=2021;,citation_fulltext_html_url=https://www.frontiersin.org/articles/10.3389/fevo.2021.623141/full;,citation_doi=10.3389/fevo.2021.623141;,citation_volume=9;,citation_language=en-US;,citation_journal_title=Frontiers in Ecology and Evolution;">
<meta name="citation_reference" content="citation_title=Network motifs: Simple building blocks of complex networks;,citation_author=R. Milo;,citation_author=S. Shen-Orr;,citation_author=S. Itzkovitz;,citation_author=N. Kashtan;,citation_author=D. Chklovskii;,citation_author=U. Alon;,citation_publication_date=2002-10-25;,citation_cover_date=2002-10-25;,citation_year=2002;,citation_fulltext_html_url=https://www.science.org/doi/10.1126/science.298.5594.824;,citation_issue=5594;,citation_doi=10.1126/science.298.5594.824;,citation_volume=298;,citation_journal_title=Science;">
<meta name="citation_reference" content="citation_title=Evidence for the existence of a robust pattern of prey selection in food webs;,citation_author=Daniel B Stouffer;,citation_author=Juan Camacho;,citation_author=Wenxin Jiang;,citation_author=Luís A Nunes Amaral;,citation_publication_date=2007-06;,citation_cover_date=2007-06;,citation_year=2007;,citation_issue=1621;,citation_doi=10.1098/rspb.2007.0571;,citation_volume=274;,citation_journal_title=Proceedings of the Royal Society B: Biological Sciences;">
<meta name="citation_reference" content="citation_title=Success and its limits among structural models of complex food webs;,citation_author=Richard J. Williams;,citation_author=Neo D. Martinez;,citation_publication_date=2008-05;,citation_cover_date=2008-05;,citation_year=2008;,citation_issue=3;,citation_doi=10.1111/j.1365-2656.2008.01362.x;,citation_volume=77;,citation_language=eng;,citation_journal_title=The Journal of Animal Ecology;">
<meta name="citation_reference" content="citation_title=Size, foraging, and food web structure;,citation_author=Owen L. Petchey;,citation_author=Andrew P. Beckerman;,citation_author=Jens O. Riede;,citation_author=Philip H. Warren;,citation_publication_date=2008-03-18;,citation_cover_date=2008-03-18;,citation_year=2008;,citation_fulltext_html_url=https://pnas.org/doi/full/10.1073/pnas.0710672105;,citation_issue=11;,citation_doi=10.1073/pnas.0710672105;,citation_volume=105;,citation_language=en;,citation_journal_title=Proceedings of the National Academy of Sciences;">
<meta name="citation_reference" content="citation_title=Constant connectance in community food webs;,citation_author=Neo D. Martinez;,citation_publication_date=1992;,citation_cover_date=1992;,citation_year=1992;,citation_fulltext_html_url=http://www.jstor.org/stable/2462337;,citation_issue=6;,citation_volume=139;,citation_journal_title=The American Naturalist;">
<meta name="citation_reference" content="citation_title=Analysing ecological networks of species interactions;,citation_author=Eva Delmas;,citation_author=Mathilde Besson;,citation_author=Marie-Hélène Brice;,citation_author=Laura A. Burkle;,citation_author=Giulio V. Dalla Riva;,citation_author=Marie-Josée Fortin;,citation_author=Dominique Gravel;,citation_author=Paulo R. Guimarães;,citation_author=David H. Hembry;,citation_author=Erica A. Newman;,citation_author=Jens M. Olesen;,citation_author=Mathias M. Pires;,citation_author=Justin D. Yeakel;,citation_author=Timothée Poisot;,citation_publication_date=2018-06-20;,citation_cover_date=2018-06-20;,citation_year=2018;,citation_fulltext_html_url=http://doi.wiley.com/10.1111/brv.12433;,citation_doi=10.1111/brv.12433;,citation_language=en;,citation_journal_title=Biological Reviews;">
<meta name="citation_reference" content="citation_title=A general model for food web structure;,citation_author=Stefano Allesina;,citation_author=David Alonso;,citation_author=Mercedes Pascual;,citation_publication_date=2008-05-02;,citation_cover_date=2008-05-02;,citation_year=2008;,citation_fulltext_html_url=https://www.science.org/doi/full/10.1126/science.1156269;,citation_issue=5876;,citation_doi=10.1126/science.1156269;,citation_volume=320;,citation_journal_title=Science;">
<meta name="citation_reference" content="citation_title=A roadmap towards predicting species interaction networks (across space and time);,citation_author=Tanya Strydom;,citation_author=Michael D. Catchen;,citation_author=Francis Banville;,citation_author=Dominique Caron;,citation_author=Gabriel Dansereau;,citation_author=Philippe Desjardins-Proulx;,citation_author=Norma R. Forero-Muñoz;,citation_author=Gracielle Higino;,citation_author=Benjamin Mercier;,citation_author=Andrew Gonzalez;,citation_author=Dominique Gravel;,citation_author=Laura Pollock;,citation_author=Timothée Poisot;,citation_publication_date=2021-11;,citation_cover_date=2021-11;,citation_year=2021;,citation_issue=1837;,citation_doi=10.1098/rstb.2021.0063;,citation_volume=376;,citation_journal_title=Philosophical Transactions of the Royal Society B: Biological Sciences;,citation_publisher=Royal Society;">
<meta name="citation_reference" content="citation_title=A comparative study of ecological specialization estimators;,citation_author=Timothée Poisot;,citation_author=Elsa Canard;,citation_author=Nicolas Mouquet;,citation_author=Michael E. Hochberg;,citation_publication_date=2012-01-19;,citation_cover_date=2012-01-19;,citation_year=2012;,citation_fulltext_html_url=http://dx.doi.org/10.1111/j.2041-210X.2011.00174.x;,citation_issue=3;,citation_doi=10.1111/j.2041-210x.2011.00174.x;,citation_volume=3;,citation_language=en;,citation_journal_title=Methods in Ecology and Evolution;">
<meta name="citation_reference" content="citation_title=Food Webs From the Small to the Large: The Robert H. MacArthur Award Lecture;,citation_author=Thomas W. Schoener;,citation_publication_date=1989-12;,citation_cover_date=1989-12;,citation_year=1989;,citation_fulltext_html_url=http://dx.doi.org/10.2307/1938088;,citation_issue=6;,citation_doi=10.2307/1938088;,citation_volume=70;,citation_language=en;,citation_journal_title=Ecology;">
<meta name="citation_reference" content="citation_title=ATNr: Allometric Trophic Network models in R;,citation_author=Benoit Gauzens;,citation_author=Ulrich Brose;,citation_author=Eva Delmas;,citation_author=Emilio Berti;,citation_publication_date=2023;,citation_cover_date=2023;,citation_year=2023;,citation_fulltext_html_url=https://onlinelibrary.wiley.com/doi/abs/10.1111/2041-210X.14212;,citation_issue=11;,citation_doi=10.1111/2041-210X.14212;,citation_volume=14;,citation_language=en;,citation_journal_title=Methods in Ecology and Evolution;">
<meta name="citation_reference" content="citation_title=Animal diversity and ecosystem functioning in dynamic food webs;,citation_author=Florian D. Schneider;,citation_author=Ulrich Brose;,citation_author=Björn C. Rall;,citation_author=Christian Guill;,citation_publication_date=2016-10-05;,citation_cover_date=2016-10-05;,citation_year=2016;,citation_fulltext_html_url=https://www.nature.com/articles/ncomms12718;,citation_issue=1;,citation_doi=10.1038/ncomms12718;,citation_volume=7;,citation_language=en;,citation_journal_title=Nature Communications;">
<meta name="citation_reference" content="citation_title=The reliability of R50 as a measure of vulnerability of food webs to sequential species deletions;,citation_author=Tomas Jonsson;,citation_author=Sofia Berg;,citation_author=Alexander Pimenov;,citation_author=Catherine Palmer;,citation_author=Mark Emmerson;,citation_publication_date=2015;,citation_cover_date=2015;,citation_year=2015;,citation_fulltext_html_url=https://onlinelibrary.wiley.com/doi/abs/10.1111/oik.01588;,citation_issue=4;,citation_doi=10.1111/oik.01588;,citation_volume=124;,citation_language=en;,citation_journal_title=Oikos;">
<meta name="citation_reference" content="citation_title=On random graphs. i.;,citation_author=P. Erdős;,citation_author=A. Rényi;,citation_publication_date=1959-07-01;,citation_cover_date=1959-07-01;,citation_year=1959;,citation_fulltext_html_url=http://dx.doi.org/10.5486/PMD.1959.6.3-4.12;,citation_issue=3-4;,citation_doi=10.5486/pmd.1959.6.3-4.12;,citation_volume=6;,citation_journal_title=Publicationes Mathematicae Debrecen;">
<meta name="citation_reference" content="citation_title=The ghost of nestedness in ecological networks;,citation_author=Phillip P. A. Staniczenko;,citation_author=Jason C. Kopp;,citation_author=Stefano Allesina;,citation_publication_date=2013-06;,citation_cover_date=2013-06;,citation_year=2013;,citation_fulltext_html_url=http://www.nature.com/articles/ncomms2422;,citation_issue=1;,citation_doi=10.1038/ncomms2422;,citation_volume=4;,citation_language=en;,citation_journal_title=Nature Communications;">
<meta name="citation_reference" content="citation_title=Stabilization of chaotic and non-permanent food-web dynamics;,citation_author=R. J. Williams;,citation_author=N. D. Martinez;,citation_publication_date=2004-03-01;,citation_cover_date=2004-03-01;,citation_year=2004;,citation_fulltext_html_url=http://link.springer.com/10.1140/epjb/e2004-00122-1;,citation_issue=2;,citation_doi=10.1140/epjb/e2004-00122-1;,citation_volume=38;,citation_language=en;,citation_journal_title=The European Physical Journal B - Condensed Matter;">
<meta name="citation_reference" content="citation_title=Simultaneously estimating food web connectance and structure with uncertainty;,citation_author=Anubhav Gupta;,citation_author=Reinhard Furrer;,citation_author=Owen L. Petchey;,citation_publication_date=2022;,citation_cover_date=2022;,citation_year=2022;,citation_fulltext_html_url=https://onlinelibrary.wiley.com/doi/abs/10.1002/ece3.8643;,citation_issue=3;,citation_doi=10.1002/ece3.8643;,citation_volume=12;,citation_language=en;,citation_journal_title=Ecology and Evolution;">
<meta name="citation_reference" content="citation_title=Highly resolved early eocene food webs show development of modern trophic structure after the end-cretaceous extinction;,citation_author=Jennifer A. Dunne;,citation_author=Conrad C. Labandeira;,citation_author=Richard J. Williams;,citation_publication_date=2014-05-07;,citation_cover_date=2014-05-07;,citation_year=2014;,citation_fulltext_html_url=https://royalsocietypublishing.org/doi/10.1098/rspb.2013.3280;,citation_issue=1782;,citation_doi=10.1098/rspb.2013.3280;,citation_volume=281;,citation_journal_title=Proceedings of the Royal Society B: Biological Sciences;">
<meta name="citation_reference" content="citation_title=Extinction cascades and catastrophe in ancient food webs;,citation_author=Peter D. Roopnarine;,citation_publication_date=2006;,citation_cover_date=2006;,citation_year=2006;,citation_fulltext_html_url=https://www.jstor.org/stable/4096814;,citation_issue=1;,citation_volume=32;,citation_journal_title=Paleobiology;">
<meta name="citation_reference" content="citation_title=Tailoring interaction network types to answer different ecological questions;,citation_author=Benoit Gauzens;,citation_author=Lise Thouvenot;,citation_author=Diane S. Srivastava;,citation_author=Pavel Kratina;,citation_author=Gustavo Q. Romero;,citation_author=Emilio Berti;,citation_author=Eoin J. O’Gorman;,citation_author=Angélica L. González;,citation_author=Olivier Dézerald;,citation_author=Nico Eisenhauer;,citation_author=Mathias Pires;,citation_author=Remo Ryser;,citation_author=Vinicius F. Farjalla;,citation_author=Pierre Rogy;,citation_author=Ulrich Brose;,citation_author=Jana S. Petermann;,citation_author=Benoît Geslin;,citation_author=Jes Hines;,citation_publication_date=2025-06-13;,citation_cover_date=2025-06-13;,citation_year=2025;,citation_fulltext_html_url=https://www.nature.com/articles/s44358-025-00056-7;,citation_doi=10.1038/s44358-025-00056-7;,citation_language=en;,citation_journal_title=Nature Reviews Biodiversity;">
<meta name="citation_reference" content="citation_title=Addressing priority questions of conservation science with palaeontological data;,citation_author=Wolfgang Kiessling;,citation_author=Nussaïbah B. Raja;,citation_author=Vanessa Julie Roden;,citation_author=Samuel T. Turvey;,citation_author=Erin E. Saupe;,citation_publication_date=2019-11-04;,citation_cover_date=2019-11-04;,citation_year=2019;,citation_fulltext_html_url=https://royalsocietypublishing.org/doi/10.1098/rstb.2019.0222;,citation_issue=1788;,citation_doi=10.1098/rstb.2019.0222;,citation_volume=374;,citation_journal_title=Philosophical Transactions of the Royal Society B: Biological Sciences;">
<meta name="citation_reference" content="citation_title=What is conservation paleobiology? Tracking 20 years of research and development;,citation_author=Erin M. Dillon;,citation_author=Jaleigh Q. Pier;,citation_author=Jansen A. Smith;,citation_author=Nussaïbah B. Raja;,citation_author=Danijela Dimitrijević;,citation_author=Elizabeth L. Austin;,citation_author=Jonathan D. Cybulski;,citation_author=Julia De Entrambasaguas;,citation_author=Stephen R. Durham;,citation_author=Carolin M. Grether;,citation_author=Himadri Sekhar Haldar;,citation_author=Kristína Kocáková;,citation_author=Chien-Hsiang Lin;,citation_author=Ilaria Mazzini;,citation_author=Alexis M. Mychajliw;,citation_author=Amy L. Ollendorf;,citation_author=Catalina Pimiento;,citation_author=Omar R. Regalado Fernández;,citation_author=Isaiah E. Smith;,citation_author=Gregory P. Dietl;,citation_publication_date=2022-12-07;,citation_cover_date=2022-12-07;,citation_year=2022;,citation_fulltext_html_url=https://www.frontiersin.org/journals/ecology-and-evolution/articles/10.3389/fevo.2022.1031483/full;,citation_doi=10.3389/fevo.2022.1031483;,citation_volume=10;,citation_language=en-US;,citation_journal_title=Frontiers in Ecology and Evolution;">
<meta name="citation_reference" content="citation_title=Global Projection of Terrestrial Vertebrate Food Webs Under Future Climate and Land-Use Changes;,citation_author=Xiyang Hao;,citation_author=Marcel Holyoak;,citation_author=Zhicheng Zhang;,citation_author=Chuan Yan;,citation_publication_date=2025;,citation_cover_date=2025;,citation_year=2025;,citation_fulltext_html_url=https://onlinelibrary.wiley.com/doi/abs/10.1111/gcb.70061;,citation_issue=2;,citation_doi=10.1111/gcb.70061;,citation_volume=31;,citation_language=en;,citation_journal_title=Global Change Biology;">
<meta name="citation_reference" content="citation_title=Chasing Ecological Interactions;,citation_author=Pedro Jordano;,citation_publication_date=2016-09-15;,citation_cover_date=2016-09-15;,citation_year=2016;,citation_fulltext_html_url=https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002559;,citation_issue=9;,citation_doi=10.1371/journal.pbio.1002559;,citation_volume=14;,citation_language=en;,citation_journal_title=PLOS Biology;">
<meta name="citation_reference" content="citation_title=Inferring biotic interactions from proxies;,citation_author=Ignacio Morales-Castilla;,citation_author=Miguel G. Matias;,citation_author=Dominique Gravel;,citation_author=Miguel B. Araújo;,citation_publication_date=2015-06;,citation_cover_date=2015-06;,citation_year=2015;,citation_fulltext_html_url=https://linkinghub.elsevier.com/retrieve/pii/S0169534715000774;,citation_issue=6;,citation_doi=10.1016/j.tree.2015.03.014;,citation_volume=30;,citation_language=en;,citation_journal_title=Trends in Ecology &amp;amp;amp; Evolution;">
<meta name="citation_reference" content="citation_title=Machine learning and deep learningA review for ecologists;,citation_author=Maximilian Pichler;,citation_author=Florian Hartig;,citation_publication_date=2023;,citation_cover_date=2023;,citation_year=2023;,citation_fulltext_html_url=https://onlinelibrary.wiley.com/doi/abs/10.1111/2041-210X.14061;,citation_issue=4;,citation_doi=10.1111/2041-210X.14061;,citation_volume=14;,citation_language=en;,citation_journal_title=Methods in Ecology and Evolution;">
<meta name="citation_reference" content="citation_title=A roadmap towards predicting species interaction networks (across space and time);,citation_author=Tanya Strydom;,citation_author=Michael D. Catchen;,citation_author=Francis Banville;,citation_author=Dominique Caron;,citation_author=Gabriel Dansereau;,citation_author=Philippe Desjardins-Proulx;,citation_author=Norma R. Forero-Muñoz;,citation_author=Gracielle Higino;,citation_author=Benjamin Mercier;,citation_author=Andrew Gonzalez;,citation_author=Dominique Gravel;,citation_author=Laura Pollock;,citation_author=Timothée Poisot;,citation_publication_date=2021-11-08;,citation_cover_date=2021-11-08;,citation_year=2021;,citation_fulltext_html_url=https://royalsocietypublishing.org/doi/abs/10.1098/rstb.2021.0063;,citation_issue=1837;,citation_doi=10.1098/rstb.2021.0063;,citation_volume=376;,citation_journal_title=Philosophical Transactions of the Royal Society B: Biological Sciences;">
<meta name="citation_reference" content="citation_title=A general model for food web structure;,citation_author=Stefano Allesina;,citation_author=David Alonso;,citation_author=Mercedes Pascual;,citation_publication_date=2008-05-02;,citation_cover_date=2008-05-02;,citation_year=2008;,citation_fulltext_html_url=https://www.science.org/doi/full/10.1126/science.1156269;,citation_issue=5876;,citation_doi=10.1126/science.1156269;,citation_volume=320;,citation_journal_title=Science;">
<meta name="citation_reference" content="citation_title=Collapse of an ecological network in ancient egypt;,citation_author=Justin D. Yeakel;,citation_author=Mathias M. Pires;,citation_author=Lars Rudolf;,citation_author=Nathaniel J. Dominy;,citation_author=Paul L. Koch;,citation_author=Paulo R. Guimarães;,citation_author=Thilo Gross;,citation_publication_date=2014-10;,citation_cover_date=2014-10;,citation_year=2014;,citation_issue=40;,citation_doi=10.1073/pnas.1408471111;,citation_volume=111;,citation_journal_title=PNAS;">
<meta name="citation_reference" content="citation_title=Food web reconstruction through phylogenetic transfer of low-rank network representation;,citation_author=Tanya Strydom;,citation_author=Salomé Bouskila;,citation_author=Francis Banville;,citation_author=Ceres Barros;,citation_author=Dominique Caron;,citation_author=Maxwell J. Farrell;,citation_author=Marie-Josée Fortin;,citation_author=Victoria Hemming;,citation_author=Benjamin Mercier;,citation_author=Laura J. Pollock;,citation_author=Rogini Runghen;,citation_author=Giulio V. Dalla Riva;,citation_author=Timothée Poisot;,citation_publication_date=2022-12;,citation_cover_date=2022-12;,citation_year=2022;,citation_fulltext_html_url=https://onlinelibrary.wiley.com/doi/abs/10.1111/2041-210X.13835;,citation_issue=12;,citation_doi=10.1111/2041-210X.13835;,citation_volume=13;,citation_language=en;,citation_journal_title=Methods in Ecology and Evolution;">
<meta name="citation_reference" content="citation_title=Navigating the integration of Biotic Interactions in Biogeography;,citation_author=Wilfried Thuiller;,citation_author=Irene Calderon-Sanou;,citation_author=Loïc Chalmandrier;,citation_author=Pierre Gaüzere;,citation_author=Marc Ohlmann;,citation_author=Giovanni Poggiato;,citation_author=Tamara Münkemüller;,citation_language=en;">
<meta name="citation_reference" content="citation_title=Robustness to secondary extinctions: Comparing trait-based sequential deletions in static and dynamic food webs;,citation_author=Alva Curtsdotter;,citation_author=Amrei Binzer;,citation_author=Ulrich Brose;,citation_author=Francisco De Castro;,citation_author=Bo Ebenman;,citation_author=Anna Eklöf;,citation_author=Jens O. Riede;,citation_author=Aaron Thierry;,citation_author=Björn C. Rall;,citation_publication_date=2011-11;,citation_cover_date=2011-11;,citation_year=2011;,citation_fulltext_html_url=https://linkinghub.elsevier.com/retrieve/pii/S1439179111001216;,citation_issue=7;,citation_doi=10.1016/j.baae.2011.09.008;,citation_volume=12;,citation_language=en;,citation_journal_title=Basic and Applied Ecology;">
<meta name="citation_reference" content="citation_title=Tailoring interaction network types to answer different ecological questions;,citation_author=Benoit Gauzens;,citation_author=Lise Thouvenot;,citation_author=Diane S. Srivastava;,citation_author=Pavel Kratina;,citation_author=Gustavo Q. Romero;,citation_author=Emilio Berti;,citation_author=Eoin J. O’Gorman;,citation_author=Angélica L. González;,citation_author=Olivier Dézerald;,citation_author=Nico Eisenhauer;,citation_author=Mathias Pires;,citation_author=Remo Ryser;,citation_author=Vinicius F. Farjalla;,citation_author=Pierre Rogy;,citation_author=Ulrich Brose;,citation_author=Jana S. Petermann;,citation_author=Benoît Geslin;,citation_author=Jes Hines;,citation_publication_date=2025-06-13;,citation_cover_date=2025-06-13;,citation_year=2025;,citation_fulltext_html_url=https://www.nature.com/articles/s44358-025-00056-7;,citation_doi=10.1038/s44358-025-00056-7;,citation_language=en;,citation_journal_title=Nature Reviews Biodiversity;">
<meta name="citation_reference" content="citation_title=Navigating the integration of biotic interactions in biogeography;,citation_author=Wilfried Thuiller;,citation_author=Irene Calderón-Sanou;,citation_author=Loïc Chalmandrier;,citation_author=Pierre Gaüzère;,citation_author=Louise M. J. O’Connor;,citation_author=Marc Ohlmann;,citation_author=Giovanni Poggiato;,citation_author=Tamara Münkemüller;,citation_publication_date=2024;,citation_cover_date=2024;,citation_year=2024;,citation_fulltext_html_url=https://onlinelibrary.wiley.com/doi/abs/10.1111/jbi.14734;,citation_issue=4;,citation_doi=10.1111/jbi.14734;,citation_volume=51;,citation_language=en;,citation_journal_title=Journal of Biogeography;">
</head>

<body class="quarto-light">

<header id="title-block-header" class="quarto-title-block default toc-left page-columns page-full">
  <div class="quarto-title-banner page-columns page-full">
    <div class="quarto-title column-body">
      <h1 class="title">Methods for Reconstructing Paleo Food Webs</h1>
          </div>

    
    <div class="quarto-title-meta-container">
      <div class="quarto-title-meta-column-start">
            <div class="quarto-title-meta-author">
          <div class="quarto-title-meta-heading">Authors</div>
          <div class="quarto-title-meta-heading">Affiliation</div>
          
                <div class="quarto-title-meta-contents">
            <p class="author">Tanya Strydom <a href="mailto:t.strydom@sheffield.ac.uk" class="quarto-title-author-email"><i class="bi bi-envelope"></i></a> <a href="https://orcid.org/0000-0001-6067-1349" class="quarto-title-author-orcid"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAA2ZpVFh0WE1MOmNvbS5hZG9iZS54bXAAAAAAADw/eHBhY2tldCBiZWdpbj0i77u/IiBpZD0iVzVNME1wQ2VoaUh6cmVTek5UY3prYzlkIj8+IDx4OnhtcG1ldGEgeG1sbnM6eD0iYWRvYmU6bnM6bWV0YS8iIHg6eG1wdGs9IkFkb2JlIFhNUCBDb3JlIDUuMC1jMDYwIDYxLjEzNDc3NywgMjAxMC8wMi8xMi0xNzozMjowMCAgICAgICAgIj4gPHJkZjpSREYgeG1sbnM6cmRmPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5LzAyLzIyLXJkZi1zeW50YXgtbnMjIj4gPHJkZjpEZXNjcmlwdGlvbiByZGY6YWJvdXQ9IiIgeG1sbnM6eG1wTU09Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC9tbS8iIHhtbG5zOnN0UmVmPSJodHRwOi8vbnMuYWRvYmUuY29tL3hhcC8xLjAvc1R5cGUvUmVzb3VyY2VSZWYjIiB4bWxuczp4bXA9Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC8iIHhtcE1NOk9yaWdpbmFsRG9jdW1lbnRJRD0ieG1wLmRpZDo1N0NEMjA4MDI1MjA2ODExOTk0QzkzNTEzRjZEQTg1NyIgeG1wTU06RG9jdW1lbnRJRD0ieG1wLmRpZDozM0NDOEJGNEZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wTU06SW5zdGFuY2VJRD0ieG1wLmlpZDozM0NDOEJGM0ZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wOkNyZWF0b3JUb29sPSJBZG9iZSBQaG90b3Nob3AgQ1M1IE1hY2ludG9zaCI+IDx4bXBNTTpEZXJpdmVkRnJvbSBzdFJlZjppbnN0YW5jZUlEPSJ4bXAuaWlkOkZDN0YxMTc0MDcyMDY4MTE5NUZFRDc5MUM2MUUwNEREIiBzdFJlZjpkb2N1bWVudElEPSJ4bXAuZGlkOjU3Q0QyMDgwMjUyMDY4MTE5OTRDOTM1MTNGNkRBODU3Ii8+IDwvcmRmOkRlc2NyaXB0aW9uPiA8L3JkZjpSREY+IDwveDp4bXBtZXRhPiA8P3hwYWNrZXQgZW5kPSJyIj8+84NovQAAAR1JREFUeNpiZEADy85ZJgCpeCB2QJM6AMQLo4yOL0AWZETSqACk1gOxAQN+cAGIA4EGPQBxmJA0nwdpjjQ8xqArmczw5tMHXAaALDgP1QMxAGqzAAPxQACqh4ER6uf5MBlkm0X4EGayMfMw/Pr7Bd2gRBZogMFBrv01hisv5jLsv9nLAPIOMnjy8RDDyYctyAbFM2EJbRQw+aAWw/LzVgx7b+cwCHKqMhjJFCBLOzAR6+lXX84xnHjYyqAo5IUizkRCwIENQQckGSDGY4TVgAPEaraQr2a4/24bSuoExcJCfAEJihXkWDj3ZAKy9EJGaEo8T0QSxkjSwORsCAuDQCD+QILmD1A9kECEZgxDaEZhICIzGcIyEyOl2RkgwAAhkmC+eAm0TAAAAABJRU5ErkJggg=="></a></p>
          </div>
                <div class="quarto-title-meta-contents">
                    <p class="affiliation">
                        School of Biosciences, University of Sheffield, Sheffield, UK
                      </p>
                  </div>
                      <div class="quarto-title-meta-contents">
            <p class="author">Andrew P. Beckerman <a href="https://orcid.org/0000-0002-7859-8394" class="quarto-title-author-orcid"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAA2ZpVFh0WE1MOmNvbS5hZG9iZS54bXAAAAAAADw/eHBhY2tldCBiZWdpbj0i77u/IiBpZD0iVzVNME1wQ2VoaUh6cmVTek5UY3prYzlkIj8+IDx4OnhtcG1ldGEgeG1sbnM6eD0iYWRvYmU6bnM6bWV0YS8iIHg6eG1wdGs9IkFkb2JlIFhNUCBDb3JlIDUuMC1jMDYwIDYxLjEzNDc3NywgMjAxMC8wMi8xMi0xNzozMjowMCAgICAgICAgIj4gPHJkZjpSREYgeG1sbnM6cmRmPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5LzAyLzIyLXJkZi1zeW50YXgtbnMjIj4gPHJkZjpEZXNjcmlwdGlvbiByZGY6YWJvdXQ9IiIgeG1sbnM6eG1wTU09Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC9tbS8iIHhtbG5zOnN0UmVmPSJodHRwOi8vbnMuYWRvYmUuY29tL3hhcC8xLjAvc1R5cGUvUmVzb3VyY2VSZWYjIiB4bWxuczp4bXA9Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC8iIHhtcE1NOk9yaWdpbmFsRG9jdW1lbnRJRD0ieG1wLmRpZDo1N0NEMjA4MDI1MjA2ODExOTk0QzkzNTEzRjZEQTg1NyIgeG1wTU06RG9jdW1lbnRJRD0ieG1wLmRpZDozM0NDOEJGNEZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wTU06SW5zdGFuY2VJRD0ieG1wLmlpZDozM0NDOEJGM0ZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wOkNyZWF0b3JUb29sPSJBZG9iZSBQaG90b3Nob3AgQ1M1IE1hY2ludG9zaCI+IDx4bXBNTTpEZXJpdmVkRnJvbSBzdFJlZjppbnN0YW5jZUlEPSJ4bXAuaWlkOkZDN0YxMTc0MDcyMDY4MTE5NUZFRDc5MUM2MUUwNEREIiBzdFJlZjpkb2N1bWVudElEPSJ4bXAuZGlkOjU3Q0QyMDgwMjUyMDY4MTE5OTRDOTM1MTNGNkRBODU3Ii8+IDwvcmRmOkRlc2NyaXB0aW9uPiA8L3JkZjpSREY+IDwveDp4bXBtZXRhPiA8P3hwYWNrZXQgZW5kPSJyIj8+84NovQAAAR1JREFUeNpiZEADy85ZJgCpeCB2QJM6AMQLo4yOL0AWZETSqACk1gOxAQN+cAGIA4EGPQBxmJA0nwdpjjQ8xqArmczw5tMHXAaALDgP1QMxAGqzAAPxQACqh4ER6uf5MBlkm0X4EGayMfMw/Pr7Bd2gRBZogMFBrv01hisv5jLsv9nLAPIOMnjy8RDDyYctyAbFM2EJbRQw+aAWw/LzVgx7b+cwCHKqMhjJFCBLOzAR6+lXX84xnHjYyqAo5IUizkRCwIENQQckGSDGY4TVgAPEaraQr2a4/24bSuoExcJCfAEJihXkWDj3ZAKy9EJGaEo8T0QSxkjSwORsCAuDQCD+QILmD1A9kECEZgxDaEZhICIzGcIyEyOl2RkgwAAhkmC+eAm0TAAAAABJRU5ErkJggg=="></a></p>
          </div>
                <div class="quarto-title-meta-contents">
                    <p class="affiliation">
                        School of Biosciences, University of Sheffield, Sheffield, UK
                      </p>
                  </div>
                    </div>
        
        <div class="quarto-title-meta">

                      
                <div>
            <div class="quarto-title-meta-heading">Published</div>
            <div class="quarto-title-meta-contents">
              <p class="date">September 29, 2025</p>
            </div>
          </div>
          
                
              </div>
      </div>
      <div class="quarto-title-meta-column-end quarto-other-formats-target">
      <div class="quarto-alternate-formats"><div class="quarto-title-meta-heading">Other Formats</div><div class="quarto-title-meta-contents"><p><a href="index.docx"><i class="bi bi-file-word"></i>MS Word</a></p></div><div class="quarto-title-meta-contents"><p><a href="index.pdf"><i class="bi bi-file-pdf"></i>PDF (preprint)</a></p></div></div></div>
    </div>

    <div>
      <div class="abstract">
        <div class="block-title">Abstract</div>
        <p>Food webs represent the feeding relationships between species and can help infer ecosystem-level processes. Alongside the development of food web theory, methods for constructing food webs have been developed to infer species interactions when empirical data is lacking. Food web construction methods are diverse, each utilising different approaches to infer species interactions —such as the use of traits to infer mechanistic relationships vs using gut content as a proxy for species diets. These methods have distinct theories, mechanisms, and data requirements. In paleoecology, where direct evidence of feeding interactions are rare, food web construction methods are especially valuable and affords us the opportunity to make inferences about paleo communities beyond simply a record of species composition. However, the limitations of paleontological data (e.g., information of species traits is limited to that which can be preserved) restrict which methods can reliably be used. By considering both ecological theory and the constraints of what can be derived from the fossil record, we identify the methods best suited for the construction of paleo food webs. Specifically, we focus on how these methods differ in the networks they produce and what these networks can reveal about species interactions. In doing so we hope to clarify the ecological nuances of network prediction and help prevent the accidental misuse or misinterpretation of paleo food webs.</p>
      </div>
    </div>

    <div>
      <div class="keywords">
        <div class="block-title">Keywords</div>
        <p>food web, network construction</p>
      </div>
    </div>

    <div class="quarto-other-links-text-target">
    <div class="quarto-code-links"><div class="quarto-title-meta-heading">Code Links</div><div class="quarto-title-meta-contents"><span><a href="https://github.com/TanyaS08/ms_paleo_webs/tree/main/code"><i class="bi bi-github"></i>GitHub</a></span></div></div></div>  </div>
</header><div id="quarto-content" class="page-columns page-rows-contents page-layout-article toc-left">
<div id="quarto-sidebar-toc-left" class="sidebar toc-left">
  <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">Table of contents</h2>
   
  <ul>
  <li><a href="#contextualising-the-prediction-of-paleo-webs-within-the-contemporary-toolbox" id="toc-contextualising-the-prediction-of-paleo-webs-within-the-contemporary-toolbox" class="nav-link active" data-scroll-target="#contextualising-the-prediction-of-paleo-webs-within-the-contemporary-toolbox"><span class="header-section-number">1</span> Contextualising the prediction of paleo webs within the contemporary toolbox</a></li>
  <li><a href="#understanding-how-networks-are-different" id="toc-understanding-how-networks-are-different" class="nav-link" data-scroll-target="#understanding-how-networks-are-different"><span class="header-section-number">2</span> Understanding how networks are different</a></li>
  <li><a href="#challenges-specific-to-building-paleo-networks" id="toc-challenges-specific-to-building-paleo-networks" class="nav-link" data-scroll-target="#challenges-specific-to-building-paleo-networks"><span class="header-section-number">3</span> Challenges specific to building paleo networks</a>
  <ul class="collapse">
  <li><a href="#approaches-to-food-web-prediction" id="toc-approaches-to-food-web-prediction" class="nav-link" data-scroll-target="#approaches-to-food-web-prediction"><span class="header-section-number">3.1</span> Approaches to food web prediction</a></li>
  </ul></li>
  <li><a href="#case-study-toarcian-mass-extinction-event" id="toc-case-study-toarcian-mass-extinction-event" class="nav-link" data-scroll-target="#case-study-toarcian-mass-extinction-event"><span class="header-section-number">4</span> Case study: Toarcian mass extinction event</a>
  <ul class="collapse">
  <li><a href="#dataset-overview" id="toc-dataset-overview" class="nav-link" data-scroll-target="#dataset-overview"><span class="header-section-number">4.1</span> Dataset overview</a>
  <ul class="collapse">
  <li><a href="#species-occurrence" id="toc-species-occurrence" class="nav-link" data-scroll-target="#species-occurrence"><span class="header-section-number">4.1.1</span> Species occurrence</a></li>
  <li><a href="#defining-modes-of-life-traits" id="toc-defining-modes-of-life-traits" class="nav-link" data-scroll-target="#defining-modes-of-life-traits"><span class="header-section-number">4.1.2</span> Defining modes of life (traits)</a></li>
  <li><a href="#constructing-networks" id="toc-constructing-networks" class="nav-link" data-scroll-target="#constructing-networks"><span class="header-section-number">4.1.3</span> Constructing networks</a></li>
  </ul></li>
  <li><a href="#models-capture-different-network-structure-but-in-unexpected-ways" id="toc-models-capture-different-network-structure-but-in-unexpected-ways" class="nav-link" data-scroll-target="#models-capture-different-network-structure-but-in-unexpected-ways"><span class="header-section-number">4.2</span> Models capture different network structure but in unexpected ways</a>
  <ul class="collapse">
  <li><a href="#macro-network-properties" id="toc-macro-network-properties" class="nav-link" data-scroll-target="#macro-network-properties"><span class="header-section-number">4.2.1</span> Macro network properties</a></li>
  <li><a href="#meso-network-properties" id="toc-meso-network-properties" class="nav-link" data-scroll-target="#meso-network-properties"><span class="header-section-number">4.2.2</span> Meso network properties</a></li>
  <li><a href="#micro-network-properties" id="toc-micro-network-properties" class="nav-link" data-scroll-target="#micro-network-properties"><span class="header-section-number">4.2.3</span> Micro network properties</a></li>
  </ul></li>
  <li><a href="#some-networks-dont-share-any-interactions-and-some-share-a-lot" id="toc-some-networks-dont-share-any-interactions-and-some-share-a-lot" class="nav-link" data-scroll-target="#some-networks-dont-share-any-interactions-and-some-share-a-lot"><span class="header-section-number">4.3</span> Some networks don’t share any interactions and some share a lot</a></li>
  <li><a href="#model-choice-changes-the-narrative" id="toc-model-choice-changes-the-narrative" class="nav-link" data-scroll-target="#model-choice-changes-the-narrative"><span class="header-section-number">4.4</span> Model choice changes the narrative</a>
  <ul class="collapse">
  <li><a href="#trends-over-time" id="toc-trends-over-time" class="nav-link" data-scroll-target="#trends-over-time"><span class="header-section-number">4.4.1</span> Trends over time</a></li>
  <li><a href="#inferred-extinction-drivers" id="toc-inferred-extinction-drivers" class="nav-link" data-scroll-target="#inferred-extinction-drivers"><span class="header-section-number">4.4.2</span> Inferred extinction drivers</a></li>
  </ul></li>
  </ul></li>
  <li><a href="#discussion-need-a-catchier-heading" id="toc-discussion-need-a-catchier-heading" class="nav-link" data-scroll-target="#discussion-need-a-catchier-heading"><span class="header-section-number">5</span> Discussion (need a catchier heading)</a></li>
  <li><a href="#references" id="toc-references" class="nav-link" data-scroll-target="#references">References</a></li>
  </ul>
<div class="quarto-alternate-notebooks"><h2>Notebooks</h2><ul><li><a href="notebooks/contemporary_webs-preview.html"><i class="bi bi-journal-code"></i>SuppMat 1: Comparing models using contemporary food webs</a></li><li><a href="notebooks/model_sumaries-preview.html"><i class="bi bi-journal-code"></i>SuppMat 1: Expanded descriptions of the food web models</a></li></ul></div></nav>
</div>
<div id="quarto-margin-sidebar" class="sidebar margin-sidebar zindex-bottom">
</div>
<main class="content quarto-banner-title-block" id="quarto-document-content">



  


<p>There has been a growing interest in looking at community responses to environmental changes across deep time events as a means to help understand current and future biodiversity changes <span class="citation" data-cites="kiessling2019 dillon2022">(<a href="#ref-kiessling2019" role="doc-biblioref">Kiessling et al. 2019</a>; <a href="#ref-dillon2022" role="doc-biblioref">Dillon et al. 2022</a>)</span>. Species interactions and and the resulting networks hav gained an interest in contemporary settings as a means to help us to understand aspects of community composition and biodiversity (eg <span class="citation" data-cites="thuiller2024">Thuiller et al. (<a href="#ref-thuiller2024" role="doc-biblioref">2024</a>)</span> and ??) and so it is perhaps unsurprising that there has been a growing interest in using paleo food webs in a similar manner <span class="citation" data-cites="dunhill2024 hao2025 yeakel2014">(*e.g.,* <a href="#ref-dunhill2024" role="doc-biblioref">Dunhill et al. 2024</a> looked at…; <a href="#ref-hao2025" role="doc-biblioref">Hao et al. 2025</a> looked at…; <a href="#ref-yeakel2014" role="doc-biblioref">Yeakel et al. 2014</a> looked at…)</span>. However, one of the core challenges and limitations of being able to effectively <em>use</em> food webs is the challenge of <em>creating</em> them <span class="citation" data-cites="jordano2016">(<a href="#ref-jordano2016" role="doc-biblioref">Jordano 2016</a>)</span>, although this is a challenge within contemporary settings it is compounded in paleo contexts as we are dependant on the fossil record (and the inherent limitation it imposes) to infer interactions from. As a way to address the challenges with recording species interactions there has been the development of a large number of models and tools that can be used to infer either species interactions <span class="citation" data-cites="morales-castilla2015 pichler2023 strydom2021">(see <em>e.g.,</em> <a href="#ref-morales-castilla2015" role="doc-biblioref">Morales-Castilla et al. 2015</a>; <a href="#ref-pichler2023" role="doc-biblioref">Pichler and Hartig 2023</a>; <a href="#ref-strydom2021" role="doc-biblioref">Strydom et al. 2021b</a> for broader reviews)</span> or networks <span class="citation" data-cites="allesina2008">(see <em>e.g.,</em> <a href="#ref-allesina2008" role="doc-biblioref">Allesina, Alonso, and Pascual 2008</a>)</span>. Although there has been the development of models and tools that are specific for inferring paelo food webs <span class="citation" data-cites="shaw2024 roopnarine2006 fricke2022">(*e.g.,* <a href="#ref-shaw2024" role="doc-biblioref">Shaw et al. 2024</a>; <a href="#ref-roopnarine2006" role="doc-biblioref">Roopnarine 2006</a>; <a href="#ref-fricke2022" role="doc-biblioref">Fricke et al. 2022</a>)</span>, it should be noted that these models only occupy a subset of the broader family of approaches that are used to predict networks as they typically use only one mechanism for determining interactions (the feasibility of the interaction being able to occur). Being able to only construct one ‘type’ of network means that we are limited in the scope of questions that we can appropriately answer with those networks [see Strydom in prep; <span class="citation" data-cites="gauzens2025">Gauzens et al. (<a href="#ref-gauzens2025" role="doc-biblioref">2025</a>)</span>]. However there is scope that models and tools that have been developed in contemporary settings have the potential to be used for paleo settings <span class="citation" data-cites="yeakel2014">(<em>e.g.,</em> <a href="#ref-yeakel2014" role="doc-biblioref">Yeakel et al. 2014</a>)</span>, which opens the door for researchers to ask a broader and more complete range of questions about community responses to environmental change.</p>
<p>Here we: want to identify the differences between models that predict interactions, and models that predict network structure. Specifically we want to look at 1) the structural difference between all models (<em>i.e.,</em> do we see a difference in the distribution of links between networks that have the same number of nodes?) and 2) the identity of pairwise links between species pairs (<em>i.e.,</em> do different models differ in which links are present (or absent) between species pairs?) Additionally we want to establish if using networks that are constructed using different models will change the the downstream inferences that are made for this we use the work from <span class="citation" data-cites="dunhill2024">Dunhill et al. (<a href="#ref-dunhill2024" role="doc-biblioref">2024</a>)</span> as a case study.</p>
<section id="contextualising-the-prediction-of-paleo-webs-within-the-contemporary-toolbox" class="level1" data-number="1">
<h1 data-number="1"><span class="header-section-number">1</span> Contextualising the prediction of paleo webs within the contemporary toolbox</h1>
<p>There is an evolving body of work that focuses on developing tools specifically for the task of predicting food webs. However as highlighted in Strydom (T4T in prep) it is important that we understand what assumptions are being embedded within the network as a result of the underlying philosophy which a model was built on. Broadly we can think about models that are nested within two different schools of thought (and thus methodological approaches, <a href="#fig-concept" class="quarto-xref">Figure&nbsp;1</a>), models that focus on assessing the <em>mechanistic</em> feasibility of an interaction being able to occur between two species or models that are more closely married to specific bodies of ecological <em>theory</em> - such as niche theory or foraging ecology. Broadly speaking the difference between these two modelling approaches is that mechanistic models typically asses interactions at a pairwise level buy determining if an interaction is feasible between a species <em>pair</em> (extended <em>e.g.,</em> here about trait-based assumptions), whereas theoretical models typically use some set of assumptions to constrain the distribution of links at the <em>community</em> scale (extended <em>e.g.,</em> here probably niche model or DBM).</p>
<div id="fig-concept" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-concept-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="figures/model_fams.png" class="lightbox" data-gallery="quarto-lightbox-gallery-1" title="Figure&nbsp;1: This obviously needs work but a variation on this to try and articulate the different approaches and broadly how they may differ."><img src="figures/model_fams.png" class="img-fluid figure-img"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-concept-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;1: This obviously needs work but a variation on this to try and articulate the different approaches and broadly how they may differ.
</figcaption>
</figure>
</div>
<p>Models that have specifically been developed in the paleo space tend to be mechanistic models that focus on using a trait-based approach to formalise feeding interactions (<em>e.g.,</em> <span class="citation" data-cites="shaw2024">Shaw et al. (<a href="#ref-shaw2024" role="doc-biblioref">2024</a>)</span>; <span class="citation" data-cites="roopnarine2006">Roopnarine (<a href="#ref-roopnarine2006" role="doc-biblioref">2006</a>)</span>), are assembled by expert opinion (<em>e.g.</em> <span class="citation" data-cites="dunne2014">Dunne, Labandeira, and Williams (<a href="#ref-dunne2014" role="doc-biblioref">2014</a>)</span>) or make assumptions based on the evolutionary signals of interactions (<em>e.g.,</em> <span class="citation" data-cites="fricke2022">Fricke et al. (<a href="#ref-fricke2022" role="doc-biblioref">2022</a>)</span>). However, there is an argument that the theoretical models that have been developed in contemporary settings should hold even for paleo communities as we expect the fundamental ‘currencies of life’ to have remained constant - <em>e.g.,</em> the energetic constraints of foraging or foraging niches (is that the right way to phrase it, also I wonder is we can link to some other paleo work that showcases <em>e.g.,</em> niche/competition is something present among some of the earliest animals). Somehow close this out by going from we should be able to use contemporary models we need to think about the constraints that are typically placed on us by paleo data as well as the assumptions that some of these models might require us to make (<em>e.g.</em> the niche model makes some heavy assumptions by constraining the connectance - which itself is often used as a metric to understand changes or differences in network structure).</p>
<blockquote class="blockquote">
<p>Additionally there are models that allow one to construct structurally sound networks that require very little input data. These are methods that are amenable to the data constraints that are prevalent in paleo communities in terms of both the completeness of fossil records as well as how the deeper in time we move the further away we might be moving from contemporary analogs. Secondly it should also be noted that different network construction approaches are encoding different processes (T4T, in prep) and there is value in showcasing how the networks construct models may differ.</p>
</blockquote>
<p>Introduce here that it is thus important to understand that not all contemporary models may actually be suitable for paleo contexts as the assumptions that they make (or the data that they require) may actually introduce uncertainty/errors into the resulting network rendering them of little use. SImilarly not all paleo methods will be suitable for all paleo communities. As a simple example the framework developed by <span class="citation" data-cites="fricke2022">Fricke et al. (<a href="#ref-fricke2022" role="doc-biblioref">2022</a>)</span> uses phylogenetic relatedness as a way to infer interactions of Pleistocene mammals by looking at how their extant relatives interact. Although this approach is ecologically sound (phylogenetic relatedness is also used in other approaches <em>e.g.,</em> <span class="citation" data-cites="strydom2022">Strydom et al. (<a href="#ref-strydom2022" role="doc-biblioref">2022</a>)</span>) there is also an argument that the further back in evolutionary time we go (and the greater the phylogenetic distance between extant and extinct communities become) there is more uncertainty introduced by the phylogenetic tree than what is introduced by assuming that interactions will be phylogenetically conserved. On the other side of the coin it can be very challenging to determine traits from the fossil record and so it may be instead by more pragmatic to use models that that are completely agnostic to the identity of the species and are instead concerned with the network structure (<em>e.g.,</em> the niche model developed by <span class="citation" data-cites="williams2008">Richard J. Williams and Martinez (<a href="#ref-williams2008" role="doc-biblioref">2008</a>)</span>). Fundamentally this means that there is a trade off between the data that is available and the type of network one is interested in creating.</p>
<blockquote class="blockquote">
<p>this section should be a box we can link to in the intro I think. We can also include some key ‘definitions’</p>
</blockquote>
</section>
<section id="understanding-how-networks-are-different" class="level1" data-number="2">
<h1 data-number="2"><span class="header-section-number">2</span> Understanding how networks are different</h1>
<p>It is important to be aware that networks can be configured in different ways depending on how the interactions are defined (Strydom, in prep) and that the configuration of the desired network should be informed by the research question <span class="citation" data-cites="gauzens2025">(<a href="#ref-gauzens2025" role="doc-biblioref">Gauzens et al. 2025</a>)</span>.</p>
<p>Think about the axes - trait-based/mechanistic model (metawebs) and then we have the statistical/theoretical models (which have their own mini axis of regression vs full theory models… - see also <a href="#fig-concept" class="quarto-xref">Figure&nbsp;1</a>). Basically we have metawebs (which represent <em>potential</em> interactions, mechanistic networks), realised networks (which represent the subset of potential that are realised as a result of community and environmental context, theory networks), and structural networks (species agnostic networks that are structurally informative, theory networks). Here also talk about the implications of these different networks types - different uses and capturing different processes. Specifically link this to models - <em>i.e.,</em> different models have been developed to construct a <em>specific</em> network representation.</p>
<section id="key-definitions" class="level4" data-number="2.0.0.1">
<h4 data-number="2.0.0.1" class="anchored" data-anchor-id="key-definitions"><span class="header-section-number">2.0.0.1</span> Key Definitions</h4>
<ul>
<li>Metaweb:</li>
<li>Realised Network:</li>
<li>Structural Network:</li>
<li>Feasible Interactions:</li>
<li>Realised interactions:</li>
</ul>
</section>
</section>
<section id="challenges-specific-to-building-paleo-networks" class="level1" data-number="3">
<h1 data-number="3"><span class="header-section-number">3</span> Challenges specific to building paleo networks</h1>
<p>Although there has been a push for the development of tools and methods that allow us to predict species interactions and networks [see <em>e.g.,</em> XXX for some reviews] they will not all be suitable for the prediction of paleo communities. This is primarily due to limitations that we are faced with in terms of the information that can be inferred from the fossil record (such as species traits abundances, and assemblages), which is needed as input data for the different models. The limited information available from the fossil record is compounded by the incomplete and biased preservation of species [REF], the spatial ambiguity of fossils found in a location [were species conserved <em>in situ</em> or were they there owing to geological processes; REF], and an increasing degree of ‘fuzziness’ the further one moves back in geological time [our understanding of both phylogenetic and functional trait space, REF]. Methodologically speaking some tools that ‘learn’ from contemporary communities (<em>e.g.,</em> <span class="citation" data-cites="strydom2023">Strydom et al. (<a href="#ref-strydom2023" role="doc-biblioref">2023</a>)</span>, <span class="citation" data-cites="caron2022">Caron et al. (<a href="#ref-caron2022" role="doc-biblioref">2022</a>)</span>) will become ‘worse’ the further one goes back in time since species then look very different from now but can still be useful for ‘recent’ communities (<em>e.g.,</em> <span class="citation" data-cites="fricke2022">Fricke et al. (<a href="#ref-fricke2022" role="doc-biblioref">2022</a>)</span>). This is not to say that it is impossible to construct paleo networks but rather identify that there are a subset of models that are probably not at all suitable for constructing paleo networks (<em>e.g.,</em> Null models, since there are fully driven by abundance), other methods will be better suited depending on the community of interest <em>e.g.,</em> for more contemporary communities that have modern analogs we can use methods rooted in phylogeny (<em>e.g.,</em> <span class="citation" data-cites="fricke2022">Fricke et al. (<a href="#ref-fricke2022" role="doc-biblioref">2022</a>)</span>, <span class="citation" data-cites="strydom2023">Strydom et al. (<a href="#ref-strydom2023" role="doc-biblioref">2023</a>)</span>) or traits (<em>e.g.,</em> <span class="citation" data-cites="caron2022">Caron et al. (<a href="#ref-caron2022" role="doc-biblioref">2022</a>)</span>), and then there is the third axis which is to think about which are the assumptions that are made and there trade off of that. This includes thinking about both assumptions you are making about the actual data <em>e.g.,</em> trying to extrapolate body size from fossil data but also assumptions across time <em>e.g.,</em> assuming modern trait-feeding modes are the same OR that assumptions about network structure will hold across deep time.</p>
<section id="approaches-to-food-web-prediction" class="level2" data-number="3.1">
<h2 data-number="3.1" class="anchored" data-anchor-id="approaches-to-food-web-prediction"><span class="header-section-number">3.1</span> Approaches to food web prediction</h2>
<p>Here we present six different models (<a href="#tbl-models" class="quarto-xref">Table&nbsp;1</a>) that can be used to construct food webs for both this specific community but are also broadly suited to paleo network prediction. These models span all facets of the network representation space (metaweb, realised, and structural network) and are suitable for an array of different paleo communities as the data requirements are ‘paleo friendly’.</p>
<div id="tbl-models" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-tbl figure">
<figcaption class="quarto-float-caption-top quarto-float-caption quarto-float-tbl" id="tbl-models-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Table&nbsp;1: A summary of the different families of tools that can be used to generate paleo food webs.
</figcaption>
<div aria-describedby="tbl-models-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<table class="caption-top table">
<colgroup>
<col style="width: 16%">
<col style="width: 16%">
<col style="width: 16%">
<col style="width: 16%">
<col style="width: 16%">
<col style="width: 16%">
</colgroup>
<thead>
<tr class="header">
<th>Model family</th>
<th>Assumptions</th>
<th>Data needs</th>
<th>‘Limitation’</th>
<th>Network type</th>
<th>Key reference</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>random</td>
<td>Links are randomly distributed within a network</td>
<td>richness, number of links</td>
<td>parameter assumptions, species agnostic</td>
<td>structural network</td>
<td><span class="citation" data-cites="erdos1959">Erdős and Rényi (<a href="#ref-erdos1959" role="doc-biblioref">1959</a>)</span></td>
</tr>
<tr class="even">
<td>niche</td>
<td>Networks are interval, species can be ordered on a ‘niche axis’</td>
<td>richness, connectance</td>
<td>parameter assumptions, species agnostic</td>
<td>structural network</td>
<td><span class="citation" data-cites="williams2008">Richard J. Williams and Martinez (<a href="#ref-williams2008" role="doc-biblioref">2008</a>)</span></td>
</tr>
<tr class="odd">
<td>allometric diet breadth model (ADBM)</td>
<td>Interactions are determined by energetic costs (foraging ecology)</td>
<td>body mass, biomass (abundance)</td>
<td>does not account for forbidden links in terms of trait compatibility, assumptions on body size and biomass (abundance) from fossil data</td>
<td>theoretical network</td>
<td><span class="citation" data-cites="petchey2008">Petchey et al. (<a href="#ref-petchey2008" role="doc-biblioref">2008</a>)</span></td>
</tr>
<tr class="even">
<td>l-matrix</td>
<td>Interactions inferred using allometric rules (ratio of body sizes between predator and prey), with links being constrained by a Ricker function</td>
<td>body mass, number of producer species</td>
<td>does not account for forbidden links in terms of trait compatibility, assumptions on body size from fossil data, assumptions as to the number of producer species</td>
<td>theoretical network</td>
<td><span class="citation" data-cites="schneider2016">Schneider et al. (<a href="#ref-schneider2016" role="doc-biblioref">2016</a>)</span></td>
</tr>
<tr class="odd">
<td>paleo food web inference model (PFIM)</td>
<td>Interactions can be inferred by a mechanistic framework/relationships</td>
<td>feeding traits for taxa, mechanistic feeding rules</td>
<td>Assumption made as to the feeding mechanisms, need to elucidate traits from models (although this is a way smaller issue)</td>
<td>mechanistic web</td>
<td><span class="citation" data-cites="shaw2024">Shaw et al. (<a href="#ref-shaw2024" role="doc-biblioref">2024</a>)</span></td>
</tr>
<tr class="even">
<td>body size ratio model</td>
<td>Interactions inferred using allometric rules (ratio of body sizes between predator and prey). :ogit of the linking probability used to further constrain links to an ‘optimal size range’ for prey.</td>
<td>body mass</td>
<td>does not account for forbidden links in terms of evolutionary compatibility, assumptions on body size from fossil data</td>
<td>theoretical network</td>
<td><span class="citation" data-cites="rohr2010">Rohr et al. (<a href="#ref-rohr2010" role="doc-biblioref">2010</a>)</span></td>
</tr>
</tbody>
</table>
</div>
</figure>
</div>
</section>
</section>
<section id="case-study-toarcian-mass-extinction-event" class="level1" data-number="4">
<h1 data-number="4"><span class="header-section-number">4</span> Case study: Toarcian mass extinction event</h1>
<section id="dataset-overview" class="level2" data-number="4.1">
<h2 data-number="4.1" class="anchored" data-anchor-id="dataset-overview"><span class="header-section-number">4.1</span> Dataset overview</h2>
<section id="species-occurrence" class="level3" data-number="4.1.1">
<h3 data-number="4.1.1" class="anchored" data-anchor-id="species-occurrence"><span class="header-section-number">4.1.1</span> Species occurrence</h3>
<p>Here we use the fossil occurrence data over an interval extends from the upper Pliensbachian (~185 Ma) to the upper Toarcian (~175 Ma) of the Cleveland Basin <span class="citation" data-cites="dunhill2024">(see <a href="#ref-dunhill2024" role="doc-biblioref">Dunhill et al. 2024</a> for a more comprehensive overview)</span>. The data set consists of a subset of four broad time periods (pre-extinction, post-extinction, early recovery, and late recovery). The assemblages are treated as communities of interacting organisms. Something about the total number of taxa as well as numbers per a time period? Probbaly also make a comment that this is a ‘deep time’ community we are looking at.</p>
</section>
<section id="defining-modes-of-life-traits" class="level3" data-number="4.1.2">
<h3 data-number="4.1.2" class="anchored" data-anchor-id="defining-modes-of-life-traits"><span class="header-section-number">4.1.2</span> Defining modes of life (traits)</h3>
<p>We used the modes of life (traits) as identified in <span class="citation" data-cites="dunhill2024">Dunhill et al. (<a href="#ref-dunhill2024" role="doc-biblioref">2024</a>)</span>, who defined four traits: motility (fast, slow, facultative, non-motile), tiering (pelagic, erect, surficial, semi-infaunal, shallow infaunal, deep infaunal), feeding (predator, suspension feeder, deposit feeder, mining, grazer), and size: gigantic (&gt;500 mm), very large (&gt;300–500 mm), large (&gt;100–300 mm), medium (&gt;50–100 mm), small (&gt;10–50 mm), tiny (≤10 mm), for each fossil species based on the ecological traits defined in the Bambach ecospace model <span class="citation" data-cites="bambach2007">(<a href="#ref-bambach2007" role="doc-biblioref">Bambach, Bush, and Erwin 2007</a>)</span>.</p>
</section>
<section id="constructing-networks" class="level3" data-number="4.1.3">
<h3 data-number="4.1.3" class="anchored" data-anchor-id="constructing-networks"><span class="header-section-number">4.1.3</span> Constructing networks</h3>
<p>For each paleo community (time bin) we constructed <strong>100</strong> networks for each model (so 6 * 100) networks. These networks were ‘simplified’ to removed any disconnected species. In total 2400 networks were constructed. When a quantitative measure of body size is needed (ADBM, bodymassratio, lmatrix) we drew a body mass for each species from a uniform distribution. The ranges were defined by the different size classes as discussed in insert cross ref to correct subsection here <em>e.g.,</em> a species classed as ‘very large’ would have a body mass drawn from <span class="math inline">\(U(300, 500)\)</span>. This was repeated for each run in order to add variation to the networks constructed, however the same body sizes were kept consistent for the relevant models (adbm, bodymassratio, l-matrix) <em>i.e.,</em> an ADBM and bodymassratio network from the same rep number would have used the same bodysizes. The PFIM networks were downsampled (see relevant section is S1). For both the random and niche model the desired connectance was randomly selected between the range 0.07 - 0.15 for each repetition but kep consistent for both models. For each network we calculated the properties listed in <a href="#tbl-properties" class="quarto-xref">Table&nbsp;2</a></p>
</section>
</section>
<section id="models-capture-different-network-structure-but-in-unexpected-ways" class="level2" data-number="4.2">
<h2 data-number="4.2" class="anchored" data-anchor-id="models-capture-different-network-structure-but-in-unexpected-ways"><span class="header-section-number">4.2</span> Models capture different network structure but in unexpected ways</h2>
<p>Why is structure important and what can it tell us? Broadly when we talk about quantifying the structure of a network we are interesting in capturing some aspect of how the links are distributed between nodes, or alternatively about properties of the nodes (specifically in terms of the number of links coming in to (prey) or out of (predators) the node). What are some things we can learn/infer from network structure: energy flows and fluxes [REF], propagation of stress [REF], roles of species in the community [REF, think trophic levels]. Some closing statement about how thus there are different facets of network structure and the value of understanding generally how different models differ in terms of the structure that they recover - link to <a href="#tbl-properties" class="quarto-xref">Table&nbsp;2</a> maybe.</p>
<p>In terms of wanting to asses and compare across the different models it is beneficial to approach this task by thinking about the different aspects of the network as well as interactions that are being predicted by the different models. It is perhaps beneficial to think of these across different ‘scales’ of organisation within the network, namely macro (the entire network), meso (smaller interacting units within the network), and micro (species-level attributes). Although there are a myriad of possible ways to ‘measure’ and analyse ecological networks <span class="citation" data-cites="delmas2018">(<a href="#ref-delmas2018" role="doc-biblioref">Delmas et al. 2018</a>)</span> we do still lack a clear set of guidelines for assessing how well models recover network structure <span class="citation" data-cites="allesina2008">(<a href="#ref-allesina2008" role="doc-biblioref">Allesina, Alonso, and Pascual 2008</a>)</span> and it is beneficial to use a small subset of metrics that can clearly be tied to broader aspects of network function or capturing a ecological process.</p>
<p>Here we used a Multivariate Analysis Of Variance (MANOVA) as it is able to capture model differences based on the combined information of the multiple structural network measures. Model defined as <code>network structure values ~ model + time period</code> and Linear Discriminant Analysis (LDA) to determine if different models produced networks with differing structure. <strong>Need to report the relevant effect of time in driving observed differences</strong></p>
<div id="tbl-properties" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-tbl figure">
<figcaption class="quarto-float-caption-top quarto-float-caption quarto-float-tbl" id="tbl-properties-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Table&nbsp;2: An informative caption about the different network properties
</figcaption>
<div aria-describedby="tbl-properties-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<table class="caption-top table">
<colgroup>
<col style="width: 24%">
<col style="width: 26%">
<col style="width: 24%">
<col style="width: 24%">
</colgroup>
<thead>
<tr class="header">
<th>Label</th>
<th>Definition</th>
<th>Scale</th>
<th>Reference (for maths), can make footnotes probs</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Connectance</td>
<td><span class="math inline">\(L/S^2\)</span>, where <span class="math inline">\(S\)</span> is the number of species and <span class="math inline">\(L\)</span> the number of links</td>
<td>Macro</td>
<td></td>
</tr>
<tr class="even">
<td>GenSD</td>
<td>Normalized standard deviation of generality of a species standardized by <span class="math inline">\(L/S\)</span></td>
<td>Micro</td>
<td><span class="citation" data-cites="williams2000">Richard J. Williams and Martinez (<a href="#ref-williams2000" role="doc-biblioref">2000</a>)</span></td>
</tr>
<tr class="odd">
<td>LinkSD</td>
<td>Normalized standard deviation of links (number of consumers plus resources per taxon)</td>
<td>Micro</td>
<td></td>
</tr>
<tr class="even">
<td>Richness</td>
<td>Number of nodes in the network</td>
<td>Macro</td>
<td></td>
</tr>
<tr class="odd">
<td>TL</td>
<td>Prey-weighted trophic level averaged across taxa</td>
<td>Macro</td>
<td><span class="citation" data-cites="williams2004">R. J. Williams and Martinez (<a href="#ref-williams2004" role="doc-biblioref">2004</a>)</span></td>
</tr>
<tr class="even">
<td>VulSD</td>
<td>Normalized standard deviation of vulnerability of a species standardized by <span class="math inline">\(L/S\)</span></td>
<td>Micro</td>
<td><span class="citation" data-cites="williams2000">Richard J. Williams and Martinez (<a href="#ref-williams2000" role="doc-biblioref">2000</a>)</span></td>
</tr>
<tr class="odd">
<td>Diameter</td>
<td>Diameter can also be measured as the average of the distances between each pair of nodes in the network</td>
<td>Macro</td>
<td><span class="citation" data-cites="delmas2018">Delmas et al. (<a href="#ref-delmas2018" role="doc-biblioref">2018</a>)</span></td>
</tr>
<tr class="even">
<td><span class="math inline">\(\rho\)</span></td>
<td>Spectral radius is a a conceptual analog to nestedness (and more appropriate for unipartite networks). It is defined as the absolute value of the largest real part of the eigenvalues of the <em>undirected</em> adjacency matrix</td>
<td>Macro</td>
<td><span class="citation" data-cites="staniczenko2013">Staniczenko, Kopp, and Allesina (<a href="#ref-staniczenko2013" role="doc-biblioref">2013</a>)</span></td>
</tr>
<tr class="odd">
<td>Complexity</td>
<td>SVD complexity of a network, defined as the Pielou entropy of its singular values</td>
<td>Macro</td>
<td><span class="citation" data-cites="strydom2021">Strydom et al. (<a href="#ref-strydom2021" role="doc-biblioref">2021b</a>)</span></td>
</tr>
<tr class="even">
<td>S1</td>
<td>Number of linear chains</td>
<td>Meso</td>
<td><span class="citation" data-cites="milo2002">Milo et al. (<a href="#ref-milo2002" role="doc-biblioref">2002</a>)</span>; <span class="citation" data-cites="stouffer2007">Stouffer et al. (<a href="#ref-stouffer2007" role="doc-biblioref">2007</a>)</span></td>
</tr>
<tr class="odd">
<td>S2</td>
<td>Number of omnivory motifs</td>
<td>Meso</td>
<td><span class="citation" data-cites="milo2002">Milo et al. (<a href="#ref-milo2002" role="doc-biblioref">2002</a>)</span>; <span class="citation" data-cites="stouffer2007">Stouffer et al. (<a href="#ref-stouffer2007" role="doc-biblioref">2007</a>)</span></td>
</tr>
<tr class="even">
<td>S4</td>
<td>Number of apparent competition motifs</td>
<td>Meso</td>
<td><span class="citation" data-cites="milo2002">Milo et al. (<a href="#ref-milo2002" role="doc-biblioref">2002</a>)</span>; <span class="citation" data-cites="stouffer2007">Stouffer et al. (<a href="#ref-stouffer2007" role="doc-biblioref">2007</a>)</span></td>
</tr>
<tr class="odd">
<td>S5</td>
<td>Number of direct competition motifs</td>
<td>Meso</td>
<td><span class="citation" data-cites="milo2002">Milo et al. (<a href="#ref-milo2002" role="doc-biblioref">2002</a>)</span>; <span class="citation" data-cites="stouffer2007">Stouffer et al. (<a href="#ref-stouffer2007" role="doc-biblioref">2007</a>)</span></td>
</tr>
</tbody>
</table>
</div>
</figure>
</div>
<section id="macro-network-properties" class="level3" data-number="4.2.1">
<h3 data-number="4.2.1" class="anchored" data-anchor-id="macro-network-properties"><span class="header-section-number">4.2.1</span> Macro network properties</h3>
<p><strong>Connectance</strong> <span class="citation" data-cites="martinez1992">(<a href="#ref-martinez1992" role="doc-biblioref">Martinez 1992</a>)</span> has been shown to be the feature of networks that underpin a series of other properties and function <span class="citation" data-cites="strydom2021b">(<a href="#ref-strydom2021b" role="doc-biblioref">Strydom et al. 2021a</a>)</span> and so it is perhaps the most important structural attribute for a model to be able to retrieve correctly. Additionally we consider the <strong>complexity</strong> of networks by calculating their SVD entropy (this gives us an estimate of the physical as opposed to behavioural complexity of networks; <span class="citation" data-cites="strydom2021">Strydom et al. (<a href="#ref-strydom2021" role="doc-biblioref">2021b</a>)</span>), we could also look at the rank/rank deficiency of networks which (theoretically) represents the number fo unique interaction strategies in the network <span class="citation" data-cites="strydom2021">(<a href="#ref-strydom2021" role="doc-biblioref">Strydom et al. 2021b</a>)</span>, which may be specifically interesting in terms of looking at pre and post extinction but also as a way to unpack ‘functional redundancy’ that some models may introduce.</p>
</section>
<section id="meso-network-properties" class="level3" data-number="4.2.2">
<h3 data-number="4.2.2" class="anchored" data-anchor-id="meso-network-properties"><span class="header-section-number">4.2.2</span> Meso network properties</h3>
<p>Motifs represent smaller subset of interactions between three species, and are argued to capture dynamics that are likely to be ecologically relevant <span class="citation" data-cites="milo2002 stouffer2007">(<a href="#ref-milo2002" role="doc-biblioref">Milo et al. 2002</a>; <a href="#ref-stouffer2007" role="doc-biblioref">Stouffer et al. 2007</a>)</span>. Here we specifically look at the number of <strong>linear chains</strong>, <strong>omnivory</strong>, <strong>apparent competition</strong>, and <strong>direct competition</strong> motifs. In the broader context the ability of a model in being able to capture these smaller motifs will inform as to its suitability of use understanding the more dynamic component of network ecology.</p>
</section>
<section id="micro-network-properties" class="level3" data-number="4.2.3">
<h3 data-number="4.2.3" class="anchored" data-anchor-id="micro-network-properties"><span class="header-section-number">4.2.3</span> Micro network properties</h3>
<p>The number of interactions established (<strong>generality</strong>) or received (<strong>vulnerability</strong>) by each species <span class="citation" data-cites="schoener1989">(<a href="#ref-schoener1989" role="doc-biblioref">Schoener 1989</a>)</span>, are (broadly) indicative of consumer-resource relationships and diet breadth of species [ref]. Although this is usually determined at the species level the standard deviation of the generality and vulnerability of species is often used when benchmarking predicted networks <span class="citation" data-cites="williams2008 petchey2008">(<em>e.g.,</em> <a href="#ref-williams2008" role="doc-biblioref">Richard J. Williams and Martinez 2008</a>; <a href="#ref-petchey2008" role="doc-biblioref">Petchey et al. 2008</a>)</span>.</p>
<p>The <strong>specificity</strong> of species in a network is measured as a function of the proportion of resources they effectively use <span class="citation" data-cites="poisot2012">(<a href="#ref-poisot2012" role="doc-biblioref">Poisot et al. 2012</a>)</span></p>
<div id="fig-structure" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-structure-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="figures/MANOVA_lda.png" class="lightbox" data-gallery="quarto-lightbox-gallery-2" title="Figure&nbsp;2: stuff…"><img src="figures/MANOVA_lda.png" class="img-fluid figure-img"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-structure-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;2: stuff…
</figcaption>
</figure>
</div>
<p>What is perhaps the most striking result in <a href="#fig-structure" class="quarto-xref">Figure&nbsp;2</a> is that although there are clear structural differences between the different models the differences are not distinct between the broader model families but rather that there is a degree of overlap between them (specifically the log ratio, PFIM, and niche models). Although the log ratio and niche models are classified as different families they are built on similar ecological background and theory and so it is perhaps not surprising that these networks capture a similar structure (the same holds for the ADBM and l-matrix models). The fact that the random model occupies a completely different space is unsurprising as it has clearly been shown that networks are non-random in nature [REF] and so we expect random models to be constructing ecologically illogical networks. What is perhaps the most interesting result is that the PFIM model constructs networks that are very similar to those that are rooted in niche-based processes despite the model being more mechanistic in nature. Not sure how to articulate but this is cool because the is <em>something</em> in network structure constraints that is straddling the trait-niche space of ecology - but also see my next point about it being ‘correct’ is still up for debate</p>
<p>Although it is not possible to confidently identify the models that are predicting the <em>‘correct’</em> network structure the fact that a models from different families are able to recover similar structures is reassuring as it suggests that it might be possible to substitute one model for another if the input data is insufficient.</p>
<blockquote class="blockquote">
<p><strong>TODO</strong> Is it sound to try and unpack the ‘pairwise differences’ between the different structural metrics as well (or some) as this will allow us to say e.g.&nbsp;Niche and PFIM might recover the same connectance but differ in vulnerability.</p>
</blockquote>
</section>
</section>
<section id="some-networks-dont-share-any-interactions-and-some-share-a-lot" class="level2" data-number="4.3">
<h2 data-number="4.3" class="anchored" data-anchor-id="some-networks-dont-share-any-interactions-and-some-share-a-lot"><span class="header-section-number">4.3</span> Some networks don’t share any interactions and some share a lot</h2>
<p>In addition to wanting to measure network structure researchers may also be interested in understanding aspects about the diets and predators of <em>specific</em> species in a community. In this instance the interest should be in understanding how the pairwise links predicted between species pairs differ between models. Here we look ath the interaction turnover <span class="citation" data-cites="poisot2012">(<a href="#ref-poisot2012" role="doc-biblioref">Poisot et al. 2012</a>)</span> both within and between the different models. This can be thought of as the equivalent of species <span class="math inline">\(\beta\)</span> turnover and tells us which interactions are ‘conserved’ (shared) across the networks but only between species pairs that are shared - <em>i.e.,</em> this turnover is only driven by interaction and not species turnover. Here we only compared networks that we constructed for the same period (as our interest is only in between model differences) and excluded the random and niche networks from consideration as these two models are essentially species agnostic.</p>
<div id="fig-beta_div" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-beta_div-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="figures/beta_div.png" class="lightbox" data-gallery="quarto-lightbox-gallery-3" title="Figure&nbsp;3: stuff… % interaction shared is calculated as number shared interactions / ((number interactions left - shared interactions) + (number interactions right - shared interactions) + shared interactions). Additionally niche and random models are excluded as it is illogical since both of these models are fundamentally species agnostic"><img src="figures/beta_div.png" class="img-fluid figure-img"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-beta_div-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;3: stuff… % interaction shared is calculated as number shared interactions / ((number interactions left - shared interactions) + (number interactions right - shared interactions) + shared interactions). Additionally niche and random models are excluded as it is illogical since both of these models are fundamentally species agnostic
</figcaption>
</figure>
</div>
<p>In <a href="#fig-beta_div" class="quarto-xref">Figure&nbsp;3</a> it is clear that some models share a large percentage of interactions <em>e.g.,</em> between ADBM and l-matrix networks and others share nothing <em>e.g.,</em> ADBM and PFIM networks. This result is unsurprising as the mechanisms that determine interactions in ADBM and l-matrix (a single trait (bodysize) + some ecological theory) is very different from the PFIM model that makes assumptions on a trait-based hierarchy.</p>
<p>The key takeaway that this needs to lead into is thinking about diet related questions and the model that is best suited to get you there. It makes sense to contextualise this in the feasible vs realised interaction spectrum - specifically that from a ‘philosophical’ basis if you are asking questions about possible diets of species then it makes sense to use models that fall firmly in the ‘feasible’ space <em>e.g.,</em> PFIM model or even something like the <span class="citation" data-cites="fricke2022">Fricke et al. (<a href="#ref-fricke2022" role="doc-biblioref">2022</a>)</span> model. How these results support that is that we can see the ADBM and PFIM are recovering (almost) totally different pairwise links.</p>
</section>
<section id="model-choice-changes-the-narrative" class="level2" data-number="4.4">
<h2 data-number="4.4" class="anchored" data-anchor-id="model-choice-changes-the-narrative"><span class="header-section-number">4.4</span> Model choice changes the narrative</h2>
<section id="simulating-extinctions" class="level4" data-number="4.4.0.1">
<h4 data-number="4.4.0.1" class="anchored" data-anchor-id="simulating-extinctions"><span class="header-section-number">4.4.0.1</span> Simulating Extinctions</h4>
<p>Extinctions were simulated using different plausible mechanisms based on both species traits (size, motility), their position within the network (generality, vulnerability), as well as randomly. Each network was subjected to <strong>50</strong> extinction runs for each extinction mechanisms. The extinctions themselves were cascading in nature meaning that after the target species was removed all species that no longer had any prey were also deemed as extinct (secondary extinction), checking for secondary extinctions was then repeated until there were no longer any species without prey. This represents one extinction event and only then would the proceeding target species be removed from the network and cascading extinctions assessed again. Note that for extinction simulations which use the network position of a species to determine extinction order we follow the protocol from <span class="citation" data-cites="curtsdotter2011">Curtsdotter et al. (<a href="#ref-curtsdotter2011" role="doc-biblioref">2011</a>)</span> and reassess the vulnerability/generality of each species after each extinction event to ‘redetermine’ the extinction order.</p>
<p>As we are using <span class="citation" data-cites="dunhill2024">Dunhill et al. (<a href="#ref-dunhill2024" role="doc-biblioref">2024</a>)</span> as a case study we followed their appraoch when simulating extinctions as well as assessing which extinction mechanism results in a simulated network that most closely matches the real post extinciton network. Extinction simulations were only run on the pre extinction networks whereby species were removed until they reached the ‘target richness’, which is the richness of the post extinction community. In order to determine which extinction mechnism creates a network most similar to the post extinction network we used the (get full name of score) TSS <span class="citation" data-cites="gupta2022">(<a href="#ref-gupta2022" role="doc-biblioref">Gupta, Furrer, and Petchey 2022</a>)</span> to assess how different the pairwise interactions are between ‘simulated’ and ‘real’ post extinction communities as well as looking at the absolute differences in network structure metrics.</p>
<blockquote class="blockquote">
<p><strong>TODO</strong> not sure if we also want to unpack/showcase robustness <span class="math inline">\(R_{50}\)</span> <span class="citation" data-cites="jonsson2015">(<a href="#ref-jonsson2015" role="doc-biblioref">Jonsson et al. 2015</a>)</span></p>
</blockquote>
<div id="fig-dunhill" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-dunhill-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="figures/dunhill_comp.png" class="lightbox" data-gallery="quarto-lightbox-gallery-4" title="Figure&nbsp;4: stuff… Recreation of the figure from Dunhill 2024. Note not 100% sold on the TSS and absolute mean calculations…"><img src="figures/dunhill_comp.png" class="img-fluid figure-img"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-dunhill-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;4: stuff… Recreation of the figure from Dunhill 2024. Note not 100% sold on the TSS and absolute mean calculations…
</figcaption>
</figure>
</div>
</section>
<section id="trends-over-time" class="level3" data-number="4.4.1">
<h3 data-number="4.4.1" class="anchored" data-anchor-id="trends-over-time"><span class="header-section-number">4.4.1</span> Trends over time</h3>
<blockquote class="blockquote">
<p><strong>TODO</strong> Not sure statistically speaking what the best way to unpack this is… 2-way ANOVA/ANCOVA explanation is valuable? There are intercept differences (<em>e.g.,</em> baseline average values are different; are the rankings among all three response variables the same?) and there are shape differences/similarities (<em>e.g.,</em> motifs are all the same shape but Co and Gen show some among model differences in pattern.)</p>
</blockquote>
<p>Visual take-away seems to suggest that we see that the values (intercepts) of the different summary statistics are different but (broadly) they are capturing the same trends. This might suggest that although we observe differences in structure (<a href="#fig-structure" class="quarto-xref">Figure&nbsp;2</a>) the general patterns still remain the same. This is good news because it means that at least the models that we have used here tend to tell us the same general story - which is worth contextualising in the space of ‘right’ vs ‘wrong’ and as long as we are not fixated on the point value but rather on understanding the trends.</p>
</section>
<section id="inferred-extinction-drivers" class="level3" data-number="4.4.2">
<h3 data-number="4.4.2" class="anchored" data-anchor-id="inferred-extinction-drivers"><span class="header-section-number">4.4.2</span> Inferred extinction drivers</h3>
<p>Points of discussion one will be to point to the mean absolute distance and how generally the ADBM/l-matrix do really badly - high mean absolute value. And this maybe makes sense though because of how we specify extinction mechanisms (trait-based) and so it sets the body-size models are not ‘talking’ the same language. In terms of the TSS scores - not sure how we should unpack it. Individually by model family to see which model agrees with which appraoch and see if different mechanisms come out stronger? <strong>TODO</strong> still need to sanity check the TSS anyway…</p>
</section>
</section>
</section>
<section id="discussion-need-a-catchier-heading" class="level1" data-number="5">
<h1 data-number="5"><span class="header-section-number">5</span> Discussion (need a catchier heading)</h1>
<p>I want this section to be more about contextualising model choice within the bigger research question discussion - i.e.&nbsp;mapping question and model choice more tightly…</p>
<p>Points to discuss:</p>
<ul>
<li>Guidlines - as a box? Can we give something concrete??</li>
<li>How to we synthesise these results? As in should we give clear directives ot is it enough to do a bit more handwaving and have the bigger message be that model choice matters?</li>
</ul>
</section>
<section id="references" class="level1 unnumbered">


</section>

<div id="quarto-appendix" class="default"><section class="quarto-appendix-contents" role="doc-bibliography" id="quarto-bibliography"><h2 class="anchored quarto-appendix-heading">References</h2><div id="refs" class="references csl-bib-body hanging-indent" data-entry-spacing="0" role="list">
<div id="ref-allesina2008" class="csl-entry" role="listitem">
Allesina, Stefano, David Alonso, and Mercedes Pascual. 2008. <span>“A General Model for Food Web Structure.”</span> <em>Science</em> 320 (5876): 658–61. <a href="https://doi.org/10.1126/science.1156269">https://doi.org/10.1126/science.1156269</a>.
</div>
<div id="ref-bambach2007" class="csl-entry" role="listitem">
Bambach, Richard K., Andrew M. Bush, and Douglas H. Erwin. 2007. <span>“Autecology and the Filling of Ecospace: Key Metazoan Radiations.”</span> <em>Palaeontology</em> 50 (1): 1–22. <a href="https://doi.org/10.1111/j.1475-4983.2006.00611.x">https://doi.org/10.1111/j.1475-4983.2006.00611.x</a>.
</div>
<div id="ref-caron2022" class="csl-entry" role="listitem">
Caron, Dominique, Luigi Maiorano, Wilfried Thuiller, and Laura J. Pollock. 2022. <span>“Addressing the Eltonian Shortfall with Trait-Based Interaction Models.”</span> <em>Ecology Letters</em> 25 (4): 889–99. <a href="https://doi.org/10.1111/ele.13966">https://doi.org/10.1111/ele.13966</a>.
</div>
<div id="ref-curtsdotter2011" class="csl-entry" role="listitem">
Curtsdotter, Alva, Amrei Binzer, Ulrich Brose, Francisco De Castro, Bo Ebenman, Anna Eklöf, Jens O. Riede, Aaron Thierry, and Björn C. Rall. 2011. <span>“Robustness to Secondary Extinctions: Comparing Trait-Based Sequential Deletions in Static and Dynamic Food Webs.”</span> <em>Basic and Applied Ecology</em> 12 (7): 571–80. <a href="https://doi.org/10.1016/j.baae.2011.09.008">https://doi.org/10.1016/j.baae.2011.09.008</a>.
</div>
<div id="ref-delmas2018" class="csl-entry" role="listitem">
Delmas, Eva, Mathilde Besson, Marie-Hélène Brice, Laura A. Burkle, Giulio V. Dalla Riva, Marie-Josée Fortin, Dominique Gravel, et al. 2018. <span>“Analysing Ecological Networks of Species Interactions.”</span> <em>Biological Reviews</em>, June, 112540. <a href="https://doi.org/10.1111/brv.12433">https://doi.org/10.1111/brv.12433</a>.
</div>
<div id="ref-dillon2022" class="csl-entry" role="listitem">
Dillon, Erin M., Jaleigh Q. Pier, Jansen A. Smith, Nussaïbah B. Raja, Danijela Dimitrijević, Elizabeth L. Austin, Jonathan D. Cybulski, et al. 2022. <span>“What Is Conservation Paleobiology? Tracking 20 Years of Research and Development.”</span> <em>Frontiers in Ecology and Evolution</em> 10 (December). <a href="https://doi.org/10.3389/fevo.2022.1031483">https://doi.org/10.3389/fevo.2022.1031483</a>.
</div>
<div id="ref-dunhill2024" class="csl-entry" role="listitem">
Dunhill, Alexander M., Karolina Zarzyczny, Jack O. Shaw, Jed W. Atkinson, Crispin T. S. Little, and Andrew P. Beckerman. 2024. <span>“Extinction Cascades, Community Collapse, and Recovery Across a Mesozoic Hyperthermal Event.”</span> <em>Nature Communications</em> 15 (1): 8599. <a href="https://doi.org/10.1038/s41467-024-53000-2">https://doi.org/10.1038/s41467-024-53000-2</a>.
</div>
<div id="ref-dunne2014" class="csl-entry" role="listitem">
Dunne, Jennifer A., Conrad C. Labandeira, and Richard J. Williams. 2014. <span>“Highly Resolved Early Eocene Food Webs Show Development of Modern Trophic Structure After the End-Cretaceous Extinction.”</span> <em>Proceedings of the Royal Society B: Biological Sciences</em> 281 (1782): 20133280. <a href="https://doi.org/10.1098/rspb.2013.3280">https://doi.org/10.1098/rspb.2013.3280</a>.
</div>
<div id="ref-erdos1959" class="csl-entry" role="listitem">
Erdős, P., and A. Rényi. 1959. <span>“On Random Graphs. i.”</span> <em>Publicationes Mathematicae Debrecen</em> 6 (3-4): 290–97. <a href="https://doi.org/10.5486/pmd.1959.6.3-4.12">https://doi.org/10.5486/pmd.1959.6.3-4.12</a>.
</div>
<div id="ref-fricke2022" class="csl-entry" role="listitem">
Fricke, Evan C., Chia Hsieh, Owen Middleton, Daniel Gorczynski, Caroline D. Cappello, Oscar Sanisidro, John Rowan, Jens-Christian Svenning, and Lydia Beaudrot. 2022. <span>“Collapse of Terrestrial Mammal Food Webs Since the Late Pleistocene.”</span> <em>Science</em> 377 (6609): 1008–11. <a href="https://doi.org/10.1126/science.abn4012">https://doi.org/10.1126/science.abn4012</a>.
</div>
<div id="ref-gauzens2025" class="csl-entry" role="listitem">
Gauzens, Benoit, Lise Thouvenot, Diane S. Srivastava, Pavel Kratina, Gustavo Q. Romero, Emilio Berti, Eoin J. O’Gorman, et al. 2025. <span>“Tailoring Interaction Network Types to Answer Different Ecological Questions.”</span> <em>Nature Reviews Biodiversity</em>, June, 1–10. <a href="https://doi.org/10.1038/s44358-025-00056-7">https://doi.org/10.1038/s44358-025-00056-7</a>.
</div>
<div id="ref-gupta2022" class="csl-entry" role="listitem">
Gupta, Anubhav, Reinhard Furrer, and Owen L. Petchey. 2022. <span>“Simultaneously Estimating Food Web Connectance and Structure with Uncertainty.”</span> <em>Ecology and Evolution</em> 12 (3): e8643. <a href="https://doi.org/10.1002/ece3.8643">https://doi.org/10.1002/ece3.8643</a>.
</div>
<div id="ref-hao2025" class="csl-entry" role="listitem">
Hao, Xiyang, Marcel Holyoak, Zhicheng Zhang, and Chuan Yan. 2025. <span>“Global Projection of Terrestrial Vertebrate Food Webs Under Future Climate and Land-Use Changes.”</span> <em>Global Change Biology</em> 31 (2): e70061. <a href="https://doi.org/10.1111/gcb.70061">https://doi.org/10.1111/gcb.70061</a>.
</div>
<div id="ref-jonsson2015" class="csl-entry" role="listitem">
Jonsson, Tomas, Sofia Berg, Alexander Pimenov, Catherine Palmer, and Mark Emmerson. 2015. <span>“The Reliability of R50 as a Measure of Vulnerability of Food Webs to Sequential Species Deletions.”</span> <em>Oikos</em> 124 (4): 446–57. <a href="https://doi.org/10.1111/oik.01588">https://doi.org/10.1111/oik.01588</a>.
</div>
<div id="ref-jordano2016" class="csl-entry" role="listitem">
Jordano, Pedro. 2016. <span>“Chasing Ecological Interactions.”</span> <em>PLOS Biology</em> 14 (9): e1002559. <a href="https://doi.org/10.1371/journal.pbio.1002559">https://doi.org/10.1371/journal.pbio.1002559</a>.
</div>
<div id="ref-kiessling2019" class="csl-entry" role="listitem">
Kiessling, Wolfgang, Nussaïbah B. Raja, Vanessa Julie Roden, Samuel T. Turvey, and Erin E. Saupe. 2019. <span>“Addressing Priority Questions of Conservation Science with Palaeontological Data.”</span> <em>Philosophical Transactions of the Royal Society B: Biological Sciences</em> 374 (1788): 20190222. <a href="https://doi.org/10.1098/rstb.2019.0222">https://doi.org/10.1098/rstb.2019.0222</a>.
</div>
<div id="ref-martinez1992" class="csl-entry" role="listitem">
Martinez, Neo D. 1992. <span>“Constant Connectance in Community Food Webs.”</span> <em>The American Naturalist</em> 139 (6): 1208–18. <a href="http://www.jstor.org/stable/2462337">http://www.jstor.org/stable/2462337</a>.
</div>
<div id="ref-milo2002" class="csl-entry" role="listitem">
Milo, R., S. Shen-Orr, S. Itzkovitz, N. Kashtan, D. Chklovskii, and U. Alon. 2002. <span>“Network Motifs: Simple Building Blocks of Complex Networks.”</span> <em>Science</em> 298 (5594): 824–27. <a href="https://doi.org/10.1126/science.298.5594.824">https://doi.org/10.1126/science.298.5594.824</a>.
</div>
<div id="ref-morales-castilla2015" class="csl-entry" role="listitem">
Morales-Castilla, Ignacio, Miguel G. Matias, Dominique Gravel, and Miguel B. Araújo. 2015. <span>“Inferring Biotic Interactions from Proxies.”</span> <em>Trends in Ecology &amp; Evolution</em> 30 (6): 347–56. <a href="https://doi.org/10.1016/j.tree.2015.03.014">https://doi.org/10.1016/j.tree.2015.03.014</a>.
</div>
<div id="ref-petchey2008" class="csl-entry" role="listitem">
Petchey, Owen L., Andrew P. Beckerman, Jens O. Riede, and Philip H. Warren. 2008. <span>“Size, Foraging, and Food Web Structure.”</span> <em>Proceedings of the National Academy of Sciences</em> 105 (11): 4191–96. <a href="https://doi.org/10.1073/pnas.0710672105">https://doi.org/10.1073/pnas.0710672105</a>.
</div>
<div id="ref-pichler2023" class="csl-entry" role="listitem">
Pichler, Maximilian, and Florian Hartig. 2023. <span>“Machine Learning and Deep Learning<span></span>A Review for Ecologists.”</span> <em>Methods in Ecology and Evolution</em> 14 (4): 994–1016. <a href="https://doi.org/10.1111/2041-210X.14061">https://doi.org/10.1111/2041-210X.14061</a>.
</div>
<div id="ref-poisot2012" class="csl-entry" role="listitem">
Poisot, Timothée, Elsa Canard, Nicolas Mouquet, and Michael E. Hochberg. 2012. <span>“A Comparative Study of Ecological Specialization Estimators.”</span> <em>Methods in Ecology and Evolution</em> 3 (3): 537–44. <a href="https://doi.org/10.1111/j.2041-210x.2011.00174.x">https://doi.org/10.1111/j.2041-210x.2011.00174.x</a>.
</div>
<div id="ref-rohr2010" class="csl-entry" role="listitem">
Rohr, Rudolf&nbsp;Philippe, Heike Scherer, Patrik Kehrli, Christian Mazza, and Louis-Félix Bersier. 2010. <span>“Modeling Food Webs: Exploring Unexplained Structure Using Latent Traits.”</span> <em>The American Naturalist</em> 176 (2): 170–77. <a href="https://doi.org/10.1086/653667">https://doi.org/10.1086/653667</a>.
</div>
<div id="ref-roopnarine2006" class="csl-entry" role="listitem">
Roopnarine, Peter D. 2006. <span>“Extinction Cascades and Catastrophe in Ancient Food Webs.”</span> <em>Paleobiology</em> 32 (1): 1–19. <a href="https://www.jstor.org/stable/4096814">https://www.jstor.org/stable/4096814</a>.
</div>
<div id="ref-schneider2016" class="csl-entry" role="listitem">
Schneider, Florian D., Ulrich Brose, Björn C. Rall, and Christian Guill. 2016. <span>“Animal Diversity and Ecosystem Functioning in Dynamic Food Webs.”</span> <em>Nature Communications</em> 7 (1): 12718. <a href="https://doi.org/10.1038/ncomms12718">https://doi.org/10.1038/ncomms12718</a>.
</div>
<div id="ref-schoener1989" class="csl-entry" role="listitem">
Schoener, Thomas W. 1989. <span>“Food Webs From the Small to the Large: The Robert H. MacArthur Award Lecture.”</span> <em>Ecology</em> 70 (6): 1559–89. <a href="https://doi.org/10.2307/1938088">https://doi.org/10.2307/1938088</a>.
</div>
<div id="ref-shaw2024" class="csl-entry" role="listitem">
Shaw, Jack O., Alexander M. Dunhill, Andrew P. Beckerman, Jennifer A. Dunne, and Pincelli M. Hull. 2024. <span>“A Framework for Reconstructing Ancient Food Webs Using Functional Trait Data.”</span> bioRxiv. <a href="https://doi.org/10.1101/2024.01.30.578036">https://doi.org/10.1101/2024.01.30.578036</a>.
</div>
<div id="ref-staniczenko2013" class="csl-entry" role="listitem">
Staniczenko, Phillip P. A., Jason C. Kopp, and Stefano Allesina. 2013. <span>“The Ghost of Nestedness in Ecological Networks.”</span> <em>Nature Communications</em> 4 (1): 1391. <a href="https://doi.org/10.1038/ncomms2422">https://doi.org/10.1038/ncomms2422</a>.
</div>
<div id="ref-stouffer2007" class="csl-entry" role="listitem">
Stouffer, Daniel B, Juan Camacho, Wenxin Jiang, and Luís A Nunes Amaral. 2007. <span>“Evidence for the Existence of a Robust Pattern of Prey Selection in Food Webs.”</span> <em>Proceedings of the Royal Society B: Biological Sciences</em> 274 (1621): 1931–40. <a href="https://doi.org/10.1098/rspb.2007.0571">https://doi.org/10.1098/rspb.2007.0571</a>.
</div>
<div id="ref-strydom2022" class="csl-entry" role="listitem">
Strydom, Tanya, Salomé Bouskila, Francis Banville, Ceres Barros, Dominique Caron, Maxwell J. Farrell, Marie-Josée Fortin, et al. 2022. <span>“Food Web Reconstruction Through Phylogenetic Transfer of Low-Rank Network Representation.”</span> <em>Methods in Ecology and Evolution</em> 13 (12). <a href="https://doi.org/10.1111/2041-210X.13835">https://doi.org/10.1111/2041-210X.13835</a>.
</div>
<div id="ref-strydom2023" class="csl-entry" role="listitem">
Strydom, Tanya, Salomé Bouskila, Francis Banville, Ceres Barros, Dominique Caron, Maxwell J. Farrell, Marie-Josée Fortin, et al. 2023. <span>“Graph Embedding and Transfer Learning Can Help Predict Potential Species Interaction Networks Despite Data Limitations.”</span> <em>Methods in Ecology and Evolution</em> 14 (12): 2917–30. <a href="https://doi.org/10.1111/2041-210X.14228">https://doi.org/10.1111/2041-210X.14228</a>.
</div>
<div id="ref-strydom2021b" class="csl-entry" role="listitem">
Strydom, Tanya, Michael D. Catchen, Francis Banville, Dominique Caron, Gabriel Dansereau, Philippe Desjardins-Proulx, Norma R. Forero-Muñoz, et al. 2021a. <span>“A Roadmap Towards Predicting Species Interaction Networks (Across Space and Time).”</span> <em>Philosophical Transactions of the Royal Society B: Biological Sciences</em> 376 (1837): 20210063. <a href="https://doi.org/10.1098/rstb.2021.0063">https://doi.org/10.1098/rstb.2021.0063</a>.
</div>
<div id="ref-strydom2021" class="csl-entry" role="listitem">
———, et al. 2021b. <span>“A Roadmap Towards Predicting Species Interaction Networks (Across Space and Time).”</span> <em>Philosophical Transactions of the Royal Society B: Biological Sciences</em> 376 (1837): 20210063. <a href="https://doi.org/10.1098/rstb.2021.0063">https://doi.org/10.1098/rstb.2021.0063</a>.
</div>
<div id="ref-thuiller2024" class="csl-entry" role="listitem">
Thuiller, Wilfried, Irene Calderón-Sanou, Loïc Chalmandrier, Pierre Gaüzère, Louise M. J. O’Connor, Marc Ohlmann, Giovanni Poggiato, and Tamara Münkemüller. 2024. <span>“Navigating the Integration of Biotic Interactions in Biogeography.”</span> <em>Journal of Biogeography</em> 51 (4): 550–59. <a href="https://doi.org/10.1111/jbi.14734">https://doi.org/10.1111/jbi.14734</a>.
</div>
<div id="ref-williams2004" class="csl-entry" role="listitem">
Williams, R. J., and N. D. Martinez. 2004. <span>“Stabilization of Chaotic and Non-Permanent Food-Web Dynamics.”</span> <em>The European Physical Journal B - Condensed Matter</em> 38 (2): 297–303. <a href="https://doi.org/10.1140/epjb/e2004-00122-1">https://doi.org/10.1140/epjb/e2004-00122-1</a>.
</div>
<div id="ref-williams2000" class="csl-entry" role="listitem">
Williams, Richard J., and Neo D. Martinez. 2000. <span>“Simple Rules Yield Complex Food Webs.”</span> <em>Nature</em> 404 (6774): 180–83. <a href="https://doi.org/10.1038/35004572">https://doi.org/10.1038/35004572</a>.
</div>
<div id="ref-williams2008" class="csl-entry" role="listitem">
———. 2008. <span>“Success and its limits among structural models of complex food webs.”</span> <em>The Journal of Animal Ecology</em> 77 (3): 512–19. <a href="https://doi.org/10.1111/j.1365-2656.2008.01362.x">https://doi.org/10.1111/j.1365-2656.2008.01362.x</a>.
</div>
<div id="ref-yeakel2014" class="csl-entry" role="listitem">
Yeakel, Justin D., Mathias M. Pires, Lars Rudolf, Nathaniel J. Dominy, Paul L. Koch, Paulo R. Guimarães, and Thilo Gross. 2014. <span>“Collapse of an Ecological Network in Ancient Egypt.”</span> <em>PNAS</em> 111 (40): 14472–77. <a href="https://doi.org/10.1073/pnas.1408471111">https://doi.org/10.1073/pnas.1408471111</a>.
</div>
</div></section><section class="quarto-appendix-contents" id="quarto-citation"><h2 class="anchored quarto-appendix-heading">Citation</h2><div><div class="quarto-appendix-secondary-label">BibTeX citation:</div><pre class="sourceCode code-with-copy quarto-appendix-bibtex"><code class="sourceCode bibtex">@online{strydom2025,
  author = {Strydom, Tanya and P. Beckerman, Andrew},
  title = {Methods for {Reconstructing} {Paleo} {Food} {Webs}},
  date = {2025-09-29},
  langid = {en},
  abstract = {Food webs represent the feeding relationships between
    species and can help infer ecosystem-level processes. Alongside the
    development of food web theory, methods for constructing food webs
    have been developed to infer species interactions when empirical
    data is lacking. Food web construction methods are diverse, each
    utilising different approaches to infer species interactions —such
    as the use of traits to infer mechanistic relationships vs using gut
    content as a proxy for species diets. These methods have distinct
    theories, mechanisms, and data requirements. In paleoecology, where
    direct evidence of feeding interactions are rare, food web
    construction methods are especially valuable and affords us the
    opportunity to make inferences about paleo communities beyond simply
    a record of species composition. However, the limitations of
    paleontological data (e.g., information of species traits is limited
    to that which can be preserved) restrict which methods can reliably
    be used. By considering both ecological theory and the constraints
    of what can be derived from the fossil record, we identify the
    methods best suited for the construction of paleo food webs.
    Specifically, we focus on how these methods differ in the networks
    they produce and what these networks can reveal about species
    interactions. In doing so we hope to clarify the ecological nuances
    of network prediction and help prevent the accidental misuse or
    misinterpretation of paleo food webs.}
}
</code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre><div class="quarto-appendix-secondary-label">For attribution, please cite this work as:</div><div id="ref-strydom2025" class="csl-entry quarto-appendix-citeas" role="listitem">
Strydom, Tanya, and Andrew P. Beckerman. 2025. <span>“Methods for
Reconstructing Paleo Food Webs.”</span> TBD. September 29, 2025.
</div></div></section></div></main>
<!-- /main column -->
<script id="quarto-html-after-body" type="application/javascript">
  window.document.addEventListener("DOMContentLoaded", function (event) {
    const icon = "";
    const anchorJS = new window.AnchorJS();
    anchorJS.options = {
      placement: 'right',
      icon: icon
    };
    anchorJS.add('.anchored');
    const isCodeAnnotation = (el) => {
      for (const clz of el.classList) {
        if (clz.startsWith('code-annotation-')) {                     
          return true;
        }
      }
      return false;
    }
    const onCopySuccess = function(e) {
      // button target
      const button = e.trigger;
      // don't keep focus
      button.blur();
      // flash "checked"
      button.classList.add('code-copy-button-checked');
      var currentTitle = button.getAttribute("title");
      button.setAttribute("title", "Copied!");
      let tooltip;
      if (window.bootstrap) {
        button.setAttribute("data-bs-toggle", "tooltip");
        button.setAttribute("data-bs-placement", "left");
        button.setAttribute("data-bs-title", "Copied!");
        tooltip = new bootstrap.Tooltip(button, 
          { trigger: "manual", 
            customClass: "code-copy-button-tooltip",
            offset: [0, -8]});
        tooltip.show();    
      }
      setTimeout(function() {
        if (tooltip) {
          tooltip.hide();
          button.removeAttribute("data-bs-title");
          button.removeAttribute("data-bs-toggle");
          button.removeAttribute("data-bs-placement");
        }
        button.setAttribute("title", currentTitle);
        button.classList.remove('code-copy-button-checked');
      }, 1000);
      // clear code selection
      e.clearSelection();
    }
    const getTextToCopy = function(trigger) {
      const outerScaffold = trigger.parentElement.cloneNode(true);
      const codeEl = outerScaffold.querySelector('code');
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
    }
    const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
      text: getTextToCopy
    });
    clipboard.on('success', onCopySuccess);
    if (window.document.getElementById('quarto-embedded-source-code-modal')) {
      const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
        text: getTextToCopy,
        container: window.document.getElementById('quarto-embedded-source-code-modal')
      });
      clipboardModal.on('success', onCopySuccess);
    }
      var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
      var mailtoRegex = new RegExp(/^mailto:/);
        var filterRegex = new RegExp('/' + window.location.host + '/');
      var isInternal = (href) => {
          return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
      }
      // Inspect non-navigation links and adorn them if external
     var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
      for (var i=0; i<links.length; i++) {
        const link = links[i];
        if (!isInternal(link.href)) {
          // undo the damage that might have been done by quarto-nav.js in the case of
          // links that we want to consider external
          if (link.dataset.originalHref !== undefined) {
            link.href = link.dataset.originalHref;
          }
        }
      }
    function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
      const config = {
        allowHTML: true,
        maxWidth: 500,
        delay: 100,
        arrow: false,
        appendTo: function(el) {
            return el.parentElement;
        },
        interactive: true,
        interactiveBorder: 10,
        theme: 'quarto',
        placement: 'bottom-start',
      };
      if (contentFn) {
        config.content = contentFn;
      }
      if (onTriggerFn) {
        config.onTrigger = onTriggerFn;
      }
      if (onUntriggerFn) {
        config.onUntrigger = onUntriggerFn;
      }
      window.tippy(el, config); 
    }
    const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
    for (var i=0; i<noterefs.length; i++) {
      const ref = noterefs[i];
      tippyHover(ref, function() {
        // use id or data attribute instead here
        let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
        try { href = new URL(href).hash; } catch {}
        const id = href.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note) {
          return note.innerHTML;
        } else {
          return "";
        }
      });
    }
    const xrefs = window.document.querySelectorAll('a.quarto-xref');
    const processXRef = (id, note) => {
      // Strip column container classes
      const stripColumnClz = (el) => {
        el.classList.remove("page-full", "page-columns");
        if (el.children) {
          for (const child of el.children) {
            stripColumnClz(child);
          }
        }
      }
      stripColumnClz(note)
      if (id === null || id.startsWith('sec-')) {
        // Special case sections, only their first couple elements
        const container = document.createElement("div");
        if (note.children && note.children.length > 2) {
          container.appendChild(note.children[0].cloneNode(true));
          for (let i = 1; i < note.children.length; i++) {
            const child = note.children[i];
            if (child.tagName === "P" && child.innerText === "") {
              continue;
            } else {
              container.appendChild(child.cloneNode(true));
              break;
            }
          }
          if (window.Quarto?.typesetMath) {
            window.Quarto.typesetMath(container);
          }
          return container.innerHTML
        } else {
          if (window.Quarto?.typesetMath) {
            window.Quarto.typesetMath(note);
          }
          return note.innerHTML;
        }
      } else {
        // Remove any anchor links if they are present
        const anchorLink = note.querySelector('a.anchorjs-link');
        if (anchorLink) {
          anchorLink.remove();
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        if (note.classList.contains("callout")) {
          return note.outerHTML;
        } else {
          return note.innerHTML;
        }
      }
    }
    for (var i=0; i<xrefs.length; i++) {
      const xref = xrefs[i];
      tippyHover(xref, undefined, function(instance) {
        instance.disable();
        let url = xref.getAttribute('href');
        let hash = undefined; 
        if (url.startsWith('#')) {
          hash = url;
        } else {
          try { hash = new URL(url).hash; } catch {}
        }
        if (hash) {
          const id = hash.replace(/^#\/?/, "");
          const note = window.document.getElementById(id);
          if (note !== null) {
            try {
              const html = processXRef(id, note.cloneNode(true));
              instance.setContent(html);
            } finally {
              instance.enable();
              instance.show();
            }
          } else {
            // See if we can fetch this
            fetch(url.split('#')[0])
            .then(res => res.text())
            .then(html => {
              const parser = new DOMParser();
              const htmlDoc = parser.parseFromString(html, "text/html");
              const note = htmlDoc.getElementById(id);
              if (note !== null) {
                const html = processXRef(id, note);
                instance.setContent(html);
              } 
            }).finally(() => {
              instance.enable();
              instance.show();
            });
          }
        } else {
          // See if we can fetch a full url (with no hash to target)
          // This is a special case and we should probably do some content thinning / targeting
          fetch(url)
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.querySelector('main.content');
            if (note !== null) {
              // This should only happen for chapter cross references
              // (since there is no id in the URL)
              // remove the first header
              if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
                note.children[0].remove();
              }
              const html = processXRef(null, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      }, function(instance) {
      });
    }
        let selectedAnnoteEl;
        const selectorForAnnotation = ( cell, annotation) => {
          let cellAttr = 'data-code-cell="' + cell + '"';
          let lineAttr = 'data-code-annotation="' +  annotation + '"';
          const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
          return selector;
        }
        const selectCodeLines = (annoteEl) => {
          const doc = window.document;
          const targetCell = annoteEl.getAttribute("data-target-cell");
          const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
          const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
          const lines = annoteSpan.getAttribute("data-code-lines").split(",");
          const lineIds = lines.map((line) => {
            return targetCell + "-" + line;
          })
          let top = null;
          let height = null;
          let parent = null;
          if (lineIds.length > 0) {
              //compute the position of the single el (top and bottom and make a div)
              const el = window.document.getElementById(lineIds[0]);
              top = el.offsetTop;
              height = el.offsetHeight;
              parent = el.parentElement.parentElement;
            if (lineIds.length > 1) {
              const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
              const bottom = lastEl.offsetTop + lastEl.offsetHeight;
              height = bottom - top;
            }
            if (top !== null && height !== null && parent !== null) {
              // cook up a div (if necessary) and position it 
              let div = window.document.getElementById("code-annotation-line-highlight");
              if (div === null) {
                div = window.document.createElement("div");
                div.setAttribute("id", "code-annotation-line-highlight");
                div.style.position = 'absolute';
                parent.appendChild(div);
              }
              div.style.top = top - 2 + "px";
              div.style.height = height + 4 + "px";
              div.style.left = 0;
              let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
              if (gutterDiv === null) {
                gutterDiv = window.document.createElement("div");
                gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
                gutterDiv.style.position = 'absolute';
                const codeCell = window.document.getElementById(targetCell);
                const gutter = codeCell.querySelector('.code-annotation-gutter');
                gutter.appendChild(gutterDiv);
              }
              gutterDiv.style.top = top - 2 + "px";
              gutterDiv.style.height = height + 4 + "px";
            }
            selectedAnnoteEl = annoteEl;
          }
        };
        const unselectCodeLines = () => {
          const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
          elementsIds.forEach((elId) => {
            const div = window.document.getElementById(elId);
            if (div) {
              div.remove();
            }
          });
          selectedAnnoteEl = undefined;
        };
          // Handle positioning of the toggle
      window.addEventListener(
        "resize",
        throttle(() => {
          elRect = undefined;
          if (selectedAnnoteEl) {
            selectCodeLines(selectedAnnoteEl);
          }
        }, 10)
      );
      function throttle(fn, ms) {
      let throttle = false;
      let timer;
        return (...args) => {
          if(!throttle) { // first call gets through
              fn.apply(this, args);
              throttle = true;
          } else { // all the others get throttled
              if(timer) clearTimeout(timer); // cancel #2
              timer = setTimeout(() => {
                fn.apply(this, args);
                timer = throttle = false;
              }, ms);
          }
        };
      }
        // Attach click handler to the DT
        const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
        for (const annoteDlNode of annoteDls) {
          annoteDlNode.addEventListener('click', (event) => {
            const clickedEl = event.target;
            if (clickedEl !== selectedAnnoteEl) {
              unselectCodeLines();
              const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
              if (activeEl) {
                activeEl.classList.remove('code-annotation-active');
              }
              selectCodeLines(clickedEl);
              clickedEl.classList.add('code-annotation-active');
            } else {
              // Unselect the line
              unselectCodeLines();
              clickedEl.classList.remove('code-annotation-active');
            }
          });
        }
    const findCites = (el) => {
      const parentEl = el.parentElement;
      if (parentEl) {
        const cites = parentEl.dataset.cites;
        if (cites) {
          return {
            el,
            cites: cites.split(' ')
          };
        } else {
          return findCites(el.parentElement)
        }
      } else {
        return undefined;
      }
    };
    var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
    for (var i=0; i<bibliorefs.length; i++) {
      const ref = bibliorefs[i];
      const citeInfo = findCites(ref);
      if (citeInfo) {
        tippyHover(citeInfo.el, function() {
          var popup = window.document.createElement('div');
          citeInfo.cites.forEach(function(cite) {
            var citeDiv = window.document.createElement('div');
            citeDiv.classList.add('hanging-indent');
            citeDiv.classList.add('csl-entry');
            var biblioDiv = window.document.getElementById('ref-' + cite);
            if (biblioDiv) {
              citeDiv.innerHTML = biblioDiv.innerHTML;
            }
            popup.appendChild(citeDiv);
          });
          return popup.innerHTML;
        });
      }
    }
  });
  </script>
</div> <!-- /content -->
<script>var lightboxQuarto = GLightbox({"closeEffect":"zoom","descPosition":"bottom","loop":false,"openEffect":"zoom","selector":".lightbox"});
(function() {
  let previousOnload = window.onload;
  window.onload = () => {
    if (previousOnload) {
      previousOnload();
    }
    lightboxQuarto.on('slide_before_load', (data) => {
      const { slideIndex, slideNode, slideConfig, player, trigger } = data;
      const href = trigger.getAttribute('href');
      if (href !== null) {
        const imgEl = window.document.querySelector(`a[href="${href}"] img`);
        if (imgEl !== null) {
          const srcAttr = imgEl.getAttribute("src");
          if (srcAttr && srcAttr.startsWith("data:")) {
            slideConfig.href = srcAttr;
          }
        }
      } 
    });
  
    lightboxQuarto.on('slide_after_load', (data) => {
      const { slideIndex, slideNode, slideConfig, player, trigger } = data;
      if (window.Quarto?.typesetMath) {
        window.Quarto.typesetMath(slideNode);
      }
    });
  
  };
  
})();
          </script>




</body></html>