<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.9.18">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">

<meta name="author" content="Tanya Strydom">
<meta name="author" content="Baran Karapunar">
<meta name="author" content="Andrew P. Beckerman">
<meta name="author" content="Alexander Dunhill">
<meta name="dcterms.date" content="2026-02-03">
<meta name="keywords" content="Paleoecological networks, Food-web reconstruction, Ecological networks, Extinction dynamics, Trophic interactions, Toarcian Oceanic Anoxic Event">

<title>Reconstructing food webs in deep time: why model choice matters for ecological inference</title>
<style>
/* Default styles provided by pandoc.
** See https://pandoc.org/MANUAL.html#variables-for-html for config info.
*/
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
/* CSS for citations */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging-indent div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}</style>


<script src="site_libs/clipboard/clipboard.min.js"></script>
<script src="site_libs/quarto-html/quarto.js" type="module"></script>
<script src="site_libs/quarto-html/tabsets/tabsets.js" type="module"></script>
<script src="site_libs/quarto-html/axe/axe-check.js" type="module"></script>
<script src="site_libs/quarto-html/popper.min.js"></script>
<script src="site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="site_libs/quarto-html/anchor.min.js"></script>
<link href="site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="site_libs/quarto-html/quarto-syntax-highlighting-e3fecfe3565d481478fb93785bd05dc0.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="site_libs/bootstrap/bootstrap.min.js"></script>
<link href="site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="site_libs/bootstrap/bootstrap-0a83c53033ed047cde592d264eaa4157.min.css" rel="stylesheet" append-hash="true" id="quarto-bootstrap" data-mode="light">
<script src="site_libs/quarto-contrib/glightbox/glightbox.min.js"></script>
<link href="site_libs/quarto-contrib/glightbox/glightbox.min.css" rel="stylesheet">
<link href="site_libs/quarto-contrib/glightbox/lightbox.css" rel="stylesheet">
<script async="" src="https://hypothes.is/embed.js"></script>
<script>
  window.document.addEventListener("DOMContentLoaded", function (_event) {
    document.body.classList.add('hypothesis-enabled');
  });
</script>

  <script src="https://cdnjs.cloudflare.com/polyfill/v3/polyfill.min.js?features=es6"></script>
  <script defer="" src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-chtml-full.js" type="text/javascript"></script>

<script type="text/javascript">
const typesetMath = (el) => {
  if (window.MathJax) {
    // MathJax Typeset
    window.MathJax.typeset([el]);
  } else if (window.katex) {
    // KaTeX Render
    var mathElements = el.getElementsByClassName("math");
    var macros = [];
    for (var i = 0; i < mathElements.length; i++) {
      var texText = mathElements[i].firstChild;
      if (mathElements[i].tagName == "SPAN" && texText && texText.data) {
        window.katex.render(texText.data, mathElements[i], {
          displayMode: mathElements[i].classList.contains('display'),
          throwOnError: false,
          macros: macros,
          fleqn: false
        });
      }
    }
  }
}
window.Quarto = {
  typesetMath
};
</script>

<meta name="citation_title" content="Reconstructing food webs in deep time: why model choice matters for ecological inference">
<meta name="citation_abstract" content="Food webs provide a powerful framework for understanding ecosystem structure and function, yet reconstructing them in paleoecological contexts remains challenging because direct evidence of species interactions is rarely preserved. A growing array of models exists for predicting interactions and inferring network structure, but these approaches differ markedly in their assumptions, mechanisms, and data requirements. Here, we evaluate how network reconstruction model choice shapes ecological inference in deep time and which approaches are most appropriate given the constraints of the fossil record. Using the Toarcian Oceanic Anoxic Event (Early Jurassic, ~183 Ma) as a case study, we reconstruct food webs for four successive community states using six contrasting modelling approaches spanning mechanistic, trait-based, and structural network representations. Despite identical taxon pools, models produced strikingly different network structures and species-level interactions, and these differences propagated into divergent inferences about extinction dynamics, interaction loss, and cascading effects. By framing food-web reconstructions as alternative ecological hypotheses (rather than interchangeable representations) this study bridges paleoecology and modern network theory, and demonstrates how model choice fundamentally conditions inference about extinction dynamics in deep time.
">
<meta name="citation_keywords" content="Paleoecological networks,Food-web reconstruction,Ecological networks,Extinction dynamics,Trophic interactions,Toarcian Oceanic Anoxic Event">
<meta name="citation_author" content="Tanya Strydom">
<meta name="citation_author" content="Baran Karapunar">
<meta name="citation_author" content="Andrew P. Beckerman">
<meta name="citation_author" content="Alexander Dunhill">
<meta name="citation_publication_date" content="2026-02-03">
<meta name="citation_cover_date" content="2026-02-03">
<meta name="citation_year" content="2026">
<meta name="citation_online_date" content="2026-02-03">
<meta name="citation_language" content="en">
<meta name="citation_journal_title" content="TBD">
<meta name="citation_reference" content="citation_title=Extinction cascades, community collapse, and recovery across a mesozoic hyperthermal event;,citation_author=Alexander M. Dunhill;,citation_author=Karolina Zarzyczny;,citation_author=Jack O. Shaw;,citation_author=Jed W. Atkinson;,citation_author=Crispin T. S. Little;,citation_author=Andrew P. Beckerman;,citation_publication_date=2024-10-04;,citation_cover_date=2024-10-04;,citation_year=2024;,citation_fulltext_html_url=https://www.nature.com/articles/s41467-024-53000-2;,citation_issue=1;,citation_doi=10.1038/s41467-024-53000-2;,citation_volume=15;,citation_language=en;,citation_journal_title=Nature Communications;">
<meta name="citation_reference" content="citation_title=Shortcomings of reusing species interaction networks created by different sets of researchers;,citation_author=Chris Brimacombe;,citation_author=Korryn Bodner;,citation_author=Matthew Michalska-Smith;,citation_author=Timothée Poisot;,citation_author=Marie-Josée Fortin;,citation_publication_date=2023-03;,citation_cover_date=2023-03;,citation_year=2023;,citation_fulltext_html_url=https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.3002068;,citation_issue=4;,citation_doi=10.1371/journal.pbio.3002068;,citation_volume=21;,citation_language=en;,citation_journal_title=PLOS Biology;">
<meta name="citation_reference" content="citation_title=A framework for reconstructing ancient food webs using functional trait data;,citation_abstract=Food webs provide quantitative insights into the structure and dynamics of ecological communities. Previous work has shown their utility in understanding community responses to modern and ancient perturbations, including anthropogenic change and mass extinctions. However, few ancient food webs have been reconstructed due to difficulties assessing trophic interactions amongst extinct species derived from an incomplete fossil record. We present and assess the Paleo Food web Inference Model (PFIM). PFIM uses functional trait data–predictive of interactions in modern ecosystems and commonly available for fossil organisms–to reconstruct ancient food webs. We test the model by (i) applying it to four modern ecosystems with empirical constrained food webs to directly compare PFIM-constructed networks to their empirical counterparts, (ii) by carefully comparing discrepancies between PFIM-inferred and empirical webs in one of those systems, and (iii) by comparing networks describing feasible trophic interactions (&amp;amp;amp;quot;feasible webs&amp;quot;) with networks to which we superimpose characteristic interaction distributions derived from modern theory (&quot;realized webs&quot;). As a proof of concept, we then apply the method to faunal data from two Cambrian fossil deposits to reconstruct ancient trophic systems. PFIM-inferred feasible food webs successfully predict ~70% of trophic interactions across four modern systems. Furthermore, inferred food webs with enforced interaction distributions (i.e., realized webs) accurately predict ~90% of interactions. Comparisons with a global database of trophic interactions and other food web models, suggest that under sampling of empirical webs accounts for up to 21% of the remaining differences between PFIM and empirical food webs. Food webs can be reasonably approximated by inferring trophic interactions based upon life habit traits. This study provides the foundation to use trait-based inference models across the fossil record to examine ancient food webs and community evolution.;,citation_author=Jack O. Shaw;,citation_author=Alexander M. Dunhill;,citation_author=Andrew P. Beckerman;,citation_author=Jennifer A. Dunne;,citation_author=Pincelli M. Hull;,citation_publication_date=2024-01;,citation_cover_date=2024-01;,citation_year=2024;,citation_doi=10.1101/2024.01.30.578036;,citation_language=en-US;,citation_publisher=bioRxiv;">
<meta name="citation_reference" content="citation_title=Extinction cascades and catastrophe in ancient food webs;,citation_author=Peter D. Roopnarine;,citation_publication_date=2006;,citation_cover_date=2006;,citation_year=2006;,citation_fulltext_html_url=https://www.jstor.org/stable/4096814;,citation_issue=1;,citation_volume=32;,citation_journal_title=Paleobiology;">
<meta name="citation_reference" content="citation_title=Size, foraging, and food web structure;,citation_author=Owen L. Petchey;,citation_author=Andrew P. Beckerman;,citation_author=Jens O. Riede;,citation_author=Philip H. Warren;,citation_publication_date=2008-03-18;,citation_cover_date=2008-03-18;,citation_year=2008;,citation_fulltext_html_url=https://pnas.org/doi/full/10.1073/pnas.0710672105;,citation_issue=11;,citation_doi=10.1073/pnas.0710672105;,citation_volume=105;,citation_language=en;,citation_journal_title=Proceedings of the National Academy of Sciences;">
<meta name="citation_reference" content="citation_title=Modeling food webs: Exploring unexplained structure using latent traits.;,citation_author=Rudolf&nbsp;Philippe Rohr;,citation_author=Heike Scherer;,citation_author=Patrik Kehrli;,citation_author=Christian Mazza;,citation_author=Louis-Félix Bersier;,citation_publication_date=2010-08;,citation_cover_date=2010-08;,citation_year=2010;,citation_fulltext_html_url=https://www.journals.uchicago.edu/doi/10.1086/653667;,citation_issue=2;,citation_doi=10.1086/653667;,citation_volume=176;,citation_journal_title=The American Naturalist;">
<meta name="citation_reference" content="citation_title=Collapse of an ecological network in ancient egypt;,citation_author=Justin D. Yeakel;,citation_author=Mathias M. Pires;,citation_author=Lars Rudolf;,citation_author=Nathaniel J. Dominy;,citation_author=Paul L. Koch;,citation_author=Paulo R. Guimarães;,citation_author=Thilo Gross;,citation_publication_date=2014-10;,citation_cover_date=2014-10;,citation_year=2014;,citation_issue=40;,citation_doi=10.1073/pnas.1408471111;,citation_volume=111;,citation_journal_title=PNAS;">
<meta name="citation_reference" content="citation_title=Simple rules yield complex food webs;,citation_author=Richard J. Williams;,citation_author=Neo D. Martinez;,citation_publication_date=2000-03;,citation_cover_date=2000-03;,citation_year=2000;,citation_fulltext_html_url=https://www.nature.com/articles/35004572;,citation_issue=6774;,citation_doi=10.1038/35004572;,citation_volume=404;,citation_language=en;,citation_journal_title=Nature;">
<meta name="citation_reference" content="citation_title=Success and its limits among structural models of complex food webs;,citation_author=Richard J. Williams;,citation_author=Neo D. Martinez;,citation_publication_date=2008-05;,citation_cover_date=2008-05;,citation_year=2008;,citation_issue=3;,citation_doi=10.1111/j.1365-2656.2008.01362.x;,citation_volume=77;,citation_language=eng;,citation_journal_title=The Journal of Animal Ecology;">
<meta name="citation_reference" content="citation_title=Graph embedding and transfer learning can help predict potential species interaction networks despite data limitations;,citation_author=Tanya Strydom;,citation_author=Salomé Bouskila;,citation_author=Francis Banville;,citation_author=Ceres Barros;,citation_author=Dominique Caron;,citation_author=Maxwell J. Farrell;,citation_author=Marie-Josée Fortin;,citation_author=Benjamin Mercier;,citation_author=Laura J. Pollock;,citation_author=Rogini Runghen;,citation_author=Giulio V. Dalla Riva;,citation_author=Timothée Poisot;,citation_publication_date=2023;,citation_cover_date=2023;,citation_year=2023;,citation_fulltext_html_url=https://onlinelibrary.wiley.com/doi/abs/10.1111/2041-210X.14228;,citation_issue=12;,citation_doi=10.1111/2041-210X.14228;,citation_volume=14;,citation_language=en;,citation_journal_title=Methods in Ecology and Evolution;">
<meta name="citation_reference" content="citation_title=Addressing the eltonian shortfall with trait-based interaction models;,citation_author=Dominique Caron;,citation_author=Luigi Maiorano;,citation_author=Wilfried Thuiller;,citation_author=Laura J. Pollock;,citation_publication_date=2022;,citation_cover_date=2022;,citation_year=2022;,citation_fulltext_html_url=https://onlinelibrary.wiley.com/doi/abs/10.1111/ele.13966;,citation_issue=4;,citation_doi=10.1111/ele.13966;,citation_volume=25;,citation_language=en;,citation_journal_title=Ecology Letters;">
<meta name="citation_reference" content="citation_title=Collapse of terrestrial mammal food webs since the late pleistocene;,citation_author=Evan C. Fricke;,citation_author=Chia Hsieh;,citation_author=Owen Middleton;,citation_author=Daniel Gorczynski;,citation_author=Caroline D. Cappello;,citation_author=Oscar Sanisidro;,citation_author=John Rowan;,citation_author=Jens-Christian Svenning;,citation_author=Lydia Beaudrot;,citation_publication_date=2022-08-26;,citation_cover_date=2022-08-26;,citation_year=2022;,citation_fulltext_html_url=http://dx.doi.org/10.1126/science.abn4012;,citation_issue=6609;,citation_doi=10.1126/science.abn4012;,citation_volume=377;,citation_language=en;,citation_journal_title=Science;">
<meta name="citation_reference" content="citation_title=Ecological modelling of paleocommunity food webs;,citation_author=Peter D Roopnarine;,citation_publication_date=2017;,citation_cover_date=2017;,citation_year=2017;,citation_language=en;,citation_inbook_title=undefined;">
<meta name="citation_reference" content="citation_title=Autecology and the filling of ecospace: Key metazoan radiations;,citation_author=Richard K. Bambach;,citation_author=Andrew M. Bush;,citation_author=Douglas H. Erwin;,citation_publication_date=2007;,citation_cover_date=2007;,citation_year=2007;,citation_fulltext_html_url=https://onlinelibrary.wiley.com/doi/abs/10.1111/j.1475-4983.2006.00611.x;,citation_issue=1;,citation_doi=10.1111/j.1475-4983.2006.00611.x;,citation_volume=50;,citation_language=en;,citation_journal_title=Palaeontology;">
<meta name="citation_reference" content="citation_title=Inferring predator-prey interactions in food webs;,citation_author=Justin PF Pomeranz;,citation_author=Ross M. Thompson;,citation_author=Timothée Poisot;,citation_author=Jon S. Harding;,citation_publication_date=2018;,citation_cover_date=2018;,citation_year=2018;,citation_fulltext_html_url=https://besjournals.onlinelibrary.wiley.com/doi/abs/10.1111/2041-210X.13125;,citation_issue=ja;,citation_doi=10.1111/2041-210X.13125;,citation_volume=0;,citation_language=en;,citation_journal_title=Methods in Ecology and Evolution;">
<meta name="citation_reference" content="citation_title=SVD entropy reveals the high complexity of ecological networks;,citation_author=Tanya Strydom;,citation_author=Giulio V. Dalla Riva;,citation_author=Timothée Poisot;,citation_publication_date=2021;,citation_cover_date=2021;,citation_year=2021;,citation_fulltext_html_url=https://www.frontiersin.org/articles/10.3389/fevo.2021.623141/full;,citation_doi=10.3389/fevo.2021.623141;,citation_volume=9;,citation_language=en-US;,citation_journal_title=Frontiers in Ecology and Evolution;">
<meta name="citation_reference" content="citation_title=Network motifs: Simple building blocks of complex networks;,citation_author=R. Milo;,citation_author=S. Shen-Orr;,citation_author=S. Itzkovitz;,citation_author=N. Kashtan;,citation_author=D. Chklovskii;,citation_author=U. Alon;,citation_publication_date=2002-10-25;,citation_cover_date=2002-10-25;,citation_year=2002;,citation_fulltext_html_url=https://www.science.org/doi/10.1126/science.298.5594.824;,citation_issue=5594;,citation_doi=10.1126/science.298.5594.824;,citation_volume=298;,citation_journal_title=Science;">
<meta name="citation_reference" content="citation_title=Evidence for the existence of a robust pattern of prey selection in food webs;,citation_author=Daniel B Stouffer;,citation_author=Juan Camacho;,citation_author=Wenxin Jiang;,citation_author=Luís A Nunes Amaral;,citation_publication_date=2007-06;,citation_cover_date=2007-06;,citation_year=2007;,citation_issue=1621;,citation_doi=10.1098/rspb.2007.0571;,citation_volume=274;,citation_journal_title=Proceedings of the Royal Society B: Biological Sciences;">
<meta name="citation_reference" content="citation_title=Success and its limits among structural models of complex food webs;,citation_author=Richard J. Williams;,citation_author=Neo D. Martinez;,citation_publication_date=2008-05;,citation_cover_date=2008-05;,citation_year=2008;,citation_issue=3;,citation_doi=10.1111/j.1365-2656.2008.01362.x;,citation_volume=77;,citation_language=eng;,citation_journal_title=The Journal of Animal Ecology;">
<meta name="citation_reference" content="citation_title=Size, foraging, and food web structure;,citation_author=Owen L. Petchey;,citation_author=Andrew P. Beckerman;,citation_author=Jens O. Riede;,citation_author=Philip H. Warren;,citation_publication_date=2008-03-18;,citation_cover_date=2008-03-18;,citation_year=2008;,citation_fulltext_html_url=https://pnas.org/doi/full/10.1073/pnas.0710672105;,citation_issue=11;,citation_doi=10.1073/pnas.0710672105;,citation_volume=105;,citation_language=en;,citation_journal_title=Proceedings of the National Academy of Sciences;">
<meta name="citation_reference" content="citation_title=Constant connectance in community food webs;,citation_author=Neo D. Martinez;,citation_publication_date=1992;,citation_cover_date=1992;,citation_year=1992;,citation_fulltext_html_url=http://www.jstor.org/stable/2462337;,citation_issue=6;,citation_volume=139;,citation_journal_title=The American Naturalist;">
<meta name="citation_reference" content="citation_title=Analysing ecological networks of species interactions;,citation_author=Eva Delmas;,citation_author=Mathilde Besson;,citation_author=Marie-Hélène Brice;,citation_author=Laura A. Burkle;,citation_author=Giulio V. Dalla Riva;,citation_author=Marie-Josée Fortin;,citation_author=Dominique Gravel;,citation_author=Paulo R. Guimarães;,citation_author=David H. Hembry;,citation_author=Erica A. Newman;,citation_author=Jens M. Olesen;,citation_author=Mathias M. Pires;,citation_author=Justin D. Yeakel;,citation_author=Timothée Poisot;,citation_publication_date=2018-06-20;,citation_cover_date=2018-06-20;,citation_year=2018;,citation_fulltext_html_url=http://doi.wiley.com/10.1111/brv.12433;,citation_doi=10.1111/brv.12433;,citation_language=en;,citation_journal_title=Biological Reviews;">
<meta name="citation_reference" content="citation_title=A general model for food web structure;,citation_author=Stefano Allesina;,citation_author=David Alonso;,citation_author=Mercedes Pascual;,citation_publication_date=2008-05-02;,citation_cover_date=2008-05-02;,citation_year=2008;,citation_fulltext_html_url=https://www.science.org/doi/full/10.1126/science.1156269;,citation_issue=5876;,citation_doi=10.1126/science.1156269;,citation_volume=320;,citation_journal_title=Science;">
<meta name="citation_reference" content="citation_title=A roadmap towards predicting species interaction networks (across space and time);,citation_author=Tanya Strydom;,citation_author=Michael D. Catchen;,citation_author=Francis Banville;,citation_author=Dominique Caron;,citation_author=Gabriel Dansereau;,citation_author=Philippe Desjardins-Proulx;,citation_author=Norma R. Forero-Muñoz;,citation_author=Gracielle Higino;,citation_author=Benjamin Mercier;,citation_author=Andrew Gonzalez;,citation_author=Dominique Gravel;,citation_author=Laura Pollock;,citation_author=Timothée Poisot;,citation_publication_date=2021-11;,citation_cover_date=2021-11;,citation_year=2021;,citation_issue=1837;,citation_doi=10.1098/rstb.2021.0063;,citation_volume=376;,citation_journal_title=Philosophical Transactions of the Royal Society B: Biological Sciences;,citation_publisher=Royal Society;">
<meta name="citation_reference" content="citation_title=A comparative study of ecological specialization estimators;,citation_author=Timothée Poisot;,citation_author=Elsa Canard;,citation_author=Nicolas Mouquet;,citation_author=Michael E. Hochberg;,citation_publication_date=2012-01-19;,citation_cover_date=2012-01-19;,citation_year=2012;,citation_fulltext_html_url=http://dx.doi.org/10.1111/j.2041-210X.2011.00174.x;,citation_issue=3;,citation_doi=10.1111/j.2041-210x.2011.00174.x;,citation_volume=3;,citation_language=en;,citation_journal_title=Methods in Ecology and Evolution;">
<meta name="citation_reference" content="citation_title=Food webs from the small to the large: The robert h. MacArthur award lecture;,citation_author=Thomas W. Schoener;,citation_publication_date=1989-12;,citation_cover_date=1989-12;,citation_year=1989;,citation_fulltext_html_url=http://dx.doi.org/10.2307/1938088;,citation_issue=6;,citation_doi=10.2307/1938088;,citation_volume=70;,citation_language=en;,citation_journal_title=Ecology;">
<meta name="citation_reference" content="citation_title=ATNr: Allometric trophic network models in r;,citation_author=Benoit Gauzens;,citation_author=Ulrich Brose;,citation_author=Eva Delmas;,citation_author=Emilio Berti;,citation_publication_date=2023;,citation_cover_date=2023;,citation_year=2023;,citation_fulltext_html_url=https://onlinelibrary.wiley.com/doi/abs/10.1111/2041-210X.14212;,citation_issue=11;,citation_doi=10.1111/2041-210X.14212;,citation_volume=14;,citation_language=en;,citation_journal_title=Methods in Ecology and Evolution;">
<meta name="citation_reference" content="citation_title=Animal diversity and ecosystem functioning in dynamic food webs;,citation_author=Florian D. Schneider;,citation_author=Ulrich Brose;,citation_author=Björn C. Rall;,citation_author=Christian Guill;,citation_publication_date=2016-10-05;,citation_cover_date=2016-10-05;,citation_year=2016;,citation_fulltext_html_url=https://www.nature.com/articles/ncomms12718;,citation_issue=1;,citation_doi=10.1038/ncomms12718;,citation_volume=7;,citation_language=en;,citation_journal_title=Nature Communications;">
<meta name="citation_reference" content="citation_title=The reliability of R50 as a measure of vulnerability of food webs to sequential species deletions;,citation_author=Tomas Jonsson;,citation_author=Sofia Berg;,citation_author=Alexander Pimenov;,citation_author=Catherine Palmer;,citation_author=Mark Emmerson;,citation_publication_date=2015;,citation_cover_date=2015;,citation_year=2015;,citation_fulltext_html_url=https://onlinelibrary.wiley.com/doi/abs/10.1111/oik.01588;,citation_issue=4;,citation_doi=10.1111/oik.01588;,citation_volume=124;,citation_language=en;,citation_journal_title=Oikos;">
<meta name="citation_reference" content="citation_title=On random graphs. i.;,citation_author=P. Erdős;,citation_author=A. Rényi;,citation_publication_date=1959-07-01;,citation_cover_date=1959-07-01;,citation_year=1959;,citation_fulltext_html_url=http://dx.doi.org/10.5486/PMD.1959.6.3-4.12;,citation_issue=3-4;,citation_doi=10.5486/pmd.1959.6.3-4.12;,citation_volume=6;,citation_journal_title=Publicationes Mathematicae Debrecen;">
<meta name="citation_reference" content="citation_title=The ghost of nestedness in ecological networks;,citation_author=Phillip P. A. Staniczenko;,citation_author=Jason C. Kopp;,citation_author=Stefano Allesina;,citation_publication_date=2013-06;,citation_cover_date=2013-06;,citation_year=2013;,citation_fulltext_html_url=http://www.nature.com/articles/ncomms2422;,citation_issue=1;,citation_doi=10.1038/ncomms2422;,citation_volume=4;,citation_language=en;,citation_journal_title=Nature Communications;">
<meta name="citation_reference" content="citation_title=Stabilization of chaotic and non-permanent food-web dynamics;,citation_author=R. J. Williams;,citation_author=N. D. Martinez;,citation_publication_date=2004-03-01;,citation_cover_date=2004-03-01;,citation_year=2004;,citation_fulltext_html_url=http://link.springer.com/10.1140/epjb/e2004-00122-1;,citation_issue=2;,citation_doi=10.1140/epjb/e2004-00122-1;,citation_volume=38;,citation_language=en;,citation_journal_title=The European Physical Journal B - Condensed Matter;">
<meta name="citation_reference" content="citation_title=Simultaneously estimating food web connectance and structure with uncertainty;,citation_author=Anubhav Gupta;,citation_author=Reinhard Furrer;,citation_author=Owen L. Petchey;,citation_publication_date=2022;,citation_cover_date=2022;,citation_year=2022;,citation_fulltext_html_url=https://onlinelibrary.wiley.com/doi/abs/10.1002/ece3.8643;,citation_issue=3;,citation_doi=10.1002/ece3.8643;,citation_volume=12;,citation_language=en;,citation_journal_title=Ecology and Evolution;">
<meta name="citation_reference" content="citation_title=Highly resolved early eocene food webs show development of modern trophic structure after the end-cretaceous extinction;,citation_author=Jennifer A. Dunne;,citation_author=Conrad C. Labandeira;,citation_author=Richard J. Williams;,citation_publication_date=2014-05-07;,citation_cover_date=2014-05-07;,citation_year=2014;,citation_fulltext_html_url=https://royalsocietypublishing.org/doi/10.1098/rspb.2013.3280;,citation_issue=1782;,citation_doi=10.1098/rspb.2013.3280;,citation_volume=281;,citation_journal_title=Proceedings of the Royal Society B: Biological Sciences;">
<meta name="citation_reference" content="citation_title=Extinction cascades and catastrophe in ancient food webs;,citation_author=Peter D. Roopnarine;,citation_publication_date=2006;,citation_cover_date=2006;,citation_year=2006;,citation_fulltext_html_url=https://www.jstor.org/stable/4096814;,citation_issue=1;,citation_volume=32;,citation_journal_title=Paleobiology;">
<meta name="citation_reference" content="citation_title=Tailoring interaction network types to answer different ecological questions;,citation_author=Benoit Gauzens;,citation_author=Lise Thouvenot;,citation_author=Diane S. Srivastava;,citation_author=Pavel Kratina;,citation_author=Gustavo Q. Romero;,citation_author=Emilio Berti;,citation_author=Eoin J. O’Gorman;,citation_author=Angélica L. González;,citation_author=Olivier Dézerald;,citation_author=Nico Eisenhauer;,citation_author=Mathias Pires;,citation_author=Remo Ryser;,citation_author=Vinicius F. Farjalla;,citation_author=Pierre Rogy;,citation_author=Ulrich Brose;,citation_author=Jana S. Petermann;,citation_author=Benoît Geslin;,citation_author=Jes Hines;,citation_publication_date=2025-06-13;,citation_cover_date=2025-06-13;,citation_year=2025;,citation_fulltext_html_url=https://www.nature.com/articles/s44358-025-00056-7;,citation_doi=10.1038/s44358-025-00056-7;,citation_language=en;,citation_journal_title=Nature Reviews Biodiversity;">
<meta name="citation_reference" content="citation_title=Addressing priority questions of conservation science with palaeontological data;,citation_author=Wolfgang Kiessling;,citation_author=Nussaïbah B. Raja;,citation_author=Vanessa Julie Roden;,citation_author=Samuel T. Turvey;,citation_author=Erin E. Saupe;,citation_publication_date=2019-11-04;,citation_cover_date=2019-11-04;,citation_year=2019;,citation_fulltext_html_url=https://royalsocietypublishing.org/doi/10.1098/rstb.2019.0222;,citation_issue=1788;,citation_doi=10.1098/rstb.2019.0222;,citation_volume=374;,citation_journal_title=Philosophical Transactions of the Royal Society B: Biological Sciences;">
<meta name="citation_reference" content="citation_title=What is conservation paleobiology? Tracking 20 years of research and development;,citation_author=Erin M. Dillon;,citation_author=Jaleigh Q. Pier;,citation_author=Jansen A. Smith;,citation_author=Nussaïbah B. Raja;,citation_author=Danijela Dimitrijević;,citation_author=Elizabeth L. Austin;,citation_author=Jonathan D. Cybulski;,citation_author=Julia De Entrambasaguas;,citation_author=Stephen R. Durham;,citation_author=Carolin M. Grether;,citation_author=Himadri Sekhar Haldar;,citation_author=Kristína Kocáková;,citation_author=Chien-Hsiang Lin;,citation_author=Ilaria Mazzini;,citation_author=Alexis M. Mychajliw;,citation_author=Amy L. Ollendorf;,citation_author=Catalina Pimiento;,citation_author=Omar R. Regalado Fernández;,citation_author=Isaiah E. Smith;,citation_author=Gregory P. Dietl;,citation_publication_date=2022-12-07;,citation_cover_date=2022-12-07;,citation_year=2022;,citation_fulltext_html_url=https://www.frontiersin.org/journals/ecology-and-evolution/articles/10.3389/fevo.2022.1031483/full;,citation_doi=10.3389/fevo.2022.1031483;,citation_volume=10;,citation_language=en-US;,citation_journal_title=Frontiers in Ecology and Evolution;">
<meta name="citation_reference" content="citation_title=Global projection of terrestrial vertebrate food webs under future climate and land-use changes;,citation_author=Xiyang Hao;,citation_author=Marcel Holyoak;,citation_author=Zhicheng Zhang;,citation_author=Chuan Yan;,citation_publication_date=2025;,citation_cover_date=2025;,citation_year=2025;,citation_fulltext_html_url=https://onlinelibrary.wiley.com/doi/abs/10.1111/gcb.70061;,citation_issue=2;,citation_doi=10.1111/gcb.70061;,citation_volume=31;,citation_language=en;,citation_journal_title=Global Change Biology;">
<meta name="citation_reference" content="citation_title=Chasing ecological interactions;,citation_author=Pedro Jordano;,citation_publication_date=2016-09-15;,citation_cover_date=2016-09-15;,citation_year=2016;,citation_fulltext_html_url=https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002559;,citation_issue=9;,citation_doi=10.1371/journal.pbio.1002559;,citation_volume=14;,citation_language=en;,citation_journal_title=PLOS Biology;">
<meta name="citation_reference" content="citation_title=Inferring biotic interactions from proxies;,citation_author=Ignacio Morales-Castilla;,citation_author=Miguel G. Matias;,citation_author=Dominique Gravel;,citation_author=Miguel B. Araújo;,citation_publication_date=2015-06;,citation_cover_date=2015-06;,citation_year=2015;,citation_fulltext_html_url=https://linkinghub.elsevier.com/retrieve/pii/S0169534715000774;,citation_issue=6;,citation_doi=10.1016/j.tree.2015.03.014;,citation_volume=30;,citation_language=en;,citation_journal_title=Trends in Ecology &amp;amp;amp; Evolution;">
<meta name="citation_reference" content="citation_title=Machine learning and deep learninga review for ecologists;,citation_author=Maximilian Pichler;,citation_author=Florian Hartig;,citation_publication_date=2023;,citation_cover_date=2023;,citation_year=2023;,citation_fulltext_html_url=https://onlinelibrary.wiley.com/doi/abs/10.1111/2041-210X.14061;,citation_issue=4;,citation_doi=10.1111/2041-210X.14061;,citation_volume=14;,citation_language=en;,citation_journal_title=Methods in Ecology and Evolution;">
<meta name="citation_reference" content="citation_title=A roadmap towards predicting species interaction networks (across space and time);,citation_author=Tanya Strydom;,citation_author=Michael D. Catchen;,citation_author=Francis Banville;,citation_author=Dominique Caron;,citation_author=Gabriel Dansereau;,citation_author=Philippe Desjardins-Proulx;,citation_author=Norma R. Forero-Muñoz;,citation_author=Gracielle Higino;,citation_author=Benjamin Mercier;,citation_author=Andrew Gonzalez;,citation_author=Dominique Gravel;,citation_author=Laura Pollock;,citation_author=Timothée Poisot;,citation_publication_date=2021-11-08;,citation_cover_date=2021-11-08;,citation_year=2021;,citation_fulltext_html_url=https://royalsocietypublishing.org/doi/abs/10.1098/rstb.2021.0063;,citation_issue=1837;,citation_doi=10.1098/rstb.2021.0063;,citation_volume=376;,citation_journal_title=Philosophical Transactions of the Royal Society B: Biological Sciences;">
<meta name="citation_reference" content="citation_title=A general model for food web structure;,citation_author=Stefano Allesina;,citation_author=David Alonso;,citation_author=Mercedes Pascual;,citation_publication_date=2008-05-02;,citation_cover_date=2008-05-02;,citation_year=2008;,citation_fulltext_html_url=https://www.science.org/doi/full/10.1126/science.1156269;,citation_issue=5876;,citation_doi=10.1126/science.1156269;,citation_volume=320;,citation_journal_title=Science;">
<meta name="citation_reference" content="citation_title=Collapse of an ecological network in ancient egypt;,citation_author=Justin D. Yeakel;,citation_author=Mathias M. Pires;,citation_author=Lars Rudolf;,citation_author=Nathaniel J. Dominy;,citation_author=Paul L. Koch;,citation_author=Paulo R. Guimarães;,citation_author=Thilo Gross;,citation_publication_date=2014-10;,citation_cover_date=2014-10;,citation_year=2014;,citation_issue=40;,citation_doi=10.1073/pnas.1408471111;,citation_volume=111;,citation_journal_title=PNAS;">
<meta name="citation_reference" content="citation_title=Food web reconstruction through phylogenetic transfer of low-rank network representation;,citation_author=Tanya Strydom;,citation_author=Salomé Bouskila;,citation_author=Francis Banville;,citation_author=Ceres Barros;,citation_author=Dominique Caron;,citation_author=Maxwell J. Farrell;,citation_author=Marie-Josée Fortin;,citation_author=Victoria Hemming;,citation_author=Benjamin Mercier;,citation_author=Laura J. Pollock;,citation_author=Rogini Runghen;,citation_author=Giulio V. Dalla Riva;,citation_author=Timothée Poisot;,citation_publication_date=2022-12;,citation_cover_date=2022-12;,citation_year=2022;,citation_fulltext_html_url=https://onlinelibrary.wiley.com/doi/abs/10.1111/2041-210X.13835;,citation_issue=12;,citation_doi=10.1111/2041-210X.13835;,citation_volume=13;,citation_language=en;,citation_journal_title=Methods in Ecology and Evolution;">
<meta name="citation_reference" content="citation_title=Navigating the integration of biotic interactions in biogeography;,citation_author=Wilfried Thuiller;,citation_author=Irene Calderon-Sanou;,citation_author=Loïc Chalmandrier;,citation_author=Pierre Gaüzere;,citation_author=Marc Ohlmann;,citation_author=Giovanni Poggiato;,citation_author=Tamara Münkemüller;,citation_language=en;">
<meta name="citation_reference" content="citation_title=Robustness to secondary extinctions: Comparing trait-based sequential deletions in static and dynamic food webs;,citation_author=Alva Curtsdotter;,citation_author=Amrei Binzer;,citation_author=Ulrich Brose;,citation_author=Francisco De Castro;,citation_author=Bo Ebenman;,citation_author=Anna Eklöf;,citation_author=Jens O. Riede;,citation_author=Aaron Thierry;,citation_author=Björn C. Rall;,citation_publication_date=2011-11;,citation_cover_date=2011-11;,citation_year=2011;,citation_fulltext_html_url=https://linkinghub.elsevier.com/retrieve/pii/S1439179111001216;,citation_issue=7;,citation_doi=10.1016/j.baae.2011.09.008;,citation_volume=12;,citation_language=en;,citation_journal_title=Basic and Applied Ecology;">
<meta name="citation_reference" content="citation_title=Tailoring interaction network types to answer different ecological questions;,citation_author=Benoit Gauzens;,citation_author=Lise Thouvenot;,citation_author=Diane S. Srivastava;,citation_author=Pavel Kratina;,citation_author=Gustavo Q. Romero;,citation_author=Emilio Berti;,citation_author=Eoin J. O’Gorman;,citation_author=Angélica L. González;,citation_author=Olivier Dézerald;,citation_author=Nico Eisenhauer;,citation_author=Mathias Pires;,citation_author=Remo Ryser;,citation_author=Vinicius F. Farjalla;,citation_author=Pierre Rogy;,citation_author=Ulrich Brose;,citation_author=Jana S. Petermann;,citation_author=Benoît Geslin;,citation_author=Jes Hines;,citation_publication_date=2025-06-13;,citation_cover_date=2025-06-13;,citation_year=2025;,citation_fulltext_html_url=https://www.nature.com/articles/s44358-025-00056-7;,citation_doi=10.1038/s44358-025-00056-7;,citation_language=en;,citation_journal_title=Nature Reviews Biodiversity;">
<meta name="citation_reference" content="citation_title=Navigating the integration of biotic interactions in biogeography;,citation_author=Wilfried Thuiller;,citation_author=Irene Calderón-Sanou;,citation_author=Loïc Chalmandrier;,citation_author=Pierre Gaüzère;,citation_author=Louise M. J. O’Connor;,citation_author=Marc Ohlmann;,citation_author=Giovanni Poggiato;,citation_author=Tamara Münkemüller;,citation_publication_date=2024;,citation_cover_date=2024;,citation_year=2024;,citation_fulltext_html_url=https://onlinelibrary.wiley.com/doi/abs/10.1111/jbi.14734;,citation_issue=4;,citation_doi=10.1111/jbi.14734;,citation_volume=51;,citation_language=en;,citation_journal_title=Journal of Biogeography;">
<meta name="citation_reference" content="citation_title=Ecological networks–beyond food webs;,citation_author=Thomas C. Ings;,citation_author=José M. Montoya;,citation_author=Jordi Bascompte;,citation_author=Nico Blüthgen;,citation_author=Lee Brown;,citation_author=Carsten F. Dormann;,citation_author=François Edwards;,citation_author=David Figueroa;,citation_author=Ute Jacob;,citation_author=J. Iwan Jones;,citation_author=Rasmus B. Lauridsen;,citation_author=Mark E. Ledger;,citation_author=Hannah M. Lewis;,citation_author=Jens M. Olesen;,citation_author=F. J. Frank Veen;,citation_author=Phil H. Warren;,citation_author=Guy Woodward;,citation_publication_date=2009-01;,citation_cover_date=2009-01;,citation_year=2009;,citation_issue=1;,citation_doi=10.1111/j.1365-2656.2008.01460.x;,citation_volume=78;,citation_language=eng;,citation_journal_title=The Journal of Animal Ecology;">
<meta name="citation_reference" content="citation_title=Compilation and network analyses of cambrian food webs;,citation_author=Jennifer A. Dunne;,citation_author=Richard J. Williams;,citation_author=Neo D. Martinez;,citation_author=Rachel A. Wood;,citation_author=Douglas H. Erwin;,citation_publication_date=2008-04-29;,citation_cover_date=2008-04-29;,citation_year=2008;,citation_fulltext_html_url=https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.0060102;,citation_issue=4;,citation_doi=10.1371/journal.pbio.0060102;,citation_volume=6;,citation_language=en;,citation_journal_title=PLOS Biology;">
<meta name="citation_reference" content="citation_title=Predatory behaviour and taphonomy of a jurassic belemnoid coleoid (diplobelida, cephalopoda);,citation_author=Dominique Jenny;,citation_author=Dirk Fuchs;,citation_author=Alexander I. Arkhipkin;,citation_author=Rolf B. Hauff;,citation_author=Barbara Fritschi;,citation_author=Christian Klug;,citation_publication_date=2019-05-28;,citation_cover_date=2019-05-28;,citation_year=2019;,citation_fulltext_html_url=https://www.nature.com/articles/s41598-019-44260-w;,citation_issue=1;,citation_doi=10.1038/s41598-019-44260-w;,citation_volume=9;,citation_language=en;,citation_journal_title=Scientific Reports;">
<meta name="citation_reference" content="citation_title=Direct evidence of hybodont shark predation on late jurassic ammonites;,citation_author=Romain Vullo;,citation_publication_date=2011-06-01;,citation_cover_date=2011-06-01;,citation_year=2011;,citation_fulltext_html_url=https://doi.org/10.1007/s00114-011-0789-9;,citation_issue=6;,citation_doi=10.1007/s00114-011-0789-9;,citation_volume=98;,citation_language=en;,citation_journal_title=Naturwissenschaften;">
<meta name="citation_reference" content="citation_title=Chasing ecological interactions;,citation_author=Pedro Jordano;,citation_publication_date=2016-09-15;,citation_cover_date=2016-09-15;,citation_year=2016;,citation_fulltext_html_url=https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002559;,citation_issue=9;,citation_doi=10.1371/journal.pbio.1002559;,citation_volume=14;,citation_language=en;,citation_journal_title=PLOS Biology;">
<meta name="citation_reference" content="citation_title=Sampling networks of ecological interactions;,citation_author=Pedro Jordano;,citation_editor=Daniel Stouffer;,citation_publication_date=2016-12;,citation_cover_date=2016-12;,citation_year=2016;,citation_fulltext_html_url=http://doi.wiley.com/10.1111/1365-2435.12763;,citation_issue=12;,citation_doi=10.1111/1365-2435.12763;,citation_volume=30;,citation_language=en;,citation_journal_title=Functional Ecology;">
<meta name="citation_reference" content="citation_title=Analysing ecological networks of species interactions;,citation_author=Eva Delmas;,citation_author=Mathilde Besson;,citation_author=Marie-Hélène Brice;,citation_author=Laura A. Burkle;,citation_author=Giulio V. Dalla Riva;,citation_author=Marie-Josée Fortin;,citation_author=Dominique Gravel;,citation_author=Paulo R. Guimarães;,citation_author=David H. Hembry;,citation_author=Erica A. Newman;,citation_author=Jens M. Olesen;,citation_author=Mathias M. Pires;,citation_author=Justin D. Yeakel;,citation_author=Timothée Poisot;,citation_publication_date=2019;,citation_cover_date=2019;,citation_year=2019;,citation_fulltext_html_url=http://onlinelibrary.wiley.com/doi/abs/10.1111/brv.12433;,citation_issue=1;,citation_doi=10.1111/brv.12433;,citation_volume=94;,citation_language=en;,citation_journal_title=Biological Reviews;">
<meta name="citation_reference" content="citation_title=Stability criteria for complex ecosystems;,citation_author=Stefano Allesina;,citation_author=Si Tang;,citation_publication_date=2012-03;,citation_cover_date=2012-03;,citation_year=2012;,citation_fulltext_html_url=https://www.nature.com/articles/nature10832;,citation_issue=7388;,citation_doi=10.1038/nature10832;,citation_volume=483;,citation_language=en;,citation_journal_title=Nature;">
<meta name="citation_reference" content="citation_title=Coral reefs in crisis: The reliability of deep-time food web reconstructions as analogs for the present;,citation_author=Peter D. Roopnarine;,citation_author=Ashley A. Dineen;,citation_editor=Carrie L. Tyler;,citation_editor=Chris L. Schneider;,citation_publication_date=2018;,citation_cover_date=2018;,citation_year=2018;,citation_fulltext_html_url=https://doi.org/10.1007/978-3-319-73795-9_6;,citation_language=en;,citation_inbook_title=undefined;">
<meta name="citation_reference" content="citation_title=Complexity and fragility in ecological networks;,citation_author=Ricard V. Solé;,citation_author=M. Montoya;,citation_publication_date=2001-10-07;,citation_cover_date=2001-10-07;,citation_year=2001;,citation_fulltext_html_url=https://royalsocietypublishing.org/doi/10.1098/rspb.2001.1767;,citation_issue=1480;,citation_doi=10.1098/rspb.2001.1767;,citation_volume=268;,citation_language=en;,citation_journal_title=Proceedings of the Royal Society of London. Series B: Biological Sciences;">
<meta name="citation_reference" content="citation_title=Network structure and biodiversity loss in food webs: Robustness increases with connectance;,citation_author=Jennifer Dunne;,citation_author=Richard J Williams;,citation_author=Néo D Martinez;,citation_publication_date=2002;,citation_cover_date=2002;,citation_year=2002;,citation_issue=4;,citation_volume=5;,citation_journal_title=Ecol. Lett.;">
<meta name="citation_reference" content="citation_title=Approaching a state shift in earth’s biosphere;,citation_author=Anthony D. Barnosky;,citation_author=Elizabeth A. Hadly;,citation_author=Jordi Bascompte;,citation_author=Eric L. Berlow;,citation_author=James H. Brown;,citation_author=Mikael Fortelius;,citation_author=Wayne M. Getz;,citation_author=John Harte;,citation_author=Alan Hastings;,citation_author=Pablo A. Marquet;,citation_author=Neo D. Martinez;,citation_author=Arne Mooers;,citation_author=Peter Roopnarine;,citation_author=Geerat Vermeij;,citation_author=John W. Williams;,citation_author=Rosemary Gillespie;,citation_author=Justin Kitzes;,citation_author=Charles Marshall;,citation_author=Nicholas Matzke;,citation_author=David P. Mindell;,citation_author=Eloy Revilla;,citation_author=Adam B. Smith;,citation_publication_date=2012-06;,citation_cover_date=2012-06;,citation_year=2012;,citation_fulltext_html_url=https://www.nature.com/articles/nature11018;,citation_issue=7401;,citation_doi=10.1038/nature11018;,citation_volume=486;,citation_language=en;,citation_journal_title=Nature;">
<meta name="citation_reference" content="citation_title=Scaling from metawebs to realised webs: A hierarchical approach to network ecology;,citation_author=Tanya Strydom;,citation_author=Alexander M. Dunhill;,citation_author=Jennifer A. Dunne;,citation_author=Timothée Poisot;,citation_author=Andrew P. Beckerman;,citation_publication_date=2026-01-21;,citation_cover_date=2026-01-21;,citation_year=2026;,citation_fulltext_html_url=https://ecoevorxiv.org/repository/view/11514/;,citation_doi=10.32942/X2JW8K;,citation_language=en;,citation_journal_title=EcoEvoRxiv;">
<meta name="citation_reference" content="citation_title=Animal diversity and ecosystem functioning in dynamic food webs;,citation_author=Florian D. Schneider;,citation_author=Ulrich Brose;,citation_author=Björn C. Rall;,citation_author=Christian Guill;,citation_publication_date=2016-10-05;,citation_cover_date=2016-10-05;,citation_year=2016;,citation_fulltext_html_url=https://www.nature.com/articles/ncomms12718;,citation_issue=1;,citation_doi=10.1038/ncomms12718;,citation_volume=7;,citation_language=en;,citation_journal_title=Nature Communications;">
<meta name="citation_reference" content="citation_title=The effects of defaunation on plants’ capacity to track climate change;,citation_author=Evan C. Fricke;,citation_author=Alejandro Ordonez;,citation_author=Haldre S. Rogers;,citation_author=Jens-Christian Svenning;,citation_publication_date=2022-01-14;,citation_cover_date=2022-01-14;,citation_year=2022;,citation_fulltext_html_url=http://www.science.org/doi/full/10.1126/science.abk3510;,citation_issue=6577;,citation_doi=10.1126/science.abk3510;,citation_volume=375;,citation_journal_title=Science;">
<meta name="citation_reference" content="citation_title=Rescuing ecosystems from extinction cascades through compensatory perturbations;,citation_author=Sagar Sahasrabudhe;,citation_author=Adilson E. Motter;,citation_publication_date=2011-01-25;,citation_cover_date=2011-01-25;,citation_year=2011;,citation_fulltext_html_url=https://www.nature.com/articles/ncomms1163;,citation_issue=1;,citation_doi=10.1038/ncomms1163;,citation_volume=2;,citation_language=en;,citation_journal_title=Nature Communications;">
<meta name="citation_reference" content="citation_title=Stability criteria for complex ecosystems;,citation_author=Stefano Allesina;,citation_author=Si Tang;,citation_publication_date=2012-03;,citation_cover_date=2012-03;,citation_year=2012;,citation_fulltext_html_url=https://www.nature.com/articles/nature10832;,citation_issue=7388;,citation_doi=10.1038/nature10832;,citation_volume=483;,citation_language=en;,citation_journal_title=Nature;">
<meta name="citation_reference" content="citation_title=All ecological models are wrong, but some are useful;,citation_author=Daniel B. Stouffer;,citation_publication_date=2019;,citation_cover_date=2019;,citation_year=2019;,citation_fulltext_html_url=http://besjournals.onlinelibrary.wiley.com/doi/abs/10.1111/1365-2656.12949;,citation_issue=2;,citation_doi=https://doi.org/10.1111/1365-2656.12949;,citation_volume=88;,citation_language=en;,citation_journal_title=Journal of Animal Ecology;">
<meta name="citation_reference" content="citation_title=Consumerresource body-size relationships in natural food webs;,citation_author=Ulrich Brose;,citation_author=Tomas Jonsson;,citation_author=Eric L. Berlow;,citation_author=Philip Warren;,citation_author=Carolin Banasek-Richter;,citation_author=Louis-Félix Bersier;,citation_author=Julia L. Blanchard;,citation_author=Thomas Brey;,citation_author=Stephen R. Carpenter;,citation_author=Marie-France Cattin Blandenier;,citation_author=Lara Cushing;,citation_author=Hassan Ali Dawah;,citation_author=Tony Dell;,citation_author=Francois Edwards;,citation_author=Sarah Harper-Smith;,citation_author=Ute Jacob;,citation_author=Mark E. Ledger;,citation_author=Neo D. Martinez;,citation_author=Jane Memmott;,citation_author=Katja Mintenbeck;,citation_author=John K. Pinnegar;,citation_author=Björn C. Rall;,citation_author=Thomas S. Rayner;,citation_author=Daniel C. Reuman;,citation_author=Liliane Ruess;,citation_author=Werner Ulrich;,citation_author=Richard J. Williams;,citation_author=Guy Woodward;,citation_author=Joel E. Cohen;,citation_publication_date=2006;,citation_cover_date=2006;,citation_year=2006;,citation_fulltext_html_url=http://esajournals.onlinelibrary.wiley.com/doi/abs/10.1890/0012-9658%282006%2987%5B2411%3ACBRINF%5D2.0.CO%3B2;,citation_issue=10;,citation_doi=https://doi.org/10.1890/0012-9658(2006)87[2411:CBRINF]2.0.CO;2;,citation_volume=87;,citation_language=en;,citation_journal_title=Ecology;">
<meta name="citation_reference" content="citation_title=Collapse of an ecological network in ancient egypt;,citation_author=Justin D. Yeakel;,citation_author=Mathias M. Pires;,citation_author=Lars Rudolf;,citation_author=Nathaniel J. Dominy;,citation_author=Paul L. Koch;,citation_author=Paulo R. Guimarães;,citation_author=Thilo Gross;,citation_publication_date=2014-10;,citation_cover_date=2014-10;,citation_year=2014;,citation_issue=40;,citation_doi=10.1073/pnas.1408471111;,citation_volume=111;,citation_journal_title=PNAS;">
<meta name="citation_reference" content="citation_title=Extinction cascades, community collapse, and recovery across a mesozoic hyperthermal event;,citation_author=Alexander M. Dunhill;,citation_author=Karolina Zarzyczny;,citation_author=Jack O. Shaw;,citation_author=Jed W. Atkinson;,citation_author=Crispin T. S. Little;,citation_author=Andrew P. Beckerman;,citation_publication_date=2024-10-04;,citation_cover_date=2024-10-04;,citation_year=2024;,citation_fulltext_html_url=https://www.nature.com/articles/s41467-024-53000-2;,citation_issue=1;,citation_doi=10.1038/s41467-024-53000-2;,citation_volume=15;,citation_language=en;,citation_journal_title=Nature Communications;">
<meta name="citation_reference" content="citation_title=The structure of probabilistic networks;,citation_author=Timothée Poisot;,citation_author=Alyssa Cirtwill;,citation_author=Kévin Cazelles;,citation_author=Dominique Gravel;,citation_author=Marie-Josée Fortin;,citation_author=Daniel Stouffer;,citation_publication_date=2016;,citation_cover_date=2016;,citation_year=2016;,citation_issue=3;,citation_doi=10;,citation_volume=7;,citation_journal_title=Methods in Ecology and Evolution;">
<meta name="citation_reference" content="citation_title=Beyond species: Why ecological interaction networks vary through space and time;,citation_author=Timothée Poisot;,citation_author=Daniel B. Stouffer;,citation_author=Dominique Gravel;,citation_publication_date=2015;,citation_cover_date=2015;,citation_year=2015;,citation_fulltext_html_url=http://onlinelibrary.wiley.com/doi/abs/10.1111/oik.01719;,citation_issue=3;,citation_doi=10.1111/oik.01719;,citation_volume=124;,citation_language=en;,citation_journal_title=Oikos;">
<meta name="citation_reference" content="citation_title=Bringing elton and grinnell together: A quantitative framework to represent the biogeography of ecological interaction networks;,citation_author=Dominique Gravel;,citation_author=Benjamin Baiser;,citation_author=Jennifer A. Dunne;,citation_author=Jens-Peter Kopelke;,citation_author=Neo D. Martinez;,citation_author=Tommi Nyman;,citation_author=Timothée Poisot;,citation_author=Daniel B. Stouffer;,citation_author=Jason M. Tylianakis;,citation_author=Spencer A. Wood;,citation_author=Tomas Roslin;,citation_publication_date=2019;,citation_cover_date=2019;,citation_year=2019;,citation_fulltext_html_url=https://onlinelibrary.wiley.com/doi/abs/10.1111/ecog.04006;,citation_issue=3;,citation_doi=https://doi.org/10.1111/ecog.04006;,citation_volume=42;,citation_language=en;,citation_journal_title=Ecography;">
<meta name="citation_reference" content="citation_title=Quantitative patterns in the structure of model and empirical food webs;,citation_author=D. B. Stouffer;,citation_author=J. Camacho;,citation_author=R. Guimerà;,citation_author=C. A. Ng;,citation_author=L. A. Nunes Amaral;,citation_publication_date=2005;,citation_cover_date=2005;,citation_year=2005;,citation_fulltext_html_url=https://onlinelibrary.wiley.com/doi/abs/10.1890/04-0957;,citation_issue=5;,citation_doi=10.1890/04-0957;,citation_volume=86;,citation_language=en;,citation_journal_title=Ecology;">
<meta name="citation_reference" content="citation_title=Robustness to secondary extinctions: Comparing trait-based sequential deletions in static and dynamic food webs;,citation_author=Alva Curtsdotter;,citation_author=Amrei Binzer;,citation_author=Ulrich Brose;,citation_author=Francisco De Castro;,citation_author=Bo Ebenman;,citation_author=Anna Eklöf;,citation_author=Jens O. Riede;,citation_author=Aaron Thierry;,citation_author=Björn C. Rall;,citation_publication_date=2011-11;,citation_cover_date=2011-11;,citation_year=2011;,citation_fulltext_html_url=https://linkinghub.elsevier.com/retrieve/pii/S1439179111001216;,citation_issue=7;,citation_doi=10.1016/j.baae.2011.09.008;,citation_volume=12;,citation_language=en;,citation_journal_title=Basic and Applied Ecology;">
<meta name="citation_reference" content="citation_title=Collapse of terrestrial mammal food webs since the late pleistocene;,citation_author=Evan C. Fricke;,citation_author=Chia Hsieh;,citation_author=Owen Middleton;,citation_author=Daniel Gorczynski;,citation_author=Caroline D. Cappello;,citation_author=Oscar Sanisidro;,citation_author=John Rowan;,citation_author=Jens-Christian Svenning;,citation_author=Lydia Beaudrot;,citation_publication_date=2022-08-26;,citation_cover_date=2022-08-26;,citation_year=2022;,citation_fulltext_html_url=http://dx.doi.org/10.1126/science.abn4012;,citation_issue=6609;,citation_doi=10.1126/science.abn4012;,citation_volume=377;,citation_language=en;,citation_journal_title=Science;">
<meta name="citation_reference" content="citation_title=Collapse of terrestrial mammal food webs since the late pleistocene;,citation_author=Evan C. Fricke;,citation_author=Chia Hsieh;,citation_author=Owen Middleton;,citation_author=Daniel Gorczynski;,citation_author=Caroline D. Cappello;,citation_author=Oscar Sanisidro;,citation_author=John Rowan;,citation_author=Jens-Christian Svenning;,citation_author=Lydia Beaudrot;,citation_publication_date=2022-08-26;,citation_cover_date=2022-08-26;,citation_year=2022;,citation_fulltext_html_url=http://dx.doi.org/10.1126/science.abn4012;,citation_issue=6609;,citation_doi=10.1126/science.abn4012;,citation_volume=377;,citation_language=en;,citation_journal_title=Science;">
<meta name="citation_reference" content="citation_title=The dissimilarity of species interaction networks;,citation_author=Timothée Poisot;,citation_author=Elsa Canard;,citation_author=David Mouillot;,citation_author=Nicolas Mouquet;,citation_author=Dominique Gravel;,citation_editor=Ferenc Jordan;,citation_publication_date=2012-09-20;,citation_cover_date=2012-09-20;,citation_year=2012;,citation_fulltext_html_url=http://dx.doi.org/10.1111/ele.12002;,citation_issue=12;,citation_doi=10.1111/ele.12002;,citation_volume=15;,citation_language=en;,citation_journal_title=Ecology Letters;">
</head>

<body class="quarto-light">

<header id="title-block-header" class="quarto-title-block default toc-left page-columns page-full">
  <div class="quarto-title-banner page-columns page-full">
    <div class="quarto-title column-body">
      <h1 class="title">Reconstructing food webs in deep time: why model choice matters for ecological inference</h1>
          </div>

    
    <div class="quarto-title-meta-container">
      <div class="quarto-title-meta-column-start">
            <div class="quarto-title-meta-author">
          <div class="quarto-title-meta-heading">Authors</div>
          <div class="quarto-title-meta-heading">Affiliations</div>
          
                <div class="quarto-title-meta-contents">
            <p class="author">Tanya Strydom <a href="mailto:t.strydom@sheffield.ac.uk" class="quarto-title-author-email"><i class="bi bi-envelope"></i></a> <a href="https://orcid.org/0000-0001-6067-1349" class="quarto-title-author-orcid"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAA2ZpVFh0WE1MOmNvbS5hZG9iZS54bXAAAAAAADw/eHBhY2tldCBiZWdpbj0i77u/IiBpZD0iVzVNME1wQ2VoaUh6cmVTek5UY3prYzlkIj8+IDx4OnhtcG1ldGEgeG1sbnM6eD0iYWRvYmU6bnM6bWV0YS8iIHg6eG1wdGs9IkFkb2JlIFhNUCBDb3JlIDUuMC1jMDYwIDYxLjEzNDc3NywgMjAxMC8wMi8xMi0xNzozMjowMCAgICAgICAgIj4gPHJkZjpSREYgeG1sbnM6cmRmPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5LzAyLzIyLXJkZi1zeW50YXgtbnMjIj4gPHJkZjpEZXNjcmlwdGlvbiByZGY6YWJvdXQ9IiIgeG1sbnM6eG1wTU09Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC9tbS8iIHhtbG5zOnN0UmVmPSJodHRwOi8vbnMuYWRvYmUuY29tL3hhcC8xLjAvc1R5cGUvUmVzb3VyY2VSZWYjIiB4bWxuczp4bXA9Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC8iIHhtcE1NOk9yaWdpbmFsRG9jdW1lbnRJRD0ieG1wLmRpZDo1N0NEMjA4MDI1MjA2ODExOTk0QzkzNTEzRjZEQTg1NyIgeG1wTU06RG9jdW1lbnRJRD0ieG1wLmRpZDozM0NDOEJGNEZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wTU06SW5zdGFuY2VJRD0ieG1wLmlpZDozM0NDOEJGM0ZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wOkNyZWF0b3JUb29sPSJBZG9iZSBQaG90b3Nob3AgQ1M1IE1hY2ludG9zaCI+IDx4bXBNTTpEZXJpdmVkRnJvbSBzdFJlZjppbnN0YW5jZUlEPSJ4bXAuaWlkOkZDN0YxMTc0MDcyMDY4MTE5NUZFRDc5MUM2MUUwNEREIiBzdFJlZjpkb2N1bWVudElEPSJ4bXAuZGlkOjU3Q0QyMDgwMjUyMDY4MTE5OTRDOTM1MTNGNkRBODU3Ii8+IDwvcmRmOkRlc2NyaXB0aW9uPiA8L3JkZjpSREY+IDwveDp4bXBtZXRhPiA8P3hwYWNrZXQgZW5kPSJyIj8+84NovQAAAR1JREFUeNpiZEADy85ZJgCpeCB2QJM6AMQLo4yOL0AWZETSqACk1gOxAQN+cAGIA4EGPQBxmJA0nwdpjjQ8xqArmczw5tMHXAaALDgP1QMxAGqzAAPxQACqh4ER6uf5MBlkm0X4EGayMfMw/Pr7Bd2gRBZogMFBrv01hisv5jLsv9nLAPIOMnjy8RDDyYctyAbFM2EJbRQw+aAWw/LzVgx7b+cwCHKqMhjJFCBLOzAR6+lXX84xnHjYyqAo5IUizkRCwIENQQckGSDGY4TVgAPEaraQr2a4/24bSuoExcJCfAEJihXkWDj3ZAKy9EJGaEo8T0QSxkjSwORsCAuDQCD+QILmD1A9kECEZgxDaEZhICIzGcIyEyOl2RkgwAAhkmC+eAm0TAAAAABJRU5ErkJggg=="></a></p>
          </div>
                <div class="quarto-title-meta-contents">
                    <p class="affiliation">
                        School of Biosciences, University of Sheffield, Sheffield, UK
                      </p>
                  </div>
                      <div class="quarto-title-meta-contents">
            <p class="author">Baran Karapunar <a href="https://orcid.org/0000-0001-9711-1492" class="quarto-title-author-orcid"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAA2ZpVFh0WE1MOmNvbS5hZG9iZS54bXAAAAAAADw/eHBhY2tldCBiZWdpbj0i77u/IiBpZD0iVzVNME1wQ2VoaUh6cmVTek5UY3prYzlkIj8+IDx4OnhtcG1ldGEgeG1sbnM6eD0iYWRvYmU6bnM6bWV0YS8iIHg6eG1wdGs9IkFkb2JlIFhNUCBDb3JlIDUuMC1jMDYwIDYxLjEzNDc3NywgMjAxMC8wMi8xMi0xNzozMjowMCAgICAgICAgIj4gPHJkZjpSREYgeG1sbnM6cmRmPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5LzAyLzIyLXJkZi1zeW50YXgtbnMjIj4gPHJkZjpEZXNjcmlwdGlvbiByZGY6YWJvdXQ9IiIgeG1sbnM6eG1wTU09Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC9tbS8iIHhtbG5zOnN0UmVmPSJodHRwOi8vbnMuYWRvYmUuY29tL3hhcC8xLjAvc1R5cGUvUmVzb3VyY2VSZWYjIiB4bWxuczp4bXA9Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC8iIHhtcE1NOk9yaWdpbmFsRG9jdW1lbnRJRD0ieG1wLmRpZDo1N0NEMjA4MDI1MjA2ODExOTk0QzkzNTEzRjZEQTg1NyIgeG1wTU06RG9jdW1lbnRJRD0ieG1wLmRpZDozM0NDOEJGNEZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wTU06SW5zdGFuY2VJRD0ieG1wLmlpZDozM0NDOEJGM0ZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wOkNyZWF0b3JUb29sPSJBZG9iZSBQaG90b3Nob3AgQ1M1IE1hY2ludG9zaCI+IDx4bXBNTTpEZXJpdmVkRnJvbSBzdFJlZjppbnN0YW5jZUlEPSJ4bXAuaWlkOkZDN0YxMTc0MDcyMDY4MTE5NUZFRDc5MUM2MUUwNEREIiBzdFJlZjpkb2N1bWVudElEPSJ4bXAuZGlkOjU3Q0QyMDgwMjUyMDY4MTE5OTRDOTM1MTNGNkRBODU3Ii8+IDwvcmRmOkRlc2NyaXB0aW9uPiA8L3JkZjpSREY+IDwveDp4bXBtZXRhPiA8P3hwYWNrZXQgZW5kPSJyIj8+84NovQAAAR1JREFUeNpiZEADy85ZJgCpeCB2QJM6AMQLo4yOL0AWZETSqACk1gOxAQN+cAGIA4EGPQBxmJA0nwdpjjQ8xqArmczw5tMHXAaALDgP1QMxAGqzAAPxQACqh4ER6uf5MBlkm0X4EGayMfMw/Pr7Bd2gRBZogMFBrv01hisv5jLsv9nLAPIOMnjy8RDDyYctyAbFM2EJbRQw+aAWw/LzVgx7b+cwCHKqMhjJFCBLOzAR6+lXX84xnHjYyqAo5IUizkRCwIENQQckGSDGY4TVgAPEaraQr2a4/24bSuoExcJCfAEJihXkWDj3ZAKy9EJGaEo8T0QSxkjSwORsCAuDQCD+QILmD1A9kECEZgxDaEZhICIzGcIyEyOl2RkgwAAhkmC+eAm0TAAAAABJRU5ErkJggg=="></a></p>
          </div>
                <div class="quarto-title-meta-contents">
                    <p class="affiliation">
                        School of Earth and Environment, University of Leeds, Leeds, UK
                      </p>
                  </div>
                      <div class="quarto-title-meta-contents">
            <p class="author">Andrew P. Beckerman <a href="https://orcid.org/0000-0002-7859-8394" class="quarto-title-author-orcid"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAA2ZpVFh0WE1MOmNvbS5hZG9iZS54bXAAAAAAADw/eHBhY2tldCBiZWdpbj0i77u/IiBpZD0iVzVNME1wQ2VoaUh6cmVTek5UY3prYzlkIj8+IDx4OnhtcG1ldGEgeG1sbnM6eD0iYWRvYmU6bnM6bWV0YS8iIHg6eG1wdGs9IkFkb2JlIFhNUCBDb3JlIDUuMC1jMDYwIDYxLjEzNDc3NywgMjAxMC8wMi8xMi0xNzozMjowMCAgICAgICAgIj4gPHJkZjpSREYgeG1sbnM6cmRmPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5LzAyLzIyLXJkZi1zeW50YXgtbnMjIj4gPHJkZjpEZXNjcmlwdGlvbiByZGY6YWJvdXQ9IiIgeG1sbnM6eG1wTU09Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC9tbS8iIHhtbG5zOnN0UmVmPSJodHRwOi8vbnMuYWRvYmUuY29tL3hhcC8xLjAvc1R5cGUvUmVzb3VyY2VSZWYjIiB4bWxuczp4bXA9Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC8iIHhtcE1NOk9yaWdpbmFsRG9jdW1lbnRJRD0ieG1wLmRpZDo1N0NEMjA4MDI1MjA2ODExOTk0QzkzNTEzRjZEQTg1NyIgeG1wTU06RG9jdW1lbnRJRD0ieG1wLmRpZDozM0NDOEJGNEZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wTU06SW5zdGFuY2VJRD0ieG1wLmlpZDozM0NDOEJGM0ZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wOkNyZWF0b3JUb29sPSJBZG9iZSBQaG90b3Nob3AgQ1M1IE1hY2ludG9zaCI+IDx4bXBNTTpEZXJpdmVkRnJvbSBzdFJlZjppbnN0YW5jZUlEPSJ4bXAuaWlkOkZDN0YxMTc0MDcyMDY4MTE5NUZFRDc5MUM2MUUwNEREIiBzdFJlZjpkb2N1bWVudElEPSJ4bXAuZGlkOjU3Q0QyMDgwMjUyMDY4MTE5OTRDOTM1MTNGNkRBODU3Ii8+IDwvcmRmOkRlc2NyaXB0aW9uPiA8L3JkZjpSREY+IDwveDp4bXBtZXRhPiA8P3hwYWNrZXQgZW5kPSJyIj8+84NovQAAAR1JREFUeNpiZEADy85ZJgCpeCB2QJM6AMQLo4yOL0AWZETSqACk1gOxAQN+cAGIA4EGPQBxmJA0nwdpjjQ8xqArmczw5tMHXAaALDgP1QMxAGqzAAPxQACqh4ER6uf5MBlkm0X4EGayMfMw/Pr7Bd2gRBZogMFBrv01hisv5jLsv9nLAPIOMnjy8RDDyYctyAbFM2EJbRQw+aAWw/LzVgx7b+cwCHKqMhjJFCBLOzAR6+lXX84xnHjYyqAo5IUizkRCwIENQQckGSDGY4TVgAPEaraQr2a4/24bSuoExcJCfAEJihXkWDj3ZAKy9EJGaEo8T0QSxkjSwORsCAuDQCD+QILmD1A9kECEZgxDaEZhICIzGcIyEyOl2RkgwAAhkmC+eAm0TAAAAABJRU5ErkJggg=="></a></p>
          </div>
                <div class="quarto-title-meta-contents">
                    <p class="affiliation">
                        School of Biosciences, University of Sheffield, Sheffield, UK
                      </p>
                  </div>
                      <div class="quarto-title-meta-contents">
            <p class="author">Alexander Dunhill <a href="https://orcid.org/0000-0002-8680-9163" class="quarto-title-author-orcid"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAA2ZpVFh0WE1MOmNvbS5hZG9iZS54bXAAAAAAADw/eHBhY2tldCBiZWdpbj0i77u/IiBpZD0iVzVNME1wQ2VoaUh6cmVTek5UY3prYzlkIj8+IDx4OnhtcG1ldGEgeG1sbnM6eD0iYWRvYmU6bnM6bWV0YS8iIHg6eG1wdGs9IkFkb2JlIFhNUCBDb3JlIDUuMC1jMDYwIDYxLjEzNDc3NywgMjAxMC8wMi8xMi0xNzozMjowMCAgICAgICAgIj4gPHJkZjpSREYgeG1sbnM6cmRmPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5LzAyLzIyLXJkZi1zeW50YXgtbnMjIj4gPHJkZjpEZXNjcmlwdGlvbiByZGY6YWJvdXQ9IiIgeG1sbnM6eG1wTU09Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC9tbS8iIHhtbG5zOnN0UmVmPSJodHRwOi8vbnMuYWRvYmUuY29tL3hhcC8xLjAvc1R5cGUvUmVzb3VyY2VSZWYjIiB4bWxuczp4bXA9Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC8iIHhtcE1NOk9yaWdpbmFsRG9jdW1lbnRJRD0ieG1wLmRpZDo1N0NEMjA4MDI1MjA2ODExOTk0QzkzNTEzRjZEQTg1NyIgeG1wTU06RG9jdW1lbnRJRD0ieG1wLmRpZDozM0NDOEJGNEZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wTU06SW5zdGFuY2VJRD0ieG1wLmlpZDozM0NDOEJGM0ZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wOkNyZWF0b3JUb29sPSJBZG9iZSBQaG90b3Nob3AgQ1M1IE1hY2ludG9zaCI+IDx4bXBNTTpEZXJpdmVkRnJvbSBzdFJlZjppbnN0YW5jZUlEPSJ4bXAuaWlkOkZDN0YxMTc0MDcyMDY4MTE5NUZFRDc5MUM2MUUwNEREIiBzdFJlZjpkb2N1bWVudElEPSJ4bXAuZGlkOjU3Q0QyMDgwMjUyMDY4MTE5OTRDOTM1MTNGNkRBODU3Ii8+IDwvcmRmOkRlc2NyaXB0aW9uPiA8L3JkZjpSREY+IDwveDp4bXBtZXRhPiA8P3hwYWNrZXQgZW5kPSJyIj8+84NovQAAAR1JREFUeNpiZEADy85ZJgCpeCB2QJM6AMQLo4yOL0AWZETSqACk1gOxAQN+cAGIA4EGPQBxmJA0nwdpjjQ8xqArmczw5tMHXAaALDgP1QMxAGqzAAPxQACqh4ER6uf5MBlkm0X4EGayMfMw/Pr7Bd2gRBZogMFBrv01hisv5jLsv9nLAPIOMnjy8RDDyYctyAbFM2EJbRQw+aAWw/LzVgx7b+cwCHKqMhjJFCBLOzAR6+lXX84xnHjYyqAo5IUizkRCwIENQQckGSDGY4TVgAPEaraQr2a4/24bSuoExcJCfAEJihXkWDj3ZAKy9EJGaEo8T0QSxkjSwORsCAuDQCD+QILmD1A9kECEZgxDaEZhICIzGcIyEyOl2RkgwAAhkmC+eAm0TAAAAABJRU5ErkJggg=="></a></p>
          </div>
                <div class="quarto-title-meta-contents">
                    <p class="affiliation">
                        School of Earth and Environment, University of Leeds, Leeds, UK
                      </p>
                  </div>
                    </div>
        
        <div class="quarto-title-meta">

                      
                <div>
            <div class="quarto-title-meta-heading">Published</div>
            <div class="quarto-title-meta-contents">
              <p class="date">February 3, 2026</p>
            </div>
          </div>
          
                
              </div>
      </div>
      <div class="quarto-title-meta-column-end quarto-other-formats-target">
      <div class="quarto-alternate-formats"><div class="quarto-title-meta-heading">Other Formats</div><div class="quarto-title-meta-contents"><p><a href="index.docx"><i class="bi bi-file-word"></i>MS Word</a></p></div><div class="quarto-title-meta-contents"><p><a href="index.pdf"><i class="bi bi-file-pdf"></i>PDF (preprint)</a></p></div></div></div>
    </div>

    <div>
      <div class="abstract">
        <div class="block-title">Abstract</div>
        <p>Food webs provide a powerful framework for understanding ecosystem structure and function, yet reconstructing them in paleoecological contexts remains challenging because direct evidence of species interactions is rarely preserved. A growing array of models exists for predicting interactions and inferring network structure, but these approaches differ markedly in their assumptions, mechanisms, and data requirements. Here, we evaluate how network reconstruction model choice shapes ecological inference in deep time and which approaches are most appropriate given the constraints of the fossil record. Using the Toarcian Oceanic Anoxic Event (Early Jurassic, ~183 Ma) as a case study, we reconstruct food webs for four successive community states using six contrasting modelling approaches spanning mechanistic, trait-based, and structural network representations. Despite identical taxon pools, models produced strikingly different network structures and species-level interactions, and these differences propagated into divergent inferences about extinction dynamics, interaction loss, and cascading effects. By framing food-web reconstructions as alternative ecological hypotheses (rather than interchangeable representations) this study bridges paleoecology and modern network theory, and demonstrates how model choice fundamentally conditions inference about extinction dynamics in deep time.</p>
      </div>
    </div>

    <div>
      <div class="keywords">
        <div class="block-title">Keywords</div>
        <p>Paleoecological networks, Food-web reconstruction, Ecological networks, Extinction dynamics, Trophic interactions, Toarcian Oceanic Anoxic Event</p>
      </div>
    </div>

    <div class="quarto-other-links-text-target">
    <div class="quarto-code-links"><div class="quarto-title-meta-heading">Code Links</div><div class="quarto-title-meta-contents"><span><a href="https://github.com/TanyaS08/ms_paleo_webs/tree/main/code"><i class="bi bi-github"></i>GitHub</a></span></div></div></div>  </div>
</header><div id="quarto-content" class="page-columns page-rows-contents page-layout-article toc-left">
<div id="quarto-sidebar-toc-left" class="sidebar toc-left">
  <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">Table of contents</h2>
   
  <ul>
  <li><a href="#introduction" id="toc-introduction" class="nav-link active" data-scroll-target="#introduction"><span class="header-section-number">1</span> Introduction</a></li>
  <li><a href="#materials-and-methods" id="toc-materials-and-methods" class="nav-link" data-scroll-target="#materials-and-methods"><span class="header-section-number">2</span> Materials and Methods</a>
  <ul class="collapse">
  <li><a href="#study-system-and-fossil-data" id="toc-study-system-and-fossil-data" class="nav-link" data-scroll-target="#study-system-and-fossil-data"><span class="header-section-number">2.1</span> Study system and fossil data</a></li>
  <li><a href="#network-reconstruction-approaches" id="toc-network-reconstruction-approaches" class="nav-link" data-scroll-target="#network-reconstruction-approaches"><span class="header-section-number">2.2</span> Network reconstruction approaches</a>
  <ul class="collapse">
  <li><a href="#conceptual-classification-of-network-types" id="toc-conceptual-classification-of-network-types" class="nav-link" data-scroll-target="#conceptual-classification-of-network-types"><span class="header-section-number">2.2.1</span> Conceptual classification of network types</a></li>
  <li><a href="#network-generation-and-replication" id="toc-network-generation-and-replication" class="nav-link" data-scroll-target="#network-generation-and-replication"><span class="header-section-number">2.2.2</span> Network generation and replication</a></li>
  </ul></li>
  <li><a href="#network-metrics-and-structural-analyses" id="toc-network-metrics-and-structural-analyses" class="nav-link" data-scroll-target="#network-metrics-and-structural-analyses"><span class="header-section-number">2.3</span> Network metrics and structural analyses</a></li>
  <li><a href="#extinction-simulations-and-model-evaluation" id="toc-extinction-simulations-and-model-evaluation" class="nav-link" data-scroll-target="#extinction-simulations-and-model-evaluation"><span class="header-section-number">2.4</span> Extinction simulations and model evaluation</a></li>
  </ul></li>
  <li><a href="#results" id="toc-results" class="nav-link" data-scroll-target="#results"><span class="header-section-number">3</span> Results</a>
  <ul class="collapse">
  <li><a href="#network-structure-differs-among-reconstruction-approaches" id="toc-network-structure-differs-among-reconstruction-approaches" class="nav-link" data-scroll-target="#network-structure-differs-among-reconstruction-approaches"><span class="header-section-number">3.1</span> Network structure differs among reconstruction approaches</a>
  <ul class="collapse">
  <li><a href="#inferred-pairwise-interactions-vary-widely-among-models" id="toc-inferred-pairwise-interactions-vary-widely-among-models" class="nav-link" data-scroll-target="#inferred-pairwise-interactions-vary-widely-among-models"><span class="header-section-number">3.1.1</span> Inferred pairwise interactions vary widely among models</a></li>
  </ul></li>
  <li><a href="#model-choice-influences-inferred-extinction-dynamics" id="toc-model-choice-influences-inferred-extinction-dynamics" class="nav-link" data-scroll-target="#model-choice-influences-inferred-extinction-dynamics"><span class="header-section-number">3.2</span> Model choice influences inferred extinction dynamics</a></li>
  </ul></li>
  <li><a href="#discussion" id="toc-discussion" class="nav-link" data-scroll-target="#discussion"><span class="header-section-number">4</span> Discussion</a></li>
  <li><a href="#model-choice-as-a-component-of-ecological-inference" id="toc-model-choice-as-a-component-of-ecological-inference" class="nav-link" data-scroll-target="#model-choice-as-a-component-of-ecological-inference"><span class="header-section-number">5</span> Model choice as a component of ecological inference</a>
  <ul class="collapse">
  <li><a href="#aligning-ecological-questions-with-model-choice" id="toc-aligning-ecological-questions-with-model-choice" class="nav-link" data-scroll-target="#aligning-ecological-questions-with-model-choice"><span class="header-section-number">5.1</span> Aligning ecological questions with model choice</a></li>
  <li><a href="#implications-for-paleoecological-network-studies" id="toc-implications-for-paleoecological-network-studies" class="nav-link" data-scroll-target="#implications-for-paleoecological-network-studies"><span class="header-section-number">5.2</span> Implications for paleoecological network studies</a></li>
  <li><a href="#recommendations-for-network-reconstruction-in-paleoecology" id="toc-recommendations-for-network-reconstruction-in-paleoecology" class="nav-link" data-scroll-target="#recommendations-for-network-reconstruction-in-paleoecology"><span class="header-section-number">5.3</span> Recommendations for network reconstruction in paleoecology</a></li>
  </ul></li>
  <li><a href="#conclusions" id="toc-conclusions" class="nav-link" data-scroll-target="#conclusions"><span class="header-section-number">6</span> Conclusions</a></li>
  <li><a href="#references" id="toc-references" class="nav-link" data-scroll-target="#references">References</a></li>
  </ul>
<div class="quarto-alternate-notebooks"><h2>Notebooks</h2><ul><li><a href="notebooks/model_summaries-preview.html"><i class="bi bi-journal-code"></i>SuppMat 1: Expanded descriptions of the food web models</a></li><li><a href="notebooks/results_expanded-preview.html"><i class="bi bi-journal-code"></i>SuppMat 2: Effects of Network Reconstruction on Food-Web Structure</a></li></ul></div></nav>
</div>
<div id="quarto-margin-sidebar" class="sidebar margin-sidebar zindex-bottom">
</div>
<main class="content quarto-banner-title-block" id="quarto-document-content">



  


<section id="introduction" class="level1" data-number="1">
<h1 data-number="1"><span class="header-section-number">1</span> Introduction</h1>
<p>There is growing interest in using deep-time fossil data and evidence of species interactions in past ecosystems (<em>e.g.,</em> <span class="citation" data-cites="dunne2008">Dunne et al. (<a href="#ref-dunne2008" role="doc-biblioref">2008</a>)</span>; <span class="citation" data-cites="dunne2014">Dunne et al. (<a href="#ref-dunne2014" role="doc-biblioref">2014</a>)</span>) as a foundation for understanding how ecological communities respond to multi-stressor environmental change, such as temperature increase, ocean acidification, and hypoxia <span class="citation" data-cites="kiessling2019 dillon2022">(<a href="#ref-kiessling2019" role="doc-biblioref">Kiessling et al. 2019</a>; <a href="#ref-dillon2022" role="doc-biblioref">Dillon et al. 2022</a>)</span>. Paleoecological networks therefore represent a particularly valuable opportunity to investigate community responses to major environmental transitions as they allow for the explicit construction of pre- and post-extinction interaction networks and enable the disentangling of extinction drivers as well as broader cascading effects <span class="citation" data-cites="dunhill2024 yeakel2014 roopnarine2006">(<a href="#ref-dunhill2024" role="doc-biblioref">Dunhill et al. 2024</a>; <a href="#ref-yeakel2014" role="doc-biblioref">Yeakel et al. 2014</a>; <a href="#ref-roopnarine2006" role="doc-biblioref">Roopnarine 2006</a>)</span>. Insights gained from these deep-time systems provide a critical context for interpreting present day ecological change and anticipating the impacts of ongoing anthropogenic stressors <span class="citation" data-cites="roopnarine2018 barnosky2012">(<a href="#ref-roopnarine2018" role="doc-biblioref">Roopnarine and Dineen 2018</a>; <a href="#ref-barnosky2012" role="doc-biblioref">Barnosky et al. 2012</a>)</span>.</p>
<p>Inference from ecological networks regarding structure and complexity is (at its core) a mathematical task and is therefore relatively independent of biological assumptions <span class="citation" data-cites="delmas2019">(<a href="#ref-delmas2019" role="doc-biblioref">Delmas et al. 2019</a>)</span>. While these structural properties can be analysed independently of biological detail, their implications for stability and dynamics depend critically on assumptions about the distribution and strength of interaction <span class="citation" data-cites="allesina2012 poisot2015">(<a href="#ref-allesina2012" role="doc-biblioref">Allesina and Tang 2012</a>; <a href="#ref-poisot2015" role="doc-biblioref">Poisot et al. 2015</a>)</span>. Thus, assumptions become meaningful once network structure and complexity are interpreted in functional terms or used as templates for simulating community and extinction dynamics. While there is a well-developed theoretical framework describing what can be inferred from network properties, a central challenge (particularly in paleoecology) lies in how networks are constructed. Unlike modern systems, paleoecological interactions are rarely directly observed, with only exceptional cases preserving explicit evidence of trophic interactions (<em>e.g.,</em> <span class="citation" data-cites="jenny2019">Jenny et al. (<a href="#ref-jenny2019" role="doc-biblioref">2019</a>)</span>; <span class="citation" data-cites="vullo2011">Vullo (<a href="#ref-vullo2011" role="doc-biblioref">2011</a>)</span>). As a result, paleo food web reconstruction depends on indirect inference, drawing on preserved traits, analogies with modern taxa, and ecological theory. Simply put, network reconstruction is not a data recovery process, but rather hypothesis generation under constraints.</p>
<p>Although numerous tools exist for inferring species interactions <span class="citation" data-cites="morales-castilla2015 pichler2023 strydom2021 allesina2008">(see <a href="#ref-morales-castilla2015" role="doc-biblioref">Morales-Castilla et al. 2015</a>; <a href="#ref-pichler2023" role="doc-biblioref">Pichler and Hartig 2023</a>; <a href="#ref-strydom2021" role="doc-biblioref">Strydom et al. 2021</a>; <a href="#ref-allesina2008" role="doc-biblioref">Allesina et al. 2008</a> for reviews)</span>, only a subset can be reliably applied in paleo contexts, where data on traits, abundances, and community composition are incomplete and systematically biased. This makes it essential to clearly articulate which reconstruction approaches are appropriate for which inferential purposes. Recent work has shown that reconstruction approaches (<em>e.g.,</em> models based on traits, abundances, or co-occurrence) can be categorised by the type of network that they predict <span class="citation" data-cites="strydom2026">(<a href="#ref-strydom2026" role="doc-biblioref">Strydom et al. 2026</a>)</span>. These include feasible networks (derived from trait matching and phylogenetic inference that produce metawebs of plausible interactions), realised networks, (constrained by ecological rules and evidence of actual feeding relationships), and purely structural networks (which reproduce ecologically plausible topologies but lack species-level node identities). In this sense, food-web reconstructions are not empirical recoveries of a single past ecosystem, but rather represent alternative, model-based, hypotheses about interaction structure constrained by the fossil record.</p>
<p>Each of these reconstruction approaches carries distinct assumptions that influence inferred network size, complexity, structure, and node-level properties, with direct consequences for ecological interpretations of extinction dynamics, stability, resilience, and ecosystem function <span class="citation" data-cites="solé2001 dunne2002 roopnarine2006 gravel2019">(<a href="#ref-solé2001" role="doc-biblioref">Solé and Montoya 2001</a>; <a href="#ref-dunne2002" role="doc-biblioref">Dunne et al. 2002</a>; <a href="#ref-roopnarine2006" role="doc-biblioref">Roopnarine 2006</a>; <a href="#ref-gravel2019" role="doc-biblioref">Gravel et al. 2019</a>)</span>. Despite this, most paleo food web studies default to constructing networks using expert knowledge (<em>e.g.,</em> <span class="citation" data-cites="dunne2008">Dunne et al. (<a href="#ref-dunne2008" role="doc-biblioref">2008</a>)</span>) or mechanistic feeding rules (<em>e.g.,</em> <span class="citation" data-cites="dunhill2024">Dunhill et al. (<a href="#ref-dunhill2024" role="doc-biblioref">2024</a>)</span>; <span class="citation" data-cites="roopnarine2017">Roopnarine (<a href="#ref-roopnarine2017" role="doc-biblioref">2017</a>)</span>; <span class="citation" data-cites="fricke2022a">Fricke et al. (<a href="#ref-fricke2022a" role="doc-biblioref">2022</a>)</span>), approaches that typically result in metawebs. This raises a critical but underexplored question: to what extent does the choice of network type (and particularly the use of alternatives to metawebs) control not only inferred food web structure but also conclusions about system behaviour, especially with respect to extinction cascades and vulnerability.</p>
<p>In this study, we address this question by explicitly evaluating how network reconstruction model choice shapes ecological inference in deep time. We link recent advances in food-web reconstruction methods to a comparative re-evaluation of primary and secondary, trait-based extinction dynamics during the early Toarcian extinction event, a volcanic-driven hyperthermal and marine crisis in the Early Jurassic (~183 Ma) <span class="citation" data-cites="dunhill2024">(<a href="#ref-dunhill2024" role="doc-biblioref">Dunhill et al. 2024</a>)</span>. We reconstruct four successive communities (pre-extinction, post-extinction, early recovery, and late recovery) from the Cleveland Basin of North Yorkshire, UK, using six contrasting network reconstruction approaches spanning feasible, realised, and structural network representations - as recognised in recent network inference frameworks <span class="citation" data-cites="morales-castilla2015 strydom2026">(<a href="#ref-morales-castilla2015" role="doc-biblioref">Morales-Castilla et al. 2015</a>; <a href="#ref-strydom2026" role="doc-biblioref">Strydom et al. 2026</a>)</span>. For each community, we compare network-level structure, species-level properties, and inferred interactions across models, allowing us to assess how reconstruction assumptions propagate into ecological interpretations. Finally, we replicate the secondary extinction analyses of Dunhill et al.&nbsp;across all six reconstruction methods to explicitly test how model choice influences inference about extinction drivers, interaction loss, and cascading dynamics. By explicitly comparing multiple reconstruction approaches within a single paleoecological system, this study provides a framework for evaluating how methodological assumptions shape interpretations of ancient food-web structure and dynamics.</p>
</section>
<section id="materials-and-methods" class="level1" data-number="2">
<h1 data-number="2"><span class="header-section-number">2</span> Materials and Methods</h1>
<section id="study-system-and-fossil-data" class="level2" data-number="2.1">
<h2 data-number="2.1" class="anchored" data-anchor-id="study-system-and-fossil-data"><span class="header-section-number">2.1</span> Study system and fossil data</h2>
<p>We used fossil occurrence data from the Cleveland Basin spanning the upper Pliensbachian to the upper Toarcian, following <span class="citation" data-cites="dunhill2024">Dunhill et al. (<a href="#ref-dunhill2024" role="doc-biblioref">2024</a>)</span>. Four paleo-communities were defined: pre-extinction, post-extinction, early recovery, and late recovery. Each assemblage was treated as a community of potentially interacting taxa. Modes of life were assigned following <span class="citation" data-cites="dunhill2024">Dunhill et al. (<a href="#ref-dunhill2024" role="doc-biblioref">2024</a>)</span> using the Bambach ecospace framework. Traits included motility, tiering, feeding mode, and size class, with size defined categorically based on maximum linear dimensions.</p>
</section>
<section id="network-reconstruction-approaches" class="level2" data-number="2.2">
<h2 data-number="2.2" class="anchored" data-anchor-id="network-reconstruction-approaches"><span class="header-section-number">2.2</span> Network reconstruction approaches</h2>
<section id="conceptual-classification-of-network-types" class="level3" data-number="2.2.1">
<h3 data-number="2.2.1" class="anchored" data-anchor-id="conceptual-classification-of-network-types"><span class="header-section-number">2.2.1</span> Conceptual classification of network types</h3>
<p>Most paleo-specific approaches currently operate within the feasibility space <span class="citation" data-cites="shaw2024 roopnarine2006 fricke2022a">(<em>e.g.,</em> <a href="#ref-shaw2024" role="doc-biblioref">Shaw et al. 2024</a>; <a href="#ref-roopnarine2006" role="doc-biblioref">Roopnarine 2006</a>; <a href="#ref-fricke2022a" role="doc-biblioref">Fricke et al. 2022</a>)</span>. Although well suited for reconstructing feasible interactions, these methods represent only a subset of the broader spectrum of network construction approaches. Here, we present a suite of models (<a href="#tbl-models" class="quarto-xref">Table&nbsp;1</a>) that enable the construction of a wider range of ecological networks and the exploration of a broader set of ecological questions, provided that their underlying assumptions are compatible with the constraints of fossil data. For example, some tools require quantitative estimates of body size, which must often be inferred from size classes or functional morphology in the fossil record. Structural models, such as the niche model, rely only on species richness and estimates/specification of connectance, but their species-agnostic nature limits their applicability to trait-based or diet-specific questions, although they do still accurately recover network structure <span class="citation" data-cites="stouffer2005">(<a href="#ref-stouffer2005" role="doc-biblioref">Stouffer et al. 2005</a>)</span> . Mechanistic approaches, by contrast, depend on accurate assignment of feeding traits or robust phylogenetic support. Recognizing how these methodological requirements intersect with the limits of fossil evidence is essential for selecting an appropriate modelling framework.</p>
<div id="tbl-models" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-tbl figure">
<figcaption class="quarto-float-caption-top quarto-float-caption quarto-float-tbl" id="tbl-models-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Table&nbsp;1: Six different models that can be used to construct food webs for both this specific community but are also broadly suited to paleo network prediction. These models span all facets of the network representation space (metaweb, realised, and structural network) and are suitable for an array of different paleo communities as the data requirements fall within the limitations set by the fossil record.
</figcaption>
<div aria-describedby="tbl-models-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<table class="caption-top table">
<colgroup>
<col style="width: 14%">
<col style="width: 14%">
<col style="width: 14%">
<col style="width: 14%">
<col style="width: 14%">
<col style="width: 14%">
<col style="width: 14%">
</colgroup>
<thead>
<tr class="header">
<th>Model family</th>
<th>Assumptions</th>
<th>Data needs</th>
<th>‘Limitation’</th>
<th>Network type</th>
<th>Key reference</th>
<th>Usage examples</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>random</td>
<td>Links are randomly distributed within a network</td>
<td>richness, number of links</td>
<td>parameter assumptions, species agnostic</td>
<td>structural network</td>
<td><span class="citation" data-cites="erdos1959">Erdős and Rényi (<a href="#ref-erdos1959" role="doc-biblioref">1959</a>)</span></td>
<td></td>
</tr>
<tr class="even">
<td>niche</td>
<td>Networks are interval, species can be ordered on a ‘niche axis’</td>
<td>richness, connectance</td>
<td>parameter assumptions, species agnostic</td>
<td>structural network</td>
<td><span class="citation" data-cites="williams2008">Williams and Martinez (<a href="#ref-williams2008" role="doc-biblioref">2008</a>)</span></td>
<td></td>
</tr>
<tr class="odd">
<td>allometric diet breadth model (ADBM)</td>
<td>Interactions are determined by energetic costs (foraging ecology)</td>
<td>body mass, biomass (abundance)</td>
<td>does not account for forbidden links in terms of trait compatibility, assumptions on body size and biomass (abundance) from fossil data</td>
<td>realised network</td>
<td><span class="citation" data-cites="petchey2008">Petchey et al. (<a href="#ref-petchey2008" role="doc-biblioref">2008</a>)</span></td>
<td></td>
</tr>
<tr class="even">
<td>Allometric trophic network (ATN)</td>
<td>Interactions inferred using allometric rules (ratio of body sizes between predator and prey), with links being constrained by a Ricker function</td>
<td>body mass, number of producer species</td>
<td>does not account for forbidden links in terms of trait compatibility, assumptions on body size from fossil data, assumptions as to the number of producer species</td>
<td>realised network</td>
<td><span class="citation" data-cites="brose2006">Brose et al. (<a href="#ref-brose2006" role="doc-biblioref">2006</a>)</span>; <span class="citation" data-cites="gauzens2023">Gauzens et al. (<a href="#ref-gauzens2023" role="doc-biblioref">2023</a>)</span></td>
<td></td>
</tr>
<tr class="odd">
<td>paleo food web inference model (PFIM)</td>
<td>Interactions can be inferred by a mechanistic framework/relationships</td>
<td>feeding traits for taxa, mechanistic feeding rules</td>
<td>Assumption made as to the feeding mechanisms, need to elucidate traits from models (although this is a way smaller issue)</td>
<td>feasibility web</td>
<td><span class="citation" data-cites="shaw2024">Shaw et al. (<a href="#ref-shaw2024" role="doc-biblioref">2024</a>)</span></td>
<td>Secondary extinctions <span class="citation" data-cites="dunhill2024">(<a href="#ref-dunhill2024" role="doc-biblioref">Dunhill et al. 2024</a>)</span></td>
</tr>
<tr class="even">
<td>body size ratio model</td>
<td>Interactions inferred using allometric rules (ratio of body sizes between predator and prey). Logit of the linking probability used to further constrain links to an ‘optimal size range’ for prey.</td>
<td>body mass</td>
<td>does not account for forbidden links in terms of evolutionary compatibility, assumptions on body size from fossil data</td>
<td>realised network</td>
<td><span class="citation" data-cites="rohr2010">Rohr et al. (<a href="#ref-rohr2010" role="doc-biblioref">2010</a>)</span></td>
<td>Network collapse <span class="citation" data-cites="yeakel2014">(<a href="#ref-yeakel2014" role="doc-biblioref">Yeakel et al. 2014</a>)</span></td>
</tr>
</tbody>
</table>
</div>
</figure>
</div>
</section>
<section id="network-generation-and-replication" class="level3" data-number="2.2.2">
<h3 data-number="2.2.2" class="anchored" data-anchor-id="network-generation-and-replication"><span class="header-section-number">2.2.2</span> Network generation and replication</h3>
<p>We evaluated six models spanning this space <a href="#tbl-models" class="quarto-xref">Table&nbsp;1</a>: random and niche models (structural network); allometric diet breadth (ADBM), allometric trophic network (ATN), and body-size ratio models (realised network); and a paleo food-web inference model (PFIM; feasibility web). More details on parameters can be found in the Supplementary Materials. For each of the four communities, we constructed 100 replicate networks using each of the six models (2400 networks total). Networks were simplified by removing disconnected species. For size-based models, body masses were drawn from uniform distributions bounded by size-class limits,allowing for variance between replicates but preserving relative sizes within replicates. Structural models were parameterised using connectance values drawn from an empirically realistic range (0.07 – 0.34) while holding richness constant. The same parameter draws were used across comparable models within each replicate.</p>
</section>
</section>
<section id="network-metrics-and-structural-analyses" class="level2" data-number="2.3">
<h2 data-number="2.3" class="anchored" data-anchor-id="network-metrics-and-structural-analyses"><span class="header-section-number">2.3</span> Network metrics and structural analyses</h2>
<p>We quantified network structure using a suite of macro-, meso-, and micro-scale metrics <a href="#tbl-properties" class="quarto-xref">Table&nbsp;2</a>, capturing global properties, motif structure, and species-level variability. Differences among models were assessed using MANOVA, followed by univariate ANOVAs, post-hoc comparisons, and linear discriminant analysis. Pairwise interaction turnover was quantified using link-based beta diversity for models that infer species-specific interactions <span class="citation" data-cites="poisot2012a">(<a href="#ref-poisot2012a" role="doc-biblioref">Poisot et al. 2012</a>)</span></p>
<div id="tbl-properties" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-tbl figure">
<figcaption class="quarto-float-caption-top quarto-float-caption quarto-float-tbl" id="tbl-properties-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Table&nbsp;2: Network properties used for analysis.
</figcaption>
<div aria-describedby="tbl-properties-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<table class="caption-top table">
<colgroup>
<col style="width: 24%">
<col style="width: 26%">
<col style="width: 24%">
<col style="width: 24%">
</colgroup>
<thead>
<tr class="header">
<th>Metric</th>
<th>Definition</th>
<th>Scale</th>
<th>Reference (for maths), can make footnotes probs</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Richness</td>
<td>Number of nodes in the network</td>
<td>Macro</td>
<td></td>
</tr>
<tr class="even">
<td>Links</td>
<td>Normalized standard deviation of links (number of consumers plus resources per taxon)</td>
<td>Micro</td>
<td></td>
</tr>
<tr class="odd">
<td>Connectance</td>
<td><span class="math inline">\(L/S^2\)</span>, where <span class="math inline">\(S\)</span> is the number of species and <span class="math inline">\(L\)</span> the number of links</td>
<td>Macro</td>
<td></td>
</tr>
<tr class="even">
<td>Max trophic level</td>
<td>Prey-weighted trophic level averaged across taxa</td>
<td>Macro</td>
<td><span class="citation" data-cites="williams2004">Williams and Martinez (<a href="#ref-williams2004" role="doc-biblioref">2004</a>)</span></td>
</tr>
<tr class="odd">
<td>S1</td>
<td>Number of linear chains, normalised</td>
<td>Meso</td>
<td><span class="citation" data-cites="milo2002">Milo et al. (<a href="#ref-milo2002" role="doc-biblioref">2002</a>)</span>; <span class="citation" data-cites="stouffer2007">Stouffer et al. (<a href="#ref-stouffer2007" role="doc-biblioref">2007</a>)</span></td>
</tr>
<tr class="even">
<td>S2</td>
<td>Number of omnivory motifs, normalised</td>
<td>Meso</td>
<td><span class="citation" data-cites="milo2002">Milo et al. (<a href="#ref-milo2002" role="doc-biblioref">2002</a>)</span>; <span class="citation" data-cites="stouffer2007">Stouffer et al. (<a href="#ref-stouffer2007" role="doc-biblioref">2007</a>)</span></td>
</tr>
<tr class="odd">
<td>S4</td>
<td>Number of apparent competition motifs, normalised</td>
<td>Meso</td>
<td><span class="citation" data-cites="milo2002">Milo et al. (<a href="#ref-milo2002" role="doc-biblioref">2002</a>)</span>; <span class="citation" data-cites="stouffer2007">Stouffer et al. (<a href="#ref-stouffer2007" role="doc-biblioref">2007</a>)</span></td>
</tr>
<tr class="even">
<td>S5</td>
<td>Number of direct competition motifs, normalised</td>
<td>Meso</td>
<td><span class="citation" data-cites="milo2002">Milo et al. (<a href="#ref-milo2002" role="doc-biblioref">2002</a>)</span>; <span class="citation" data-cites="stouffer2007">Stouffer et al. (<a href="#ref-stouffer2007" role="doc-biblioref">2007</a>)</span></td>
</tr>
<tr class="odd">
<td>Generality</td>
<td>Normalized standard deviation of generality of a species standardized by <span class="math inline">\(L/S\)</span></td>
<td>Micro</td>
<td><span class="citation" data-cites="williams2000">Williams and Martinez (<a href="#ref-williams2000" role="doc-biblioref">2000</a>)</span></td>
</tr>
<tr class="even">
<td>Vulnerability</td>
<td>Normalized standard deviation of vulnerability of a species standardized by <span class="math inline">\(L/S\)</span></td>
<td>Micro</td>
<td><span class="citation" data-cites="williams2000">Williams and Martinez (<a href="#ref-williams2000" role="doc-biblioref">2000</a>)</span></td>
</tr>
</tbody>
</table>
</div>
</figure>
</div>
</section>
<section id="extinction-simulations-and-model-evaluation" class="level2" data-number="2.4">
<h2 data-number="2.4" class="anchored" data-anchor-id="extinction-simulations-and-model-evaluation"><span class="header-section-number">2.4</span> Extinction simulations and model evaluation</h2>
<p>Following <span class="citation" data-cites="dunhill2024">Dunhill et al. (<a href="#ref-dunhill2024" role="doc-biblioref">2024</a>)</span> and using the pre-extinction and post-extinction networks, we simulated species loss under multiple extinction scenarios, including trait-based, network-position-based, and random removals, allowing for secondary extinctions. Simulated post-extinction networks were compared to empirical post-extinction communities using mean absolute differences (MAD) in network metrics and a modified true skill statistic (TSS) at both node and link levels. Scenario rankings were compared across models using Kendall’s rank correlation coefficient.</p>
</section>
</section>
<section id="results" class="level1" data-number="3">
<h1 data-number="3"><span class="header-section-number">3</span> Results</h1>
<p>Across six reconstruction approaches, both global network structure and species-level interactions differed substantially, with implications for interpreting past extinction dynamics. Deterministic models (e.g., PFIM) tended to produce more consistent network-level patterns and smoother extinction trajectories, whereas stochastic or theory-driven models (<em>e.g.,</em> ADBM, niche, ATN) showed greater variability in inferred interactions and temporal extinction dynamics. Models with similar macro-level metrics sometimes differed in their specification of pairwise interactions, highlighting that agreement in global structure does not guarantee concordance at the species level. Consequently, inferred extinction pathways and secondary extinctions were highly sensitive to model choice, emphasizing the importance of evaluating multiple network reconstructions when interpreting ecological dynamics in deep time.</p>
<section id="network-structure-differs-among-reconstruction-approaches" class="level2" data-number="3.1">
<h2 data-number="3.1" class="anchored" data-anchor-id="network-structure-differs-among-reconstruction-approaches"><span class="header-section-number">3.1</span> Network structure differs among reconstruction approaches</h2>
<p>To test whether network reconstruction approach influences inferred food-web structure, we compared multivariate patterns of network metrics across all six models using a MANOVA. Network structure differed strongly among reconstruction approaches (MANOVA, Pillai’s trace = 3.81, approximate <span class="math inline">\(F_{40,11955}\)</span> = 962.12, p &lt; 0.001). Univariate analyses showed that model choice explained a large proportion of variance in most network metrics, with high partial η² values for connectance, generality, vulnerability, and motif frequencies (η² = 0.66–0.92). In contrast, maximum trophic level exhibited a much smaller effect of model choice (η² = 0.04). Estimated marginal means and Tukey-adjusted comparisons indicated consistent differences among reconstruction approaches, with PFIM differing significantly from all other models (p ≤ 0.026), while ADBM and ATN did not differ from one another (p = 1.00). Linear discriminant analysis (LDA) further helped visualise distinctions among reconstruction approaches in multivariate network space <a href="#fig-structure" class="quarto-xref">Figure&nbsp;1</a>, with the first two axes explaining 86.5% of between-model variance (LD1 = 53.9%, LD2 = 32.6%). LD1 was most strongly correlated with vulnerability, connectance, and the number of direct competition motifs (S4), whereas LD2 was associated primarily with generality, number of omnivory motifs (S2), and number of apparent competition motifs (S5). Higher-order axes each explained less than 9% of the remaining variance. This demonstrates that the reconstruction approach leaves a strong multivariate signature independent of community composition.</p>
<div id="fig-structure" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-structure-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="figures/MANOVA_lda.png" class="lightbox" data-gallery="quarto-lightbox-gallery-1" title="Figure&nbsp;1: Linear discriminant analysis (LDA) of ecological network metrics for six model types. Each point represents a replicate, and ellipses indicate 95% confidence regions for each model. The second column represents the correlation of the various network metrics with the respective LDA axes."><img src="figures/MANOVA_lda.png" class="img-fluid figure-img"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-structure-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;1: Linear discriminant analysis (LDA) of ecological network metrics for six model types. Each point represents a replicate, and ellipses indicate 95% confidence regions for each model. The second column represents the correlation of the various network metrics with the respective LDA axes.
</figcaption>
</figure>
</div>
<section id="inferred-pairwise-interactions-vary-widely-among-models" class="level3" data-number="3.1.1">
<h3 data-number="3.1.1" class="anchored" data-anchor-id="inferred-pairwise-interactions-vary-widely-among-models"><span class="header-section-number">3.1.1</span> Inferred pairwise interactions vary widely among models</h3>
<p>Building on differences in global network structure, we next examined how reconstruction approach influences species-level ecological inference by quantifying turnover in inferred pairwise interactions among networks constructed from the same taxon pool. While models that produced similar global metrics sometimes agreed broadly on network structure, they often differed sharply in the specific interactions they inferred.</p>
<p>Pairwise β-turnover revealed that some model pairs shared very few links despite comparable macro- or meso-scale properties <a href="#fig-beta_div" class="quarto-xref">Figure&nbsp;2</a>. ADBM and ATN were highly concordant, reflecting similar underlying assumptions despite different generative rules, whereas the body-size ratio model consistently exhibited high differences in pairwise interactions relative to all other approaches. PFIM showed intermediate overlap with size-based theoretical models. These patterns indicate that agreement in global network metrics does not guarantee agreement in species-level diets or trophic roles, highlighting the importance of evaluating both network- and species-level outcomes when comparing reconstruction methods.</p>
<div id="fig-beta_div" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-beta_div-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="figures/beta_div.png" class="lightbox" data-gallery="quarto-lightbox-gallery-2" title="Figure&nbsp;2: Pairwise beta turnover in species interactions among four ecological network models (ADBM, lmatrix, body-size ratio, and pfim). Each cell represents the mean turnover value between a pair of models, with warmer colors indicating greater dissimilarity in inferred interactions. The diagonal is omitted. High turnover values (yellow) indicate strong disagreement in network structure between models, whereas lower values (blue–purple) indicate greater similarity."><img src="figures/beta_div.png" class="img-fluid figure-img"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-beta_div-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;2: Pairwise beta turnover in species interactions among four ecological network models (ADBM, lmatrix, body-size ratio, and pfim). Each cell represents the mean turnover value between a pair of models, with warmer colors indicating greater dissimilarity in inferred interactions. The diagonal is omitted. High turnover values (yellow) indicate strong disagreement in network structure between models, whereas lower values (blue–purple) indicate greater similarity.
</figcaption>
</figure>
</div>
</section>
</section>
<section id="model-choice-influences-inferred-extinction-dynamics" class="level2" data-number="3.2">
<h2 data-number="3.2" class="anchored" data-anchor-id="model-choice-influences-inferred-extinction-dynamics"><span class="header-section-number">3.2</span> Model choice influences inferred extinction dynamics</h2>
<p>To quantify how network structure changed over time during extinction simulations and whether these dynamics differed among reconstruction models, we fit generalized additive models (GAMs) to time series of network-level metrics. GAMs capture nonlinear temporal trajectories, allowing formal tests of whether the shape of these trajectories differs among models. These model-specific temporal trajectories are shown in <a href="#fig-gam" class="quarto-xref">Figure&nbsp;3</a>. For connectance, the model-specific smooth terms were all significant (<em>e.g.,</em> PFIM: EDF = 2.99, F = 113.8, p &lt; 0.001; ADBM: EDF = 2.92, F = 22.6, p &lt; 0.001), indicating distinct temporal patterns across reconstruction approaches. Trophic level trajectories also differed among models (PFIM: EDF = 2.96, F = 36.7, p &lt; 0.001; Niche: EDF = 2.96, F = 94.9, p &lt; 0.001). Deterministic approaches, such as PFIM, produced more consistent and smooth trajectories, whereas stochastic, theory-driven models (niche, ADBM, ATN) displayed greater variability in timing and magnitude of structural change. These results demonstrate that inferred pathways of collapse, trophic bottlenecks, and secondary extinctions are highly sensitive to model choice. Corresponding raw temporal trajectories are shown in Fig. S3.</p>
<div id="fig-gam" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-gam-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="figures/GAM_predictions.png" class="lightbox" data-gallery="quarto-lightbox-gallery-3" title="Figure&nbsp;3: GAM-predicted trajectories of network structure during extinction simulations reveal pronounced differences in the timing and magnitude of change across reconstruction models. Lines show model-specific smooths and shaded areas indicate 95% confidence intervals. Deterministic approaches produce smoother, more consistent dynamics, whereas stochastic models exhibit greater variability, underscoring the sensitivity of inferred collapse pathways to reconstruction assumptions."><img src="figures/GAM_predictions.png" class="img-fluid figure-img"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-gam-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;3: GAM-predicted trajectories of network structure during extinction simulations reveal pronounced differences in the timing and magnitude of change across reconstruction models. Lines show model-specific smooths and shaded areas indicate 95% confidence intervals. Deterministic approaches produce smoother, more consistent dynamics, whereas stochastic models exhibit greater variability, underscoring the sensitivity of inferred collapse pathways to reconstruction assumptions.
</figcaption>
</figure>
</div>
<p>To evaluate how model choice affects inferred extinction dynamics, we compared simulated post-extinction networks to observed networks using mean absolute differences (MAD) for network-level metrics and total sum-of-squares (TSS) for node- and link-level outcomes <a href="#fig-mad" class="quarto-xref">Figure&nbsp;4</a>. Overall, models were more consistent in ranking extinction scenarios at the network level: Kendall’s <span class="math inline">\(\tau\)</span> values for MAD-based rankings were generally positive, with strong agreement between ADBM and ATN models (<span class="math inline">\(\tau\)</span> ≈ 0.82) and weaker or inconsistent correlations involving stochastic models such as log ratio or random (<span class="math inline">\(\tau\)</span> ranging from −0.26 to 0.44). Node-level TSS scores similarly showed broad consistency across models, reflecting comparable species removal sequences (<span class="math inline">\(\tau\)</span> ≈ 0.38–0.46 between ADBM and niche or PFIM). In contrast, link-level outcomes were more variable: deterministic models like PFIM maintained moderate agreement with observed post-extinction links (<span class="math inline">\(\tau\)</span> ≈ 0.44), whereas stochastic, theory-driven models exhibited low or inconsistent correlations (<span class="math inline">\(\tau\)</span> as low as −0.44). These results indicate that while different models often recover similar species-level extinction patterns, inferred interaction loss and cascade dynamics are highly sensitive to model choice.</p>
<div id="fig-mad" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-mad-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="figures/kendal_tau.png" class="lightbox" data-gallery="quarto-lightbox-gallery-4" title="Figure&nbsp;4: Heatmaps showing pairwise Kendall rank correlation coefficients (\tau) between models for each network metric. Each panel corresponds to a different metric and displays the degree of agreement in extinction-scenario rankings across models based on mean absolute differences (MAD) between observed and predicted network values. Positive \tau values (blue) indicate concordant rankings between models, whereas negative \tau values (red) indicate opposing rankings. Warmer colours approaching zero represent little or no agreement. Panels illustrate how consistently different modelling approaches evaluate the relative realism of extinction scenarios across multiple network properties."><img src="figures/kendal_tau.png" class="img-fluid figure-img"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-mad-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;4: Heatmaps showing pairwise Kendall rank correlation coefficients (<span class="math inline">\(\tau\)</span>) between models for each network metric. Each panel corresponds to a different metric and displays the degree of agreement in extinction-scenario rankings across models based on mean absolute differences (MAD) between observed and predicted network values. Positive <span class="math inline">\(\tau\)</span> values (blue) indicate concordant rankings between models, whereas negative <span class="math inline">\(\tau\)</span> values (red) indicate opposing rankings. Warmer colours approaching zero represent little or no agreement. Panels illustrate how consistently different modelling approaches evaluate the relative realism of extinction scenarios across multiple network properties.
</figcaption>
</figure>
</div>
</section>
</section>
<section id="discussion" class="level1" data-number="4">
<h1 data-number="4"><span class="header-section-number">4</span> Discussion</h1>
</section>
<section id="model-choice-as-a-component-of-ecological-inference" class="level1" data-number="5">
<h1 data-number="5"><span class="header-section-number">5</span> Model choice as a component of ecological inference</h1>
<p>Reconstructing food webs from fossil data is inherently an exercise in inference under uncertainty, where ecological conclusions depend as much on modelling assumptions as on preserved evidence <span class="citation" data-cites="dunne2008 morales-castilla2015">(<a href="#ref-dunne2008" role="doc-biblioref">Dunne et al. 2008</a>; <a href="#ref-morales-castilla2015" role="doc-biblioref">Morales-Castilla et al. 2015</a>)</span>. Our results demonstrate that the choice of network reconstruction model is itself a major ecological decision, shaping not only the structural properties of inferred networks but also downstream interpretations of extinction dynamics <span class="citation" data-cites="allesina2012 solé2001">(<a href="#ref-allesina2012" role="doc-biblioref">Allesina and Tang 2012</a>; <a href="#ref-solé2001" role="doc-biblioref">Solé and Montoya 2001</a>)</span>. Even with identical taxon pools, different models yield networks that diverge in global metrics, species-level interactions, and temporal patterns of collapse. These divergences do not stem from differences in fossil evidence or taxon composition, but from the assumptions embedded within each reconstruction approach <span class="citation" data-cites="pichler2023 strydom2021">(<a href="#ref-pichler2023" role="doc-biblioref">Pichler and Hartig 2023</a>; <a href="#ref-strydom2021" role="doc-biblioref">Strydom et al. 2021</a>)</span>. Consequently, network reconstruction is not a neutral methodological step; model choice shapes the ecological narratives we extract from ancient ecosystems.</p>
<p>While previous studies have emphasized the role of model assumptions in metaweb reconstruction <span class="citation" data-cites="roopnarine2006 dunhill2024">(<a href="#ref-roopnarine2006" role="doc-biblioref">Roopnarine 2006</a>; <a href="#ref-dunhill2024" role="doc-biblioref">Dunhill et al. 2024</a>)</span>, our results demonstrate that these assumptions create distinct, predictable clusters of network properties. These clusters map directly onto the conceptual divide between feasible, realised, and structural network types <span class="citation" data-cites="strydom2026">(<a href="#ref-strydom2026" role="doc-biblioref">Strydom et al. 2026</a>)</span>. Specifically, mechanistic models (PFIM) identify a broad landscape of trait-compatible interactions, theoretical models (ADBM, ATN) impose energetic filters to approximate realised energy flow, and structural models (niche, random) prioritise topological patterns over species identity. Our Pairwise β-turnover analysis confirms that these differences are not merely quantitative; models fundamentally disagree on link identity, reinforcing concerns that traditional metawebs may significantly overestimate realised interaction diversity <span class="citation" data-cites="fricke2022a shaw2024">(<a href="#ref-fricke2022a" role="doc-biblioref">Fricke et al. 2022</a>; <a href="#ref-shaw2024" role="doc-biblioref">Shaw et al. 2024</a>)</span>. This distinction is central to paleoecological inference, where feasible interactions are often mistaken for realised trophic relationships.</p>
<p>The downstream consequences of model choice are most pronounced when evaluating extinction dynamics <span class="citation" data-cites="dunne2002 sahasrabudhe2011">(<a href="#ref-dunne2002" role="doc-biblioref">Dunne et al. 2002</a>; <a href="#ref-sahasrabudhe2011" role="doc-biblioref">Sahasrabudhe and Motter 2011</a>)</span>. While broad, trait-driven patterns of species loss are relatively robust across models, the identity of lost interactions, secondary extinctions, and cascade dynamics are sensitive to the type of network reconstructed. Node-level patterns tend to be more consistent across models because they primarily reflect intrinsic species traits, whereas link-level outcomes are emergent properties of model assumptions. This distinction mirrors findings in modern food-web studies, where deterministic and stochastic model assumptions influence the magnitude and timing of secondary extinctions <span class="citation" data-cites="dunne2002 yeakel2014 allesina2012 curtsdotter2011">(<a href="#ref-dunne2002" role="doc-biblioref">Dunne et al. 2002</a>; <a href="#ref-yeakel2014" role="doc-biblioref">Yeakel et al. 2014</a>; <a href="#ref-allesina2012" role="doc-biblioref">Allesina and Tang 2012</a>; <a href="#ref-curtsdotter2011" role="doc-biblioref">Curtsdotter et al. 2011</a>)</span>. Our results suggest that interpretations of trophic collapse, bottlenecks, and recovery trajectories are shaped not only by the fossil record but also by the choice of network model. Consequently, paleoecologists must carefully consider which ecological signals they aim to recover (potential interactions, realized diets, or macro-scale structural properties) before selecting a reconstruction approach. Importantly, disagreement among models does not imply that any single approach is ‘wrong’, but rather reflects the fact that different models capture different ecological signals <span class="citation" data-cites="stouffer2019">(<a href="#ref-stouffer2019" role="doc-biblioref">Stouffer 2019</a>)</span>. The challenge therefore lies not in identifying a universally correct model, but rather in aligning model choice with the ecological question being asked.</p>
<section id="aligning-ecological-questions-with-model-choice" class="level2" data-number="5.1">
<h2 data-number="5.1" class="anchored" data-anchor-id="aligning-ecological-questions-with-model-choice"><span class="header-section-number">5.1</span> Aligning ecological questions with model choice</h2>
<p>A central insight from our study is that different ecological questions require different network representations. Feasibility networks are most appropriate for evaluating potential trophic links or dietary breadth; realised networks capture energetically constrained, ecologically plausible interactions; and structural networks are suited for exploring generic topological patterns such as connectance or trophic depth. Link-level dynamics and cascade sensitivity are particularly sensitive to network choice, whereas higher-level patterns of species loss are less so.</p>
<p>This alignment between question and model type reflects a broader trend in network ecology emphasizing the role of assumptions in determining which inferences are valid <span class="citation" data-cites="delmas2019 petchey2008 schneider2016">(<a href="#ref-delmas2019" role="doc-biblioref">Delmas et al. 2019</a>; <a href="#ref-petchey2008" role="doc-biblioref">Petchey et al. 2008</a>; <a href="#ref-schneider2016" role="doc-biblioref">Schneider et al. 2016</a>)</span>. By explicitly matching network representation to research question, paleoecologists can avoid overinterpreting model-dependent artifacts as biological signals.</p>
<p>Below is a conceptual summary:</p>
<table class="caption-top table">
<colgroup>
<col style="width: 33%">
<col style="width: 33%">
<col style="width: 33%">
</colgroup>
<thead>
<tr class="header">
<th><strong>Ecological question</strong></th>
<th><strong>Recommended network type</strong></th>
<th><strong>Rationale</strong></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>What <em>could</em> have eaten what? (feasible interactions; trophic potential)</td>
<td>Feasibility web (PFIM, trait-based, evolutionary-signal models)</td>
<td>Captures all trait-feasible links; appropriate for reconstructing dietary breadth or assessing potential niches.</td>
</tr>
<tr class="even">
<td>What interactions were <em>likely realised</em>?</td>
<td>Realised network (ADBM, ATN)</td>
<td>Encodes foraging ecology and energetic constraints; provides ecologically plausible ‘realised’ networks.</td>
</tr>
<tr class="odd">
<td>How do global macro structural properties vary?</td>
<td>Structural network (niche, random as baseline)</td>
<td>Useful for understanding generic patterns (connectance, trophic depth) without trait reliance.</td>
</tr>
<tr class="even">
<td>How do traits influence dynamics (e.g., cascade sensitivity)?</td>
<td>Realised network</td>
<td>Capture body-size scaling, energetic constraints, and trophic dependencies.</td>
</tr>
<tr class="odd">
<td>How robust are inferences to uncertainty?</td>
<td>Ensembles across multiple network types</td>
<td>Allows quantifying how conclusions depend on assumptions.</td>
</tr>
</tbody>
</table>
</section>
<section id="implications-for-paleoecological-network-studies" class="level2" data-number="5.2">
<h2 data-number="5.2" class="anchored" data-anchor-id="implications-for-paleoecological-network-studies"><span class="header-section-number">5.2</span> Implications for paleoecological network studies</h2>
<p>Our analysis underscores three important implications for the broader field:</p>
<ol type="1">
<li><p><strong>Interpretations of ancient food webs must explicitly acknowledge model assumptions:</strong> Without this, differences in reconstructed networks may be mistakenly attributed to biological change rather than methodological artifacts.</p></li>
<li><p><strong>Comparisons across studies must be standardised by model family:</strong> Comparing a metaweb from one study to a realised network from another risks generating misleading conclusions about ecological differences across space or time.</p></li>
<li><p><strong>Integrating theory-driven models expands inference:</strong> Mechanistic paleo-specific models alone capture only a subset of feasible networks. Incorporating modern theory-based approaches allows researchers to explore alternative network archetypes, extending the range of questions that can be addressed <span class="citation" data-cites="dunne2014 solé2001">(<a href="#ref-dunne2014" role="doc-biblioref">Dunne et al. 2014</a>; <a href="#ref-solé2001" role="doc-biblioref">Solé and Montoya 2001</a>)</span>.</p></li>
</ol>
</section>
<section id="recommendations-for-network-reconstruction-in-paleoecology" class="level2" data-number="5.3">
<h2 data-number="5.3" class="anchored" data-anchor-id="recommendations-for-network-reconstruction-in-paleoecology"><span class="header-section-number">5.3</span> Recommendations for network reconstruction in paleoecology</h2>
<p>To support consistent, transparent use of network reconstruction methods in paleoecology, we propose the following guidelines :</p>
<ol type="1">
<li><p><strong>Establish the Inferential Goal First:</strong>&nbsp;Explicitly define whether the study aims to capture potential interactions (feasibility), energetic flows (realised), or global patterns (structural). This choice dictates the appropriate model family and prevents the common error of treating metawebs as local diet realisations.</p></li>
<li><p><strong>Implement an Ensemble Modelling Framework:</strong> Rather than just using multiple models, advocate for an ensemble approach. Evaluating a suite of models allows researchers to quantify model uncertainty identifying which ecological conclusions are robust biological signals and which are methodological artifacts.</p></li>
<li><p><strong>Standardise Comparative Analyses:</strong> To avoid misleading comparisons across studies, researchers should only compare networks constructed within the same model family. If comparing a new metaweb to a published realised web, the differences must be explicitly attributed to the representation gap rather than environmental change.</p></li>
<li><p><strong>Prioritize Scale-Specific Interpretation:</strong> Distinguish clearly between node- and link-level results. Since node-level patterns (who goes extinct) are generally more robust to model choice, they can be interpreted with higher confidence than specific link-level pathways (the exact cascade route), which are highly sensitive to model assumptions.</p></li>
</ol>
</section>
</section>
<section id="conclusions" class="level1" data-number="6">
<h1 data-number="6"><span class="header-section-number">6</span> Conclusions</h1>
<p>Ecological network reconstruction in deep time is not merely a technical step but a fundamental component of ecological inference. By explicitly comparing six contrasting reconstruction approaches within a single extinction event and location we show that model choice strongly shapes inferred food-web structure, species interactions, and extinction dynamics, even when underlying fossil data are identical. While broad, trait-based patterns of species loss appear relatively robust, conclusions about pairwise interactions, secondary extinctions, and cascading dynamics depend critically on the assumptions embedded in the chosen network reconstruction approach. These results underscore the need for paleoecological studies to align reconstruction methods with specific ecological questions and to evaluate the sensitivity of key conclusions to alternative network representations. More broadly, our findings highlight that understanding past ecosystem collapse requires not only better fossil data, but also transparent, question-driven modelling frameworks that make explicit the assumptions underlying ecological inference.</p>
</section>
<section id="references" class="level1 unnumbered">


</section>

<div id="quarto-appendix" class="default"><section class="quarto-appendix-contents" role="doc-bibliography" id="quarto-bibliography"><h2 class="anchored quarto-appendix-heading">References</h2><div id="refs" class="references csl-bib-body hanging-indent" role="list">
<div id="ref-allesina2008" class="csl-entry" role="listitem">
Allesina, Stefano, David Alonso, and Mercedes Pascual. 2008. <span>“A General Model for Food Web Structure.”</span> <em>Science</em> 320 (5876): 658–61. <a href="https://doi.org/10.1126/science.1156269">https://doi.org/10.1126/science.1156269</a>.
</div>
<div id="ref-allesina2012" class="csl-entry" role="listitem">
Allesina, Stefano, and Si Tang. 2012. <span>“Stability Criteria for Complex Ecosystems.”</span> <em>Nature</em> 483 (7388): 205–8. <a href="https://doi.org/10.1038/nature10832">https://doi.org/10.1038/nature10832</a>.
</div>
<div id="ref-barnosky2012" class="csl-entry" role="listitem">
Barnosky, Anthony D., Elizabeth A. Hadly, Jordi Bascompte, et al. 2012. <span>“Approaching a State Shift in Earth<span>’</span>s Biosphere.”</span> <em>Nature</em> 486 (7401): 52–58. <a href="https://doi.org/10.1038/nature11018">https://doi.org/10.1038/nature11018</a>.
</div>
<div id="ref-brose2006" class="csl-entry" role="listitem">
Brose, Ulrich, Tomas Jonsson, Eric L. Berlow, et al. 2006. <span>“Consumer<span></span>resource Body-Size Relationships in Natural Food Webs.”</span> <em>Ecology</em> 87 (10): 2411–17. https://doi.org/<a href="https://doi.org/10.1890/0012-9658(2006)87[2411:CBRINF]2.0.CO;2">https://doi.org/10.1890/0012-9658(2006)87[2411:CBRINF]2.0.CO;2</a>.
</div>
<div id="ref-curtsdotter2011" class="csl-entry" role="listitem">
Curtsdotter, Alva, Amrei Binzer, Ulrich Brose, et al. 2011. <span>“Robustness to Secondary Extinctions: Comparing Trait-Based Sequential Deletions in Static and Dynamic Food Webs.”</span> <em>Basic and Applied Ecology</em> 12 (7): 571–80. <a href="https://doi.org/10.1016/j.baae.2011.09.008">https://doi.org/10.1016/j.baae.2011.09.008</a>.
</div>
<div id="ref-delmas2019" class="csl-entry" role="listitem">
Delmas, Eva, Mathilde Besson, Marie-Hélène Brice, et al. 2019. <span>“Analysing Ecological Networks of Species Interactions.”</span> <em>Biological Reviews</em> 94 (1): 16–36. <a href="https://doi.org/10.1111/brv.12433">https://doi.org/10.1111/brv.12433</a>.
</div>
<div id="ref-dillon2022" class="csl-entry" role="listitem">
Dillon, Erin M., Jaleigh Q. Pier, Jansen A. Smith, et al. 2022. <span>“What Is Conservation Paleobiology? Tracking 20 Years of Research and Development.”</span> <em>Frontiers in Ecology and Evolution</em> 10 (December). <a href="https://doi.org/10.3389/fevo.2022.1031483">https://doi.org/10.3389/fevo.2022.1031483</a>.
</div>
<div id="ref-dunhill2024" class="csl-entry" role="listitem">
Dunhill, Alexander M., Karolina Zarzyczny, Jack O. Shaw, Jed W. Atkinson, Crispin T. S. Little, and Andrew P. Beckerman. 2024. <span>“Extinction Cascades, Community Collapse, and Recovery Across a Mesozoic Hyperthermal Event.”</span> <em>Nature Communications</em> 15 (1): 8599. <a href="https://doi.org/10.1038/s41467-024-53000-2">https://doi.org/10.1038/s41467-024-53000-2</a>.
</div>
<div id="ref-dunne2014" class="csl-entry" role="listitem">
Dunne, Jennifer A., Conrad C. Labandeira, and Richard J. Williams. 2014. <span>“Highly Resolved Early Eocene Food Webs Show Development of Modern Trophic Structure After the End-Cretaceous Extinction.”</span> <em>Proceedings of the Royal Society B: Biological Sciences</em> 281 (1782): 20133280. <a href="https://doi.org/10.1098/rspb.2013.3280">https://doi.org/10.1098/rspb.2013.3280</a>.
</div>
<div id="ref-dunne2008" class="csl-entry" role="listitem">
Dunne, Jennifer A., Richard J. Williams, Neo D. Martinez, Rachel A. Wood, and Douglas H. Erwin. 2008. <span>“Compilation and Network Analyses of Cambrian Food Webs.”</span> <em>PLOS Biology</em> 6 (4): e102. <a href="https://doi.org/10.1371/journal.pbio.0060102">https://doi.org/10.1371/journal.pbio.0060102</a>.
</div>
<div id="ref-dunne2002" class="csl-entry" role="listitem">
Dunne, Jennifer, Richard J Williams, and Néo D Martinez. 2002. <span>“Network Structure and Biodiversity Loss in Food Webs: Robustness Increases with Connectance.”</span> <em>Ecol. Lett.</em> 5 (4): 558–67.
</div>
<div id="ref-erdos1959" class="csl-entry" role="listitem">
Erdős, P., and A. Rényi. 1959. <span>“On Random Graphs. i.”</span> <em>Publicationes Mathematicae Debrecen</em> 6 (3-4): 290–97. <a href="https://doi.org/10.5486/pmd.1959.6.3-4.12">https://doi.org/10.5486/pmd.1959.6.3-4.12</a>.
</div>
<div id="ref-fricke2022a" class="csl-entry" role="listitem">
Fricke, Evan C., Chia Hsieh, Owen Middleton, et al. 2022. <span>“Collapse of Terrestrial Mammal Food Webs Since the Late Pleistocene.”</span> <em>Science</em> 377 (6609): 1008–11. <a href="https://doi.org/10.1126/science.abn4012">https://doi.org/10.1126/science.abn4012</a>.
</div>
<div id="ref-gauzens2023" class="csl-entry" role="listitem">
Gauzens, Benoit, Ulrich Brose, Eva Delmas, and Emilio Berti. 2023. <span>“ATNr: Allometric Trophic Network Models in r.”</span> <em>Methods in Ecology and Evolution</em> 14 (11): 2766–73. <a href="https://doi.org/10.1111/2041-210X.14212">https://doi.org/10.1111/2041-210X.14212</a>.
</div>
<div id="ref-gravel2019" class="csl-entry" role="listitem">
Gravel, Dominique, Benjamin Baiser, Jennifer A. Dunne, et al. 2019. <span>“Bringing Elton and Grinnell Together: A Quantitative Framework to Represent the Biogeography of Ecological Interaction Networks.”</span> <em>Ecography</em> 42 (3): 401–15. https://doi.org/<a href="https://doi.org/10.1111/ecog.04006">https://doi.org/10.1111/ecog.04006</a>.
</div>
<div id="ref-jenny2019" class="csl-entry" role="listitem">
Jenny, Dominique, Dirk Fuchs, Alexander I. Arkhipkin, Rolf B. Hauff, Barbara Fritschi, and Christian Klug. 2019. <span>“Predatory Behaviour and Taphonomy of a Jurassic Belemnoid Coleoid (Diplobelida, Cephalopoda).”</span> <em>Scientific Reports</em> 9 (1): 7944. <a href="https://doi.org/10.1038/s41598-019-44260-w">https://doi.org/10.1038/s41598-019-44260-w</a>.
</div>
<div id="ref-kiessling2019" class="csl-entry" role="listitem">
Kiessling, Wolfgang, Nussaïbah B. Raja, Vanessa Julie Roden, Samuel T. Turvey, and Erin E. Saupe. 2019. <span>“Addressing Priority Questions of Conservation Science with Palaeontological Data.”</span> <em>Philosophical Transactions of the Royal Society B: Biological Sciences</em> 374 (1788): 20190222. <a href="https://doi.org/10.1098/rstb.2019.0222">https://doi.org/10.1098/rstb.2019.0222</a>.
</div>
<div id="ref-milo2002" class="csl-entry" role="listitem">
Milo, R., S. Shen-Orr, S. Itzkovitz, N. Kashtan, D. Chklovskii, and U. Alon. 2002. <span>“Network Motifs: Simple Building Blocks of Complex Networks.”</span> <em>Science</em> 298 (5594): 824–27. <a href="https://doi.org/10.1126/science.298.5594.824">https://doi.org/10.1126/science.298.5594.824</a>.
</div>
<div id="ref-morales-castilla2015" class="csl-entry" role="listitem">
Morales-Castilla, Ignacio, Miguel G. Matias, Dominique Gravel, and Miguel B. Araújo. 2015. <span>“Inferring Biotic Interactions from Proxies.”</span> <em>Trends in Ecology &amp; Evolution</em> 30 (6): 347–56. <a href="https://doi.org/10.1016/j.tree.2015.03.014">https://doi.org/10.1016/j.tree.2015.03.014</a>.
</div>
<div id="ref-petchey2008" class="csl-entry" role="listitem">
Petchey, Owen L., Andrew P. Beckerman, Jens O. Riede, and Philip H. Warren. 2008. <span>“Size, Foraging, and Food Web Structure.”</span> <em>Proceedings of the National Academy of Sciences</em> 105 (11): 4191–96. <a href="https://doi.org/10.1073/pnas.0710672105">https://doi.org/10.1073/pnas.0710672105</a>.
</div>
<div id="ref-pichler2023" class="csl-entry" role="listitem">
Pichler, Maximilian, and Florian Hartig. 2023. <span>“Machine Learning and Deep Learning<span></span>a Review for Ecologists.”</span> <em>Methods in Ecology and Evolution</em> 14 (4): 994–1016. <a href="https://doi.org/10.1111/2041-210X.14061">https://doi.org/10.1111/2041-210X.14061</a>.
</div>
<div id="ref-poisot2012a" class="csl-entry" role="listitem">
Poisot, Timothée, Elsa Canard, David Mouillot, Nicolas Mouquet, and Dominique Gravel. 2012. <span>“The Dissimilarity of Species Interaction Networks.”</span> <em>Ecology Letters</em> 15 (12): 1353–61. <a href="https://doi.org/10.1111/ele.12002">https://doi.org/10.1111/ele.12002</a>.
</div>
<div id="ref-poisot2015" class="csl-entry" role="listitem">
Poisot, Timothée, Daniel B. Stouffer, and Dominique Gravel. 2015. <span>“Beyond Species: Why Ecological Interaction Networks Vary Through Space and Time.”</span> <em>Oikos</em> 124 (3): 243–51. <a href="https://doi.org/10.1111/oik.01719">https://doi.org/10.1111/oik.01719</a>.
</div>
<div id="ref-rohr2010" class="csl-entry" role="listitem">
Rohr, Rudolf&nbsp;Philippe, Heike Scherer, Patrik Kehrli, Christian Mazza, and Louis-Félix Bersier. 2010. <span>“Modeling Food Webs: Exploring Unexplained Structure Using Latent Traits.”</span> <em>The American Naturalist</em> 176 (2): 170–77. <a href="https://doi.org/10.1086/653667">https://doi.org/10.1086/653667</a>.
</div>
<div id="ref-roopnarine2017" class="csl-entry" role="listitem">
Roopnarine, Peter D. 2017. <em>Ecological Modelling of Paleocommunity Food Webs</em>. University of Chicago Press.
</div>
<div id="ref-roopnarine2006" class="csl-entry" role="listitem">
Roopnarine, Peter D. 2006. <span>“Extinction Cascades and Catastrophe in Ancient Food Webs.”</span> <em>Paleobiology</em> 32 (1): 1–19. <a href="https://www.jstor.org/stable/4096814">https://www.jstor.org/stable/4096814</a>.
</div>
<div id="ref-roopnarine2018" class="csl-entry" role="listitem">
Roopnarine, Peter D., and Ashley A. Dineen. 2018. <em>Coral Reefs in Crisis: The Reliability of Deep-Time Food Web Reconstructions as Analogs for the Present</em>. Edited by Carrie L. Tyler and Chris L. Schneider. Springer International Publishing. <a href="https://doi.org/10.1007/978-3-319-73795-9_6">https://doi.org/10.1007/978-3-319-73795-9_6</a>.
</div>
<div id="ref-sahasrabudhe2011" class="csl-entry" role="listitem">
Sahasrabudhe, Sagar, and Adilson E. Motter. 2011. <span>“Rescuing Ecosystems from Extinction Cascades Through Compensatory Perturbations.”</span> <em>Nature Communications</em> 2 (1): 170. <a href="https://doi.org/10.1038/ncomms1163">https://doi.org/10.1038/ncomms1163</a>.
</div>
<div id="ref-schneider2016" class="csl-entry" role="listitem">
Schneider, Florian D., Ulrich Brose, Björn C. Rall, and Christian Guill. 2016. <span>“Animal Diversity and Ecosystem Functioning in Dynamic Food Webs.”</span> <em>Nature Communications</em> 7 (1): 12718. <a href="https://doi.org/10.1038/ncomms12718">https://doi.org/10.1038/ncomms12718</a>.
</div>
<div id="ref-shaw2024" class="csl-entry" role="listitem">
Shaw, Jack O., Alexander M. Dunhill, Andrew P. Beckerman, Jennifer A. Dunne, and Pincelli M. Hull. 2024. <em>A Framework for Reconstructing Ancient Food Webs Using Functional Trait Data</em>. bioRxiv. <a href="https://doi.org/10.1101/2024.01.30.578036">https://doi.org/10.1101/2024.01.30.578036</a>.
</div>
<div id="ref-solé2001" class="csl-entry" role="listitem">
Solé, Ricard V., and M. Montoya. 2001. <span>“Complexity and Fragility in Ecological Networks.”</span> <em>Proceedings of the Royal Society of London. Series B: Biological Sciences</em> 268 (1480): 2039–45. <a href="https://doi.org/10.1098/rspb.2001.1767">https://doi.org/10.1098/rspb.2001.1767</a>.
</div>
<div id="ref-stouffer2005" class="csl-entry" role="listitem">
Stouffer, D. B., J. Camacho, R. Guimerà, C. A. Ng, and L. A. Nunes Amaral. 2005. <span>“Quantitative Patterns in the Structure of Model and Empirical Food Webs.”</span> <em>Ecology</em> 86 (5): 1301–11. <a href="https://doi.org/10.1890/04-0957">https://doi.org/10.1890/04-0957</a>.
</div>
<div id="ref-stouffer2019" class="csl-entry" role="listitem">
Stouffer, Daniel B. 2019. <span>“All Ecological Models Are Wrong, but Some Are Useful.”</span> <em>Journal of Animal Ecology</em> 88 (2): 192–95. https://doi.org/<a href="https://doi.org/10.1111/1365-2656.12949">https://doi.org/10.1111/1365-2656.12949</a>.
</div>
<div id="ref-stouffer2007" class="csl-entry" role="listitem">
Stouffer, Daniel B, Juan Camacho, Wenxin Jiang, and Luís A Nunes Amaral. 2007. <span>“Evidence for the Existence of a Robust Pattern of Prey Selection in Food Webs.”</span> <em>Proceedings of the Royal Society B: Biological Sciences</em> 274 (1621): 1931–40. <a href="https://doi.org/10.1098/rspb.2007.0571">https://doi.org/10.1098/rspb.2007.0571</a>.
</div>
<div id="ref-strydom2021" class="csl-entry" role="listitem">
Strydom, Tanya, Michael D. Catchen, Francis Banville, et al. 2021. <span>“A Roadmap Towards Predicting Species Interaction Networks (Across Space and Time).”</span> <em>Philosophical Transactions of the Royal Society B: Biological Sciences</em> 376 (1837): 20210063. <a href="https://doi.org/10.1098/rstb.2021.0063">https://doi.org/10.1098/rstb.2021.0063</a>.
</div>
<div id="ref-strydom2026" class="csl-entry" role="listitem">
Strydom, Tanya, Alexander M. Dunhill, Jennifer A. Dunne, Timothée Poisot, and Andrew P. Beckerman. 2026. <span>“Scaling from Metawebs to Realised Webs: A Hierarchical Approach to Network Ecology.”</span> <em>EcoEvoRxiv</em>, ahead of print, January 21. <a href="https://doi.org/10.32942/X2JW8K">https://doi.org/10.32942/X2JW8K</a>.
</div>
<div id="ref-vullo2011" class="csl-entry" role="listitem">
Vullo, Romain. 2011. <span>“Direct Evidence of Hybodont Shark Predation on Late Jurassic Ammonites.”</span> <em>Naturwissenschaften</em> 98 (6): 545–49. <a href="https://doi.org/10.1007/s00114-011-0789-9">https://doi.org/10.1007/s00114-011-0789-9</a>.
</div>
<div id="ref-williams2004" class="csl-entry" role="listitem">
Williams, R. J., and N. D. Martinez. 2004. <span>“Stabilization of Chaotic and Non-Permanent Food-Web Dynamics.”</span> <em>The European Physical Journal B - Condensed Matter</em> 38 (2): 297–303. <a href="https://doi.org/10.1140/epjb/e2004-00122-1">https://doi.org/10.1140/epjb/e2004-00122-1</a>.
</div>
<div id="ref-williams2000" class="csl-entry" role="listitem">
Williams, Richard J., and Neo D. Martinez. 2000. <span>“Simple Rules Yield Complex Food Webs.”</span> <em>Nature</em> 404 (6774): 180–83. <a href="https://doi.org/10.1038/35004572">https://doi.org/10.1038/35004572</a>.
</div>
<div id="ref-williams2008" class="csl-entry" role="listitem">
Williams, Richard J., and Neo D. Martinez. 2008. <span>“Success and its limits among structural models of complex food webs.”</span> <em>The Journal of Animal Ecology</em> 77 (3): 512–19. <a href="https://doi.org/10.1111/j.1365-2656.2008.01362.x">https://doi.org/10.1111/j.1365-2656.2008.01362.x</a>.
</div>
<div id="ref-yeakel2014" class="csl-entry" role="listitem">
Yeakel, Justin D., Mathias M. Pires, Lars Rudolf, et al. 2014. <span>“Collapse of an Ecological Network in Ancient Egypt.”</span> <em>PNAS</em> 111 (40): 14472–77. <a href="https://doi.org/10.1073/pnas.1408471111">https://doi.org/10.1073/pnas.1408471111</a>.
</div>
</div></section><section class="quarto-appendix-contents" id="quarto-citation"><h2 class="anchored quarto-appendix-heading">Citation</h2><div><div class="quarto-appendix-secondary-label">BibTeX citation:</div><pre class="sourceCode code-with-copy quarto-appendix-bibtex"><code class="sourceCode bibtex">@online{strydom2026,
  author = {Strydom, Tanya and Karapunar, Baran and P. Beckerman, Andrew
    and Dunhill, Alexander},
  title = {Reconstructing Food Webs in Deep Time: Why Model Choice
    Matters for Ecological Inference},
  date = {2026-02-03},
  langid = {en},
  abstract = {Food webs provide a powerful framework for understanding
    ecosystem structure and function, yet reconstructing them in
    paleoecological contexts remains challenging because direct evidence
    of species interactions is rarely preserved. A growing array of
    models exists for predicting interactions and inferring network
    structure, but these approaches differ markedly in their
    assumptions, mechanisms, and data requirements. Here, we evaluate
    how network reconstruction model choice shapes ecological inference
    in deep time and which approaches are most appropriate given the
    constraints of the fossil record. Using the Toarcian Oceanic Anoxic
    Event (Early Jurassic, \textasciitilde183 Ma) as a case study, we
    reconstruct food webs for four successive community states using six
    contrasting modelling approaches spanning mechanistic, trait-based,
    and structural network representations. Despite identical taxon
    pools, models produced strikingly different network structures and
    species-level interactions, and these differences propagated into
    divergent inferences about extinction dynamics, interaction loss,
    and cascading effects. By framing food-web reconstructions as
    alternative ecological hypotheses (rather than interchangeable
    representations) this study bridges paleoecology and modern network
    theory, and demonstrates how model choice fundamentally conditions
    inference about extinction dynamics in deep time.}
}
</code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre><div class="quarto-appendix-secondary-label">For attribution, please cite this work as:</div><div id="ref-strydom2026" class="csl-entry quarto-appendix-citeas" role="listitem">
Strydom, Tanya, Baran Karapunar, Andrew P. Beckerman, and Alexander
Dunhill. 2026. <span>“Reconstructing Food Webs in Deep Time: Why Model
Choice Matters for Ecological Inference.”</span> TBD, February 3.
</div></div></section></div></main>
<!-- /main column -->
<script id="quarto-html-after-body" type="application/javascript">
  window.document.addEventListener("DOMContentLoaded", function (event) {
    const icon = "";
    const anchorJS = new window.AnchorJS();
    anchorJS.options = {
      placement: 'right',
      icon: icon
    };
    anchorJS.add('.anchored');
    const isCodeAnnotation = (el) => {
      for (const clz of el.classList) {
        if (clz.startsWith('code-annotation-')) {                     
          return true;
        }
      }
      return false;
    }
    const onCopySuccess = function(e) {
      // button target
      const button = e.trigger;
      // don't keep focus
      button.blur();
      // flash "checked"
      button.classList.add('code-copy-button-checked');
      var currentTitle = button.getAttribute("title");
      button.setAttribute("title", "Copied!");
      let tooltip;
      if (window.bootstrap) {
        button.setAttribute("data-bs-toggle", "tooltip");
        button.setAttribute("data-bs-placement", "left");
        button.setAttribute("data-bs-title", "Copied!");
        tooltip = new bootstrap.Tooltip(button, 
          { trigger: "manual", 
            customClass: "code-copy-button-tooltip",
            offset: [0, -8]});
        tooltip.show();    
      }
      setTimeout(function() {
        if (tooltip) {
          tooltip.hide();
          button.removeAttribute("data-bs-title");
          button.removeAttribute("data-bs-toggle");
          button.removeAttribute("data-bs-placement");
        }
        button.setAttribute("title", currentTitle);
        button.classList.remove('code-copy-button-checked');
      }, 1000);
      // clear code selection
      e.clearSelection();
    }
    const getTextToCopy = function(trigger) {
      const outerScaffold = trigger.parentElement.cloneNode(true);
      const codeEl = outerScaffold.querySelector('code');
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
    }
    const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
      text: getTextToCopy
    });
    clipboard.on('success', onCopySuccess);
    if (window.document.getElementById('quarto-embedded-source-code-modal')) {
      const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
        text: getTextToCopy,
        container: window.document.getElementById('quarto-embedded-source-code-modal')
      });
      clipboardModal.on('success', onCopySuccess);
    }
      var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
      var mailtoRegex = new RegExp(/^mailto:/);
        var filterRegex = new RegExp('/' + window.location.host + '/');
      var isInternal = (href) => {
          return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
      }
      // Inspect non-navigation links and adorn them if external
     var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
      for (var i=0; i<links.length; i++) {
        const link = links[i];
        if (!isInternal(link.href)) {
          // undo the damage that might have been done by quarto-nav.js in the case of
          // links that we want to consider external
          if (link.dataset.originalHref !== undefined) {
            link.href = link.dataset.originalHref;
          }
        }
      }
    function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
      const config = {
        allowHTML: true,
        maxWidth: 500,
        delay: 100,
        arrow: false,
        appendTo: function(el) {
            return el.parentElement;
        },
        interactive: true,
        interactiveBorder: 10,
        theme: 'quarto',
        placement: 'bottom-start',
      };
      if (contentFn) {
        config.content = contentFn;
      }
      if (onTriggerFn) {
        config.onTrigger = onTriggerFn;
      }
      if (onUntriggerFn) {
        config.onUntrigger = onUntriggerFn;
      }
      window.tippy(el, config); 
    }
    const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
    for (var i=0; i<noterefs.length; i++) {
      const ref = noterefs[i];
      tippyHover(ref, function() {
        // use id or data attribute instead here
        let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
        try { href = new URL(href).hash; } catch {}
        const id = href.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note) {
          return note.innerHTML;
        } else {
          return "";
        }
      });
    }
    const xrefs = window.document.querySelectorAll('a.quarto-xref');
    const processXRef = (id, note) => {
      // Strip column container classes
      const stripColumnClz = (el) => {
        el.classList.remove("page-full", "page-columns");
        if (el.children) {
          for (const child of el.children) {
            stripColumnClz(child);
          }
        }
      }
      stripColumnClz(note)
      if (id === null || id.startsWith('sec-')) {
        // Special case sections, only their first couple elements
        const container = document.createElement("div");
        if (note.children && note.children.length > 2) {
          container.appendChild(note.children[0].cloneNode(true));
          for (let i = 1; i < note.children.length; i++) {
            const child = note.children[i];
            if (child.tagName === "P" && child.innerText === "") {
              continue;
            } else {
              container.appendChild(child.cloneNode(true));
              break;
            }
          }
          if (window.Quarto?.typesetMath) {
            window.Quarto.typesetMath(container);
          }
          return container.innerHTML
        } else {
          if (window.Quarto?.typesetMath) {
            window.Quarto.typesetMath(note);
          }
          return note.innerHTML;
        }
      } else {
        // Remove any anchor links if they are present
        const anchorLink = note.querySelector('a.anchorjs-link');
        if (anchorLink) {
          anchorLink.remove();
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        if (note.classList.contains("callout")) {
          return note.outerHTML;
        } else {
          return note.innerHTML;
        }
      }
    }
    for (var i=0; i<xrefs.length; i++) {
      const xref = xrefs[i];
      tippyHover(xref, undefined, function(instance) {
        instance.disable();
        let url = xref.getAttribute('href');
        let hash = undefined; 
        if (url.startsWith('#')) {
          hash = url;
        } else {
          try { hash = new URL(url).hash; } catch {}
        }
        if (hash) {
          const id = hash.replace(/^#\/?/, "");
          const note = window.document.getElementById(id);
          if (note !== null) {
            try {
              const html = processXRef(id, note.cloneNode(true));
              instance.setContent(html);
            } finally {
              instance.enable();
              instance.show();
            }
          } else {
            // See if we can fetch this
            fetch(url.split('#')[0])
            .then(res => res.text())
            .then(html => {
              const parser = new DOMParser();
              const htmlDoc = parser.parseFromString(html, "text/html");
              const note = htmlDoc.getElementById(id);
              if (note !== null) {
                const html = processXRef(id, note);
                instance.setContent(html);
              } 
            }).finally(() => {
              instance.enable();
              instance.show();
            });
          }
        } else {
          // See if we can fetch a full url (with no hash to target)
          // This is a special case and we should probably do some content thinning / targeting
          fetch(url)
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.querySelector('main.content');
            if (note !== null) {
              // This should only happen for chapter cross references
              // (since there is no id in the URL)
              // remove the first header
              if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
                note.children[0].remove();
              }
              const html = processXRef(null, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      }, function(instance) {
      });
    }
        let selectedAnnoteEl;
        const selectorForAnnotation = ( cell, annotation) => {
          let cellAttr = 'data-code-cell="' + cell + '"';
          let lineAttr = 'data-code-annotation="' +  annotation + '"';
          const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
          return selector;
        }
        const selectCodeLines = (annoteEl) => {
          const doc = window.document;
          const targetCell = annoteEl.getAttribute("data-target-cell");
          const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
          const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
          const lines = annoteSpan.getAttribute("data-code-lines").split(",");
          const lineIds = lines.map((line) => {
            return targetCell + "-" + line;
          })
          let top = null;
          let height = null;
          let parent = null;
          if (lineIds.length > 0) {
              //compute the position of the single el (top and bottom and make a div)
              const el = window.document.getElementById(lineIds[0]);
              top = el.offsetTop;
              height = el.offsetHeight;
              parent = el.parentElement.parentElement;
            if (lineIds.length > 1) {
              const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
              const bottom = lastEl.offsetTop + lastEl.offsetHeight;
              height = bottom - top;
            }
            if (top !== null && height !== null && parent !== null) {
              // cook up a div (if necessary) and position it 
              let div = window.document.getElementById("code-annotation-line-highlight");
              if (div === null) {
                div = window.document.createElement("div");
                div.setAttribute("id", "code-annotation-line-highlight");
                div.style.position = 'absolute';
                parent.appendChild(div);
              }
              div.style.top = top - 2 + "px";
              div.style.height = height + 4 + "px";
              div.style.left = 0;
              let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
              if (gutterDiv === null) {
                gutterDiv = window.document.createElement("div");
                gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
                gutterDiv.style.position = 'absolute';
                const codeCell = window.document.getElementById(targetCell);
                const gutter = codeCell.querySelector('.code-annotation-gutter');
                gutter.appendChild(gutterDiv);
              }
              gutterDiv.style.top = top - 2 + "px";
              gutterDiv.style.height = height + 4 + "px";
            }
            selectedAnnoteEl = annoteEl;
          }
        };
        const unselectCodeLines = () => {
          const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
          elementsIds.forEach((elId) => {
            const div = window.document.getElementById(elId);
            if (div) {
              div.remove();
            }
          });
          selectedAnnoteEl = undefined;
        };
          // Handle positioning of the toggle
      window.addEventListener(
        "resize",
        throttle(() => {
          elRect = undefined;
          if (selectedAnnoteEl) {
            selectCodeLines(selectedAnnoteEl);
          }
        }, 10)
      );
      function throttle(fn, ms) {
      let throttle = false;
      let timer;
        return (...args) => {
          if(!throttle) { // first call gets through
              fn.apply(this, args);
              throttle = true;
          } else { // all the others get throttled
              if(timer) clearTimeout(timer); // cancel #2
              timer = setTimeout(() => {
                fn.apply(this, args);
                timer = throttle = false;
              }, ms);
          }
        };
      }
        // Attach click handler to the DT
        const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
        for (const annoteDlNode of annoteDls) {
          annoteDlNode.addEventListener('click', (event) => {
            const clickedEl = event.target;
            if (clickedEl !== selectedAnnoteEl) {
              unselectCodeLines();
              const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
              if (activeEl) {
                activeEl.classList.remove('code-annotation-active');
              }
              selectCodeLines(clickedEl);
              clickedEl.classList.add('code-annotation-active');
            } else {
              // Unselect the line
              unselectCodeLines();
              clickedEl.classList.remove('code-annotation-active');
            }
          });
        }
    const findCites = (el) => {
      const parentEl = el.parentElement;
      if (parentEl) {
        const cites = parentEl.dataset.cites;
        if (cites) {
          return {
            el,
            cites: cites.split(' ')
          };
        } else {
          return findCites(el.parentElement)
        }
      } else {
        return undefined;
      }
    };
    var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
    for (var i=0; i<bibliorefs.length; i++) {
      const ref = bibliorefs[i];
      const citeInfo = findCites(ref);
      if (citeInfo) {
        tippyHover(citeInfo.el, function() {
          var popup = window.document.createElement('div');
          citeInfo.cites.forEach(function(cite) {
            var citeDiv = window.document.createElement('div');
            citeDiv.classList.add('hanging-indent');
            citeDiv.classList.add('csl-entry');
            var biblioDiv = window.document.getElementById('ref-' + cite);
            if (biblioDiv) {
              citeDiv.innerHTML = biblioDiv.innerHTML;
            }
            popup.appendChild(citeDiv);
          });
          return popup.innerHTML;
        });
      }
    }
  });
  </script>
</div> <!-- /content -->
<script>var lightboxQuarto = GLightbox({"closeEffect":"zoom","descPosition":"bottom","loop":false,"openEffect":"zoom","selector":".lightbox"});
(function() {
  let previousOnload = window.onload;
  window.onload = () => {
    if (previousOnload) {
      previousOnload();
    }
    lightboxQuarto.on('slide_before_load', (data) => {
      const { slideIndex, slideNode, slideConfig, player, trigger } = data;
      const href = trigger.getAttribute('href');
      if (href !== null) {
        const imgEl = window.document.querySelector(`a[href="${href}"] img`);
        if (imgEl !== null) {
          const srcAttr = imgEl.getAttribute("src");
          if (srcAttr && srcAttr.startsWith("data:")) {
            slideConfig.href = srcAttr;
          }
        }
      } 
    });
  
    lightboxQuarto.on('slide_after_load', (data) => {
      const { slideIndex, slideNode, slideConfig, player, trigger } = data;
      if (window.Quarto?.typesetMath) {
        window.Quarto.typesetMath(slideNode);
      }
    });
  
  };
  
})();
          </script>




</body></html>