<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.9.12">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">

<meta name="author" content="Tanya Strydom">
<meta name="author" content="Baran Karapunar">
<meta name="author" content="Andrew P. Beckerman">
<meta name="author" content="Alexander Dunhill">
<meta name="dcterms.date" content="2025-12-09">
<meta name="keywords" content="food web, network construction">

<title>Methods for Reconstructing Paleo Food Webs</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
/* CSS for citations */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
  margin-bottom: 0em;
}
.hanging-indent div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}</style>


<script src="site_libs/clipboard/clipboard.min.js"></script>
<script src="site_libs/quarto-html/quarto.js" type="module"></script>
<script src="site_libs/quarto-html/tabsets/tabsets.js" type="module"></script>
<script src="site_libs/quarto-html/axe/axe-check.js" type="module"></script>
<script src="site_libs/quarto-html/popper.min.js"></script>
<script src="site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="site_libs/quarto-html/anchor.min.js"></script>
<link href="site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="site_libs/quarto-html/quarto-syntax-highlighting-cdaacfc258cb6f151192107f105ac881.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="site_libs/bootstrap/bootstrap.min.js"></script>
<link href="site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="site_libs/bootstrap/bootstrap-0a61af393cc41664e89a7addda21b354.min.css" rel="stylesheet" append-hash="true" id="quarto-bootstrap" data-mode="light">
<script src="site_libs/quarto-contrib/glightbox/glightbox.min.js"></script>
<link href="site_libs/quarto-contrib/glightbox/glightbox.min.css" rel="stylesheet">
<link href="site_libs/quarto-contrib/glightbox/lightbox.css" rel="stylesheet">
<script async="" src="https://hypothes.is/embed.js"></script>
<script>
  window.document.addEventListener("DOMContentLoaded", function (_event) {
    document.body.classList.add('hypothesis-enabled');
  });
</script>

  <script src="https://cdnjs.cloudflare.com/polyfill/v3/polyfill.min.js?features=es6"></script>
  <script src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-chtml-full.js" type="text/javascript"></script>

<script type="text/javascript">
const typesetMath = (el) => {
  if (window.MathJax) {
    // MathJax Typeset
    window.MathJax.typeset([el]);
  } else if (window.katex) {
    // KaTeX Render
    var mathElements = el.getElementsByClassName("math");
    var macros = [];
    for (var i = 0; i < mathElements.length; i++) {
      var texText = mathElements[i].firstChild;
      if (mathElements[i].tagName == "SPAN" && texText && texText.data) {
        window.katex.render(texText.data, mathElements[i], {
          displayMode: mathElements[i].classList.contains('display'),
          throwOnError: false,
          macros: macros,
          fleqn: false
        });
      }
    }
  }
}
window.Quarto = {
  typesetMath
};
</script>

<meta name="citation_title" content="Methods for Reconstructing Paleo Food Webs">
<meta name="citation_abstract" content="Food webs provide a powerful lens for understanding ecosystem structure and function, but reconstructing them in paleoecological contexts remains challenging because direct evidence of feeding interactions is rarely preserved. A wide range of models now exist for predicting interactions and inferring network structure, yet these models differ substantially in their assumptions, mechanisms, and data requirements. Here, we evaluate which network construction approaches are most suitable for paleo food webs given the constraints of the fossil record, and we assess how model choice influences the networks we infer. Using the Toarcian Oceanic Anoxic Event as a case study, we compare six modelling approaches encompassing mechanistic, structural, and theory-based methods. We show that the models produce strikingly different network structures and pairwise interactions, and that these differences propagate into ecological inference—including conclusions about extinction dynamics. Our results highlight the importance of aligning model choice with research questions and underscore the interpretative risks of treating all food-web reconstruction methods as interchangeable.
">
<meta name="citation_keywords" content="food web,network construction">
<meta name="citation_author" content="Tanya Strydom">
<meta name="citation_author" content="Baran Karapunar">
<meta name="citation_author" content="Andrew P. Beckerman">
<meta name="citation_author" content="Alexander Dunhill">
<meta name="citation_publication_date" content="2025-12-09">
<meta name="citation_cover_date" content="2025-12-09">
<meta name="citation_year" content="2025">
<meta name="citation_online_date" content="2025-12-09">
<meta name="citation_language" content="en">
<meta name="citation_journal_title" content="TBD">
<meta name="citation_reference" content="citation_title=Extinction cascades, community collapse, and recovery across a Mesozoic hyperthermal event;,citation_author=Alexander M. Dunhill;,citation_author=Karolina Zarzyczny;,citation_author=Jack O. Shaw;,citation_author=Jed W. Atkinson;,citation_author=Crispin T. S. Little;,citation_author=Andrew P. Beckerman;,citation_publication_date=2024-10-04;,citation_cover_date=2024-10-04;,citation_year=2024;,citation_fulltext_html_url=https://www.nature.com/articles/s41467-024-53000-2;,citation_issue=1;,citation_doi=10.1038/s41467-024-53000-2;,citation_volume=15;,citation_language=en;,citation_journal_title=Nature Communications;">
<meta name="citation_reference" content="citation_title=Shortcomings of reusing species interaction networks created by different sets of researchers;,citation_author=Chris Brimacombe;,citation_author=Korryn Bodner;,citation_author=Matthew Michalska-Smith;,citation_author=Timothée Poisot;,citation_author=Marie-Josée Fortin;,citation_publication_date=2023-03;,citation_cover_date=2023-03;,citation_year=2023;,citation_fulltext_html_url=https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.3002068;,citation_issue=4;,citation_doi=10.1371/journal.pbio.3002068;,citation_volume=21;,citation_language=en;,citation_journal_title=PLOS Biology;">
<meta name="citation_reference" content="citation_title=A framework for reconstructing ancient food webs using functional trait data;,citation_abstract=Food webs provide quantitative insights into the structure and dynamics of ecological communities. Previous work has shown their utility in understanding community responses to modern and ancient perturbations, including anthropogenic change and mass extinctions. However, few ancient food webs have been reconstructed due to difficulties assessing trophic interactions amongst extinct species derived from an incomplete fossil record. We present and assess the Paleo Food web Inference Model (PFIM). PFIM uses functional trait data–predictive of interactions in modern ecosystems and commonly available for fossil organisms–to reconstruct ancient food webs. We test the model by (i) applying it to four modern ecosystems with empirical constrained food webs to directly compare PFIM-constructed networks to their empirical counterparts, (ii) by carefully comparing discrepancies between PFIM-inferred and empirical webs in one of those systems, and (iii) by comparing networks describing feasible trophic interactions (&amp;amp;amp;quot;feasible webs&amp;quot;) with networks to which we superimpose characteristic interaction distributions derived from modern theory (&quot;realized webs&quot;). As a proof of concept, we then apply the method to faunal data from two Cambrian fossil deposits to reconstruct ancient trophic systems. PFIM-inferred feasible food webs successfully predict ~70% of trophic interactions across four modern systems. Furthermore, inferred food webs with enforced interaction distributions (i.e., realized webs) accurately predict ~90% of interactions. Comparisons with a global database of trophic interactions and other food web models, suggest that under sampling of empirical webs accounts for up to 21% of the remaining differences between PFIM and empirical food webs. Food webs can be reasonably approximated by inferring trophic interactions based upon life habit traits. This study provides the foundation to use trait-based inference models across the fossil record to examine ancient food webs and community evolution.;,citation_author=Jack O. Shaw;,citation_author=Alexander M. Dunhill;,citation_author=Andrew P. Beckerman;,citation_author=Jennifer A. Dunne;,citation_author=Pincelli M. Hull;,citation_publication_date=2024-01;,citation_cover_date=2024-01;,citation_year=2024;,citation_doi=10.1101/2024.01.30.578036;,citation_language=en-US;,citation_publisher=bioRxiv;">
<meta name="citation_reference" content="citation_title=Extinction cascades and catastrophe in ancient food webs;,citation_author=Peter D. Roopnarine;,citation_publication_date=2006;,citation_cover_date=2006;,citation_year=2006;,citation_fulltext_html_url=https://www.jstor.org/stable/4096814;,citation_issue=1;,citation_volume=32;,citation_journal_title=Paleobiology;">
<meta name="citation_reference" content="citation_title=Size, foraging, and food web structure;,citation_author=Owen L. Petchey;,citation_author=Andrew P. Beckerman;,citation_author=Jens O. Riede;,citation_author=Philip H. Warren;,citation_publication_date=2008-03-18;,citation_cover_date=2008-03-18;,citation_year=2008;,citation_fulltext_html_url=https://pnas.org/doi/full/10.1073/pnas.0710672105;,citation_issue=11;,citation_doi=10.1073/pnas.0710672105;,citation_volume=105;,citation_language=en;,citation_journal_title=Proceedings of the National Academy of Sciences;">
<meta name="citation_reference" content="citation_title=Modeling food webs: Exploring unexplained structure using latent traits.;,citation_author=Rudolf&nbsp;Philippe Rohr;,citation_author=Heike Scherer;,citation_author=Patrik Kehrli;,citation_author=Christian Mazza;,citation_author=Louis-Félix Bersier;,citation_publication_date=2010-08;,citation_cover_date=2010-08;,citation_year=2010;,citation_fulltext_html_url=https://www.journals.uchicago.edu/doi/10.1086/653667;,citation_issue=2;,citation_doi=10.1086/653667;,citation_volume=176;,citation_journal_title=The American Naturalist;">
<meta name="citation_reference" content="citation_title=Collapse of an ecological network in ancient egypt;,citation_author=Justin D. Yeakel;,citation_author=Mathias M. Pires;,citation_author=Lars Rudolf;,citation_author=Nathaniel J. Dominy;,citation_author=Paul L. Koch;,citation_author=Paulo R. Guimarães;,citation_author=Thilo Gross;,citation_publication_date=2014-10;,citation_cover_date=2014-10;,citation_year=2014;,citation_issue=40;,citation_doi=10.1073/pnas.1408471111;,citation_volume=111;,citation_journal_title=PNAS;">
<meta name="citation_reference" content="citation_title=Simple rules yield complex food webs;,citation_author=Richard J. Williams;,citation_author=Neo D. Martinez;,citation_publication_date=2000-03;,citation_cover_date=2000-03;,citation_year=2000;,citation_fulltext_html_url=https://www.nature.com/articles/35004572;,citation_issue=6774;,citation_doi=10.1038/35004572;,citation_volume=404;,citation_language=en;,citation_journal_title=Nature;">
<meta name="citation_reference" content="citation_title=Success and its limits among structural models of complex food webs;,citation_author=Richard J. Williams;,citation_author=Neo D. Martinez;,citation_publication_date=2008-05;,citation_cover_date=2008-05;,citation_year=2008;,citation_issue=3;,citation_doi=10.1111/j.1365-2656.2008.01362.x;,citation_volume=77;,citation_language=eng;,citation_journal_title=The Journal of Animal Ecology;">
<meta name="citation_reference" content="citation_title=Graph embedding and transfer learning can help predict potential species interaction networks despite data limitations;,citation_author=Tanya Strydom;,citation_author=Salomé Bouskila;,citation_author=Francis Banville;,citation_author=Ceres Barros;,citation_author=Dominique Caron;,citation_author=Maxwell J. Farrell;,citation_author=Marie-Josée Fortin;,citation_author=Benjamin Mercier;,citation_author=Laura J. Pollock;,citation_author=Rogini Runghen;,citation_author=Giulio V. Dalla Riva;,citation_author=Timothée Poisot;,citation_publication_date=2023;,citation_cover_date=2023;,citation_year=2023;,citation_fulltext_html_url=https://onlinelibrary.wiley.com/doi/abs/10.1111/2041-210X.14228;,citation_issue=12;,citation_doi=10.1111/2041-210X.14228;,citation_volume=14;,citation_language=en;,citation_journal_title=Methods in Ecology and Evolution;">
<meta name="citation_reference" content="citation_title=Addressing the Eltonian shortfall with trait-based interaction models;,citation_author=Dominique Caron;,citation_author=Luigi Maiorano;,citation_author=Wilfried Thuiller;,citation_author=Laura J. Pollock;,citation_publication_date=2022;,citation_cover_date=2022;,citation_year=2022;,citation_fulltext_html_url=https://onlinelibrary.wiley.com/doi/abs/10.1111/ele.13966;,citation_issue=4;,citation_doi=10.1111/ele.13966;,citation_volume=25;,citation_language=en;,citation_journal_title=Ecology Letters;">
<meta name="citation_reference" content="citation_title=Collapse of terrestrial mammal food webs since the Late Pleistocene;,citation_author=Evan C. Fricke;,citation_author=Chia Hsieh;,citation_author=Owen Middleton;,citation_author=Daniel Gorczynski;,citation_author=Caroline D. Cappello;,citation_author=Oscar Sanisidro;,citation_author=John Rowan;,citation_author=Jens-Christian Svenning;,citation_author=Lydia Beaudrot;,citation_publication_date=2022-08-26;,citation_cover_date=2022-08-26;,citation_year=2022;,citation_fulltext_html_url=http://dx.doi.org/10.1126/science.abn4012;,citation_issue=6609;,citation_doi=10.1126/science.abn4012;,citation_volume=377;,citation_language=en;,citation_journal_title=Science;">
<meta name="citation_reference" content="citation_title=Ecological Modelling of Paleocommunity Food Webs;,citation_author=Peter D Roopnarine;,citation_publication_date=2017;,citation_cover_date=2017;,citation_year=2017;,citation_language=en;,citation_inbook_title=undefined;">
<meta name="citation_reference" content="citation_title=Autecology and the Filling of Ecospace: Key Metazoan Radiations;,citation_author=Richard K. Bambach;,citation_author=Andrew M. Bush;,citation_author=Douglas H. Erwin;,citation_publication_date=2007;,citation_cover_date=2007;,citation_year=2007;,citation_fulltext_html_url=https://onlinelibrary.wiley.com/doi/abs/10.1111/j.1475-4983.2006.00611.x;,citation_issue=1;,citation_doi=10.1111/j.1475-4983.2006.00611.x;,citation_volume=50;,citation_language=en;,citation_journal_title=Palaeontology;">
<meta name="citation_reference" content="citation_title=Inferring predator-prey interactions in food webs;,citation_author=Justin PF Pomeranz;,citation_author=Ross M. Thompson;,citation_author=Timothée Poisot;,citation_author=Jon S. Harding;,citation_publication_date=2018;,citation_cover_date=2018;,citation_year=2018;,citation_fulltext_html_url=https://besjournals.onlinelibrary.wiley.com/doi/abs/10.1111/2041-210X.13125;,citation_issue=ja;,citation_doi=10.1111/2041-210X.13125;,citation_volume=0;,citation_language=en;,citation_journal_title=Methods in Ecology and Evolution;">
<meta name="citation_reference" content="citation_title=SVD entropy reveals the high complexity of ecological networks;,citation_author=Tanya Strydom;,citation_author=Giulio V. Dalla Riva;,citation_author=Timothée Poisot;,citation_publication_date=2021;,citation_cover_date=2021;,citation_year=2021;,citation_fulltext_html_url=https://www.frontiersin.org/articles/10.3389/fevo.2021.623141/full;,citation_doi=10.3389/fevo.2021.623141;,citation_volume=9;,citation_language=en-US;,citation_journal_title=Frontiers in Ecology and Evolution;">
<meta name="citation_reference" content="citation_title=Network motifs: Simple building blocks of complex networks;,citation_author=R. Milo;,citation_author=S. Shen-Orr;,citation_author=S. Itzkovitz;,citation_author=N. Kashtan;,citation_author=D. Chklovskii;,citation_author=U. Alon;,citation_publication_date=2002-10-25;,citation_cover_date=2002-10-25;,citation_year=2002;,citation_fulltext_html_url=https://www.science.org/doi/10.1126/science.298.5594.824;,citation_issue=5594;,citation_doi=10.1126/science.298.5594.824;,citation_volume=298;,citation_journal_title=Science;">
<meta name="citation_reference" content="citation_title=Evidence for the existence of a robust pattern of prey selection in food webs;,citation_author=Daniel B Stouffer;,citation_author=Juan Camacho;,citation_author=Wenxin Jiang;,citation_author=Luís A Nunes Amaral;,citation_publication_date=2007-06;,citation_cover_date=2007-06;,citation_year=2007;,citation_issue=1621;,citation_doi=10.1098/rspb.2007.0571;,citation_volume=274;,citation_journal_title=Proceedings of the Royal Society B: Biological Sciences;">
<meta name="citation_reference" content="citation_title=Success and its limits among structural models of complex food webs;,citation_author=Richard J. Williams;,citation_author=Neo D. Martinez;,citation_publication_date=2008-05;,citation_cover_date=2008-05;,citation_year=2008;,citation_issue=3;,citation_doi=10.1111/j.1365-2656.2008.01362.x;,citation_volume=77;,citation_language=eng;,citation_journal_title=The Journal of Animal Ecology;">
<meta name="citation_reference" content="citation_title=Size, foraging, and food web structure;,citation_author=Owen L. Petchey;,citation_author=Andrew P. Beckerman;,citation_author=Jens O. Riede;,citation_author=Philip H. Warren;,citation_publication_date=2008-03-18;,citation_cover_date=2008-03-18;,citation_year=2008;,citation_fulltext_html_url=https://pnas.org/doi/full/10.1073/pnas.0710672105;,citation_issue=11;,citation_doi=10.1073/pnas.0710672105;,citation_volume=105;,citation_language=en;,citation_journal_title=Proceedings of the National Academy of Sciences;">
<meta name="citation_reference" content="citation_title=Constant connectance in community food webs;,citation_author=Neo D. Martinez;,citation_publication_date=1992;,citation_cover_date=1992;,citation_year=1992;,citation_fulltext_html_url=http://www.jstor.org/stable/2462337;,citation_issue=6;,citation_volume=139;,citation_journal_title=The American Naturalist;">
<meta name="citation_reference" content="citation_title=Analysing ecological networks of species interactions;,citation_author=Eva Delmas;,citation_author=Mathilde Besson;,citation_author=Marie-Hélène Brice;,citation_author=Laura A. Burkle;,citation_author=Giulio V. Dalla Riva;,citation_author=Marie-Josée Fortin;,citation_author=Dominique Gravel;,citation_author=Paulo R. Guimarães;,citation_author=David H. Hembry;,citation_author=Erica A. Newman;,citation_author=Jens M. Olesen;,citation_author=Mathias M. Pires;,citation_author=Justin D. Yeakel;,citation_author=Timothée Poisot;,citation_publication_date=2018-06-20;,citation_cover_date=2018-06-20;,citation_year=2018;,citation_fulltext_html_url=http://doi.wiley.com/10.1111/brv.12433;,citation_doi=10.1111/brv.12433;,citation_language=en;,citation_journal_title=Biological Reviews;">
<meta name="citation_reference" content="citation_title=A general model for food web structure;,citation_author=Stefano Allesina;,citation_author=David Alonso;,citation_author=Mercedes Pascual;,citation_publication_date=2008-05-02;,citation_cover_date=2008-05-02;,citation_year=2008;,citation_fulltext_html_url=https://www.science.org/doi/full/10.1126/science.1156269;,citation_issue=5876;,citation_doi=10.1126/science.1156269;,citation_volume=320;,citation_journal_title=Science;">
<meta name="citation_reference" content="citation_title=A roadmap towards predicting species interaction networks (across space and time);,citation_author=Tanya Strydom;,citation_author=Michael D. Catchen;,citation_author=Francis Banville;,citation_author=Dominique Caron;,citation_author=Gabriel Dansereau;,citation_author=Philippe Desjardins-Proulx;,citation_author=Norma R. Forero-Muñoz;,citation_author=Gracielle Higino;,citation_author=Benjamin Mercier;,citation_author=Andrew Gonzalez;,citation_author=Dominique Gravel;,citation_author=Laura Pollock;,citation_author=Timothée Poisot;,citation_publication_date=2021-11;,citation_cover_date=2021-11;,citation_year=2021;,citation_issue=1837;,citation_doi=10.1098/rstb.2021.0063;,citation_volume=376;,citation_journal_title=Philosophical Transactions of the Royal Society B: Biological Sciences;,citation_publisher=Royal Society;">
<meta name="citation_reference" content="citation_title=A comparative study of ecological specialization estimators;,citation_author=Timothée Poisot;,citation_author=Elsa Canard;,citation_author=Nicolas Mouquet;,citation_author=Michael E. Hochberg;,citation_publication_date=2012-01-19;,citation_cover_date=2012-01-19;,citation_year=2012;,citation_fulltext_html_url=http://dx.doi.org/10.1111/j.2041-210X.2011.00174.x;,citation_issue=3;,citation_doi=10.1111/j.2041-210x.2011.00174.x;,citation_volume=3;,citation_language=en;,citation_journal_title=Methods in Ecology and Evolution;">
<meta name="citation_reference" content="citation_title=Food Webs From the Small to the Large: The Robert H. MacArthur Award Lecture;,citation_author=Thomas W. Schoener;,citation_publication_date=1989-12;,citation_cover_date=1989-12;,citation_year=1989;,citation_fulltext_html_url=http://dx.doi.org/10.2307/1938088;,citation_issue=6;,citation_doi=10.2307/1938088;,citation_volume=70;,citation_language=en;,citation_journal_title=Ecology;">
<meta name="citation_reference" content="citation_title=ATNr: Allometric Trophic Network models in R;,citation_author=Benoit Gauzens;,citation_author=Ulrich Brose;,citation_author=Eva Delmas;,citation_author=Emilio Berti;,citation_publication_date=2023;,citation_cover_date=2023;,citation_year=2023;,citation_fulltext_html_url=https://onlinelibrary.wiley.com/doi/abs/10.1111/2041-210X.14212;,citation_issue=11;,citation_doi=10.1111/2041-210X.14212;,citation_volume=14;,citation_language=en;,citation_journal_title=Methods in Ecology and Evolution;">
<meta name="citation_reference" content="citation_title=Animal diversity and ecosystem functioning in dynamic food webs;,citation_author=Florian D. Schneider;,citation_author=Ulrich Brose;,citation_author=Björn C. Rall;,citation_author=Christian Guill;,citation_publication_date=2016-10-05;,citation_cover_date=2016-10-05;,citation_year=2016;,citation_fulltext_html_url=https://www.nature.com/articles/ncomms12718;,citation_issue=1;,citation_doi=10.1038/ncomms12718;,citation_volume=7;,citation_language=en;,citation_journal_title=Nature Communications;">
<meta name="citation_reference" content="citation_title=The reliability of R50 as a measure of vulnerability of food webs to sequential species deletions;,citation_author=Tomas Jonsson;,citation_author=Sofia Berg;,citation_author=Alexander Pimenov;,citation_author=Catherine Palmer;,citation_author=Mark Emmerson;,citation_publication_date=2015;,citation_cover_date=2015;,citation_year=2015;,citation_fulltext_html_url=https://onlinelibrary.wiley.com/doi/abs/10.1111/oik.01588;,citation_issue=4;,citation_doi=10.1111/oik.01588;,citation_volume=124;,citation_language=en;,citation_journal_title=Oikos;">
<meta name="citation_reference" content="citation_title=On random graphs. i.;,citation_author=P. Erdős;,citation_author=A. Rényi;,citation_publication_date=1959-07-01;,citation_cover_date=1959-07-01;,citation_year=1959;,citation_fulltext_html_url=http://dx.doi.org/10.5486/PMD.1959.6.3-4.12;,citation_issue=3-4;,citation_doi=10.5486/pmd.1959.6.3-4.12;,citation_volume=6;,citation_journal_title=Publicationes Mathematicae Debrecen;">
<meta name="citation_reference" content="citation_title=The ghost of nestedness in ecological networks;,citation_author=Phillip P. A. Staniczenko;,citation_author=Jason C. Kopp;,citation_author=Stefano Allesina;,citation_publication_date=2013-06;,citation_cover_date=2013-06;,citation_year=2013;,citation_fulltext_html_url=http://www.nature.com/articles/ncomms2422;,citation_issue=1;,citation_doi=10.1038/ncomms2422;,citation_volume=4;,citation_language=en;,citation_journal_title=Nature Communications;">
<meta name="citation_reference" content="citation_title=Stabilization of chaotic and non-permanent food-web dynamics;,citation_author=R. J. Williams;,citation_author=N. D. Martinez;,citation_publication_date=2004-03-01;,citation_cover_date=2004-03-01;,citation_year=2004;,citation_fulltext_html_url=http://link.springer.com/10.1140/epjb/e2004-00122-1;,citation_issue=2;,citation_doi=10.1140/epjb/e2004-00122-1;,citation_volume=38;,citation_language=en;,citation_journal_title=The European Physical Journal B - Condensed Matter;">
<meta name="citation_reference" content="citation_title=Simultaneously estimating food web connectance and structure with uncertainty;,citation_author=Anubhav Gupta;,citation_author=Reinhard Furrer;,citation_author=Owen L. Petchey;,citation_publication_date=2022;,citation_cover_date=2022;,citation_year=2022;,citation_fulltext_html_url=https://onlinelibrary.wiley.com/doi/abs/10.1002/ece3.8643;,citation_issue=3;,citation_doi=10.1002/ece3.8643;,citation_volume=12;,citation_language=en;,citation_journal_title=Ecology and Evolution;">
<meta name="citation_reference" content="citation_title=Highly resolved early eocene food webs show development of modern trophic structure after the end-cretaceous extinction;,citation_author=Jennifer A. Dunne;,citation_author=Conrad C. Labandeira;,citation_author=Richard J. Williams;,citation_publication_date=2014-05-07;,citation_cover_date=2014-05-07;,citation_year=2014;,citation_fulltext_html_url=https://royalsocietypublishing.org/doi/10.1098/rspb.2013.3280;,citation_issue=1782;,citation_doi=10.1098/rspb.2013.3280;,citation_volume=281;,citation_journal_title=Proceedings of the Royal Society B: Biological Sciences;">
<meta name="citation_reference" content="citation_title=Extinction cascades and catastrophe in ancient food webs;,citation_author=Peter D. Roopnarine;,citation_publication_date=2006;,citation_cover_date=2006;,citation_year=2006;,citation_fulltext_html_url=https://www.jstor.org/stable/4096814;,citation_issue=1;,citation_volume=32;,citation_journal_title=Paleobiology;">
<meta name="citation_reference" content="citation_title=Tailoring interaction network types to answer different ecological questions;,citation_author=Benoit Gauzens;,citation_author=Lise Thouvenot;,citation_author=Diane S. Srivastava;,citation_author=Pavel Kratina;,citation_author=Gustavo Q. Romero;,citation_author=Emilio Berti;,citation_author=Eoin J. O’Gorman;,citation_author=Angélica L. González;,citation_author=Olivier Dézerald;,citation_author=Nico Eisenhauer;,citation_author=Mathias Pires;,citation_author=Remo Ryser;,citation_author=Vinicius F. Farjalla;,citation_author=Pierre Rogy;,citation_author=Ulrich Brose;,citation_author=Jana S. Petermann;,citation_author=Benoît Geslin;,citation_author=Jes Hines;,citation_publication_date=2025-06-13;,citation_cover_date=2025-06-13;,citation_year=2025;,citation_fulltext_html_url=https://www.nature.com/articles/s44358-025-00056-7;,citation_doi=10.1038/s44358-025-00056-7;,citation_language=en;,citation_journal_title=Nature Reviews Biodiversity;">
<meta name="citation_reference" content="citation_title=Addressing priority questions of conservation science with palaeontological data;,citation_author=Wolfgang Kiessling;,citation_author=Nussaïbah B. Raja;,citation_author=Vanessa Julie Roden;,citation_author=Samuel T. Turvey;,citation_author=Erin E. Saupe;,citation_publication_date=2019-11-04;,citation_cover_date=2019-11-04;,citation_year=2019;,citation_fulltext_html_url=https://royalsocietypublishing.org/doi/10.1098/rstb.2019.0222;,citation_issue=1788;,citation_doi=10.1098/rstb.2019.0222;,citation_volume=374;,citation_journal_title=Philosophical Transactions of the Royal Society B: Biological Sciences;">
<meta name="citation_reference" content="citation_title=What is conservation paleobiology? Tracking 20 years of research and development;,citation_author=Erin M. Dillon;,citation_author=Jaleigh Q. Pier;,citation_author=Jansen A. Smith;,citation_author=Nussaïbah B. Raja;,citation_author=Danijela Dimitrijević;,citation_author=Elizabeth L. Austin;,citation_author=Jonathan D. Cybulski;,citation_author=Julia De Entrambasaguas;,citation_author=Stephen R. Durham;,citation_author=Carolin M. Grether;,citation_author=Himadri Sekhar Haldar;,citation_author=Kristína Kocáková;,citation_author=Chien-Hsiang Lin;,citation_author=Ilaria Mazzini;,citation_author=Alexis M. Mychajliw;,citation_author=Amy L. Ollendorf;,citation_author=Catalina Pimiento;,citation_author=Omar R. Regalado Fernández;,citation_author=Isaiah E. Smith;,citation_author=Gregory P. Dietl;,citation_publication_date=2022-12-07;,citation_cover_date=2022-12-07;,citation_year=2022;,citation_fulltext_html_url=https://www.frontiersin.org/journals/ecology-and-evolution/articles/10.3389/fevo.2022.1031483/full;,citation_doi=10.3389/fevo.2022.1031483;,citation_volume=10;,citation_language=en-US;,citation_journal_title=Frontiers in Ecology and Evolution;">
<meta name="citation_reference" content="citation_title=Global Projection of Terrestrial Vertebrate Food Webs Under Future Climate and Land-Use Changes;,citation_author=Xiyang Hao;,citation_author=Marcel Holyoak;,citation_author=Zhicheng Zhang;,citation_author=Chuan Yan;,citation_publication_date=2025;,citation_cover_date=2025;,citation_year=2025;,citation_fulltext_html_url=https://onlinelibrary.wiley.com/doi/abs/10.1111/gcb.70061;,citation_issue=2;,citation_doi=10.1111/gcb.70061;,citation_volume=31;,citation_language=en;,citation_journal_title=Global Change Biology;">
<meta name="citation_reference" content="citation_title=Chasing Ecological Interactions;,citation_author=Pedro Jordano;,citation_publication_date=2016-09-15;,citation_cover_date=2016-09-15;,citation_year=2016;,citation_fulltext_html_url=https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002559;,citation_issue=9;,citation_doi=10.1371/journal.pbio.1002559;,citation_volume=14;,citation_language=en;,citation_journal_title=PLOS Biology;">
<meta name="citation_reference" content="citation_title=Inferring biotic interactions from proxies;,citation_author=Ignacio Morales-Castilla;,citation_author=Miguel G. Matias;,citation_author=Dominique Gravel;,citation_author=Miguel B. Araújo;,citation_publication_date=2015-06;,citation_cover_date=2015-06;,citation_year=2015;,citation_fulltext_html_url=https://linkinghub.elsevier.com/retrieve/pii/S0169534715000774;,citation_issue=6;,citation_doi=10.1016/j.tree.2015.03.014;,citation_volume=30;,citation_language=en;,citation_journal_title=Trends in Ecology &amp;amp;amp; Evolution;">
<meta name="citation_reference" content="citation_title=Machine learning and deep learningA review for ecologists;,citation_author=Maximilian Pichler;,citation_author=Florian Hartig;,citation_publication_date=2023;,citation_cover_date=2023;,citation_year=2023;,citation_fulltext_html_url=https://onlinelibrary.wiley.com/doi/abs/10.1111/2041-210X.14061;,citation_issue=4;,citation_doi=10.1111/2041-210X.14061;,citation_volume=14;,citation_language=en;,citation_journal_title=Methods in Ecology and Evolution;">
<meta name="citation_reference" content="citation_title=A roadmap towards predicting species interaction networks (across space and time);,citation_author=Tanya Strydom;,citation_author=Michael D. Catchen;,citation_author=Francis Banville;,citation_author=Dominique Caron;,citation_author=Gabriel Dansereau;,citation_author=Philippe Desjardins-Proulx;,citation_author=Norma R. Forero-Muñoz;,citation_author=Gracielle Higino;,citation_author=Benjamin Mercier;,citation_author=Andrew Gonzalez;,citation_author=Dominique Gravel;,citation_author=Laura Pollock;,citation_author=Timothée Poisot;,citation_publication_date=2021-11-08;,citation_cover_date=2021-11-08;,citation_year=2021;,citation_fulltext_html_url=https://royalsocietypublishing.org/doi/abs/10.1098/rstb.2021.0063;,citation_issue=1837;,citation_doi=10.1098/rstb.2021.0063;,citation_volume=376;,citation_journal_title=Philosophical Transactions of the Royal Society B: Biological Sciences;">
<meta name="citation_reference" content="citation_title=A general model for food web structure;,citation_author=Stefano Allesina;,citation_author=David Alonso;,citation_author=Mercedes Pascual;,citation_publication_date=2008-05-02;,citation_cover_date=2008-05-02;,citation_year=2008;,citation_fulltext_html_url=https://www.science.org/doi/full/10.1126/science.1156269;,citation_issue=5876;,citation_doi=10.1126/science.1156269;,citation_volume=320;,citation_journal_title=Science;">
<meta name="citation_reference" content="citation_title=Collapse of an ecological network in ancient egypt;,citation_author=Justin D. Yeakel;,citation_author=Mathias M. Pires;,citation_author=Lars Rudolf;,citation_author=Nathaniel J. Dominy;,citation_author=Paul L. Koch;,citation_author=Paulo R. Guimarães;,citation_author=Thilo Gross;,citation_publication_date=2014-10;,citation_cover_date=2014-10;,citation_year=2014;,citation_issue=40;,citation_doi=10.1073/pnas.1408471111;,citation_volume=111;,citation_journal_title=PNAS;">
<meta name="citation_reference" content="citation_title=Food web reconstruction through phylogenetic transfer of low-rank network representation;,citation_author=Tanya Strydom;,citation_author=Salomé Bouskila;,citation_author=Francis Banville;,citation_author=Ceres Barros;,citation_author=Dominique Caron;,citation_author=Maxwell J. Farrell;,citation_author=Marie-Josée Fortin;,citation_author=Victoria Hemming;,citation_author=Benjamin Mercier;,citation_author=Laura J. Pollock;,citation_author=Rogini Runghen;,citation_author=Giulio V. Dalla Riva;,citation_author=Timothée Poisot;,citation_publication_date=2022-12;,citation_cover_date=2022-12;,citation_year=2022;,citation_fulltext_html_url=https://onlinelibrary.wiley.com/doi/abs/10.1111/2041-210X.13835;,citation_issue=12;,citation_doi=10.1111/2041-210X.13835;,citation_volume=13;,citation_language=en;,citation_journal_title=Methods in Ecology and Evolution;">
<meta name="citation_reference" content="citation_title=Navigating the integration of Biotic Interactions in Biogeography;,citation_author=Wilfried Thuiller;,citation_author=Irene Calderon-Sanou;,citation_author=Loïc Chalmandrier;,citation_author=Pierre Gaüzere;,citation_author=Marc Ohlmann;,citation_author=Giovanni Poggiato;,citation_author=Tamara Münkemüller;,citation_language=en;">
<meta name="citation_reference" content="citation_title=Robustness to secondary extinctions: Comparing trait-based sequential deletions in static and dynamic food webs;,citation_author=Alva Curtsdotter;,citation_author=Amrei Binzer;,citation_author=Ulrich Brose;,citation_author=Francisco De Castro;,citation_author=Bo Ebenman;,citation_author=Anna Eklöf;,citation_author=Jens O. Riede;,citation_author=Aaron Thierry;,citation_author=Björn C. Rall;,citation_publication_date=2011-11;,citation_cover_date=2011-11;,citation_year=2011;,citation_fulltext_html_url=https://linkinghub.elsevier.com/retrieve/pii/S1439179111001216;,citation_issue=7;,citation_doi=10.1016/j.baae.2011.09.008;,citation_volume=12;,citation_language=en;,citation_journal_title=Basic and Applied Ecology;">
<meta name="citation_reference" content="citation_title=Tailoring interaction network types to answer different ecological questions;,citation_author=Benoit Gauzens;,citation_author=Lise Thouvenot;,citation_author=Diane S. Srivastava;,citation_author=Pavel Kratina;,citation_author=Gustavo Q. Romero;,citation_author=Emilio Berti;,citation_author=Eoin J. O’Gorman;,citation_author=Angélica L. González;,citation_author=Olivier Dézerald;,citation_author=Nico Eisenhauer;,citation_author=Mathias Pires;,citation_author=Remo Ryser;,citation_author=Vinicius F. Farjalla;,citation_author=Pierre Rogy;,citation_author=Ulrich Brose;,citation_author=Jana S. Petermann;,citation_author=Benoît Geslin;,citation_author=Jes Hines;,citation_publication_date=2025-06-13;,citation_cover_date=2025-06-13;,citation_year=2025;,citation_fulltext_html_url=https://www.nature.com/articles/s44358-025-00056-7;,citation_doi=10.1038/s44358-025-00056-7;,citation_language=en;,citation_journal_title=Nature Reviews Biodiversity;">
<meta name="citation_reference" content="citation_title=Navigating the integration of biotic interactions in biogeography;,citation_author=Wilfried Thuiller;,citation_author=Irene Calderón-Sanou;,citation_author=Loïc Chalmandrier;,citation_author=Pierre Gaüzère;,citation_author=Louise M. J. O’Connor;,citation_author=Marc Ohlmann;,citation_author=Giovanni Poggiato;,citation_author=Tamara Münkemüller;,citation_publication_date=2024;,citation_cover_date=2024;,citation_year=2024;,citation_fulltext_html_url=https://onlinelibrary.wiley.com/doi/abs/10.1111/jbi.14734;,citation_issue=4;,citation_doi=10.1111/jbi.14734;,citation_volume=51;,citation_language=en;,citation_journal_title=Journal of Biogeography;">
<meta name="citation_reference" content="citation_title=Ecological networks–beyond food webs;,citation_author=Thomas C. Ings;,citation_author=José M. Montoya;,citation_author=Jordi Bascompte;,citation_author=Nico Blüthgen;,citation_author=Lee Brown;,citation_author=Carsten F. Dormann;,citation_author=François Edwards;,citation_author=David Figueroa;,citation_author=Ute Jacob;,citation_author=J. Iwan Jones;,citation_author=Rasmus B. Lauridsen;,citation_author=Mark E. Ledger;,citation_author=Hannah M. Lewis;,citation_author=Jens M. Olesen;,citation_author=F. J. Frank Veen;,citation_author=Phil H. Warren;,citation_author=Guy Woodward;,citation_publication_date=2009-01;,citation_cover_date=2009-01;,citation_year=2009;,citation_issue=1;,citation_doi=10.1111/j.1365-2656.2008.01460.x;,citation_volume=78;,citation_language=eng;,citation_journal_title=The Journal of Animal Ecology;">
</head>

<body class="quarto-light">

<header id="title-block-header" class="quarto-title-block default toc-left page-columns page-full">
  <div class="quarto-title-banner page-columns page-full">
    <div class="quarto-title column-body">
      <h1 class="title">Methods for Reconstructing Paleo Food Webs</h1>
          </div>

    
    <div class="quarto-title-meta-container">
      <div class="quarto-title-meta-column-start">
            <div class="quarto-title-meta-author">
          <div class="quarto-title-meta-heading">Authors</div>
          <div class="quarto-title-meta-heading">Affiliations</div>
          
                <div class="quarto-title-meta-contents">
            <p class="author">Tanya Strydom <a href="mailto:t.strydom@sheffield.ac.uk" class="quarto-title-author-email"><i class="bi bi-envelope"></i></a> <a href="https://orcid.org/0000-0001-6067-1349" class="quarto-title-author-orcid"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAA2ZpVFh0WE1MOmNvbS5hZG9iZS54bXAAAAAAADw/eHBhY2tldCBiZWdpbj0i77u/IiBpZD0iVzVNME1wQ2VoaUh6cmVTek5UY3prYzlkIj8+IDx4OnhtcG1ldGEgeG1sbnM6eD0iYWRvYmU6bnM6bWV0YS8iIHg6eG1wdGs9IkFkb2JlIFhNUCBDb3JlIDUuMC1jMDYwIDYxLjEzNDc3NywgMjAxMC8wMi8xMi0xNzozMjowMCAgICAgICAgIj4gPHJkZjpSREYgeG1sbnM6cmRmPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5LzAyLzIyLXJkZi1zeW50YXgtbnMjIj4gPHJkZjpEZXNjcmlwdGlvbiByZGY6YWJvdXQ9IiIgeG1sbnM6eG1wTU09Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC9tbS8iIHhtbG5zOnN0UmVmPSJodHRwOi8vbnMuYWRvYmUuY29tL3hhcC8xLjAvc1R5cGUvUmVzb3VyY2VSZWYjIiB4bWxuczp4bXA9Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC8iIHhtcE1NOk9yaWdpbmFsRG9jdW1lbnRJRD0ieG1wLmRpZDo1N0NEMjA4MDI1MjA2ODExOTk0QzkzNTEzRjZEQTg1NyIgeG1wTU06RG9jdW1lbnRJRD0ieG1wLmRpZDozM0NDOEJGNEZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wTU06SW5zdGFuY2VJRD0ieG1wLmlpZDozM0NDOEJGM0ZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wOkNyZWF0b3JUb29sPSJBZG9iZSBQaG90b3Nob3AgQ1M1IE1hY2ludG9zaCI+IDx4bXBNTTpEZXJpdmVkRnJvbSBzdFJlZjppbnN0YW5jZUlEPSJ4bXAuaWlkOkZDN0YxMTc0MDcyMDY4MTE5NUZFRDc5MUM2MUUwNEREIiBzdFJlZjpkb2N1bWVudElEPSJ4bXAuZGlkOjU3Q0QyMDgwMjUyMDY4MTE5OTRDOTM1MTNGNkRBODU3Ii8+IDwvcmRmOkRlc2NyaXB0aW9uPiA8L3JkZjpSREY+IDwveDp4bXBtZXRhPiA8P3hwYWNrZXQgZW5kPSJyIj8+84NovQAAAR1JREFUeNpiZEADy85ZJgCpeCB2QJM6AMQLo4yOL0AWZETSqACk1gOxAQN+cAGIA4EGPQBxmJA0nwdpjjQ8xqArmczw5tMHXAaALDgP1QMxAGqzAAPxQACqh4ER6uf5MBlkm0X4EGayMfMw/Pr7Bd2gRBZogMFBrv01hisv5jLsv9nLAPIOMnjy8RDDyYctyAbFM2EJbRQw+aAWw/LzVgx7b+cwCHKqMhjJFCBLOzAR6+lXX84xnHjYyqAo5IUizkRCwIENQQckGSDGY4TVgAPEaraQr2a4/24bSuoExcJCfAEJihXkWDj3ZAKy9EJGaEo8T0QSxkjSwORsCAuDQCD+QILmD1A9kECEZgxDaEZhICIzGcIyEyOl2RkgwAAhkmC+eAm0TAAAAABJRU5ErkJggg=="></a></p>
          </div>
                <div class="quarto-title-meta-contents">
                    <p class="affiliation">
                        School of Biosciences, University of Sheffield, Sheffield, UK
                      </p>
                  </div>
                      <div class="quarto-title-meta-contents">
            <p class="author">Baran Karapunar <a href="https://orcid.org/0000-0001-9711-1492" class="quarto-title-author-orcid"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAA2ZpVFh0WE1MOmNvbS5hZG9iZS54bXAAAAAAADw/eHBhY2tldCBiZWdpbj0i77u/IiBpZD0iVzVNME1wQ2VoaUh6cmVTek5UY3prYzlkIj8+IDx4OnhtcG1ldGEgeG1sbnM6eD0iYWRvYmU6bnM6bWV0YS8iIHg6eG1wdGs9IkFkb2JlIFhNUCBDb3JlIDUuMC1jMDYwIDYxLjEzNDc3NywgMjAxMC8wMi8xMi0xNzozMjowMCAgICAgICAgIj4gPHJkZjpSREYgeG1sbnM6cmRmPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5LzAyLzIyLXJkZi1zeW50YXgtbnMjIj4gPHJkZjpEZXNjcmlwdGlvbiByZGY6YWJvdXQ9IiIgeG1sbnM6eG1wTU09Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC9tbS8iIHhtbG5zOnN0UmVmPSJodHRwOi8vbnMuYWRvYmUuY29tL3hhcC8xLjAvc1R5cGUvUmVzb3VyY2VSZWYjIiB4bWxuczp4bXA9Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC8iIHhtcE1NOk9yaWdpbmFsRG9jdW1lbnRJRD0ieG1wLmRpZDo1N0NEMjA4MDI1MjA2ODExOTk0QzkzNTEzRjZEQTg1NyIgeG1wTU06RG9jdW1lbnRJRD0ieG1wLmRpZDozM0NDOEJGNEZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wTU06SW5zdGFuY2VJRD0ieG1wLmlpZDozM0NDOEJGM0ZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wOkNyZWF0b3JUb29sPSJBZG9iZSBQaG90b3Nob3AgQ1M1IE1hY2ludG9zaCI+IDx4bXBNTTpEZXJpdmVkRnJvbSBzdFJlZjppbnN0YW5jZUlEPSJ4bXAuaWlkOkZDN0YxMTc0MDcyMDY4MTE5NUZFRDc5MUM2MUUwNEREIiBzdFJlZjpkb2N1bWVudElEPSJ4bXAuZGlkOjU3Q0QyMDgwMjUyMDY4MTE5OTRDOTM1MTNGNkRBODU3Ii8+IDwvcmRmOkRlc2NyaXB0aW9uPiA8L3JkZjpSREY+IDwveDp4bXBtZXRhPiA8P3hwYWNrZXQgZW5kPSJyIj8+84NovQAAAR1JREFUeNpiZEADy85ZJgCpeCB2QJM6AMQLo4yOL0AWZETSqACk1gOxAQN+cAGIA4EGPQBxmJA0nwdpjjQ8xqArmczw5tMHXAaALDgP1QMxAGqzAAPxQACqh4ER6uf5MBlkm0X4EGayMfMw/Pr7Bd2gRBZogMFBrv01hisv5jLsv9nLAPIOMnjy8RDDyYctyAbFM2EJbRQw+aAWw/LzVgx7b+cwCHKqMhjJFCBLOzAR6+lXX84xnHjYyqAo5IUizkRCwIENQQckGSDGY4TVgAPEaraQr2a4/24bSuoExcJCfAEJihXkWDj3ZAKy9EJGaEo8T0QSxkjSwORsCAuDQCD+QILmD1A9kECEZgxDaEZhICIzGcIyEyOl2RkgwAAhkmC+eAm0TAAAAABJRU5ErkJggg=="></a></p>
          </div>
                <div class="quarto-title-meta-contents">
                    <p class="affiliation">
                        School of Earth and Environment, University of Leeds, Leeds, UK
                      </p>
                  </div>
                      <div class="quarto-title-meta-contents">
            <p class="author">Andrew P. Beckerman <a href="https://orcid.org/0000-0002-7859-8394" class="quarto-title-author-orcid"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAA2ZpVFh0WE1MOmNvbS5hZG9iZS54bXAAAAAAADw/eHBhY2tldCBiZWdpbj0i77u/IiBpZD0iVzVNME1wQ2VoaUh6cmVTek5UY3prYzlkIj8+IDx4OnhtcG1ldGEgeG1sbnM6eD0iYWRvYmU6bnM6bWV0YS8iIHg6eG1wdGs9IkFkb2JlIFhNUCBDb3JlIDUuMC1jMDYwIDYxLjEzNDc3NywgMjAxMC8wMi8xMi0xNzozMjowMCAgICAgICAgIj4gPHJkZjpSREYgeG1sbnM6cmRmPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5LzAyLzIyLXJkZi1zeW50YXgtbnMjIj4gPHJkZjpEZXNjcmlwdGlvbiByZGY6YWJvdXQ9IiIgeG1sbnM6eG1wTU09Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC9tbS8iIHhtbG5zOnN0UmVmPSJodHRwOi8vbnMuYWRvYmUuY29tL3hhcC8xLjAvc1R5cGUvUmVzb3VyY2VSZWYjIiB4bWxuczp4bXA9Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC8iIHhtcE1NOk9yaWdpbmFsRG9jdW1lbnRJRD0ieG1wLmRpZDo1N0NEMjA4MDI1MjA2ODExOTk0QzkzNTEzRjZEQTg1NyIgeG1wTU06RG9jdW1lbnRJRD0ieG1wLmRpZDozM0NDOEJGNEZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wTU06SW5zdGFuY2VJRD0ieG1wLmlpZDozM0NDOEJGM0ZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wOkNyZWF0b3JUb29sPSJBZG9iZSBQaG90b3Nob3AgQ1M1IE1hY2ludG9zaCI+IDx4bXBNTTpEZXJpdmVkRnJvbSBzdFJlZjppbnN0YW5jZUlEPSJ4bXAuaWlkOkZDN0YxMTc0MDcyMDY4MTE5NUZFRDc5MUM2MUUwNEREIiBzdFJlZjpkb2N1bWVudElEPSJ4bXAuZGlkOjU3Q0QyMDgwMjUyMDY4MTE5OTRDOTM1MTNGNkRBODU3Ii8+IDwvcmRmOkRlc2NyaXB0aW9uPiA8L3JkZjpSREY+IDwveDp4bXBtZXRhPiA8P3hwYWNrZXQgZW5kPSJyIj8+84NovQAAAR1JREFUeNpiZEADy85ZJgCpeCB2QJM6AMQLo4yOL0AWZETSqACk1gOxAQN+cAGIA4EGPQBxmJA0nwdpjjQ8xqArmczw5tMHXAaALDgP1QMxAGqzAAPxQACqh4ER6uf5MBlkm0X4EGayMfMw/Pr7Bd2gRBZogMFBrv01hisv5jLsv9nLAPIOMnjy8RDDyYctyAbFM2EJbRQw+aAWw/LzVgx7b+cwCHKqMhjJFCBLOzAR6+lXX84xnHjYyqAo5IUizkRCwIENQQckGSDGY4TVgAPEaraQr2a4/24bSuoExcJCfAEJihXkWDj3ZAKy9EJGaEo8T0QSxkjSwORsCAuDQCD+QILmD1A9kECEZgxDaEZhICIzGcIyEyOl2RkgwAAhkmC+eAm0TAAAAABJRU5ErkJggg=="></a></p>
          </div>
                <div class="quarto-title-meta-contents">
                    <p class="affiliation">
                        School of Biosciences, University of Sheffield, Sheffield, UK
                      </p>
                  </div>
                      <div class="quarto-title-meta-contents">
            <p class="author">Alexander Dunhill <a href="https://orcid.org/0000-0002-8680-9163" class="quarto-title-author-orcid"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAA2ZpVFh0WE1MOmNvbS5hZG9iZS54bXAAAAAAADw/eHBhY2tldCBiZWdpbj0i77u/IiBpZD0iVzVNME1wQ2VoaUh6cmVTek5UY3prYzlkIj8+IDx4OnhtcG1ldGEgeG1sbnM6eD0iYWRvYmU6bnM6bWV0YS8iIHg6eG1wdGs9IkFkb2JlIFhNUCBDb3JlIDUuMC1jMDYwIDYxLjEzNDc3NywgMjAxMC8wMi8xMi0xNzozMjowMCAgICAgICAgIj4gPHJkZjpSREYgeG1sbnM6cmRmPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5LzAyLzIyLXJkZi1zeW50YXgtbnMjIj4gPHJkZjpEZXNjcmlwdGlvbiByZGY6YWJvdXQ9IiIgeG1sbnM6eG1wTU09Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC9tbS8iIHhtbG5zOnN0UmVmPSJodHRwOi8vbnMuYWRvYmUuY29tL3hhcC8xLjAvc1R5cGUvUmVzb3VyY2VSZWYjIiB4bWxuczp4bXA9Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC8iIHhtcE1NOk9yaWdpbmFsRG9jdW1lbnRJRD0ieG1wLmRpZDo1N0NEMjA4MDI1MjA2ODExOTk0QzkzNTEzRjZEQTg1NyIgeG1wTU06RG9jdW1lbnRJRD0ieG1wLmRpZDozM0NDOEJGNEZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wTU06SW5zdGFuY2VJRD0ieG1wLmlpZDozM0NDOEJGM0ZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wOkNyZWF0b3JUb29sPSJBZG9iZSBQaG90b3Nob3AgQ1M1IE1hY2ludG9zaCI+IDx4bXBNTTpEZXJpdmVkRnJvbSBzdFJlZjppbnN0YW5jZUlEPSJ4bXAuaWlkOkZDN0YxMTc0MDcyMDY4MTE5NUZFRDc5MUM2MUUwNEREIiBzdFJlZjpkb2N1bWVudElEPSJ4bXAuZGlkOjU3Q0QyMDgwMjUyMDY4MTE5OTRDOTM1MTNGNkRBODU3Ii8+IDwvcmRmOkRlc2NyaXB0aW9uPiA8L3JkZjpSREY+IDwveDp4bXBtZXRhPiA8P3hwYWNrZXQgZW5kPSJyIj8+84NovQAAAR1JREFUeNpiZEADy85ZJgCpeCB2QJM6AMQLo4yOL0AWZETSqACk1gOxAQN+cAGIA4EGPQBxmJA0nwdpjjQ8xqArmczw5tMHXAaALDgP1QMxAGqzAAPxQACqh4ER6uf5MBlkm0X4EGayMfMw/Pr7Bd2gRBZogMFBrv01hisv5jLsv9nLAPIOMnjy8RDDyYctyAbFM2EJbRQw+aAWw/LzVgx7b+cwCHKqMhjJFCBLOzAR6+lXX84xnHjYyqAo5IUizkRCwIENQQckGSDGY4TVgAPEaraQr2a4/24bSuoExcJCfAEJihXkWDj3ZAKy9EJGaEo8T0QSxkjSwORsCAuDQCD+QILmD1A9kECEZgxDaEZhICIzGcIyEyOl2RkgwAAhkmC+eAm0TAAAAABJRU5ErkJggg=="></a></p>
          </div>
                <div class="quarto-title-meta-contents">
                    <p class="affiliation">
                        School of Earth and Environment, University of Leeds, Leeds, UK
                      </p>
                  </div>
                    </div>
        
        <div class="quarto-title-meta">

                      
                <div>
            <div class="quarto-title-meta-heading">Published</div>
            <div class="quarto-title-meta-contents">
              <p class="date">December 9, 2025</p>
            </div>
          </div>
          
                
              </div>
      </div>
      <div class="quarto-title-meta-column-end quarto-other-formats-target">
      <div class="quarto-alternate-formats"><div class="quarto-title-meta-heading">Other Formats</div><div class="quarto-title-meta-contents"><p><a href="index.docx"><i class="bi bi-file-word"></i>MS Word</a></p></div><div class="quarto-title-meta-contents"><p><a href="index.pdf"><i class="bi bi-file-pdf"></i>PDF (preprint)</a></p></div></div></div>
    </div>

    <div>
      <div class="abstract">
        <div class="block-title">Abstract</div>
        <p>Food webs provide a powerful lens for understanding ecosystem structure and function, but reconstructing them in paleoecological contexts remains challenging because direct evidence of feeding interactions is rarely preserved. A wide range of models now exist for predicting interactions and inferring network structure, yet these models differ substantially in their assumptions, mechanisms, and data requirements. Here, we evaluate which network construction approaches are most suitable for paleo food webs given the constraints of the fossil record, and we assess how model choice influences the networks we infer. Using the Toarcian Oceanic Anoxic Event as a case study, we compare six modelling approaches encompassing mechanistic, structural, and theory-based methods. We show that the models produce strikingly different network structures and pairwise interactions, and that these differences propagate into ecological inference—including conclusions about extinction dynamics. Our results highlight the importance of aligning model choice with research questions and underscore the interpretative risks of treating all food-web reconstruction methods as interchangeable.</p>
      </div>
    </div>

    <div>
      <div class="keywords">
        <div class="block-title">Keywords</div>
        <p>food web, network construction</p>
      </div>
    </div>

    <div class="quarto-other-links-text-target">
    <div class="quarto-code-links"><div class="quarto-title-meta-heading">Code Links</div><div class="quarto-title-meta-contents"><span><a href="https://github.com/TanyaS08/ms_paleo_webs/tree/main/code"><i class="bi bi-github"></i>GitHub</a></span></div></div></div>  </div>
</header><div id="quarto-content" class="page-columns page-rows-contents page-layout-article toc-left">
<div id="quarto-sidebar-toc-left" class="sidebar toc-left">
  <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">Table of contents</h2>
   
  <ul>
  <li><a href="#constructing-paleo-webs" id="toc-constructing-paleo-webs" class="nav-link active" data-scroll-target="#constructing-paleo-webs"><span class="header-section-number">1</span> Constructing paleo webs</a>
  <ul class="collapse">
  <li><a href="#challenges-specific-to-building-paleo-networks" id="toc-challenges-specific-to-building-paleo-networks" class="nav-link" data-scroll-target="#challenges-specific-to-building-paleo-networks"><span class="header-section-number">1.1</span> Challenges specific to building paleo networks</a></li>
  <li><a href="#understanding-the-approaches-to-network-construction" id="toc-understanding-the-approaches-to-network-construction" class="nav-link" data-scroll-target="#understanding-the-approaches-to-network-construction"><span class="header-section-number">1.2</span> Understanding the approaches to network construction</a></li>
  </ul></li>
  <li><a href="#case-study-toarcian-mass-extinction-event" id="toc-case-study-toarcian-mass-extinction-event" class="nav-link" data-scroll-target="#case-study-toarcian-mass-extinction-event"><span class="header-section-number">2</span> Case study: Toarcian mass extinction event</a>
  <ul class="collapse">
  <li><a href="#dataset-overview" id="toc-dataset-overview" class="nav-link" data-scroll-target="#dataset-overview"><span class="header-section-number">2.1</span> Dataset overview</a>
  <ul class="collapse">
  <li><a href="#species-occurrence" id="toc-species-occurrence" class="nav-link" data-scroll-target="#species-occurrence"><span class="header-section-number">2.1.1</span> Species occurrence</a></li>
  <li><a href="#defining-modes-of-life-traits" id="toc-defining-modes-of-life-traits" class="nav-link" data-scroll-target="#defining-modes-of-life-traits"><span class="header-section-number">2.1.2</span> Defining modes of life (traits)</a></li>
  <li><a href="#constructing-networks" id="toc-constructing-networks" class="nav-link" data-scroll-target="#constructing-networks"><span class="header-section-number">2.1.3</span> Constructing networks</a></li>
  </ul></li>
  <li><a href="#models-capture-different-network-structure-but-in-unexpected-ways" id="toc-models-capture-different-network-structure-but-in-unexpected-ways" class="nav-link" data-scroll-target="#models-capture-different-network-structure-but-in-unexpected-ways"><span class="header-section-number">2.2</span> Models capture different network structure but in unexpected ways</a></li>
  <li><a href="#some-networks-dont-share-any-interactions-and-some-share-a-lot" id="toc-some-networks-dont-share-any-interactions-and-some-share-a-lot" class="nav-link" data-scroll-target="#some-networks-dont-share-any-interactions-and-some-share-a-lot"><span class="header-section-number">2.3</span> Some networks don’t share any interactions and some share a lot</a></li>
  <li><a href="#model-choice-changes-the-narrative" id="toc-model-choice-changes-the-narrative" class="nav-link" data-scroll-target="#model-choice-changes-the-narrative"><span class="header-section-number">2.4</span> Model choice changes the narrative</a>
  <ul class="collapse">
  <li><a href="#inferred-extinction-drivers" id="toc-inferred-extinction-drivers" class="nav-link" data-scroll-target="#inferred-extinction-drivers"><span class="header-section-number">2.4.1</span> Inferred extinction drivers</a></li>
  </ul></li>
  </ul></li>
  <li><a href="#model-choice-as-an-ecological-inference-decision" id="toc-model-choice-as-an-ecological-inference-decision" class="nav-link" data-scroll-target="#model-choice-as-an-ecological-inference-decision"><span class="header-section-number">3</span> Model Choice as an Ecological Inference Decision</a>
  <ul class="collapse">
  <li><a href="#what-our-results-demonstrate-about-model-families" id="toc-what-our-results-demonstrate-about-model-families" class="nav-link" data-scroll-target="#what-our-results-demonstrate-about-model-families"><span class="header-section-number">3.1</span> What our results demonstrate about model families</a></li>
  <li><a href="#consequences-for-interpreting-paleoecological-dynamics" id="toc-consequences-for-interpreting-paleoecological-dynamics" class="nav-link" data-scroll-target="#consequences-for-interpreting-paleoecological-dynamics"><span class="header-section-number">3.2</span> Consequences for interpreting paleoecological dynamics</a></li>
  <li><a href="#aligning-ecological-questions-with-model-choice" id="toc-aligning-ecological-questions-with-model-choice" class="nav-link" data-scroll-target="#aligning-ecological-questions-with-model-choice"><span class="header-section-number">3.3</span> Aligning ecological questions with model choice</a></li>
  <li><a href="#implications-for-the-use-of-food-webs-in-paleoecology" id="toc-implications-for-the-use-of-food-webs-in-paleoecology" class="nav-link" data-scroll-target="#implications-for-the-use-of-food-webs-in-paleoecology"><span class="header-section-number">3.4</span> Implications for the use of food webs in paleoecology</a></li>
  <li><a href="#recommendations-and-best-practices" id="toc-recommendations-and-best-practices" class="nav-link" data-scroll-target="#recommendations-and-best-practices"><span class="header-section-number">3.5</span> Recommendations and best practices</a></li>
  <li><a href="#concluding-remarks" id="toc-concluding-remarks" class="nav-link" data-scroll-target="#concluding-remarks"><span class="header-section-number">3.6</span> Concluding remarks</a></li>
  </ul></li>
  <li><a href="#references" id="toc-references" class="nav-link" data-scroll-target="#references">References</a></li>
  </ul>
</nav>
</div>
<div id="quarto-margin-sidebar" class="sidebar margin-sidebar zindex-bottom">
</div>
<main class="content quarto-banner-title-block" id="quarto-document-content">



  


<p>There has been growing interest in using deep-time fossil data to understand how ecological communities respond to environmental change <span class="citation" data-cites="kiessling2019 dillon2022">(<a href="#ref-kiessling2019" role="doc-biblioref">Kiessling et al. 2019</a>; <a href="#ref-dillon2022" role="doc-biblioref">Dillon et al. 2022</a>)</span>. In modern systems, species interactions and the networks that they form have become central to studying biodiversity, energy flow, and community stability (<em>e.g.,</em> <span class="citation" data-cites="thuiller2024">Thuiller et al. (<a href="#ref-thuiller2024" role="doc-biblioref">2024</a>)</span>). Consequently, paleoecologists have increasingly sought to reconstruct ancient food webs to gain similar insights across major environmental transitions (<em>e.g.,</em> <span class="citation" data-cites="dunhill2024">Dunhill et al. (<a href="#ref-dunhill2024" role="doc-biblioref">2024</a>)</span>; <span class="citation" data-cites="hao2025">Hao et al. (<a href="#ref-hao2025" role="doc-biblioref">2025</a>)</span>; <span class="citation" data-cites="yeakel2014">Yeakel et al. (<a href="#ref-yeakel2014" role="doc-biblioref">2014</a>)</span>). However, we are faced by the limitation that interactions cannot be directly observed in the fossil record (with the exception of rare instances) and as a result, the construction of paleo food webs depends on models that allow us to infer feeding relationships from preserved traits, analogies to modern taxa, or ecological theory. While numerous models exist for predicting interactions (see <span class="citation" data-cites="morales-castilla2015">Morales-Castilla et al. (<a href="#ref-morales-castilla2015" role="doc-biblioref">2015</a>)</span>; <span class="citation" data-cites="pichler2023">Pichler and Hartig (<a href="#ref-pichler2023" role="doc-biblioref">2023</a>)</span>; <span class="citation" data-cites="strydom2021">Strydom et al. (<a href="#ref-strydom2021" role="doc-biblioref">2021</a>)</span>; <span class="citation" data-cites="allesina2008">Allesina, Alonso, and Pascual (<a href="#ref-allesina2008" role="doc-biblioref">2008</a>)</span> for broader reviews), only a subset can reliably be applied in paleo contexts, where data on traits, abundances, and community composition are inherently incomplete and biased.</p>
<p>The growing interest in paleo food webs has outpaced a clear discussion of <em>which</em> construction methods are suitable for which purposes. Different models generate different kinds of networks (feasible, realised, or purely structural) and these differences can fundamentally alter ecological interpretations. In this study, we evaluate a suite of methods that can be feasibly applied to paleo communities and explore how their underlying assumptions shape both network structure and ecological inference. Specifically we focus on identifying a suite of models that are appropriate for use with paleo data that can feasibly be constructed within the limitations that are imposed by fossil data while still spanning the larger network space. Here we use the data from <span class="citation" data-cites="dunhill2024">Dunhill et al. (<a href="#ref-dunhill2024" role="doc-biblioref">2024</a>)</span> as a case study to understand how different models recover different networks, both in terms of structure as well as pairwise interactions and establish if there are consequences for using networks that are constructed using different models in terms of making inferences about the behaviour of the system by looking at how the model type influences what we infer to be the dominant driver of extinctions across a mass extinction event.</p>
<section id="constructing-paleo-webs" class="level1" data-number="1">
<h1 data-number="1"><span class="header-section-number">1</span> Constructing paleo webs</h1>
<section id="challenges-specific-to-building-paleo-networks" class="level2" data-number="1.1">
<h2 data-number="1.1" class="anchored" data-anchor-id="challenges-specific-to-building-paleo-networks"><span class="header-section-number">1.1</span> Challenges specific to building paleo networks</h2>
<p>Reconstructing paleo food webs presents challenges that differ from those encountered in modern systems. First, the fossil record provides an incomplete and selectively preserved subset of the original community. Preservation biases (driven by habitat, skeletal composition, and sedimentary environment) mean that some trophic groups are over-represented (<em>e.g.,</em> hard-shelled organisms), while others (<em>e.g.,</em> soft-bodied taxa, plankton) are systematically under-sampled. This directly constrains the kinds of models that can be applied, because models requiring complete assemblages or accurate guild representation will perform poorly when preservation is uneven. Second, there is inherent uncertainty about the true community boundaries. Fossil assemblages may represent time-averaged accumulations, transported material, or mixed habitats. As a result, any species list is best interpreted as a set of taxa that could have interacted, rather than a snapshot of a specific subset of interaction species. Third, many extinct taxa have ambiguous trait states, especially regarding diet and behaviour. Even when functional morphology is preserved, ecological behaviour is seldom directly evident. Such uncertainty propagates differently across model families: mechanistic models tend to accommodate broad trait assignments, whereas theory-driven models are more sensitive to uncertainty in body size, foraging mode, or feeding constraints. These limitations do not render reconstruction impossible but highlight the importance of choosing a model whose assumptions match the type of ecological inference being attempted.</p>
</section>
<section id="understanding-the-approaches-to-network-construction" class="level2" data-number="1.2">
<h2 data-number="1.2" class="anchored" data-anchor-id="understanding-the-approaches-to-network-construction"><span class="header-section-number">1.2</span> Understanding the approaches to network construction</h2>
<p>Network construction approaches can be broadly grouped into three methodological and conceptual approaches <a href="#fig-concept" class="quarto-xref">Figure&nbsp;1</a>. The first are&nbsp;mechanistic models evaluate whether an interaction is&nbsp;<em>feasibly possible</em>. These models typically use trait-based rules (<em>e.g.,</em> feeding mode, body size, or functional morphology) or evolutionary relationships to determine whether a species&nbsp;<em>could</em>&nbsp;consume another. Mechanistic approaches tend to produce metawebs - the full set of all plausible interactions given biological constraints. Theory-driven models, embed assumptions from ecological theory (such as niche theory or foraging ecology) to generate realised interactions and networks. Structural models are similar to theory-driven models, with the exception that these models are species agnostic and as such can only be used to make inferences about network structure. Both theory-driven and structural models aim to reproduce characteristic patterns observed in modern food webs, such as intervality, trophic hierarchies, or body-size-scaled feeding ranges. They do not necessarily require detailed trait information and instead rely on ecological rules or statistical distributions consistent with empirical food webs.</p>
<div id="fig-concept" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-concept-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="figures/model_fams.png" class="lightbox" data-gallery="quarto-lightbox-gallery-1" title="Figure&nbsp;1: This obviously needs work but a variation on this to try and articulate the different approaches and broadly how they may differ."><img src="figures/model_fams.png" class="img-fluid figure-img"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-concept-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;1: This obviously needs work but a variation on this to try and articulate the different approaches and broadly how they may differ.
</figcaption>
</figure>
</div>
<p>Most existing paleo-specific approaches fall within the mechanistic tradition (e.g., <span class="citation" data-cites="shaw2024">Shaw et al. (<a href="#ref-shaw2024" role="doc-biblioref">2024</a>)</span>; <span class="citation" data-cites="roopnarine2006">Roopnarine (<a href="#ref-roopnarine2006" role="doc-biblioref">2006</a>)</span>; <span class="citation" data-cites="fricke2022">Fricke et al. (<a href="#ref-fricke2022" role="doc-biblioref">2022</a>)</span>). While these are well-suited for reconstructing feasible interactions, they represent only a subset of the broader space of network construction methods. Incorporating theory-driven models allows paleoecologists to explore realised interaction structures and address a wider suite of ecological questions—provided their assumptions are compatible with the limitations of fossil data. Here we present a range of models <a href="#tbl-models" class="quarto-xref">Table&nbsp;1</a> that carry specific assumptions and data requirements. For instance, allometric models depend on quantitative body-size estimates, which must be inferred from size classes or functional morphology in the fossil record. Structural models such as the niche model require only richness and connectance, but their species-agnostic nature limits their usefulness for trait-based or diet-specific questions. Mechanistic models rely on accurate assignment of feeding traits, which may be uncertain for extinct taxa but are often more tractable than estimating abundances or interaction strengths. Understanding how these limitations intersect with what fossil data can reliably provide is essential for selecting an appropriate modelling approach.</p>
<div id="tbl-models" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-tbl figure">
<figcaption class="quarto-float-caption-top quarto-float-caption quarto-float-tbl" id="tbl-models-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Table&nbsp;1: Six different models that can be used to construct food webs for both this specific community but are also broadly suited to paleo network prediction. These models span all facets of the network representation space (metaweb, realised, and structural network) and are suitable for an array of different paleo communities as the data requirements fall within the limitations set by the fossil record.
</figcaption>
<div aria-describedby="tbl-models-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<table class="caption-top table">
<colgroup>
<col style="width: 16%">
<col style="width: 16%">
<col style="width: 16%">
<col style="width: 16%">
<col style="width: 16%">
<col style="width: 16%">
</colgroup>
<thead>
<tr class="header">
<th>Model family</th>
<th>Assumptions</th>
<th>Data needs</th>
<th>‘Limitation’</th>
<th>Network type</th>
<th>Key reference</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>random</td>
<td>Links are randomly distributed within a network</td>
<td>richness, number of links</td>
<td>parameter assumptions, species agnostic</td>
<td>structural network</td>
<td><span class="citation" data-cites="erdos1959">Erdős and Rényi (<a href="#ref-erdos1959" role="doc-biblioref">1959</a>)</span></td>
</tr>
<tr class="even">
<td>niche</td>
<td>Networks are interval, species can be ordered on a ‘niche axis’</td>
<td>richness, connectance</td>
<td>parameter assumptions, species agnostic</td>
<td>structural network</td>
<td><span class="citation" data-cites="williams2008">Richard J. Williams and Martinez (<a href="#ref-williams2008" role="doc-biblioref">2008</a>)</span></td>
</tr>
<tr class="odd">
<td>allometric diet breadth model (ADBM)</td>
<td>Interactions are determined by energetic costs (foraging ecology)</td>
<td>body mass, biomass (abundance)</td>
<td>does not account for forbidden links in terms of trait compatibility, assumptions on body size and biomass (abundance) from fossil data</td>
<td>theoretical network</td>
<td><span class="citation" data-cites="petchey2008">Petchey et al. (<a href="#ref-petchey2008" role="doc-biblioref">2008</a>)</span></td>
</tr>
<tr class="even">
<td>l-matrix</td>
<td>Interactions inferred using allometric rules (ratio of body sizes between predator and prey), with links being constrained by a Ricker function</td>
<td>body mass, number of producer species</td>
<td>does not account for forbidden links in terms of trait compatibility, assumptions on body size from fossil data, assumptions as to the number of producer species</td>
<td>theoretical network</td>
<td><span class="citation" data-cites="schneider2016">Schneider et al. (<a href="#ref-schneider2016" role="doc-biblioref">2016</a>)</span></td>
</tr>
<tr class="odd">
<td>paleo food web inference model (PFIM)</td>
<td>Interactions can be inferred by a mechanistic framework/relationships</td>
<td>feeding traits for taxa, mechanistic feeding rules</td>
<td>Assumption made as to the feeding mechanisms, need to elucidate traits from models (although this is a way smaller issue)</td>
<td>mechanistic web</td>
<td><span class="citation" data-cites="shaw2024">Shaw et al. (<a href="#ref-shaw2024" role="doc-biblioref">2024</a>)</span></td>
</tr>
<tr class="even">
<td>body size ratio model</td>
<td>Interactions inferred using allometric rules (ratio of body sizes between predator and prey). Logit of the linking probability used to further constrain links to an ‘optimal size range’ for prey.</td>
<td>body mass</td>
<td>does not account for forbidden links in terms of evolutionary compatibility, assumptions on body size from fossil data</td>
<td>theoretical network</td>
<td><span class="citation" data-cites="rohr2010">Rohr et al. (<a href="#ref-rohr2010" role="doc-biblioref">2010</a>)</span></td>
</tr>
</tbody>
</table>
</div>
</figure>
</div>
</section>
</section>
<section id="case-study-toarcian-mass-extinction-event" class="level1" data-number="2">
<h1 data-number="2"><span class="header-section-number">2</span> Case study: Toarcian mass extinction event</h1>
<section id="dataset-overview" class="level2" data-number="2.1">
<h2 data-number="2.1" class="anchored" data-anchor-id="dataset-overview"><span class="header-section-number">2.1</span> Dataset overview</h2>
<section id="species-occurrence" class="level3" data-number="2.1.1">
<h3 data-number="2.1.1" class="anchored" data-anchor-id="species-occurrence"><span class="header-section-number">2.1.1</span> Species occurrence</h3>
<p>We used fossil occurrence data spanning the upper Pliensbachian (~185 Ma) to the upper Toarcian (~175 Ma) of the Cleveland Basin, following <span class="citation" data-cites="dunhill2024">Dunhill et al. (<a href="#ref-dunhill2024" role="doc-biblioref">2024</a>)</span> . The dataset comprises four paleo-communities representing the pre-extinction, post-extinction, early recovery, and late recovery intervals of the Toarcian Oceanic Anoxic Event. Each assemblage was treated as a community of potentially interacting taxa. <strong><em>([Insert here]: total number of taxa; numbers per time bin; brief note on sampling completeness?)</em></strong> Although the Toarcian occurs within “deep time,” the depositional environment of the Cleveland Basin provides an unusually well-resolved record of community change, making it a suitable system for comparing network reconstruction methods.</p>
</section>
<section id="defining-modes-of-life-traits" class="level3" data-number="2.1.2">
<h3 data-number="2.1.2" class="anchored" data-anchor-id="defining-modes-of-life-traits"><span class="header-section-number">2.1.2</span> Defining modes of life (traits)</h3>
<p>We used the modes of life (traits) as identified in <span class="citation" data-cites="dunhill2024">Dunhill et al. (<a href="#ref-dunhill2024" role="doc-biblioref">2024</a>)</span>, who defined four traits: motility (fast, slow, facultative, non-motile), tiering (pelagic, erect, surficial, semi-infaunal, shallow infaunal, deep infaunal), feeding (predator, suspension feeder, deposit feeder, mining, grazer), and size: gigantic (&gt;500 mm), very large (&gt;300–500 mm), large (&gt;100–300 mm), medium (&gt;50–100 mm), small (&gt;10–50 mm), tiny (≤10 mm), for each fossil species based on the ecological traits defined in the Bambach ecospace model <span class="citation" data-cites="bambach2007">(<a href="#ref-bambach2007" role="doc-biblioref">Bambach, Bush, and Erwin 2007</a>)</span>.</p>
</section>
<section id="constructing-networks" class="level3" data-number="2.1.3">
<h3 data-number="2.1.3" class="anchored" data-anchor-id="constructing-networks"><span class="header-section-number">2.1.3</span> Constructing networks</h3>
<p>For each paleo community, we constructed 100 networks using each of the modles listed in <a href="#tbl-models" class="quarto-xref">Table&nbsp;1</a> (6 models × 4 time intervals × 100 replicates = 2,400 networks). Networks were then simplified by removing disconnected species, ensuring that all nodes participated in at least one interaction. Models requiring body-size inputs (ADBM, l-matrix, and body-size ratio models) were parameterised by drawing body masses from uniform distributions bounded by the size-class limits assigned in <span class="citation" data-cites="dunhill2024">Dunhill et al. (<a href="#ref-dunhill2024" role="doc-biblioref">2024</a>)</span>. This approach propagates uncertainty inherent in fossil size estimates while preserving consistent relative sizes among species within a replicate. For each replicate, the same set of body masses was used across models that depend on size. For structural models (random and niche), connectance was drawn uniformly from 0.07–0.34 to ensure networks spanned a realistic range of empirical food-web connectances, while holding richness constant. The same connectance value was used for both models within a replicate to facilitate direct comparison. For each network, we calculated the metrics listed in <a href="#tbl-properties" class="quarto-xref">Table&nbsp;2</a>, capturing macro-, meso-, and micro-scale structural properties.</p>
</section>
</section>
<section id="models-capture-different-network-structure-but-in-unexpected-ways" class="level2" data-number="2.2">
<h2 data-number="2.2" class="anchored" data-anchor-id="models-capture-different-network-structure-but-in-unexpected-ways"><span class="header-section-number">2.2</span> Models capture different network structure but in unexpected ways</h2>
<p>When quantifying network structure, we are essentially asking how interactions are distributed across species and how these patterns scale from individual nodes to the whole community. Structural metrics are informative because they reflect underlying ecological processes: how energy flows through trophic levels, how disturbances propagate, where redundancy or fragility exists, and how species specialise or generalise in their diets. To capture these different facets, we evaluated a suite of macro-, meso-, and micro-scale metrics <a href="#tbl-properties" class="quarto-xref">Table&nbsp;2</a>, ranging from global properties like connectance and complexity to motifs and species-level generality and vulnerability.</p>
<div id="tbl-properties" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-tbl figure">
<figcaption class="quarto-float-caption-top quarto-float-caption quarto-float-tbl" id="tbl-properties-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Table&nbsp;2: Network properties used for analysis.
</figcaption>
<div aria-describedby="tbl-properties-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<table class="caption-top table">
<colgroup>
<col style="width: 24%">
<col style="width: 26%">
<col style="width: 24%">
<col style="width: 24%">
</colgroup>
<thead>
<tr class="header">
<th>Metric</th>
<th>Definition</th>
<th>Scale</th>
<th>Reference (for maths), can make footnotes probs</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Richness</td>
<td>Number of nodes in the network</td>
<td>Macro</td>
<td></td>
</tr>
<tr class="even">
<td>Links</td>
<td>Normalized standard deviation of links (number of consumers plus resources per taxon)</td>
<td>Micro</td>
<td></td>
</tr>
<tr class="odd">
<td>Connectance</td>
<td><span class="math inline">\(L/S^2\)</span>, where <span class="math inline">\(S\)</span> is the number of species and <span class="math inline">\(L\)</span> the number of links</td>
<td>Macro</td>
<td></td>
</tr>
<tr class="even">
<td>Max trophic level</td>
<td>Prey-weighted trophic level averaged across taxa</td>
<td>Macro</td>
<td><span class="citation" data-cites="williams2004">R. J. Williams and Martinez (<a href="#ref-williams2004" role="doc-biblioref">2004</a>)</span></td>
</tr>
<tr class="odd">
<td>Diameter</td>
<td>Diameter can also be measured as the average of the distances between each pair of nodes in the network</td>
<td>Macro</td>
<td><span class="citation" data-cites="delmas2018">Delmas et al. (<a href="#ref-delmas2018" role="doc-biblioref">2018</a>)</span></td>
</tr>
<tr class="even">
<td>Complexity</td>
<td>SVD complexity of a network, defined as the Pielou entropy of its singular values</td>
<td>Macro</td>
<td><span class="citation" data-cites="strydom2021">Strydom et al. (<a href="#ref-strydom2021" role="doc-biblioref">2021</a>)</span></td>
</tr>
<tr class="odd">
<td>Redundancy</td>
<td><span class="math inline">\((L - (S - 1))/S\)</span>, where <span class="math inline">\(S\)</span> is the number of species and <span class="math inline">\(L\)</span> the number of links. Indicates the number of edges beyond what is needed for a minimum-connected tree</td>
<td>Macro</td>
<td></td>
</tr>
<tr class="even">
<td>S1</td>
<td>Number of linear chains, normalised</td>
<td>Meso</td>
<td><span class="citation" data-cites="milo2002">Milo et al. (<a href="#ref-milo2002" role="doc-biblioref">2002</a>)</span>; <span class="citation" data-cites="stouffer2007">Stouffer et al. (<a href="#ref-stouffer2007" role="doc-biblioref">2007</a>)</span></td>
</tr>
<tr class="odd">
<td>S2</td>
<td>Number of omnivory motifs, normalised</td>
<td>Meso</td>
<td><span class="citation" data-cites="milo2002">Milo et al. (<a href="#ref-milo2002" role="doc-biblioref">2002</a>)</span>; <span class="citation" data-cites="stouffer2007">Stouffer et al. (<a href="#ref-stouffer2007" role="doc-biblioref">2007</a>)</span></td>
</tr>
<tr class="even">
<td>S4</td>
<td>Number of apparent competition motifs, normalised</td>
<td>Meso</td>
<td><span class="citation" data-cites="milo2002">Milo et al. (<a href="#ref-milo2002" role="doc-biblioref">2002</a>)</span>; <span class="citation" data-cites="stouffer2007">Stouffer et al. (<a href="#ref-stouffer2007" role="doc-biblioref">2007</a>)</span></td>
</tr>
<tr class="odd">
<td>S5</td>
<td>Number of direct competition motifs, normalised</td>
<td>Meso</td>
<td><span class="citation" data-cites="milo2002">Milo et al. (<a href="#ref-milo2002" role="doc-biblioref">2002</a>)</span>; <span class="citation" data-cites="stouffer2007">Stouffer et al. (<a href="#ref-stouffer2007" role="doc-biblioref">2007</a>)</span></td>
</tr>
<tr class="even">
<td>Generality</td>
<td>Normalized standard deviation of generality of a species standardized by <span class="math inline">\(L/S\)</span></td>
<td>Micro</td>
<td><span class="citation" data-cites="williams2000">Richard J. Williams and Martinez (<a href="#ref-williams2000" role="doc-biblioref">2000</a>)</span></td>
</tr>
<tr class="odd">
<td>Vulnerability</td>
<td>Normalized standard deviation of vulnerability of a species standardized by <span class="math inline">\(L/S\)</span></td>
<td>Micro</td>
<td><span class="citation" data-cites="williams2000">Richard J. Williams and Martinez (<a href="#ref-williams2000" role="doc-biblioref">2000</a>)</span></td>
</tr>
</tbody>
</table>
</div>
</figure>
</div>
<p>Despite being supplied with the same species pools, the different models generated networks with&nbsp;systematically different structural signatures. A MANOVA revealed a strong multivariate effect of model type on network structure (Pillai’s Trace = 3.89, p &lt; 0.001), indicating that each modelling approach produces a distinct ‘structural fingerprint’. Follow-up ANOVAs confirmed that model choice had substantial effects on every metric we examined, with effect sizes typically exceeding 0.80. The only exception was maximum trophic level (η² = 0.19), suggesting some convergence in vertical structure even when structure differs widely.</p>
<p>Post-hoc comparisons <a href="#fig-marginal" class="quarto-xref">Figure&nbsp;2</a> revealed three broad clusters. PFIM consistently yielded the most connected, dense networks, reflecting its mechanistic emphasis on trait-based feasibility. Niche, random, and body-size ratio models formed an intermediate group, producing networks with moderate connectance and motif frequencies. ADBM and l-matrix formed a tight cluster characterised by constrained feeding ranges, reflecting their shared basis in energetic and allometric theory. Although these groupings broadly align with <em>a priori</em> expectations about model families, several patterns emerged that were less intuitive. Most notably, the body-size ratio model which is theoretically grounded but aligned more closely with structural models than with the fully allometric ones. This suggests that even slight differences in how body-size constraints are implemented can shift a model’s position within the network-structure landscape. A Linear Discriminant Analysis further illustrated the distinctiveness of the model families <a href="#fig-structure" class="quarto-xref">Figure&nbsp;3</a>. Classification accuracy of the LDA was 85%, demonstrating that the combination of dependent variables effectively discriminates among model types, indicating the strong imprint that model assumptions leave on inferred ecological patterns. The PFIM model is strongly separated, while ADBM and l-matrix networks cluster closely together. The niche and random models occupy intermediate positions. Importantly these results indicate that model choice is the dominant driver of inferred network structure, often overwhelming the ecological signal embedded within the species pool itself. Models designed to reproduce feasible interactions and those designed to generate realised niche structure occupy fundamentally different portions of ‘network space’, even when operating on identical taxa. Such structural divergences could have direct implications for ecological inference, particularly when comparing networks across time or when using network metrics to infer processes such as community stability, trophic organisation, or susceptibility to cascading extinctions.</p>
<div id="fig-marginal" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-marginal-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="figures/marginal_mean.png" class="lightbox" data-gallery="quarto-lightbox-gallery-2" title="Figure&nbsp;2: Estimated marginal means (EMMs) of ecological network metrics across six model types with 95% confidence intervals. Bars represent the predicted values for each model, and error bars indicate the 95% confidence limits. Letters above each bar denote Tukey-adjusted pairwise significance: models sharing the same letter are not significantly different, while models with different letters are significantly different (p < 0.05). The plot reveals three tiers of model performance, with pfim consistently higher, log ratio, niche, and random at intermediate levels, and adbm and lmatrix lower, consistent with the MANOVA and post-hoc analyses."><img src="figures/marginal_mean.png" class="img-fluid figure-img"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-marginal-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;2: Estimated marginal means (EMMs) of ecological network metrics across six model types with 95% confidence intervals. Bars represent the predicted values for each model, and error bars indicate the 95% confidence limits. Letters above each bar denote Tukey-adjusted pairwise significance: models sharing the same letter are not significantly different, while models with different letters are significantly different (p &lt; 0.05). The plot reveals three tiers of model performance, with pfim consistently higher, log ratio, niche, and random at intermediate levels, and adbm and lmatrix lower, consistent with the MANOVA and post-hoc analyses.
</figcaption>
</figure>
</div>
<div id="fig-structure" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-structure-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="figures/MANOVA_3_panel.png" class="lightbox" data-gallery="quarto-lightbox-gallery-3" title="Figure&nbsp;3: Linear discriminant analysis (LDA) of ecological network metrics for six model types. Each point represents a replicate, and ellipses indicate 95% confidence regions for each model. The second column represents the correlation of the various network metrics with the respective LDA axes."><img src="figures/MANOVA_3_panel.png" class="img-fluid figure-img"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-structure-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;3: Linear discriminant analysis (LDA) of ecological network metrics for six model types. Each point represents a replicate, and ellipses indicate 95% confidence regions for each model. The second column represents the correlation of the various network metrics with the respective LDA axes.
</figcaption>
</figure>
</div>
</section>
<section id="some-networks-dont-share-any-interactions-and-some-share-a-lot" class="level2" data-number="2.3">
<h2 data-number="2.3" class="anchored" data-anchor-id="some-networks-dont-share-any-interactions-and-some-share-a-lot"><span class="header-section-number">2.3</span> Some networks don’t share any interactions and some share a lot</h2>
<p>Beyond differences in global structure, researchers are often interested in&nbsp;specific ecological relationships, <em>e.g.,</em> who eats who, which species share predators, and how trophic roles change across communities or time. For these types of questions, it is essential to understand how models differ at the level of pairwise interactions. To quantify this, we measured interaction turnover between networks to allow us to assess the degree to which two models predict the same or different links for the same set of species. This is analogous to β-diversity but applied to links rather than species <span class="citation" data-cites="poisot2012">(<a href="#ref-poisot2012" role="doc-biblioref">Poisot et al. 2012</a>)</span>. Specifically we only looked at the dissimilarity of interaction between shared species. Even when supplied with identical species pools, models varied dramatically in the interactions they inferred <a href="#fig-beta_div" class="quarto-xref">Figure&nbsp;4</a>. Some pairs of models showed substantial agreement, whereas others shared almost no interactions at all. Note here that we did not include the Random or Niche models as these networks are species agnostic and as such are not designed for inferring species-specific pairwise links.</p>
<p>The body-size ratio model had consistently&nbsp;high&nbsp;turnover relative to all others, indicating that it inferred diets that were largely distinct. This reflects the strong constraints imposed by its logit-based linking rule, which sharply restricts prey to a narrow ‘optimal’ size range. Small differences in body-size estimates or functional groupings therefore lead to disproportionately large changes in inferred interactions. In contrast, the ADBM and L-matrix showed low turnover between each other, reflecting their shared theoretical foundations (operationalising foraging decisions and energetic constraints using similar allometric principles). As a result, they tend to produce similar pairwise interactions even when implemented independently. The PFIM exhibited intermediate turnover, sharing more interactions with size-based theoretical models than with the body-size ratio model. This makes sense: although PFIM uses categorical traits and hierarchical feeding rules rather than quantitative foraging theory, these constraints will still produce broadly similar trophic groupings.</p>
<p>Taken together, these patterns show that&nbsp;pairwise interactions differ far more across models than global metrics alone might suggest. Two models with superficially similar connectance or trophic structure may nonetheless infer completely different diets for individual taxa. This has large implications for any question focused on species-level ecology, including predator–prey specialisation, trophic niche breadth, or the identity of keystone consumers. These findings reinforce the importance of selecting a model whose assumptions align with the intended inference. If the goal is to explore the full set of <em>possible</em> interactions a species could have had, a mechanistic model such as PFIM is appropriate. If instead the goal is to infer the likely <em>realised</em> interactions or energy pathways, models grounded in allometric foraging theory (ADBM, L-matrix) will provide more ecologically coherent results. Conversely, models like the body-size ratio may be too restrictive or idiosyncratic for diet-based questions because they force interactions into narrow, trait-determined windows.</p>
<div id="fig-beta_div" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-beta_div-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="figures/beta_div.png" class="lightbox" data-gallery="quarto-lightbox-gallery-4" title="Figure&nbsp;4: Pairwise beta turnover in species interactions among four ecological network models (adbm, lmatrix, log-ratio, and pfim). Each cell represents the mean turnover value between a pair of models, with warmer colors indicating greater dissimilarity in inferred interactions. The diagonal is omitted. High turnover values (yellow) indicate strong disagreement in network structure between models, whereas lower values (blue–purple) indicate greater similarity."><img src="figures/beta_div.png" class="img-fluid figure-img"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-beta_div-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;4: Pairwise beta turnover in species interactions among four ecological network models (adbm, lmatrix, log-ratio, and pfim). Each cell represents the mean turnover value between a pair of models, with warmer colors indicating greater dissimilarity in inferred interactions. The diagonal is omitted. High turnover values (yellow) indicate strong disagreement in network structure between models, whereas lower values (blue–purple) indicate greater similarity.
</figcaption>
</figure>
</div>
</section>
<section id="model-choice-changes-the-narrative" class="level2" data-number="2.4">
<h2 data-number="2.4" class="anchored" data-anchor-id="model-choice-changes-the-narrative"><span class="header-section-number">2.4</span> Model choice changes the narrative</h2>
<p>The structural and interaction-level differences documented above raise a central question: do different models also lead to different interpretations of ecological dynamics? In other words, does model choice merely affect the architecture of the reconstructed networks, or does it shape the actual stories we tell about how communities collapsed and recovered during the Toarcian extinction event? Using the pre-extinction networks as starting points, we simulated species losses under a suite of ecologically plausible extinction scenarios, including trait-based removals (e.g., body size, motility), network-position removals (e.g., vulnerability, generality), and random extinctions. In each case, we allowed for cascading secondary extinctions. For each model and scenario, we then measured how closely the simulated post-extinction network resembled the real fossil community.</p>
<section id="inferred-extinction-drivers" class="level3" data-number="2.4.1">
<h3 data-number="2.4.1" class="anchored" data-anchor-id="inferred-extinction-drivers"><span class="header-section-number">2.4.1</span> Inferred extinction drivers</h3>
<p>To assess how consistently different modelling approaches evaluate extinction scenarios, we quantified the agreement in scenario rankings produced by multiple models across several network metrics. For each model, each extinction scenario, and each network metric we calculated the mean absolute difference (MAD) between the observed metric value and the value predicted following the simulated extinction sequence. Lower MAD values indicate a closer match to the empirical network structure and therefore a better-performing extinction scenario for that model and metric. Additionally, we used a derivative of <span class="citation" data-cites="gupta2022">Gupta, Furrer, and Petchey (<a href="#ref-gupta2022" role="doc-biblioref">2022</a>)</span> true skill statistic (TSS, see <a href="#eq-1" class="quarto-xref">Equation&nbsp;1</a>), where a score below zero indicates that the simulated extinction performs no better than random, and a score of one indicates a perfect match between real and simulated. Here we calculated both a node-level TSS as well as link-level TSS, by parsing out the TSS into two components we are able to assess if differences between real and simulated networks are due to node-level (the wrong species being removed) or link-level (the wrong links be recovered) mismatches. Because the extinction simulations do not allow for the origination of species, when calculating the TSS we only retained species that were present in both the pre and post extinction community and so any node-level mismatches between real and simulated networks was due to the wrong species being removed and not because new species were not.</p>
<p><span id="eq-1"><span class="math display">\[
TSS = \frac{True Positive}{True Positive + False Negative} + \frac{True Negative}{True Negative + False Positive} - 1
\tag{1}\]</span></span></p>
<p>For each network metric, we treated each model as an independent evaluator of scenario performance. MAD and TSS values were converted to within-model rankings, with rank 1 assigned to the scenario with the smallest MAD (<em>i.e.,</em> the closest match to the empirical value) or highest TSS score. Ranking was performed independently for each combination of model and network metric to avoid assumptions about comparability across metrics. To evaluate whether different models produced consistent rankings of extinction scenarios, we quantified rank correlation among models separately for each network metric. Agreement among model rankings was assessed using Kendall’s rank correlation coefficient (τ), which measures the degree of agreement between two ordinal rankings. Kendall’s τ was selected because it is robust for small sample sizes, handles tied ranks appropriately, and provides a direct measure of the probability that model pairs agree or disagree on the relative ordering of scenarios. Kendall’s τ ranges from –1 to +1, where +1 indicates perfect agreement between rankings, 0 reflects no relationship, and –1 represents complete disagreement such that one ranking is the exact reverse of the other.</p>
<p>When we look at Kendall’s τ for the MAD across network structure and models <a href="#fig-mad" class="quarto-xref">Figure&nbsp;5</a> we see that generally there is a positive correlation between the different different models. This implies that different models are often recovering a similar ranking of extinction mechanisms (as in the ‘signal’ as to which extinction mechanisms may be the most plausible are the same). Although there is not a strong agreement between models as values tend to be low it is positive to observe that it is not often that we have a completely different ranking of extinction mechanisms, with the exception of complexity and the number of direct competition motifs. When looking at the macro-level network properties the random model often showcases a disagreement in terms of the MAD. This is unsurprising as we expect random networks to produce networks that are not ecologically sound and thus will not behave as one may expect <span class="citation" data-cites="ings2009">(<a href="#ref-ings2009" role="doc-biblioref">Ings et al. 2009</a>)</span>. Interestingly we once again see the strong similarity between the L-matrix and the ADBM (have a high Kendall’s τ). meaning that they recover a similar ranking of extinction mechanisms, this is unsurprising given that we know these networks tend to recover a similar structure <a href="#fig-marginal" class="quarto-xref">Figure&nbsp;2</a>. Broadly when we look at the behaviour of the different model families (with the exception of the Random model) we see that they recover similar structural signals with regards to the mechanisms potentially driving extinctions.</p>
<div id="fig-mad" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-mad-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="figures/kendal_tau.png" class="lightbox" data-gallery="quarto-lightbox-gallery-5" title="Figure&nbsp;5: Heatmaps showing pairwise Kendall rank correlation coefficients (τ) between models for each network metric. Each panel corresponds to a different metric and displays the degree of agreement in extinction-scenario rankings across models based on mean absolute differences (MAD) between observed and predicted network values. Positive τ values (blue) indicate concordant rankings between models, whereas negative τ values (red) indicate opposing rankings. Warmer colours approaching zero represent little or no agreement. Panels illustrate how consistently different modelling approaches evaluate the relative realism of extinction scenarios across multiple network properties."><img src="figures/kendal_tau.png" class="img-fluid figure-img"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-mad-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;5: Heatmaps showing pairwise Kendall rank correlation coefficients (τ) between models for each network metric. Each panel corresponds to a different metric and displays the degree of agreement in extinction-scenario rankings across models based on mean absolute differences (MAD) between observed and predicted network values. Positive τ values (blue) indicate concordant rankings between models, whereas negative τ values (red) indicate opposing rankings. Warmer colours approaching zero represent little or no agreement. Panels illustrate how consistently different modelling approaches evaluate the relative realism of extinction scenarios across multiple network properties.
</figcaption>
</figure>
</div>
<p>When looking at the node-level TSS scores (<a href="#fig-mad" class="quarto-xref">Figure&nbsp;5</a>, TSS, panel 2) we see that in general the signal of the extinction mechanism is maintained across the different models. However as many of the extinction mechanisms are determine by the <em>traits</em> of the node it is not surprising that we see a similar signal as the species are being removed in the exact same order. Link-level TSS scores (<a href="#fig-mad" class="quarto-xref">Figure&nbsp;5</a>, TSS, panel 1) do not show the same within extinction mechanism ranking/signal. We see that the Random and PFIM models have high TSS scores (<em>i.e.,</em> have a ‘good fit’), however in the case of the PFIM this is to be expected as the links are deterministic and so if you have the same two species pools you will recover the same links. The ‘stochastic’ element of the theoretical models (ADBM, l-matrix, and log-ratio) means that they create a degree of noise at the link-level and thus they are probably inappropriate to use for the type of extinction mechanism question we are asking here - specifically does the real and the simulated network look the same. Link-level TSS is perhaps also not an appropriate approach to determine the ‘best fit’ extinction mechanism if used in isolation and we advocate that the node-level TSS score (or alternatively some measure of <span class="math inline">\(\beta\)</span> diversity is used). Finally, if we were to focus only on node level TSS we do not observe any strong differences between the models and it suggests that node-level driven (topological) extinction processes are insensitive to model type.</p>
<p>Broad inferred extinction mechanisms were relatively robust across models. This is probably in part because species are removed in the same order, node-level outcomes (which species survive) tended to agree. However, at the link level, where secondary cascades depend sensitively on inferred interactions, models often showed limited agreement. PFIM produced consistent link-level outcomes due to its deterministic rules, whereas theory-driven models (ADBM, L-matrix, body-size ratio) generated more variable trajectories due to stochastic link assignment. As a result, different models sometimes reconstructed different pathways of collapse, inferred different trophic groups as being the most affected, or ranked extinction scenarios differently. In other words, while the high-level narrative (e.g., traits matter) is stable, the fine-grained story of ecosystem disruption (who lost interactions first, how cascades unfolded, which species acted as bottlenecks) changes depending on the chosen model. Thus, the Toarcian mass extinction looks subtly but meaningfully different through the lens of each modelling framework. Model choice therefore must be treated as a core component of ecological inference, not a neutral preprocessing step. Some broad signals are robust—especially those driven by species traits—but many of the finer details that paleoecologists care about, such as trophic cascading pathways, keystone taxa, or the ordering of collapse, depend strongly on the chosen model. Researchers must therefore treat model choice not as a technical detail but as a central component of ecological inference.</p>
</section>
</section>
</section>
<section id="model-choice-as-an-ecological-inference-decision" class="level1" data-number="3">
<h1 data-number="3"><span class="header-section-number">3</span> Model Choice as an Ecological Inference Decision</h1>
<p>Reconstructing food webs from fossil data is an exercise in inference under uncertainty, and our results demonstrate that the&nbsp;choice of network construction model is itself a major ecological decision. Despite using the same species pools, different models produced networks with profoundly different structural properties, interaction patterns, and inferred extinction dynamics. These differences emerge not from the fossil data themselves but from the assumptions embedded within each modelling approach. As a consequence, network reconstruction cannot be treated as a neutral methodological step: model choice fundamentally shapes the ecological narratives we extract from the fossil record.</p>
<section id="what-our-results-demonstrate-about-model-families" class="level2" data-number="3.1">
<h2 data-number="3.1" class="anchored" data-anchor-id="what-our-results-demonstrate-about-model-families"><span class="header-section-number">3.1</span> What our results demonstrate about model families</h2>
<p>Across every structural metric we measured, model identity explained the majority of variation. PFIM-produced networks were consistently the most connected, while the ADBM and l-matrix produced sparser networks with tighter energetic constraints. Structural models (niche and random) fell between these extremes. These clusters reflect the conceptual differences between feasible, realised, and structural network representations. Importantly, these differences were not superficial. Pairwise interaction turnover revealed that models often disagreed not just on the&nbsp;number&nbsp;of links but on&nbsp;who interacts with whom. Such discrepancies directly affect diet-based ecological questions and interpretations of trophic roles.</p>
</section>
<section id="consequences-for-interpreting-paleoecological-dynamics" class="level2" data-number="3.2">
<h2 data-number="3.2" class="anchored" data-anchor-id="consequences-for-interpreting-paleoecological-dynamics"><span class="header-section-number">3.2</span> Consequences for interpreting paleoecological dynamics</h2>
<p>The downstream effects of model choice were most evident when evaluating extinction dynamics. Models often recovered comparable high-level patterns (e.g., trait-driven extinction signals remained robust across models), but the specific ranking of extinction mechanisms varied. Node-level patterns were relatively stable because extinction scenarios based on intrinsic traits remove species in the same order regardless of network structure. However, link-level outcomes—particularly those concerning secondary extinctions, network collapse, and recovery trajectories—were highly model-dependent.</p>
<p>This means that conclusions about&nbsp;<em>how</em>&nbsp;an extinction unfolded,&nbsp;<em>which</em>&nbsp;interactions disappeared first, or&nbsp;<em>what</em>&nbsp;drove cascading losses are not simply properties of the fossil data but emerge from the assumptions embedded in the chosen model. Researchers therefore need to be explicit about the type of ecological signal they are trying to recover and select models accordingly.</p>
</section>
<section id="aligning-ecological-questions-with-model-choice" class="level2" data-number="3.3">
<h2 data-number="3.3" class="anchored" data-anchor-id="aligning-ecological-questions-with-model-choice"><span class="header-section-number">3.3</span> Aligning ecological questions with model choice</h2>
<p>Our findings support a simple but crucial principle:&nbsp;different ecological questions require different types of network representations. The assumption that all food-web models are interchangeable is not supported by our results.</p>
<p>Below is a conceptual summary (corresponding to a possible <a href="#fig-guidelines" class="quarto-xref">Figure&nbsp;6</a>):</p>
<table class="caption-top table">
<colgroup>
<col style="width: 33%">
<col style="width: 33%">
<col style="width: 33%">
</colgroup>
<thead>
<tr class="header">
<th><strong>Ecological question</strong></th>
<th><strong>Recommended model type</strong></th>
<th><strong>Rationale</strong></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>What&nbsp;<em>could</em>&nbsp;have eaten what? (feasible interactions; trophic potential)</td>
<td>Mechanistic (PFIM, trait-based, evolutionary-signal models)</td>
<td>Captures all trait-feasible links; appropriate for reconstructing dietary breadth or assessing potential niches.</td>
</tr>
<tr class="even">
<td>What interactions were&nbsp;<em>likely realised</em>?</td>
<td>Theory-driven (ADBM, l-matrix)</td>
<td>Encodes foraging ecology and energetic constraints; provides ecologically plausible realised networks.</td>
</tr>
<tr class="odd">
<td>How do global structural properties vary?</td>
<td>Structural models (niche, random as baseline)</td>
<td>Useful for understanding generic patterns (connectance, trophic depth) without trait reliance.</td>
</tr>
<tr class="even">
<td>How do traits influence dynamics (e.g., cascade sensitivity)?</td>
<td>Theory-driven models</td>
<td>Capture body-size scaling, energetic constraints, and trophic dependencies.</td>
</tr>
<tr class="odd">
<td>How robust are inferences to uncertainty?</td>
<td>Ensembles across multiple model families</td>
<td>Allows quantifying how conclusions depend on assumptions.</td>
</tr>
</tbody>
</table>
</section>
<section id="implications-for-the-use-of-food-webs-in-paleoecology" class="level2" data-number="3.4">
<h2 data-number="3.4" class="anchored" data-anchor-id="implications-for-the-use-of-food-webs-in-paleoecology"><span class="header-section-number">3.4</span> Implications for the use of food webs in paleoecology</h2>
<p>Our analysis underscores three important implications for the broader field:</p>
<ol type="1">
<li><p><strong>Interpretations of ancient food webs must explicitly acknowledge model assumptions:</strong> Without this, differences in reconstructed networks may be mistakenly attributed to biological change rather than methodological artifacts.</p></li>
<li><p><strong>Comparisons across studies must be standardised by model family:</strong> Comparing a metaweb from one study to a realised network from another risks generating misleading conclusions about ecological differences across space or time.</p></li>
<li><p><strong>Mechanistic paleo-specific models alone cannot capture the full spectrum of possible network archetypes:</strong> Incorporating theory-driven models allows paleoecology to leverage decades of research on modern food-web structure and dynamics, broadening the range of ecological questions that can be asked.</p></li>
</ol>
</section>
<section id="recommendations-and-best-practices" class="level2" data-number="3.5">
<h2 data-number="3.5" class="anchored" data-anchor-id="recommendations-and-best-practices"><span class="header-section-number">3.5</span> Recommendations and best practices</h2>
<p>To support consistent, transparent use of network reconstruction methods in paleoecology, we propose the following guidelines (which could be highlighted as a boxed summary):</p>
<ol type="1">
<li><p><strong>Start with the ecological question:</strong>, then identify the network representation (feasible, realised, structural) best aligned with it.</p></li>
<li><p><strong>Use multiple models whenever possible:</strong>&nbsp;to evaluate whether key conclusions are model-dependent.</p></li>
<li><p><strong>Report all assumptions explicitly:</strong>, including body-size estimation methods, trait inference procedures, and parameter choices.</p></li>
<li><p><strong>Perform sensitivity analyses:</strong>, particularly for size-based models and connectance assumptions.</p></li>
<li><p><strong>Avoid comparing networks built with different model families</strong>&nbsp;unless explicitly testing model effects.</p></li>
<li><p><strong>Distinguish between node-level and link-level inference</strong>, as they respond differently to model choice.</p></li>
</ol>
</section>
<section id="concluding-remarks" class="level2" data-number="3.6">
<h2 data-number="3.6" class="anchored" data-anchor-id="concluding-remarks"><span class="header-section-number">3.6</span> Concluding remarks</h2>
<p>Our results reveal that network construction models differ far more than is commonly appreciated in paleoecology. Because food webs are increasingly used to infer ecological resilience, collapse dynamics, and responses to environmental change, it is crucial to recognise that these inferences are shaped as much by modelling decisions as by fossil data. By mapping model families to ecological questions and demonstrating how model assumptions influence inferred patterns, our study provides a framework for more transparent, robust, and question-driven reconstruction of paleo food webs.</p>
<div id="fig-guidelines" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-guidelines-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="figures/guidelines.png" class="lightbox" data-gallery="quarto-lightbox-gallery-6" title="Figure&nbsp;6: TODO."><img src="figures/guidelines.png" class="img-fluid figure-img"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-guidelines-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;6: TODO.
</figcaption>
</figure>
</div>
</section>
</section>
<section id="references" class="level1 unnumbered">


</section>

<div id="quarto-appendix" class="default"><section class="quarto-appendix-contents" role="doc-bibliography" id="quarto-bibliography"><h2 class="anchored quarto-appendix-heading">References</h2><div id="refs" class="references csl-bib-body hanging-indent" data-entry-spacing="0" role="list">
<div id="ref-allesina2008" class="csl-entry" role="listitem">
Allesina, Stefano, David Alonso, and Mercedes Pascual. 2008. <span>“A General Model for Food Web Structure.”</span> <em>Science</em> 320 (5876): 658–61. <a href="https://doi.org/10.1126/science.1156269">https://doi.org/10.1126/science.1156269</a>.
</div>
<div id="ref-bambach2007" class="csl-entry" role="listitem">
Bambach, Richard K., Andrew M. Bush, and Douglas H. Erwin. 2007. <span>“Autecology and the Filling of Ecospace: Key Metazoan Radiations.”</span> <em>Palaeontology</em> 50 (1): 1–22. <a href="https://doi.org/10.1111/j.1475-4983.2006.00611.x">https://doi.org/10.1111/j.1475-4983.2006.00611.x</a>.
</div>
<div id="ref-delmas2018" class="csl-entry" role="listitem">
Delmas, Eva, Mathilde Besson, Marie-Hélène Brice, Laura A. Burkle, Giulio V. Dalla Riva, Marie-Josée Fortin, Dominique Gravel, et al. 2018. <span>“Analysing Ecological Networks of Species Interactions.”</span> <em>Biological Reviews</em>, June, 112540. <a href="https://doi.org/10.1111/brv.12433">https://doi.org/10.1111/brv.12433</a>.
</div>
<div id="ref-dillon2022" class="csl-entry" role="listitem">
Dillon, Erin M., Jaleigh Q. Pier, Jansen A. Smith, Nussaïbah B. Raja, Danijela Dimitrijević, Elizabeth L. Austin, Jonathan D. Cybulski, et al. 2022. <span>“What Is Conservation Paleobiology? Tracking 20 Years of Research and Development.”</span> <em>Frontiers in Ecology and Evolution</em> 10 (December). <a href="https://doi.org/10.3389/fevo.2022.1031483">https://doi.org/10.3389/fevo.2022.1031483</a>.
</div>
<div id="ref-dunhill2024" class="csl-entry" role="listitem">
Dunhill, Alexander M., Karolina Zarzyczny, Jack O. Shaw, Jed W. Atkinson, Crispin T. S. Little, and Andrew P. Beckerman. 2024. <span>“Extinction Cascades, Community Collapse, and Recovery Across a Mesozoic Hyperthermal Event.”</span> <em>Nature Communications</em> 15 (1): 8599. <a href="https://doi.org/10.1038/s41467-024-53000-2">https://doi.org/10.1038/s41467-024-53000-2</a>.
</div>
<div id="ref-erdos1959" class="csl-entry" role="listitem">
Erdős, P., and A. Rényi. 1959. <span>“On Random Graphs. i.”</span> <em>Publicationes Mathematicae Debrecen</em> 6 (3-4): 290–97. <a href="https://doi.org/10.5486/pmd.1959.6.3-4.12">https://doi.org/10.5486/pmd.1959.6.3-4.12</a>.
</div>
<div id="ref-fricke2022" class="csl-entry" role="listitem">
Fricke, Evan C., Chia Hsieh, Owen Middleton, Daniel Gorczynski, Caroline D. Cappello, Oscar Sanisidro, John Rowan, Jens-Christian Svenning, and Lydia Beaudrot. 2022. <span>“Collapse of Terrestrial Mammal Food Webs Since the Late Pleistocene.”</span> <em>Science</em> 377 (6609): 1008–11. <a href="https://doi.org/10.1126/science.abn4012">https://doi.org/10.1126/science.abn4012</a>.
</div>
<div id="ref-gupta2022" class="csl-entry" role="listitem">
Gupta, Anubhav, Reinhard Furrer, and Owen L. Petchey. 2022. <span>“Simultaneously Estimating Food Web Connectance and Structure with Uncertainty.”</span> <em>Ecology and Evolution</em> 12 (3): e8643. <a href="https://doi.org/10.1002/ece3.8643">https://doi.org/10.1002/ece3.8643</a>.
</div>
<div id="ref-hao2025" class="csl-entry" role="listitem">
Hao, Xiyang, Marcel Holyoak, Zhicheng Zhang, and Chuan Yan. 2025. <span>“Global Projection of Terrestrial Vertebrate Food Webs Under Future Climate and Land-Use Changes.”</span> <em>Global Change Biology</em> 31 (2): e70061. <a href="https://doi.org/10.1111/gcb.70061">https://doi.org/10.1111/gcb.70061</a>.
</div>
<div id="ref-ings2009" class="csl-entry" role="listitem">
Ings, Thomas C., José M. Montoya, Jordi Bascompte, Nico Blüthgen, Lee Brown, Carsten F. Dormann, François Edwards, et al. 2009. <span>“Ecological networks–beyond food webs.”</span> <em>The Journal of Animal Ecology</em> 78 (1): 253–69. <a href="https://doi.org/10.1111/j.1365-2656.2008.01460.x">https://doi.org/10.1111/j.1365-2656.2008.01460.x</a>.
</div>
<div id="ref-kiessling2019" class="csl-entry" role="listitem">
Kiessling, Wolfgang, Nussaïbah B. Raja, Vanessa Julie Roden, Samuel T. Turvey, and Erin E. Saupe. 2019. <span>“Addressing Priority Questions of Conservation Science with Palaeontological Data.”</span> <em>Philosophical Transactions of the Royal Society B: Biological Sciences</em> 374 (1788): 20190222. <a href="https://doi.org/10.1098/rstb.2019.0222">https://doi.org/10.1098/rstb.2019.0222</a>.
</div>
<div id="ref-milo2002" class="csl-entry" role="listitem">
Milo, R., S. Shen-Orr, S. Itzkovitz, N. Kashtan, D. Chklovskii, and U. Alon. 2002. <span>“Network Motifs: Simple Building Blocks of Complex Networks.”</span> <em>Science</em> 298 (5594): 824–27. <a href="https://doi.org/10.1126/science.298.5594.824">https://doi.org/10.1126/science.298.5594.824</a>.
</div>
<div id="ref-morales-castilla2015" class="csl-entry" role="listitem">
Morales-Castilla, Ignacio, Miguel G. Matias, Dominique Gravel, and Miguel B. Araújo. 2015. <span>“Inferring Biotic Interactions from Proxies.”</span> <em>Trends in Ecology &amp; Evolution</em> 30 (6): 347–56. <a href="https://doi.org/10.1016/j.tree.2015.03.014">https://doi.org/10.1016/j.tree.2015.03.014</a>.
</div>
<div id="ref-petchey2008" class="csl-entry" role="listitem">
Petchey, Owen L., Andrew P. Beckerman, Jens O. Riede, and Philip H. Warren. 2008. <span>“Size, Foraging, and Food Web Structure.”</span> <em>Proceedings of the National Academy of Sciences</em> 105 (11): 4191–96. <a href="https://doi.org/10.1073/pnas.0710672105">https://doi.org/10.1073/pnas.0710672105</a>.
</div>
<div id="ref-pichler2023" class="csl-entry" role="listitem">
Pichler, Maximilian, and Florian Hartig. 2023. <span>“Machine Learning and Deep Learning<span></span>A Review for Ecologists.”</span> <em>Methods in Ecology and Evolution</em> 14 (4): 994–1016. <a href="https://doi.org/10.1111/2041-210X.14061">https://doi.org/10.1111/2041-210X.14061</a>.
</div>
<div id="ref-poisot2012" class="csl-entry" role="listitem">
Poisot, Timothée, Elsa Canard, Nicolas Mouquet, and Michael E. Hochberg. 2012. <span>“A Comparative Study of Ecological Specialization Estimators.”</span> <em>Methods in Ecology and Evolution</em> 3 (3): 537–44. <a href="https://doi.org/10.1111/j.2041-210x.2011.00174.x">https://doi.org/10.1111/j.2041-210x.2011.00174.x</a>.
</div>
<div id="ref-rohr2010" class="csl-entry" role="listitem">
Rohr, Rudolf&nbsp;Philippe, Heike Scherer, Patrik Kehrli, Christian Mazza, and Louis-Félix Bersier. 2010. <span>“Modeling Food Webs: Exploring Unexplained Structure Using Latent Traits.”</span> <em>The American Naturalist</em> 176 (2): 170–77. <a href="https://doi.org/10.1086/653667">https://doi.org/10.1086/653667</a>.
</div>
<div id="ref-roopnarine2006" class="csl-entry" role="listitem">
Roopnarine, Peter D. 2006. <span>“Extinction Cascades and Catastrophe in Ancient Food Webs.”</span> <em>Paleobiology</em> 32 (1): 1–19. <a href="https://www.jstor.org/stable/4096814">https://www.jstor.org/stable/4096814</a>.
</div>
<div id="ref-schneider2016" class="csl-entry" role="listitem">
Schneider, Florian D., Ulrich Brose, Björn C. Rall, and Christian Guill. 2016. <span>“Animal Diversity and Ecosystem Functioning in Dynamic Food Webs.”</span> <em>Nature Communications</em> 7 (1): 12718. <a href="https://doi.org/10.1038/ncomms12718">https://doi.org/10.1038/ncomms12718</a>.
</div>
<div id="ref-shaw2024" class="csl-entry" role="listitem">
Shaw, Jack O., Alexander M. Dunhill, Andrew P. Beckerman, Jennifer A. Dunne, and Pincelli M. Hull. 2024. <span>“A Framework for Reconstructing Ancient Food Webs Using Functional Trait Data.”</span> bioRxiv. <a href="https://doi.org/10.1101/2024.01.30.578036">https://doi.org/10.1101/2024.01.30.578036</a>.
</div>
<div id="ref-stouffer2007" class="csl-entry" role="listitem">
Stouffer, Daniel B, Juan Camacho, Wenxin Jiang, and Luís A Nunes Amaral. 2007. <span>“Evidence for the Existence of a Robust Pattern of Prey Selection in Food Webs.”</span> <em>Proceedings of the Royal Society B: Biological Sciences</em> 274 (1621): 1931–40. <a href="https://doi.org/10.1098/rspb.2007.0571">https://doi.org/10.1098/rspb.2007.0571</a>.
</div>
<div id="ref-strydom2021" class="csl-entry" role="listitem">
Strydom, Tanya, Michael D. Catchen, Francis Banville, Dominique Caron, Gabriel Dansereau, Philippe Desjardins-Proulx, Norma R. Forero-Muñoz, et al. 2021. <span>“A Roadmap Towards Predicting Species Interaction Networks (Across Space and Time).”</span> <em>Philosophical Transactions of the Royal Society B: Biological Sciences</em> 376 (1837): 20210063. <a href="https://doi.org/10.1098/rstb.2021.0063">https://doi.org/10.1098/rstb.2021.0063</a>.
</div>
<div id="ref-thuiller2024" class="csl-entry" role="listitem">
Thuiller, Wilfried, Irene Calderón-Sanou, Loïc Chalmandrier, Pierre Gaüzère, Louise M. J. O’Connor, Marc Ohlmann, Giovanni Poggiato, and Tamara Münkemüller. 2024. <span>“Navigating the Integration of Biotic Interactions in Biogeography.”</span> <em>Journal of Biogeography</em> 51 (4): 550–59. <a href="https://doi.org/10.1111/jbi.14734">https://doi.org/10.1111/jbi.14734</a>.
</div>
<div id="ref-williams2004" class="csl-entry" role="listitem">
Williams, R. J., and N. D. Martinez. 2004. <span>“Stabilization of Chaotic and Non-Permanent Food-Web Dynamics.”</span> <em>The European Physical Journal B - Condensed Matter</em> 38 (2): 297–303. <a href="https://doi.org/10.1140/epjb/e2004-00122-1">https://doi.org/10.1140/epjb/e2004-00122-1</a>.
</div>
<div id="ref-williams2000" class="csl-entry" role="listitem">
Williams, Richard J., and Neo D. Martinez. 2000. <span>“Simple Rules Yield Complex Food Webs.”</span> <em>Nature</em> 404 (6774): 180–83. <a href="https://doi.org/10.1038/35004572">https://doi.org/10.1038/35004572</a>.
</div>
<div id="ref-williams2008" class="csl-entry" role="listitem">
———. 2008. <span>“Success and its limits among structural models of complex food webs.”</span> <em>The Journal of Animal Ecology</em> 77 (3): 512–19. <a href="https://doi.org/10.1111/j.1365-2656.2008.01362.x">https://doi.org/10.1111/j.1365-2656.2008.01362.x</a>.
</div>
<div id="ref-yeakel2014" class="csl-entry" role="listitem">
Yeakel, Justin D., Mathias M. Pires, Lars Rudolf, Nathaniel J. Dominy, Paul L. Koch, Paulo R. Guimarães, and Thilo Gross. 2014. <span>“Collapse of an Ecological Network in Ancient Egypt.”</span> <em>PNAS</em> 111 (40): 14472–77. <a href="https://doi.org/10.1073/pnas.1408471111">https://doi.org/10.1073/pnas.1408471111</a>.
</div>
</div></section><section class="quarto-appendix-contents" id="quarto-citation"><h2 class="anchored quarto-appendix-heading">Citation</h2><div><div class="quarto-appendix-secondary-label">BibTeX citation:</div><pre class="sourceCode code-with-copy quarto-appendix-bibtex"><code class="sourceCode bibtex">@online{strydom2025,
  author = {Strydom, Tanya and Karapunar, Baran and P. Beckerman, Andrew
    and Dunhill, Alexander},
  title = {Methods for {Reconstructing} {Paleo} {Food} {Webs}},
  date = {2025-12-09},
  langid = {en},
  abstract = {Food webs provide a powerful lens for understanding
    ecosystem structure and function, but reconstructing them in
    paleoecological contexts remains challenging because direct evidence
    of feeding interactions is rarely preserved. A wide range of models
    now exist for predicting interactions and inferring network
    structure, yet these models differ substantially in their
    assumptions, mechanisms, and data requirements. Here, we evaluate
    which network construction approaches are most suitable for paleo
    food webs given the constraints of the fossil record, and we assess
    how model choice influences the networks we infer. Using the
    Toarcian Oceanic Anoxic Event as a case study, we compare six
    modelling approaches encompassing mechanistic, structural, and
    theory-based methods. We show that the models produce strikingly
    different network structures and pairwise interactions, and that
    these differences propagate into ecological inference—including
    conclusions about extinction dynamics. Our results highlight the
    importance of aligning model choice with research questions and
    underscore the interpretative risks of treating all food-web
    reconstruction methods as interchangeable.}
}
</code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre><div class="quarto-appendix-secondary-label">For attribution, please cite this work as:</div><div id="ref-strydom2025" class="csl-entry quarto-appendix-citeas" role="listitem">
Strydom, Tanya, Baran Karapunar, Andrew P. Beckerman, and Alexander
Dunhill. 2025. <span>“Methods for Reconstructing Paleo Food
Webs.”</span> TBD. December 9, 2025.
</div></div></section></div></main>
<!-- /main column -->
<script id="quarto-html-after-body" type="application/javascript">
  window.document.addEventListener("DOMContentLoaded", function (event) {
    const icon = "";
    const anchorJS = new window.AnchorJS();
    anchorJS.options = {
      placement: 'right',
      icon: icon
    };
    anchorJS.add('.anchored');
    const isCodeAnnotation = (el) => {
      for (const clz of el.classList) {
        if (clz.startsWith('code-annotation-')) {                     
          return true;
        }
      }
      return false;
    }
    const onCopySuccess = function(e) {
      // button target
      const button = e.trigger;
      // don't keep focus
      button.blur();
      // flash "checked"
      button.classList.add('code-copy-button-checked');
      var currentTitle = button.getAttribute("title");
      button.setAttribute("title", "Copied!");
      let tooltip;
      if (window.bootstrap) {
        button.setAttribute("data-bs-toggle", "tooltip");
        button.setAttribute("data-bs-placement", "left");
        button.setAttribute("data-bs-title", "Copied!");
        tooltip = new bootstrap.Tooltip(button, 
          { trigger: "manual", 
            customClass: "code-copy-button-tooltip",
            offset: [0, -8]});
        tooltip.show();    
      }
      setTimeout(function() {
        if (tooltip) {
          tooltip.hide();
          button.removeAttribute("data-bs-title");
          button.removeAttribute("data-bs-toggle");
          button.removeAttribute("data-bs-placement");
        }
        button.setAttribute("title", currentTitle);
        button.classList.remove('code-copy-button-checked');
      }, 1000);
      // clear code selection
      e.clearSelection();
    }
    const getTextToCopy = function(trigger) {
      const outerScaffold = trigger.parentElement.cloneNode(true);
      const codeEl = outerScaffold.querySelector('code');
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
    }
    const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
      text: getTextToCopy
    });
    clipboard.on('success', onCopySuccess);
    if (window.document.getElementById('quarto-embedded-source-code-modal')) {
      const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
        text: getTextToCopy,
        container: window.document.getElementById('quarto-embedded-source-code-modal')
      });
      clipboardModal.on('success', onCopySuccess);
    }
      var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
      var mailtoRegex = new RegExp(/^mailto:/);
        var filterRegex = new RegExp('/' + window.location.host + '/');
      var isInternal = (href) => {
          return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
      }
      // Inspect non-navigation links and adorn them if external
     var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
      for (var i=0; i<links.length; i++) {
        const link = links[i];
        if (!isInternal(link.href)) {
          // undo the damage that might have been done by quarto-nav.js in the case of
          // links that we want to consider external
          if (link.dataset.originalHref !== undefined) {
            link.href = link.dataset.originalHref;
          }
        }
      }
    function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
      const config = {
        allowHTML: true,
        maxWidth: 500,
        delay: 100,
        arrow: false,
        appendTo: function(el) {
            return el.parentElement;
        },
        interactive: true,
        interactiveBorder: 10,
        theme: 'quarto',
        placement: 'bottom-start',
      };
      if (contentFn) {
        config.content = contentFn;
      }
      if (onTriggerFn) {
        config.onTrigger = onTriggerFn;
      }
      if (onUntriggerFn) {
        config.onUntrigger = onUntriggerFn;
      }
      window.tippy(el, config); 
    }
    const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
    for (var i=0; i<noterefs.length; i++) {
      const ref = noterefs[i];
      tippyHover(ref, function() {
        // use id or data attribute instead here
        let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
        try { href = new URL(href).hash; } catch {}
        const id = href.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note) {
          return note.innerHTML;
        } else {
          return "";
        }
      });
    }
    const xrefs = window.document.querySelectorAll('a.quarto-xref');
    const processXRef = (id, note) => {
      // Strip column container classes
      const stripColumnClz = (el) => {
        el.classList.remove("page-full", "page-columns");
        if (el.children) {
          for (const child of el.children) {
            stripColumnClz(child);
          }
        }
      }
      stripColumnClz(note)
      if (id === null || id.startsWith('sec-')) {
        // Special case sections, only their first couple elements
        const container = document.createElement("div");
        if (note.children && note.children.length > 2) {
          container.appendChild(note.children[0].cloneNode(true));
          for (let i = 1; i < note.children.length; i++) {
            const child = note.children[i];
            if (child.tagName === "P" && child.innerText === "") {
              continue;
            } else {
              container.appendChild(child.cloneNode(true));
              break;
            }
          }
          if (window.Quarto?.typesetMath) {
            window.Quarto.typesetMath(container);
          }
          return container.innerHTML
        } else {
          if (window.Quarto?.typesetMath) {
            window.Quarto.typesetMath(note);
          }
          return note.innerHTML;
        }
      } else {
        // Remove any anchor links if they are present
        const anchorLink = note.querySelector('a.anchorjs-link');
        if (anchorLink) {
          anchorLink.remove();
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        if (note.classList.contains("callout")) {
          return note.outerHTML;
        } else {
          return note.innerHTML;
        }
      }
    }
    for (var i=0; i<xrefs.length; i++) {
      const xref = xrefs[i];
      tippyHover(xref, undefined, function(instance) {
        instance.disable();
        let url = xref.getAttribute('href');
        let hash = undefined; 
        if (url.startsWith('#')) {
          hash = url;
        } else {
          try { hash = new URL(url).hash; } catch {}
        }
        if (hash) {
          const id = hash.replace(/^#\/?/, "");
          const note = window.document.getElementById(id);
          if (note !== null) {
            try {
              const html = processXRef(id, note.cloneNode(true));
              instance.setContent(html);
            } finally {
              instance.enable();
              instance.show();
            }
          } else {
            // See if we can fetch this
            fetch(url.split('#')[0])
            .then(res => res.text())
            .then(html => {
              const parser = new DOMParser();
              const htmlDoc = parser.parseFromString(html, "text/html");
              const note = htmlDoc.getElementById(id);
              if (note !== null) {
                const html = processXRef(id, note);
                instance.setContent(html);
              } 
            }).finally(() => {
              instance.enable();
              instance.show();
            });
          }
        } else {
          // See if we can fetch a full url (with no hash to target)
          // This is a special case and we should probably do some content thinning / targeting
          fetch(url)
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.querySelector('main.content');
            if (note !== null) {
              // This should only happen for chapter cross references
              // (since there is no id in the URL)
              // remove the first header
              if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
                note.children[0].remove();
              }
              const html = processXRef(null, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      }, function(instance) {
      });
    }
        let selectedAnnoteEl;
        const selectorForAnnotation = ( cell, annotation) => {
          let cellAttr = 'data-code-cell="' + cell + '"';
          let lineAttr = 'data-code-annotation="' +  annotation + '"';
          const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
          return selector;
        }
        const selectCodeLines = (annoteEl) => {
          const doc = window.document;
          const targetCell = annoteEl.getAttribute("data-target-cell");
          const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
          const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
          const lines = annoteSpan.getAttribute("data-code-lines").split(",");
          const lineIds = lines.map((line) => {
            return targetCell + "-" + line;
          })
          let top = null;
          let height = null;
          let parent = null;
          if (lineIds.length > 0) {
              //compute the position of the single el (top and bottom and make a div)
              const el = window.document.getElementById(lineIds[0]);
              top = el.offsetTop;
              height = el.offsetHeight;
              parent = el.parentElement.parentElement;
            if (lineIds.length > 1) {
              const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
              const bottom = lastEl.offsetTop + lastEl.offsetHeight;
              height = bottom - top;
            }
            if (top !== null && height !== null && parent !== null) {
              // cook up a div (if necessary) and position it 
              let div = window.document.getElementById("code-annotation-line-highlight");
              if (div === null) {
                div = window.document.createElement("div");
                div.setAttribute("id", "code-annotation-line-highlight");
                div.style.position = 'absolute';
                parent.appendChild(div);
              }
              div.style.top = top - 2 + "px";
              div.style.height = height + 4 + "px";
              div.style.left = 0;
              let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
              if (gutterDiv === null) {
                gutterDiv = window.document.createElement("div");
                gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
                gutterDiv.style.position = 'absolute';
                const codeCell = window.document.getElementById(targetCell);
                const gutter = codeCell.querySelector('.code-annotation-gutter');
                gutter.appendChild(gutterDiv);
              }
              gutterDiv.style.top = top - 2 + "px";
              gutterDiv.style.height = height + 4 + "px";
            }
            selectedAnnoteEl = annoteEl;
          }
        };
        const unselectCodeLines = () => {
          const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
          elementsIds.forEach((elId) => {
            const div = window.document.getElementById(elId);
            if (div) {
              div.remove();
            }
          });
          selectedAnnoteEl = undefined;
        };
          // Handle positioning of the toggle
      window.addEventListener(
        "resize",
        throttle(() => {
          elRect = undefined;
          if (selectedAnnoteEl) {
            selectCodeLines(selectedAnnoteEl);
          }
        }, 10)
      );
      function throttle(fn, ms) {
      let throttle = false;
      let timer;
        return (...args) => {
          if(!throttle) { // first call gets through
              fn.apply(this, args);
              throttle = true;
          } else { // all the others get throttled
              if(timer) clearTimeout(timer); // cancel #2
              timer = setTimeout(() => {
                fn.apply(this, args);
                timer = throttle = false;
              }, ms);
          }
        };
      }
        // Attach click handler to the DT
        const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
        for (const annoteDlNode of annoteDls) {
          annoteDlNode.addEventListener('click', (event) => {
            const clickedEl = event.target;
            if (clickedEl !== selectedAnnoteEl) {
              unselectCodeLines();
              const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
              if (activeEl) {
                activeEl.classList.remove('code-annotation-active');
              }
              selectCodeLines(clickedEl);
              clickedEl.classList.add('code-annotation-active');
            } else {
              // Unselect the line
              unselectCodeLines();
              clickedEl.classList.remove('code-annotation-active');
            }
          });
        }
    const findCites = (el) => {
      const parentEl = el.parentElement;
      if (parentEl) {
        const cites = parentEl.dataset.cites;
        if (cites) {
          return {
            el,
            cites: cites.split(' ')
          };
        } else {
          return findCites(el.parentElement)
        }
      } else {
        return undefined;
      }
    };
    var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
    for (var i=0; i<bibliorefs.length; i++) {
      const ref = bibliorefs[i];
      const citeInfo = findCites(ref);
      if (citeInfo) {
        tippyHover(citeInfo.el, function() {
          var popup = window.document.createElement('div');
          citeInfo.cites.forEach(function(cite) {
            var citeDiv = window.document.createElement('div');
            citeDiv.classList.add('hanging-indent');
            citeDiv.classList.add('csl-entry');
            var biblioDiv = window.document.getElementById('ref-' + cite);
            if (biblioDiv) {
              citeDiv.innerHTML = biblioDiv.innerHTML;
            }
            popup.appendChild(citeDiv);
          });
          return popup.innerHTML;
        });
      }
    }
  });
  </script>
</div> <!-- /content -->
<script>var lightboxQuarto = GLightbox({"closeEffect":"zoom","descPosition":"bottom","loop":false,"openEffect":"zoom","selector":".lightbox"});
(function() {
  let previousOnload = window.onload;
  window.onload = () => {
    if (previousOnload) {
      previousOnload();
    }
    lightboxQuarto.on('slide_before_load', (data) => {
      const { slideIndex, slideNode, slideConfig, player, trigger } = data;
      const href = trigger.getAttribute('href');
      if (href !== null) {
        const imgEl = window.document.querySelector(`a[href="${href}"] img`);
        if (imgEl !== null) {
          const srcAttr = imgEl.getAttribute("src");
          if (srcAttr && srcAttr.startsWith("data:")) {
            slideConfig.href = srcAttr;
          }
        }
      } 
    });
  
    lightboxQuarto.on('slide_after_load', (data) => {
      const { slideIndex, slideNode, slideConfig, player, trigger } = data;
      if (window.Quarto?.typesetMath) {
        window.Quarto.typesetMath(slideNode);
      }
    });
  
  };
  
})();
          </script>




</body></html>